<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | SFB 680 / FOR 1078 Joint Conference</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="randomimage" type="text/javascript" charset="utf-8" 
    src="scripts/randomimage.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem02Active"><div 
        class="mainMenuItem02ActiveText"><a class="mainMenuLink02" 
        href="conferences.html">conferences</a></div></li>
      <li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="seminar_days.html">seminar days</a></li>
      <li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="colloquia.html">colloquia</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="journal_club.html">journal club</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h3>SFB 680 / FOR 1078 Joint Conference</h3>
    <h2>Scientific Program</h2>
<div class="scheduleAbstract">
<h3><a name="alcazar"></a>Immune-triggered hybrid incompatibilities between European and central Asian Arabidopsis accessions: a by-product of local selection?</h3>
<p><strong>Rubén Alcázar</strong>, Maarten Koornneef and Jane E. Parker<br />MPI for Plant Breeding Research, Cologne</p>
<p>Accumulation of genetic incompatibilities within species can lead to reproductive isolation and potentially, speciation. The Dobzhansky-Muller model of genetic incompatibilities postulates that reproductive isolation is driven by deleterious epistatic interactions between alleles arising from evolutionary divergent populations. Whether incompatibilities arise as a by-product of selection or drift is an intriguing question in evolutionary biology. In this study, we show that a common genetic framework drives many hybrid incompatibilities in the model plant species <em>Arabidopsis thaliana</em>. The geographical distribution of specific alleles of genes involved in innate immunity correlates with the occurrence of incompatibilities. Variation in disease susceptibility of plants expressing different alleles correlates with signatures of selection. We provide an example of a local adaptation process likely driven by the co-evolution between plant host and pathogen that may contribute to the occurrence of incompatibility between geographically distant populations.</p>
<p style="font-size:90%;"><strong>References:</strong><br/>
Alcázar R, García AV, Kronholm I, de Meaux J, Koornneef M, Parker JE, Reymond 
M. Natural variation at Strubbelig Receptor Kinase 3 drives immune-triggered
incompatibilities between Arabidopsis thaliana accessions. <strong>Nat Genet. 2010 42(12):1135-9</strong></p>
<p style="font-size:90%;">Alcázar R, García AV, Parker JE, Reymond M. Incremental steps toward
incompatibility revealed by Arabidopsis epistatic interactions modulating
salicylic acid pathway activation. <strong>Proc Natl Acad Sci U S A. 2009 106:334-9</strong></p>
</div>
<div class="scheduleAbstract">
<h3><a name="baines"></a>Evolutionary and ecological forces shaping diversity in the mammalian intestinal microbiota</h3>
<p><strong>John Baines</strong><br/>Christian-Albrechts-University Kiel</p>
<p>The microbial communities inhabiting the mammalian intestinal tract play an important role in diverse aspects of host biology. However, little is known regarding the forces shaping variation in these communities and its influence on host fitness. To shed light on the  relative influence of host genetics, environmental factors and natural selection on the diversity in microbial communities between individuals (i.e. beta diversity), we performed a survey of intestinal microbial communities in a panel of house mouse populations across western Europe using high throughput pyrosequencing of the bacterial 16S rRNA gene. The host factors studied included population structure as estimated by microsatellite loci, mitochondrial haplotypes, geography and the candidate gene B4galnt2, a blood group-related glycosyltransferase under selection in house mice. Overall, host population structure best explains microbial community composition and clustering. However, individual bacterial species displayed significant associations with geography, maternal lineage and B4galnt2 genotype.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="bataillon"></a>Extensive X-linked adaptive evolution in central chimpanzees</h3>
<p><strong>Thomas Bataillon</strong><br/>Aarhus University</p>
<p>Recent surveys in <em>Drosophila</em> suggest that extensive adaptation drives between-species divergence, but despite several studies, the generality of this finding, in particular for humans, remains unclear. Here we capture and sequence the complete exomes of 12 central chimpanzees (Pan troglodytes troglodytes). We identify more than 60,000 SNPs in 15.3Mb of coding DNA for which high confidence genotypes can be called in all 12 individuals. We report extensive adaptive evolution targeting the X chromosome of chimpanzees with about 30% of all amino acid replacements being adaptive. We also find much stronger purifying selection than observed in humans, and in contrast to humans, purifying selection is stronger on the X chromosome than on the autosomes in chimpanzees. Adaptive evolution is barely detectable on the autosomes except for a few striking cases of recent selective sweeps associated with immunity gene clusters, the largest spanning &gt;6 Mb. We therefore conclude that most adaptive mutations are recessive. We also document dramatically reduced synonymous diversity in the chimpanzee X chromosome (piS=0.00093) relative to autosomes (piS=0.00204) and stronger purifying selection than for the human X chromosome. Our results therefore also provide an explanation for the much-discussed reduction in the human-chimpanzee divergence at the X chromosome.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="chan"></a>Selecting for Giants: Genetic Dissection of an Adaptive Complex Trait in Laboratory and Wild Mice</h3>
<p><strong>Yingguang Frank Chan<span class="sfbSuper">1</span></strong>, Lutz Bunger<span class="sfbSuper">2</span> and Diethard Tautz<span class="sfbSuper">1</span><br/><span class="sfbSuper">1</span>MPI for Evolutionary Biology, Plön<br/><span class="sfbSuper">2</span>Scottish Agricultural College, Animal Breeding and Development Team, Midlothian</p>
<p>Adaptation underlies much evolutionary diversity. Yet, we know little about the genetic architecture underlying adaptation. We used growth as a model complex trait to ask, how repeatable is genetic adaptation? Is body size controlled by many loci of infinitesimal effects? Or by few loci of large effects? Gigantism in house mouse represents an ideal system: it occurs repeatedly during island colonisation and can be readily mimicked by artificial selection. Using mice from 11 lines, long-term divergently selected for body weight and from 2 control lines, we conducted a specialized form of association mapping referred to as “parallel association”. We have identified 67 loci (FDR 0.01) broadly distributed across the whole genome showing allele-sharing across all seven high body weight selected lines, yet rare or missing among their sibling control or low body weight lines. To confirm these regions, we also genotyped mice from one long-term selection experiment at different generations during the selection experiment  to detect allele frequency change through time. Consistent with adaptation, these regions show substantial  evidence for selective sweeps. The mapping resolution achieved by parallel association compares favorably with conventional QTL mapping, often achieving sub-megabase resolution. These loci nonetheless show strong clustering and contain key “orchestrator” genes regulating growth and metabolic pathways, suggesting that clusters of favorable alleles may be repeatedly involved in adaptation. By applying these loci to a recombinant inbred mouse dataset, we are able to predict body weight with &gt;90% accuracy. These results show that adaptive strategies may rely broadly on standing variations and adaptive evolution may proceed through many loci of significant effects.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="coupland"></a>Evolution of life history traits in plants</h3>
<p><strong>George Coupland</strong><br/>Max Planck Institute for Plant Breeding Research</p>
<p>Life history traits evolve rapidly in plants.  An example is the distinction between annual and perennial species. Annual plants complete their life cycle in a single year and reproduce only once, whilst perennials live for many years and reproduce several times. The distinction between annuals and perennials has evolved independently many times. We aim to understand the genetic and mechanistic basis of the distinction between annuals and perennials within an evolutionary framework. Species of the Brassicaceae family are used because they are genetically tractable and have small genomes. Our phylogenetic analysis suggests that annuals are derived from perennials. We are using next generation sequencing to assemble the genomic sequences of annual and perennial sister species and combining this with genetic analysis to identify the genes that confer differences in life history. This analysis has provided evidence for cis-acting changes in expression of key transcription factors conferring differences between annual and perennial traits. In addition, ecological studies of natural populations of perennials provide evidence for the selective advantage conferred by perennial traits.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="foitzik"></a>Can kin selection explain the evolution of slave rebellion?</h3>
<p><strong>Susanne Foitzik</strong><br/>Johannes Gutenberg-University, Mainz</p>
<p>The evolution of parasite defense traits can depend on the population structures of the hosts and the parasites on a local scale. Slave-making ants, such as the obligate social parasite <em>Protomognathus americanus</em>, enslave workers of other ant species, which carry out all routine task including taking care of the slavemakers' brood. Slaves of <em>Temnothorax longispinosus</em>, the main host species of <em>P. americanus</em>, have been observed to attack and kill slavemaker pupae - a behavior that has been described as slave rebellion. It has not been thoroughly studied whether this killing behavior can be interpreted as a defense trait that evolved in the coevolutionary arms race with the parasite. In principle, slave rebellion could reduce the raiding pressure of the slavemaker and thus increase the productivity of free-living relatives of relatives in the proximity of the slavemaker nest. We studied under which conditions this kin selection argument holds, and whether these are given in the field. A simulation parameterized according to observations of the <em>Protomognathus</em> - <em>Temnothorax</em> system indicates that kin selection can explain slave rebellion, but only if the costs of the rebellion traits are low and if host nests that contain close relatives of the slaves are in the raiding range of slavemaker nests. Small scale spatial genetic analyses of <em>Temnothorax</em> ants from two US study sites have shown that the latter condition is fulfilled in the field. A co-variation of killing rates in parasite and host colonies indicates potential costs of the rebellion trait.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="gordo_sousa"></a>Adaptation and compensation in <em>Escherichia coli</em></h3>
<p><strong>Isabel Gordo, Ana Sousa</strong><br/>Instituto Gulbenkian, Lisbon</p>
<p>One of the key questions in evolutionary biology is: what is the rate at which evolutionary change occurs and to what extent does this rate vary. In this talk, we will study different theoretical frameworks to understand rates of phenotypic and genetic change in the longest experimental evolution towards adaptation to a simple environment, performed in <em>E. coli</em> (Lenski et al. 1991). We explore the predictions of Fisher’s geometric model in the context of this experiment and extend this framework to account for the main patterns in the experimental data. Another important issue is how rates of evolutionary change depend on the genetic background. We will provide estimates for the distribution of effects of beneficial mutations that compensate for antibiotic resistance. This distribution is key for understanding the maintenance of resistance in bacterial populations. We show that the mean effect of compensatory mutations depends of the genetic background.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="hallatschek"></a>The evolutionary advantage of being round</h3>
<p><strong>Oskar Hallatschek</strong><br/>MPI for Dynamics and Self-Organization, Göttingen</p>
<p>Bacterial species display an astonishing variety of shapes, such as round, rod-like, comma- or spiral-shaped.  Shape is thought to influence several functions, such as nutrient take-up, swimming and the attachment to surfaces.  Here, we study a possible impact of cell shape on adaptation. We show that, due to a biophysical buckling instability, rod-like bacteria exhibit much higher levels of genetic drift in growing colonies than round-like microbes. Consequently,  the establishment of beneficial mutations is strongly suppressed in colonies of rod-like bacteria. Our experiments and model thus support the hypothesis that shape strongly influences adaptability in growing biofilms.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="howard"></a>Coevolution in a natural host-pathogen system between two model organisms, the house mouse and <em>Toxoplasma gondii</em></h3>
<p><strong>Jonathan Howard</strong><br/>University of Cologne</p>
<p>All true cats are the definitive hosts for the protozoal pathogen, <em>Toxoplasma gondii</em>. With the domestication of the wild cat, <em>Felis sylvestris libica</em>, about 10,000 years ago, the population size of this  single cat subspecies has vastly outnumbered even the sum of all other  cat species world-wide. This transformative event has presumably increased selection pressure on the normal prey species of the domestic cat, and world-wide, mice and rats will have taken the brunt  of this. We shall show that an unusual resistance mechanism against <em>T. gondii</em>, the Immunity-Related GTPases, is highly evolved in the mouse. Furthermore the genes and proteins of this system show an extraordinary polymorphism in wild mice that matches that of the MHC  in complexity. That the IRG system exerts a reciprocal selection pressure on T. gondii will be demonstrated by the identification of  polymorphic virulence genes that specifically inactivate IRG proteins. We shall present evidence of a complex "gene-for-gene" relationship between the mouse IRG system and the Toxoplasma virulence system  resembling in some respects the relationship between polymorphic plant R-genes and their antagonistic pathovars.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="hu_berg"></a>Natural variation in miR824 synthesis modifies stomata patterning and fitness in <em>Arabidopsis thaliana</em>: experiment and modelling</h3>
<p><strong>Jinyong Hu<span class="sfbSuper">1</span>, Filippos Klironomos<span class="sfbSuper">2</span></strong>, Li Lei<span class="sfbSuper">1</span>, Fei He<span class="sfbSuper">1</span>, Johannes Berg<span class="sfbSuper">2</span> and Juliette de Meaux<span class="sfbSuper">3</span><br/><span class="sfbSuper">1</span>MPI for Plant Breeding Research<br/><span class="sfbSuper">2</span>University of Cologne<br/><span class="sfbSuper">3</span>University of Muenster</p>
<p>Mutations which change how genes are regulated are thought to contribute prominently to phenotypic change and adaptive evolution. Micro-RNAs are post-transcriptional regulators of gene expression whose role in evolution is an open question. Here we consider the miRNA824-AGL16 pathway of <em>A. thaliana</em>, which plays a key role  in stomata patterning and other traits important for fitness. We find that structural polymorphisms in the micro-RNA precursor molecule affects the synthesis of mature miRNA and hence expression levels of AGL16. With a combination of mutants, near-isogenic and transgenic lines, we demonstrate that this polymorphism also influences stomata patterning and subject to fluctuating selection. Based on these experimental results, we construct a simple model of the miRNA824-AGL16 pathway. We show how feedback in this pathway leads to some intriguing effects on gene expression levels and we discuss how the model accounts for experimental data in different mutant lines. Our work highlights the unique adaptive properties of non-coding RNAs for the adjustment of gene expression at the post-transcriptional level and illustrates how these properties come into play in natural populations of <em>A. thaliana</em>.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="jensen"></a>Characterizing the influence of effective population size on the rate of adaptation</h3>
<p><strong>Jeffrey D. Jensen</strong><br/>University of Massachusetts Medical School</p>
<p>Characterizing the role of effective population size in dictating the rate of adaptive evolution remains a major challenge in evolutionary biology. Depending on the underlying distribution of fitness effects of new mutations, populations of different sizes may differ vastly in their rate of adaptation. Here, we collect polymorphism data across hundreds of loci for two closely related Drosophila species with different current effective population sizes (Ne), Drosophila miranda and <em>Drosophila pseudoobscura</em>, to evaluate the prevalence of adaptive evolution versus genetic drift in molecular evolution. Utilizing these large and consistently sampled datasets, we obtain greatly improved estimates of the demographic histories of both species. Specifically, while current Ne differs between these species, their ancestral sizes were much more similar. We find that statistical approaches capturing recent adaptive evolution (using patterns of polymorphisms) detect higher rates of adaptive evolution in the larger <em>D. pseudoobscura</em> population. In contrast, methods aimed at detecting selection over longer time periods (i.e., those relying on divergence data) estimate more similar rates of adaptation between the two species. Thus, our results suggest an important role of effective population size in dictating rates of adaptation and highlight how complicated population histories - as is probably the case for most species - can effect rates of adaptation. Additionally, we also show how different methodologies to detect positive selection can reveal information about different time scales of adaptive evolution.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="lassig"></a>Survival of the Flattest or the Fattest? Mutational load on complex traits</h3>
<p>Ralf Bundschuh<span class="sfbSuper">1</span>, Juliette de Meaux<span class="sfbSuper">2</span> and <strong>Michael Lässig<span class="sfbSuper">3</span></strong><br/><span class="sfbSuper">1</span>Ohio State University<br/><span class="sfbSuper">2</span>University of Münster<br/><span class="sfbSuper">3</span>University of Cologne</p>
<p>Complex molecular traits often evolve under significant mutational load, even in organisms with low point mutation rate. These system pose an old, but unresolved question: Does mutational load cause robustness against deleterious mutations? Here, we address this question for processing of miRNAs, a molecular function that is universal across plant genomes. We show that mutational load generates genomic modularity: the processing mechanism uses an extended stem in the pre-miRNA secondary structure, and mutational load shortens the DNA sequence segment encoding this stem. Moreover, mutational load directly affects the stem phenotype: it leads to more compact stem configurations with an increased average binding energy per base pair. Our analysis is based on genomic sequence data of pre-miRNAs across three Arabidopsis species. From these data, we infer an empirical, biophysically grounded fitness landscape for mi-RNA processing, and we compare the evolution of populations with different mutation rates in this landscape. A theoretical model of mutational load explains the emergence of genomic modularity and shows that it is generic for quantitative traits evolving in a complex fitness landscape. Genomic modularity, in turn, generates mutational robustness and facilitates the independent evolution of different functions. We conclude that mutational load has direct effects on genomic architecture and molecular functions.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="metzler"></a>Estimating demographic parameters from population genetic data</h3>
<p><strong>Dirk Metzler</strong><br/>LMU Munich</p>
<p>Given population genetic data from related populations or species we aim to estimate parameters like population split times, population growth rates and migration rates. Maximum-Likelihood and Bayesian methods based on importance sampling and MCMC are expected to give the best results but often need several months of computer run-time for just one dataset. More-over, some dataset require models which are not incorporated in the available implementations of these methods, and the development of such a software package may also take years due to the complex data structures involved. We discuss heuristics that are fast and easy to implement.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="mustonen"></a>Quantifying selection acting on a complex trait using allele frequency time-series data</h3>
<p><strong>Ville Mustonen</strong><br/>Welcome Trust Sanger Institute</p>
<p>When selection is acting on a large, genetically diverse population, beneﬁcial alleles increase in frequency. This fact can be used to map quantitative trait loci by sequencing the pooled DNA from the population at consecutive time points, and observing allele frequency changes. Here we present a population genetic method to analyse time-series data of allele frequencies from such an experiment. Beginning with a range of proposed evolutionary scenarios, the method measures the consistency of each with the observed frequency changes. Evolutionary theory is utilized to formulate equations of motion for the allele frequencies, following which likelihoods for having observed the sequencing data under each scenario are derived. Comparison of these likelihoods gives an insight into the prevailing dynamics of the system under study. We illustrate the method by quantifying selective effects from an experiment in which two phenotypically different yeast strains were ﬁrst crossed and then propagated under heat stress (Parts et al., Genome Res. 2011). From these data we discover that about 6% of polymorphic sites evolve non-neutrally under heat stress condition, either because of their linkage to beneﬁcial (driver) alleles or because they are drivers themselves. We further identify 44 genomic regions containing one or more candidate driver alleles, quantify their apparent selective advantage, obtain estimates of recombination rates within the regions, and show that the dynamics of the drivers display a strong signature of selection going beyond additive models. Our approach is applicable to study adaptation in a range of systems under different evolutionary pressures.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="parsch"></a>Population transcriptomics of <em>Drosophila melanogaster</em></h3>
<p><strong>John Parsch</strong><br/>LMU-Munich</p>
<p>Variation at the level of gene expression is abundant in natural populations and is thought to contribute to the adaptive divergence of populations and species. Gene expression also differs considerably between the sexes. We performed a microarray analysis of gene expression variation among males and females of 16 <em>Drosophila melanogaster</em> strains derived from natural populations, including eight strains from sub-Saharan Africa and eight strains from Europe. Females showed much less among-strain expression variation than males, but much greater between-population expression divergence than males. There was little overlap among genes expressed differently between populations in females and males, suggesting that most regulatory adaptation has been sex-specific. For some candidate genes implicated in local adaptation, including <em>Cyp6g1</em> and <em>CG9509</em>, population genetic and functional analyses indicate that linked <em>cis</em>-regulatory elements have been targets of recent positive selection.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="rosenzweig"></a>Rapid evolution of simple microbial communities in the laboratory</h3>
<p>M. Kinnersley<span class="sfbSuper">1</span>, J. Wenger<span class="sfbSuper">2</span>, G. Sherlock<span class="sfbSuper">2</span> and <strong>Frank Rosenzweig<span class="sfbSuper">1</span></strong><br/><span class="sfbSuper">1</span>University of Montana<br/><span class="sfbSuper">2</span>Stanford University School of Medicine</p>
<p>Classical models predict that asexual populations evolve in simple unstructured environments by clonal replacement, yet laboratory evolutionary studies have uncovered persistent polymorphism, driven either by frequency dependent selection or mutualistic interactions.  We have studied the evolution of microbes in simple unstructured environments as a way to illuminate the evolution of biodiversity.  Specifically, we sought to understand how complexity arises in an <em>Escherichia coli</em> population founded by a single clone and propagated under glucose limitation for nearly 800 generations.  When co-evolved clones are cultured separately their transcriptional profiles differ markedly from their common ancestor, and in ways consistent with our understanding of how <em>E. coli</em> adapts to glucose limitation.  These include increased expression of glycoporin LamB, galactose transporter MglABC.  A majority of 180 shared differentially expressed genes is controlled by global regulators RpoS, Crp and CpxR.  Clone-specific expression differences include up-regulation of genes whose products scavenge overflow metabolites such as acetate, enabling cross-feeding.  Unexpectedly, we find that when co-evolved clones are cultured together, the community expression profile more closely resembles those of minority clones cultured in isolation rather than that of the majority clone cultured in isolation.  We attribute this to feedback arising from consumption of overflow metabolites by niche specialist that feed off metabolites secreted by the dominant clone. Targeted and whole genome sequencing reveal acs, glpR and rpoS mutations in the founder that predispose evolution of niche specialists.  Subsequent mutations bringing about specialization are compensatory rather than gain-of-function.  Biocomplexity can therefore arise on a single limiting resource if consumption of that resource results in others that are differentially accessible to adaptive mutants. Altogether, our observations highlight the interplay of founder genotype, biotic environment, regulatory mutations and compensatory changes in the adaptive  tion of microbes.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="schloetterer"></a>Adaptation of <em>Drosophila</em> to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles</h3>
<p><strong>Christian Schl&ouml;tterer</strong><br/>University of Veterinary Medicine, Vienna</p>
<p>The genomic basis of adaptation to novel environments is a fundamental problem in evolutionary biology that has gained additional importance in the light of the recent global change discussion. Here we combined laboratory natural selection ("experimental evolution") in <em>Drosophila melanogaster</em> with genome-wide next generation sequencing of DNA pools ("Pool-Seq") to identify alleles that are favorable in a novel laboratory environment and traced their trajectories during the adaptive process. Already after 15 generations we identified a pronounced genomic response to selection, with 8029 single nucleotide polymorphisms (genome-wide FDR &lt;0.001%) deviating from neutral expectation. Importantly, the evolutionary trajectories of the selected alleles were heterogeneous, with the alleles falling into two distinct classes: (1) alleles that continuously rise in frequency, and (2) alleles that at first increase rapidly but whose frequencies then reach a plateau. Independent of their temporal dynamics, selected SNPs were overrepresented in introns, thereby underscoring the importance of regulatory variation for the adaptive process.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="schmid"></a>Positive and purifying selection in natural populations
of <em>Arabidopsis thaliana</em></h3>
<p><strong>Karl Schmid</strong><br/>University of Hohenheim</p>
<p>The model plant <em>Arabidopsis thaliana</em> is now an important model organism for studying plant adaptation because it occurs in a wide diversity of ecosystems throughout the Northern Hemisphere. We are contributing to analysing genome-wide patterns of variation by genome resequencing to characterize demographic processes and natural selection. The resequencing of 80 individuals from 10 different geographic regions identified nearly 5 Million SNPs as well as other numerous variants. The differences in levels of genetic variation reflect the demographic history since the last ice age. Only few genomic regions show strong species-wide or population-specific selective sweeps. In contrast, there is a highly significant negative correlation with the level of nucleotide diversity and the strength of purifying selection on putatively deleterious amino acid polymorphisms as predicted from the nearly neutral theory of natural selection. Analysis of a candidate gene for balancing selection indicates that despite its selfing breeding system, there is enough recombination to separate the selected region from its genomic neighbourhood so that footprints of selection are restricted to short genomic regions.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="sunyaev"></a>Learning from human population sequencing data</h3>
<p><strong>Shamil Sunyaev</strong><br/>Harvard Medical School</p>
<p>Rapidly advancing DNA sequencing technologies provide a complete picture of population genetic variation. Sequencing data are informative about patterns of mutations and selection in humans. Evolutionary models can inform studies aiming to uncover genetic basis of complex human traits. At the same time, sequencing studies on well-phenotyped populations are informative about evolution and architecture of complex traits.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="tautz"></a>Evolution and the genomics revolution: what are the challenges?</h3>
<p><strong>Diethard Tautz</strong><br/>MPI for Evolutionary Biology, Plön</p>
<p>I will try to trace the history of population genetic data generation and the questions that were associated with this. Based on this, I will ask what the genomics revolution has contributed to this and whether there is actually the chance to ask completely new questions that were perviously not accessible. I will add several results from our ongoing work to this.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="vekemans"></a>Combining demographic inference from population genomic data with evolutionary inference on the targets of adaptation to learn about ecological speciation in plants</h3>
<p><strong>Xavier Vekemans</strong><br/>Université Lille 1</p>
<p>Speciation is considered as "ecological" when reproductive barriers between two entities appear as a direct or indirect consequence of differential selection acting on phenotypic traits associated with contrasted environments. We introduce an original approach to identify ecological speciation a posteriori. This approach is based on a combination of a population genomic approach aimed at inferring a demographic model of divergence between two entities, and a molecular evolutionary analysis of the targets of adaptation involved in environmental specialization of the two entities. The combined approach allows comparing the timescale of the adaptation process with that of the divergence/speciation process. We apply this approach to the study of the speciation process of the species <em>Arabidopsis lyrata</em> and <em>A. halleri</em>, which show striking differences in the ability to colonize soils with high concentrations of zinc and cadmium.</p>
</div>
<p><a href="conference_2011.html"><b>back</b><img class="linkMarker" src="images/arrow_r_01.png" /></a></p>
        </div>
      </div>
    </div>
  </div>
</body>
</html>
