<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | SFB conference 2010</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="randomimage" type="text/javascript" charset="utf-8" 
    src="scripts/randomimage.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem02Active"><div 
        class="mainMenuItem02ActiveText"><a class="mainMenuLink02" 
        href="conferences.html">conferences</a></div></li>
      <li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="seminar_days.html">seminar days</a></li>
      <li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="colloquia.html">colloquia</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="journal_club.html">journal club</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h3>SFB 680 Conference</h3>
    <h2>Scientific Program</h2><div class="scheduleAbstract">
<h3><a name="ralcazar"></a>Natural variation at pathogen resistance gene drives incompatibility within <i>Arabidopsis thaliana</i> accessions</h3>
<p><strong>Rubén Alcázar</strong>, Jane Parker and Matthieu Reymond</p>
<p>Accumulation of genetic incompatibilities within species can lead to reproductive isolation and potentially, speciation. In this study, we show that allelic variation at pathogen resistance gene conditions the occurrence of incompatibility between Arabidopsis thaliana accessions. Geographical partitioning of incompatible alleles correlates with the occurrence of incompatibilities. Variation in disease susceptibility also correlates with a signature for selection at this locus. We provide a first example of a local adaptation process evolving in incompatibility within plant species.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="rbundschuh"></a>Fitness and structure landscapes for pre-miRNA processing</h3>
<p><strong>Ralf Bundschuh</strong>, Juliette de Meaux, and Michael L&auml;ssig</p><p>The processing from pre-miRNA to mature miRNA in plants involves a mechanism, which depends on an extended stem in the secondary structure of the pre-miRNA. Here, we show how natural selection acts on this secondary structure to produce evolutionary conservation
of the processing mechanism together with modularity of the pre-miRNA molecules, making this molecular function independent of others. Our main results are:</p><p>1. Selection on miRNA processing can be described by a fitness landscape which depends on the free energy &Delta;G of the processing  stem as quantitative molecular phenotype. Stem structures with &Delta;G>17kcal/mol have a constant fitness value; less stable stems incur a relative fitness cost. We infer this fitness landscape by a genome-wide analysis of Arabidopsis thaliana, comparing the stem free energy distribution of 130 functional pre-miRNAs with a background ensemble of random RNA molecules.</p><p>2. This fitness landscape (together with the overall sequence divergence) predicts the divergence of the free energy phenotype, as well as the selection coefficients of pre-miRNA sequence changes. These predictions are in excellent quantitative agreement with the results of a genome-wide cross-species comparison of orthologous pre-miRNAs between A. thaliana and A. lyrata. We conclude that there is a strong evolutionary constraint on processing phenotype and function, although individual sequence changes are only under weak selection. Our analysis shows how natural selection interacts with the complex genotype-phenotype mapping induced by RNA folding.</p><p>3. Actual pre-miRNA structures are modular: their stem free energy is significantly less affected by deleterious mutations in the remainder of the molecule than for random RNA molecules. This modularity is driven by mutational load favoring structures with more stable stems. This suggests a general evolutionary mechanism by which selection, mutations, and genetic drift generate modularity, a feature that is important for the independence of molecular functions in RNA, proteins, and extended molecular networks.</p></div>
<div class="scheduleAbstract">
<h3><a name="ccallan"></a>Deep sequencing, mutual information and the thermodynamics of gene regulation</h3>
<p><strong>Curtis Callan</strong></p>
<p>The regulation of gene expression is thought to be largely governed by the binding thermodynamics of regulatory proteins. I will argue that high-throughput sequencing, combined with a novel analysis technique based on mutual information, can give a precise thermodynamic account of gene regulatory function. I will describe the application of this method to the bacterial lac promoter, showing how one can infer the in vivo interaction energy between two proteins in the cell from sequence data alone. This approach to experiment design and data analysis is inspired by modern practice in the physical sciences.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="aclark"></a>Dysregulation of gene expression and allelic imbalance in mammalian interspecific hybrids</h3>
<p><strong>Andrew G. Clark</strong> and Xu Wang</p>
<p>Interspecific hybrids provide an excellent opportunity to study the effects of a profound but well-defined and replicable genome-wide perturbation.  To the extent that gene expression regulation has evolved in an adaptive manner, hybrid dysregulation may reveal attributes of the adaptive process.  The successful physiological function of interspecific hybrids is a testament to the robustness of many biological processes, and is surprising given the profound dysregulation at the transcriptional level (e.g. in Drosophila).  In this study we obtain a quantitative genome-wide picture of the degree of dysregulation in the horse, donkey and reciprocal hybrids, we sequenced the transcriptomes of invasive trophoblast cells (placenta) from each. This tissue was chosen because it represents a focal point of interaction between the mother and fetus in the developing placenta.  A total of 11.37 Gbps of trophoblast sequences were obtained from the four animals, and approximately 70% of the short-read sequences were uniquely mapped to the horse RefSeq database. The expression profile of both mule and hinny placenta strongly resemble the horse more than the donkey, and we found many instances of genes whose expression level in mule and hinny was outside the range of the parental horse and donkey (transgressive variation).  With the novel SNPs we discovered between horse and donkey, we have been able for the first time to quantify the allelic imbalance in these F1 hybrid mammals. Our data provide a detailed quantitative assessment of the dysregulation of gene expression in interspecific hybrids, and identify many instances of cis-acting SNPs that drive the species-specific imbalance.  Whole biological pathways generally express alleles from both species, but significant skews in horse vs. donkey allelic expression are rampant.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="jcolbourne"></a>Environmental gene regulation of an expansive aquatic arthropod genome</h3>
<p><strong>John Colbourne</strong></p>
<p>Understanding
the ecological context of genome structure, function and evolution
begins by sequencing the genomes of species that are sentinels within
ecosystems. The recent investigations by the <I>Daphnia </I>Genomics
Consortium are uncovering how this micro-crustacean's gene inventory
and regulation are products of the many challenges common in aquatic
environments. A surprising result from the <I>D. pulex </I>genome
annotation project is its impressive catalog of genes; more than 1/3
of genes have no detectable homologs in any other available proteome.
The large gene count is due to numerous gene duplications that yield
an abundance of tandem gene clusters that are specific to the <I>Daphnia
</I>lineage.</p><p>
Using tiling-path and multi-plex high-throughput microarrays that probe the entire sequenced genome, we conduct experiments that sample the transcriptome during development and reproduction, and during distinct ecological challenges. These experiments simultaneously annotate the boundaries of gene models, identify alternative transcripts, increase the gene count by over 40% and reveal the functional diversity of orphaned and expanded gene families. Our investigation of gene expression under varying environmental conditions shows that the lineage-specific genes are the most responsive to ecological challenges.</p><p>
These studies contribute to and benefit from the <a href="http://daphnia.cgb.indiana.edu/">Daphnia Genomics Consortium.</a></p>
</div>
<div class="scheduleAbstract">
<h3><a name="edekel"></a>On the evolutionary theory of cost and benefit</h3>
<p><strong>Erez Dekel</strong></p>
<p>Protein
production is a costly process that reduces cell’s fitness.
Proteins produced in an appropriate environment confer a benefit to
the cell. Here we investigate the cost and benefit of protein
production in bacteria. Cost is measured as the reduction in growth
rate when bacteria produce un-needed proteins, and benefit is
measured by the relative increase in growth. We show that bacteria
can evolutionary tune their protein production level as a function of
the environment. We use full genome sequence to find the mutations
that confer optimal cost-benefit. We also find that cost is
transient, and that bacteria can reduce cost of protein production to
almost zero. We show that bacteria reduce the cost of protein
production by using the ppGpp system of ribosomal gene regulation.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="selena"></a>Emerging viruses: changes in viral genome result in a better manipulation of host regulatory networks</h3>
<p><strong>Santiago F. Elena</strong></p>
<p>Understanding
the underlying mechanisms by which viruses are able to jump species
barriers and adapt to new hosts has been a challenging problem for
virologists because the large number of cellular factors involved in
the process and the complexity of interactions established during of
viral infectious cycle.  The classic approach has been the
identification, after tremendous efforts, of one or a few host genes
involved in the interaction.  However, the generalization of the use
of “omic” techniques for the analysis of host-virus interaction
has open for the first time the possibility of producing a whole
picture of the interaction.  In this contribution, I will present
results from an evolution experiment in which we artificially adapted
<I>Tobacco etch potyvirus</I> (TEV) to a new host, <I>Arabidopsis
thaliana</I>.  We found that a few changes in the viral genome had a
deep effect on the way the virus interacts with the regulatory
transcriptional network of the new host.  Next, we sought for
specificities and commonalities in the way different viruses interact
with a common host.  We use <I>A. thaliana</I> as model host to
analyze the sets of over/under-expressed genes from infection
experiments with the potyviruses TEV, <I>Turnip mosaic virus</I> and
<I>Plum pox virus</I>, and the phylogenetically unrelated <I>Turnip
crinkle virus</I>.  We firstly analyze data in terms of biological
functions  (GO/KEGG).  Then, taking advantage of the published
regulatory networks of <I>A. thaliana</I>, we dissect the viral mode
of action showing a directed mechanism by altering the expression of
key genes on the interactome.  The set of genes specifically
responding for phylogenetically related viruses represents
interactions that have been acquired during the evolutionary
diversification of a viral family.  While those interactions shared
by phylogenetically unrelated viruses should represent a non-specific
response of the plant to virus infection.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="lhurst"></a>Why there is more to gene evolution than protein function: splicing, nucleosomes and dual-coding sequence</h3>
<p><strong>Laurence Hurst</strong></p>
<p>There is considerable variation in the rate at which different genes evolve.  Why is this?  Classically it has been considered that the density of functionally important sites must predict rates of protein evolution.  Likewise, amino acid choice is usually assumed to reflect optimal protein function and codon choice is random.  Here I briefly present evidence suggesting that this view is too simplistic.  In particular I concentrate on how selection acting during the protein’s production history can also affect gene evolutionary rates as well as amino acid and codon choice. Exploring the role of selection at the DNA and RNA level, I specifically address how the need 1) to specify exonic splice enhancer motifs in pre-mRNA and 2) to ensure nucleosome positioning on DNA, impacts amino acid choice, codon choice and rates of evolution at both synonymous and non-synonymous sites.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="ekoonin"></a>Molecular innovations at the origin of eukaryotes</h3>
<p><strong>Eugene V. Koonin</strong></p>
<p>The origin of eukaryotes is linked to numerous and striking molecular innovations such as the endomembrane system including the nucleus with the elaborate nuclear pore complex, the distinct type of cytoskeleton, the RNA intereference system, ubiquitin signaling, and more. The emergence of these novel structures and functional systems involved elaboration of pre-existing analogous systems of archaea and bacteria, exaptation, and reassortment of diverse components, in some cases, of distinct prokaryotic origins.  I will present an overview of molecular novelties associated with eukaryogenesis and argue that many of these innovations can be linked into a single causal chain of events triggered by mitochondrial endosymbiosis.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="mkreitman"></a>Viability and Development</h3>
<p><strong>Martin Kreitman</strong></p>
<p>In <i>Drosophila</i>, as in many other organisms, mutations that disrupt developmental processes, such as embryogenesis, also reduce the ability an individual to survive to adulthood, <i>i.e.</i>, viability. Null alleles for every gene in the segmentation pathway, which establishes segmental identity in the developing fly embryo, for example, are classic recessive lethal mutations, as might be expected. Mild mutations in these same genes, however, can have both subtle effects on both gene expression and viability. But with reduced viability, some individuals survive while others die, and survivors often appear to be phenotypically “normal”. Reduced viability, in this case, might be more appropriately described as “incompletely penetrant lethality”. In this talk I explore incompletely penetrant lethality in a developmental context, building on experimental results produced by M. Ludwig and Manu with site-directed mutations in the <i>Drosophila even-skipped</i> (eve) stripe 2 enhancer. The Eve protein is fused to a yellow fluorescent reporter protein (YFP), which allows us to visualize and measure the formation of Eve stripes in live embryos, including Eve stripe 2 misexpression. We also investigate downstream consequences of these mild eve regulatory mutations, both on the segmentation process and also on viability. From these experiments, I suggest that reduced viability, a.k.a. incompletely penetrant lethality, is not property of mean Eve stripe 2 performance, but rather is a property of its variance. Based on this idea, I discuss the role of stochasticity and limits of canalization in the propagation of mutant phenotypes in complex genetic traits.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="jkrug"></a>Accessibility of mutational pathways in random and empirical fitness landscapes</h3>
<p>J. Franke, A. Kloezer, J.A.G.M. de Visser and <strong>J. Krug</strong></p>
<p>After
decades of work devoted to the mathematical modeling of fitness
landscapes, empirical data on epistatic interactions in multi-locus
systems are now becoming available for single genes as well as on the
organismic level. Such studies have shown that adaptive processes are
often remarkably constrained, in the sense that only a small fraction
of the possible evolutionary pathways connecting two genotypes are
accessible to an adaptive dynamics which requires fitness to increase
in each mutational step. Here we ask whether this behavior is
specific to the considered examples, or whether it reflects a typical
feature of multidimensional genotype spaces. To address this
question, we investigate the statistical properties of selectively
accessible paths for different models of random fitness landscapes.
We identify several generic features of the distribution of the
number of accessible paths between a genotype and its reversal
sequence and study its variation with the number of loci. The
theoretical predictions are compared to an ensemble of landscapes
derived from an empirical 8-locus data set of fitness values for the
asexual fungus Aspergillus niger. We conclude that  most features of
the data set are consistent with a model of random epistasis in which
each locus interacts with approximately 4 other loci. We also
elucidate the structure of lethal genotypes in this system.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="mlaessig"></a>Clonal interference in the evolution of influenza</h3>
<p>Natalja Strelkowa and <strong>Michael L&auml;ssig</strong></p>
<p>The seasonal inﬂuenza A virus undergoes rapid evolution to escape human immune response. Adaptive changes occur primarily in the viral epitope, the antibody- binding domain of the surface protein haemagglutinin. Here, we show that the genome of inﬂuenza A (H3N2) evolves by strong clonal interference. This mode of evolution explains its recurrent selective sweeps with clusters of simultaneous ﬁxations occurring about every 4 years. We ﬁnd an average of at least 1.3 strongly beneﬁcial aminoacid substitutions (driving mutations) per year, and a given selective sweep has about 4 driving mutations on average. Inﬂuenza’s tempo of evolution is shown to be governed not only by positive selection within the epitope, but also by background selection outside the epitope: antigenic adaptation and conservation of other viral functions interfere with each other. Our analysis is based on frequency time-series of polymorphisms, which are obtained from a coalescence tree of inﬂuenza sequences over 39 years. From these data, we estimate strength and time-dependence of natural selection acting on inﬂuenza by means of a multi-strain sequence evolution model. Our results imply that a quantitative understanding of influenza's antigenic and epidemiological dynamics will have to be based on a broader spectrum of genetically linked viral functions.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="sleibler"></a>Cooperation in bacteria: a synthetic perspective</h3>
<p><strong>Stanislas Leibler</strong></p>
<p>Synthetic microbial systems present a unique opportunity for a quantitative study of selection  in dynamic populations. I will present a short review of some classical arguments in theory of  natural selection, in particular of those connected with origins of cooperation. I will show how simple experiments with bacteria could help to make these arguments precise and to "demystify" somewhat the subject.</p><p>
This work has been done in collaboration with John Chuang, Olivier Rivoire and Edo Kussell.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="tlengauer"></a>Statistical models for predicting HIV phenotypes and effectiveness of antiretroviral therapies</h3>
<p><strong>Thomas Lengauer</strong></p>
<p>The prime problem in applying antiviral drug therapy to AIDS patients is the variability of the AIDS virus HIV. Presented with a given drug therapy the virus follows evolutionary escape paths to drug resistance, which renders the drug therapy ineffective. Statistical computer models can help estimate the level of resistance of a given viral variant against a given antiviral drug. Furthermore, such models can estimate the evolutionary path to resistance followed by the virus.</p><p>
We describe the science behind the <a href="http://www.geno2pheno.org/">geno2pheno</a> Server which offers such models freely via the internet: Given the relevant portion of an HIV genome, geno2pheno predicts the resistance of HIV to any of a number of antiviral drugs that are in clinical use. Furthermore the server ranks combination drug therapies with respect to their expected effectiveness against the given HIV variant</p><p>Entry inhibitors aim at preventing HIV from entering the host immune cell. Here, we are confronted with a different computational challenge: When entering the human host cell, HIV uses one of two coreceptor molecules on the cell surface. We present a statistical model offered via the geno2pheno server that predicts which of the two coreceptors the virus uses. Determining this “viral tropism” is essential for disease prognosis and in the context of applying the new entry inhibitors.</p><p>Both model classes are trained using various linear and nonlinear statistical learning procedures. The training data are carefully assembled databases comprising relevant genotypic, phenotypic and clinical parameters.</p><p>The geno2pheno server has been developed in the context of the Arevir consortium, a German National research consortium targeted at the bioinformatical analysis of HIV resistance data, and is currently in prototypical use for research purposes. Members of the consortium and their associated practices treat about two thirds of the AIDS patients in Germany. In recent years, data collection and method development has been lifted to a European level in the <a href="http://www.euresist.org/">EuResist project</a>.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="smaerkl"></a>Systematic characterization of transcription factor function in vivo and in vitro</h3>
<p><strong>Sebastian Maerkl</strong></p>
<p>We
have developed several high-throughput microfluidic platforms for the
quantitative analysis of transcription factors both <I>in vitro </I>and
<I>in vivo</I>. Using our recently developed MITOMI platform we are
able to obtain absolute binding affinities of transcription factors
to hundreds of target DNA sequences. More recently we have been
developing a kinetic MITOMI version capable of taking 768 on- and
off-rate measurements in parallel. For <I>in vivo </I>analysis of
transcription factor function we have employed synthetic biology to
synthesize hundreds of defined promoter constructs and developed a
microfluidic platform capable of measuring protein expression
dynamics of over 1000 yeast strains in parallel with high
spatio-temporal resolution. Our current experiments are geared
towards bridging the gap in our understanding between <I>in vitro</I>
measurements and how those quantitatively translate into <I>in vivo
</I>promoter outputs in the yeast <I>S. cerevisiae</I>.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="cmarx"></a>Diminishing
returns epistasis between genes during adaptation</h3>
<p><strong>Christopher Marx</strong></p>
<p>Genetic
interactions have proven extremely useful in inferring gene function
or physiological connections between cellular functions but little is
known regarding epistasis between beneficial mutations. We generated
all possible combinations of alleles that arose during the rapid
adaptation of a strain of <I>Methylobacterium</I> whose growth on
methanol depended upon a newly-introduced pathway. The proportional
selective benefit for three of the four loci consistently decreased
when introduced upon more fit backgrounds. These same three alleles
were all shown to reduce morphological defects that were caused by
expression of a foreign pathway. A simple model segregating the
apparent contribution of individual alleles to costs and benefits
effectively predicted the interactions between them. These results
suggest that diminishing returns epistasis may be a major component
underlying the consistent observation that the rate of fitness
increase slows down dramatically following introduction into new
environments.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="jdemeaux"></a>Maintenance of allelic polymorphism in miRNA824 processing</h3>
<p>Jin-Yong Hu, Li Lei and <strong>Juliette de Meaux</strong></p>
<p>In plants and animals, gene expression can be down-regulated at the post
transcriptional level by microRNAs (miRNAs), a class of small
endogenous RNA. Comparative analysis of miRNA content across species
indicates continuous birth and death of these loci in the course of
evolution. However, little is known about the microevolutionary
dynamics of these genetic elements, especially in plants. We examined
polymorphism at the miRNA encoding locus in  <I>Arabidopsis
thaliana</I>, miR824, which is found only in the Brassicacae. We compare their diversity to other
miRNA-encoding loci conserved across distant taxa. We find that this
locus presents a pattern of diversity deviating from neutrality:
miR824 displays signs of balancing selection. A detailed examination
of structural variation among alleles suggests non random evolution
of a thermo-resistant substructure in the precursor. Expression
analysis of pre-miR824 and its target, AGL16, indicates that these
structural differences impact the processing of mature miR824. In
order to elucidate the cause for maintenance of miRNA824 alleles, we
analyzed the phenotypic consequences of this polymorphism, both in
the greenhouse and in the field. We find that this polymorphism
impacts multiple phenotypes presumably subject to fluctuating
selection in the field.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="lmirny"></a>Evolutionary innovation and selection in cancer development</h3>
<p>Christopher McFarland and <strong>Leonid Mirny</strong></p>
<p>TThe
development of cancer can be considered as an evolutionary process
within an organism: cells acquire mutations, compete for resources,
and are subject of selection.  Somatic evolution of cancer proceeds
at an elevated mutation rate and leads to accumulation of two classes
of mutations: a few <I>driver</I> mutations, which constitute
innovations essential for cancer development; and many more random
<I>passenger</I> mutations. Our hypothesis is that passenger
mutations can be deleterious to cancer, but nevertheless hitchhike to
fixation making a population of cancer cells more vulnerable for
certain types of treatments.</p><p>We
(i) develop a computational model of cancer somatic evolution that
considers both driver and passenger mutations and is amenable to
analytical treatment; (ii) and characterize mutations found in
sequenced cancers. Simulations demonstrate that deleterious passenger
mutations can evade purifying selection and accumulate in cancer by
hitchhiking. Analysis of cancer genomes supports this finding by
demonstrating that sequenced passenger mutations are damaging to
harboring proteins. Theory and simulations show that competition
between passenger and driver mutation lead to the emergence of the
two regimes of cancer progression: one characterized by a long period
of dormancy and possible regression, and the other showing
exponential growth of a tumor. Simulated treatment, which increases
deleterious effect of passengers, leads to switching form the growth
to shrinkage and rapid cancer meltdown. Possible molecular mechanisms
that could elevate deleterious effect of passenger mutations via
inhibition of assisted protein folding are discussed.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="vmustonen"></a>Analysis of variation in molecular phenotypes</h3>
<p><strong>Ville Mustonen</strong></p>
<p>The
main objective of our group's work is to understand functional and
evolutionary properties of naturally occurring variationIn this talk
I discuss two model systems which enable us to address this
objective:</p><p>1)
Several high-throughput technologies are available to measure
interaction energies between transcription factor (TF) molecules and
their target sequences, e.g. protein binding microarrays (Badis <I>et
al</I>., 2009) and mechanically induced trapping of molecular
interactions (Maerkl and Quake, 2007). We have extracted the sequence
specific part of the interaction for two TFs from such data and used
them to analyse inter- and intra-specific variation in yeast
<I>S.cerevisiae</I> and <I>S.paradoxus.</I></p><p>2)
Protein kinases are a large and important protein family controlling
many signaling pathways and other cellular processes e.g.
transcription and apoptosis. A vast body of functional information
on, for example, domain structure of kinases exists and mutations in
kinases are known to cause disease. We have analysed somatic and
germline variation in human cancer samples, (data set from (Greenman
<I>et al. </I> 2007)), based on evolutionarily grounded scoring of
mutations combined with the existing functional annotation of these
proteins.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="cpal"></a>Causes and evolution of gene dispensability</h3>
<p><strong>Csaba Pal</strong></p>
<p>Perhaps one of the most striking discoveries of modern molecular genetics was
the extent by which organisms appear to tolerate mutations or even
complete loss of their genes. Systematic single gene deletion studies
on microbes have revealed that 70-80% of the single mutant strains
are viable with no apparent phenotypic deformation. Our
research concentrates largely on yeast (<I>S.
cerevisiae</I>) and <I>E. coli</I>, and we seek to understand the
physiological and evolutionary mechanisms behind this pattern. The
following questions sum up our research: (i) Are these seemingly
dispensable genes redundant or do they have important contribution
under special environmental conditions not yet tested in the
laboratory? (ii) How far the deleterious impacts of gene deletions can
be mitigated during evolution, and what factors limit the extent of
compensatory evolution? (iii) Is it likely that some of these genes
increase the rate of evolutionary adaptation? To address these issues, we combine evolutionary genomics with systems biology and laboratory experimental evolution protocols.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="dpetrov"></a>Evidence that mutation is universally biased towards AT in bacteria</h3>
<p><strong>Dmitri Petrov</strong></p>
<p>Mutation is the engine that drives evolution and adaptation forward in that it generates
the variation on which natural selection acts. Mutation is a random process that
nevertheless occurs according to certain biases. Elucidating mutational biases and the
way they vary across species and within genomes is crucial to understanding evolution
and adaptation. Here we demonstrate that clonal pathogens that evolve under severely
relaxed selection are uniquely suitable for studying mutational biases in bacteria. We
estimate mutational patterns using sequence datasets from five such clonal pathogens
belonging to four diverse bacterial clades that span most of the range of genomic
nucleotide content. We demonstrate that across different types of sites and in all four
clades mutation is consistently biased towards AT. This is true even in lineages that have
high genomic GC content. In all studied cases the mutational bias towards AT is
primarily due to the high rate of C/G to T/A transitions. These results suggest that
bacterial mutational biases are far less variable than previously thought. They further
demonstrate that variation in nucleotide content cannot stem entirely from variation in
mutational biases and that natural selection and/or a natural selection-like process such as
biased gene conversion strongly affect nucleotide content.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="bshraiman"></a>Recombination, interaction and the rate of adaptation</h3>
<p><strong>Boris Shraiman</strong></p>
<p>The talk will examine the effect of recombination in three closely related contexts: 1)Acceleration of the rate of adaptation in large populations, 2) Allele versus genotype selection in the presence of genetic interactions, 3) The hitch-hiking effect and "quasi-neutrality" in genetically diverse adapting populations.</p>
</div>
<div class="scheduleAbstract">
<h3><a name="ssunyaev"></a>Learning about interactions and selection from sequence data</h3>
<p><strong>Shamil Sunyaev</strong></p>
<p>Data
from the 1000 Genomes project enables a new method to characterize
deleterious genetic variation in humans . The method uses allelic age
conditioned on frequency. The effect described by Maruyama in 1974
predicts that, at a given frequency, neutral variants are older than
either deleterious or advantageous variants. Because low frequency
non-synonymous variation is expected to be enriched in deleterious
alleles, the effect predicts that non-synonymous variants are, on
average, younger than synonymous or non-coding variants of the same
frequency. Sequencing data enable new approaches for estimating
relative allelic age: mutation and recombination &quot;clocks&quot;.
The mutation clock estimates, for each variant, the number of likely
younger (less frequent and completely linked) variants. The
recombination clock estimates the length of the haplotype block on
which the variant exists. The two clocks can be combined to improve
power. Analysis of data generated by the 1000 Genomes project agrees
with theory and computer simulations. The mutation and recombination
clocks show that non-synonymous alleles are younger than synonymous
alleles. Consistently with simulations, the mutation clock is more
sensitive, and the effect is stronger for damaging variants than for
benign non-synonymous variants (we used PolyPhen-2 for predictions of
the functional effect of non-synonymous variants). The effect
disappears at higher frequencies, because few deleterious variants
reach higher frequencies. Consistently with the Maruyama effect,
population-specific variants are more damaging than variants shared
between populations (thus older), when conditioned on frequency. As
expected, however, benign variants are not significantly more
population specific than synonymous variants. We also tested presence
of epistatic selection using non-monotonic LD decay. Early results
suggest that epistasis is an important feature of natural selection
in humans</p>
</div>
<div class="scheduleAbstract">
<h3><a name="dtautz"></a>Phylostratigraphic
approaches to trace evolutionary innovations</h3>
<p>Tomislav Domazet-Lošo and <strong>Diethard Tautz</strong></p>
<p>Phylostratigraphy
is a method to correlate the evolutionary origin of founder genes of
gene families with particular macroevolutionary transitions (1, 2).
It is based on a model of genome evolution that suggests that the
origin of complex phenotypic innovations will be accompanied by the
emergence of such founder genes, the descendants of which can still
be traced in extant organisms. For example, the origin of
multicellularity can be considered to be a macroevolutionary
transition, for which new gene functions would have been required.
Cancer should be tightly connected to multicellular life since it can
be viewed as a malfunction of interaction between cells in a
multicellular organism.  A phylostratigraphic tracking of the origin
of cancer genes should therefore also provide insights into the
origin of multicellularity. There are indeed two strong peaks of
emergence of cancer related genes, one at the time of the origin of
the first cell and the other around the time of the evolution of the
multicellular metazoan organisms. These peaks correlate with two
major classes of cancer genes, the &quot;caretakers&quot;, which are
involved in general functions that support genome stability and the
&quot;gatekeepers&quot;, which are involved in cellular signaling and
growth processes. Interestingly, this phylogenetic succession mirrors
the ontogenetic succession of tumor progression, where mutations in
caretakers are thought to precede mutations in gatekeepers (3). I
will also show new data where we use phylostratigraphy to calculate a
transcriptomic age index. Plotting the age index along ontogenetic
development shows that the phylotypic stage of different organisms is
characterized by particularly old genes, while new genes have arisen
mainly in the context of juvenile and adult development. I will
discuss these findings with respect to the conrasting theories of
&quot;opportunities of selection&quot; versus &quot;developmental
constraint&quot;.</p><p>1.	Domazet-Lošo T, Brajković J, Tautz D: A phylostratigraphy approach to uncover the genomic history of major adaptations in metazoan lineages. Trends Genet 2007, 23:533–539.<br />
2.	Domazet-Lošo T, Tautz D: An ancient evolutionary origin of genes associated with human genetic diseases. Mol Biol Evol 2008, 25:2699-2707.<br />
3.	Domazet-Lošo T, Tautz D: Phylostratigraphic tracking of cancer genes suggests a link to the emergence of multicellularity in metazoa. BMC Biology (in press)</p>
</div>
<p><a href="conference_2010.html"><b>back</b><img class="linkMarker" src="images/arrow_r_01.png" /></a></p>
        </div>
      </div>
    </div>
  </div>
</body>
</html>
