<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | project area B</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="randomimage" type="text/javascript" charset="utf-8" 
    src="scripts/randomimage.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" 
        href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02Active"><div 
        class="mainMenuItem02ActiveText">project area B</div></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>Project area B: Evolution within species</h2>
        
      	<p class="projectHeader"><span class="projectName"><span class="projectNumber"><a name="b3">B3</span> Co-evolution of pathogen virulence and host defense</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td 
      class="projectLeaderCell"><a href="people.html#jhoward">Jonathan C. Howard</a>, Institut für Genetik, Universität zu Köln / Instituto Gulbenkian de Ciência</td></tr></table><div class="projectImage"><img src="projects/b3/b3_s.jpg" /></div>
      <p class="projectDesc">Polymorphic virulence proteins of the intracellular protozoal parasite, Toxoplasma gondii, interact with polymorphic resistance proteins (IRG proteins) of the mouse. We will analyse the genetic and phenotypic diversity of mouse IRG proteins and known Toxoplasma virulence factors from infected and uninfected wild mice. The results of these experiments will inform a co-evolutionary model that describes the ecological fitness interactions between host-pathogen genotype combinations and predicts the resulting evolutionary dynamics.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_b3.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      	<p><span class="projectName"><span class="projectNumber"><a name="b10">B10</span> Temperature modulation of plant immune responsiveness and fitness</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td 
      class="projectLeaderCell"><a href="people.html#jparker">Jane E. Parker</a>, Max-Planck-Institut für Züchtungsforschung, Köln</td></tr></table><div class="projectImage"><img src="projects/b10/b10_s.jpg" /></div>
      <p class="projectDesc">Mechanisms of plant environmental adaptation are poorly understood. Here we explore evolutionary and molecular processes underlying selection of immunity genes in Arabidopsis local populations. We will measure defence, growth and fitness traits associated with locally adapted immune receptor allelic forms and the influence of temperature on the trade-off between their induced resistance pathways and fitness. </p><div class="projectLinkBox"><a class="projectLink"  href="publications_b10.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      	<p><span class="projectName"><span class="projectNumber"><a name="b11">B11</span> Cost and benefit of bacterial transformation</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td 
      class="projectLeaderCell"><a href="people.html#bmaier">Berenike Maier</a>, Institut für Theoretische Physik und Biozentrum, Universität zu Köln</td></tr></table><div class="projectImage"><img src="projects/b11/b11_s.jpg" /></div>
      <p class="projectDesc">Bacterial transformation enables bacteria to exchange genetic information and can therefore speed up adaptive evolution. It is unclear however, under which conditions the benefit of transformation outweighs its cost. The goal of this project will be to experimentally quantify the costs and benefits of transformation in Neisseria gonorrhoeae. In particular we will test the influence of reversible mutations in genes that generate costs in transformable strains. In the long run, this project will contribute to our understanding of how recombination has evolved and is stably maintained.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_b11.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>

      	<p><span class="projectName"><span class="projectNumber"><a name="b12">B12</span> The adaptive significance of small-effect mutations in yeast and plants</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leader: <a href="people.html#beyer">Andreas Beyer</a>, <a href="institutions.html#genetik">Institut für Genetik, Universität zu Köln</a></td><td 
      class="projectLeaderCell">&nbsp;</td></tr></table><div class="projectImage"><img src="projects/b12/b12_s.png" /></div>
      <p class="projectDesc">Mutations often only have small effects on molecular traits such as mRNA or protein levels. Yet, many such mutations might together have evolutionary relevant effects on physiological parameters. Detecting adaptation through small effect mutations is a statistically challenging problem. We propose using innovative computational methods that combine molecular data at the level of networks to overcome the statistical difficulties associated with small effects. This approach is expected to reveal new insights into how small effect mutations translate to molecular and pathway changes that ultimately affect cellular and tissue traits.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_b12.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>

      	<p><span class="projectName"><span class="projectNumber"><a name="b13">Associated project B13</span> Adaptation of <em>Escherichia coli</em> to a complex ecosystem</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td 
      class="projectLeaderCell"><a href="people.html#igordo">Isabel Gordo</a>, <a href="institutions.html#igc">Instituto Gulbenkian de Ciência</a></td></tr></table><div class="projectImage"><img src="projects/b13/b13_s.jpg" /></div>
      <p class="projectDesc">Microbial evolution in complex ecosystems is a subject of growing interest. Laboratory mice models constitute a compromise between a truly natural environment and in vitro setups such as chemostats. We will study bacterial adaptation in the ecosystem comprising the mouse gut and focus on how mutation rate and effective population size affect genomic and phenotypic patterns of adaptive evolution. We will test predictions of theoretical models of adaptive evolution in collaboration with project C2. </p><div class="projectLinkBox"><a class="projectLink"  href="publications_b13.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>

      <h2>Past projects</h2>
      
      	<p class="projectHeader"><span class="projectName"><span class="projectNumber"><a name="b1">B1</span> Characterization of potential adaptive trait loci in house mouse populations (2006-2008)</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td 
      class="projectLeaderCell"><a href="people.html#dtautz">Diethard Tautz</a>, Max-Planck-Institut für Evolutionsbiologie, Plön</td></tr></table><div class="projectImage"><img src="projects/b1/b1_s.jpg" /></div><p class="projectDesc">Using genomic scans for signatures of positive selection in wild populations of the house mouse, we have identified a number of candidate genes that may have contributed to very recent adaptations. Some of these loci were further studied based on population genetical parameters and functional approaches. A special focus was given on changes of regulation, as well as on genes expressed in testis. A systematic comparative study on the role of positive selection in shaping expression divergence was conducted between subspecies of the mouse. A novel gene was identified that has evolved within the subgenus Mus and that appears to play a role in sperm motility.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_b1.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      	<p><span class="projectName"><span class="projectNumber"><a name="b2">B2</span> The role of regulatory changes in speciation prosesses of the house mouse 
 (2006-2008)</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td 
      class="projectLeaderCell"><a href="people.html#bharr">Bettina Harr</a>, Max-Planck-Institut für Evolutionsbiologie, Plön</td></tr></table><div class="projectImage"><img src="projects/b2/b2_s.jpg" /></div><p class="projectDesc">Under the biological species concept, the generation of reproductive isolation is synonymous with speciation. Understanding the genes that contribute to reproductive isolation is essential to understanding speciation, but isolating such genes has proven very difficult. In this study, we propose to use natural hybrid zones and expression profiling to identify genes that are miss-regulated in hybrids between two house mouse subspecies, M. m. domesticus and M. m. musculus. This approach will give an insight into both the number and kind of genes involved in reproductive isolation.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_b2.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      	<p><span class="projectName"><span class="projectNumber"><a name="b4">B4</span> Identification of Genes involved in Local Selection and Genetic Adaptation in Humans (2006-2009)</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leaders:</td><td 
      class="projectLeaderCell"><a href="people.html#pnuernberg">Peter Nürnberg</a>, Cologne Center for Genomics and Institute for Genetics, Universität zu Köln<br /><a href="people.html#mkayser">Manfred Kayser</a>, Erasmus University Medical Center, Rotterdam</td></tr></table><div class="projectImage"><img src="projects/b4/b4_s.jpg" /></div><p class="projectDesc">The genetic process under which humans have adapted to new environmental factors during their migration history remains largely unknown. We use high-density genome-wide SNP scans to pinpoint genomic regions that are likely to have been shaped by local positive selection in Pacific Islanders. Applying a battery of test statistics we are searching for partial and complete selective sweeps and estimate their statistical significance by means of simulations considering ascertainment bias as well as population histories. Candidate regions are scrutinized for genes and other functional elements that might have been the selection targets. As we want to test the thrifty gene hypothesis we are focussing on genes involved in obesity and obesity-related diseases such as diabetes type 2. The most convincing candidates will be followed-up by detailed analyses including functional studies. Verified functional variants are supposed to explain the process of recent adaptation to varying environmental sources on the molecular level.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_b4.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      	<p><span class="projectName"><span class="projectNumber"><a name="b5">B5</span> The adaptive value of pleiotropic variation: miR824 and the MADS-Box Gene Network (2006-2013)</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leaders:</td><td 
      class="projectLeaderCell"><a href="people.html#jdemeaux">Juliette de Meaux</a>, Institut für Evolution und Biodiversität, Westfälische Wilhelms-Universität Münster<br /><a href="people.html#mkoorneef">Maarten Koorneef</a>, Max-Planck-Institut für Züchtungsforschung, Köln</td></tr></table><div class="projectImage"><img src="projects/b5/b5_s.jpg" /></div><p class="projectDesc">The current challenge of evolutionary theory is to identify the molecular mechanism by which populations respond to natural selection and adapt to new environments. This project aims at elucidating the ecological cause and functional consequences of variation segregating at high frequency in the miR824 gene, encoding a microRNA known to control one member of a complex transcription factor network. For this, molecular constructs will be developed to identify the phenotypic consequences of miR824 polymorphism. Theoretical models will be constructed to explain the maintenance of polymorphism at this locus and the outcome of the model will be tested with field assays of fitness under ecologically relevant conditions. </p><div class="projectLinkBox"><a class="projectLink"  href="publications_b5.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      	<p><span class="projectName"><span class="projectNumber"><a name="b7">B7</span> Evolutionary adaptation of Daphnia to protease inhibitors in cyanobacteria (2010-2013)</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" 
      cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td 
      class="projectLeaderCell"><a href="people.html#evelert">Eric von Elert</a>, Zoologisches Institut, Universität zu Köln</td></tr></table><div class="projectImage"><img src="projects/b7/b7_s.jpg" /></div><p class="projectDesc">For <i>D. magna</i>, the most important herbivores in lakes, digestive trypsins and chymotrypsins have been shown to be the targets of dietary cyanobacterial protease inhibitors, and the respective protease genes have been identified. In a <i>D. magna</i> population it will be investigated if the seasonal appearance of protease inhibitors affects the frequency of protease alleles and what the molecular mechanisms of these micro-evolutionary changes are. In clonal cultures of <i>D. magna</i> the tolerance of protease inhibitors will be linked to mutations in and expression levels of protease genes. In a correlative approach, copy number variation in protease families will be examined for their potential source of adaptive mutations. </p><div class="projectLinkBox"><a class="projectLink"  href="publications_b7.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
        </div>
      </div>
    </div>
  </div>
</body>
</html>
