<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | project area C</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="randomimage" type="text/javascript" charset="utf-8" 
    src="scripts/randomimage.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" 
        href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02Active"><div 
        class="mainMenuItem02ActiveText">project area C</div></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>Project area C: Theory and computational analysis</h2>
      
      <p class="projectHeader"><span class="projectName"><span class="projectNumber"><a name="c1">C1</span> Evolution of gene expression</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" cellspacing="0"><tr><td class="projectLeaderHeader">Project leaders:</td><td class="projectLeaderCell"><a href="people.html#jberg">Johannes Berg</a>, Institut für Theoretische Physik, Universität zu Köln<br /><a href="people.html#mlaessig">Michael Lässig</a>, Institut für Theoretische Physik, Universität zu Köln</td></tr></table><div class="projectImage"><img src="projects/c1/c1_s.jpg" /></div><p class="projectDesc">Gene expression levels are important molecular traits of an organism. They are determined by the processes of transcription, translation, and degradation of proteins and mRNA. We will infer selection pressures on gene expression and on the underlying processes, using new transcriptomic and proteomic data in yeast. For example, we will ask how selection drives the correlation of the expression levels of the genes in a cellular pathway. The long-term goal is to achieve a comprehensive map of the evolutionary forces shaping gene expression levels in yeast.<div class="projectLinkBox"><a class="projectLink"  href="publications_c1.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      <p><span class="projectName"><span class="projectNumber"><a name="c2">C2</span> Adaptive evolution of microbial and viral systems</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td class="projectLeaderCell"><a href="people.html#mlaessig">Michael Lässig</a>, Institut für Theoretische Physik, Universität zu Köln</td></tr></table><div class="projectImage"><img src="projects/c2/c2_s.jpg" /></div><p class="projectDesc">Adaptive evolution is driven by changes in an organism's environment. We will develop quantitative models of adaptive evolution in microbial and viral systems, focusing on two systems: (a) Evolution of Escherichia coli in a mouse gut ecosystem, in collaboration with project B13. (b) Evolution of the human influenza A virus (the picture shows a tree of influenza strains over a period of seven years). We will build predictive models of influenza evolution, which establish a link to the epidemiology of this system and can inform vaccine selection.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_c2.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      <p><span class="projectName"><span class="projectNumber"><a name="c3">C3</span> Epistasis, recombination and predictability in adaptive evolution</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td class="projectLeaderCell"><a href="people.html#jkrug">Joachim Krug</a>, Institut für Theoretische Physik, Universität zu Köln<br /><a href="people.html#adevisser">Arjan de Visser</a>, Laboratory of Genetics, Wageningen University</td></tr></table><div class="projectImage"><img src="projects/c3/c3_s.jpg" /></div><p class="projectDesc">The project addresses fundamental aspects of evolutionary adaptation through a combination of population genetic theory and experimental evolution. Using the antibiotic-resistance enzyme TEM-1 β-lactamase, we study interactions between mutations increasing resistance, and construct models that explain the mapping from genotype to fitness through intermediate phenotypes. Based on such empirical fitness landscapes, we investigate the factors that determine the repeatability of evolution, including population size and recombination rate. Our long-term goal is to understand the roles of contingency and determinism in adaptation, and how they limit the predictability of evolution.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_c3.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      <p><span class="projectName"><span class="projectNumber"><a name="c4">C4</span> Evolutionary innovations – a case study on chromatin insulators</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td class="projectLeaderCell"><a href="people.html#twiehe">Thomas Wiehe</a>, Institute for Genetics, Universität zu Köln</td></tr></table><div class="projectImage"><img src="projects/c4/c4_s.jpg" /></div>
      <p class="projectDesc">Chromatin insulators play a fundamental role in the spatio-temporal expression of genes and in adaptive evolution of phenotypic traits. Interestingly, the insulator protein complement of different lineages, e.g. of arthropods and vertebrates, shows considerable variation. To understand the causes and consequences  of this variation, we will integrate macro- and micro-evolutionary analyses of genetic diversity and divergence, of gene expression profiles, of DNA- and protein-interaction networks. In addition, we will use a bioinformatics approach to search for new insulator proteins. Results are expected to reveal how these important chromatin regulators contribute to animal evolution.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_c4.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      <p><span class="projectName"><span class="projectNumber"><a name="c7">C7</span> The role of horizontal gene transfer in life style changes and metabolic adaptations</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td class="projectLeaderCell"><a href="people.html#mlercher">Martin Lercher</a>, Institut für Informatik, Heinrich-Heine-Universität Düsseldorf</td></tr></table><div class="projectImage"><img src="projects/c7/c7_s.jpg" /></div><p class="projectDesc">Anecdotal evidence shows that horizontal gene transfers, i.e., the acquisition of individual genes from other species, have boosted the metabolic and extremophilic capabilities of bacteria, archaea, and even eukaryotes. To examine this issue systematically, we will combine comparative genomics with ecological data and detailed metabolic network analyses. We will test for correlations of phenotypic data with gene transfers as well as gene duplications, thereby increasing our understanding of how these processes contributed to environmental adaptation.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_c7.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      <p><span class="projectName"><span class="projectNumber"><a name="c9">C9</span> The evolution of repetitive DNA and its methylation</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" cellspacing="0"><tr><td class="projectLeaderHeader">Project leader: <a href="people.html#tresch">Achim Tresch</a>, <a href="institutions.html#bi">Botanisches Institut, Universität zu Köln</a></td><td class="projectLeaderCell">&nbsp;</td></tr></table><div class="projectImage"><img src="projects/c9/c9_s.jpg" /></div><p class="projectDesc">Alu retrotransposons are the most abundant class of selfish, short DNA elements that proliferate by RNA-mediated reverse transcription. We will measure the evolutionary fitness of Alu sequence variants in terms of their reintegration rates into the genome. Our goal is to identify the molecular causes of evolutionary fitness, such as germline transcription, stability of Alu RNA, and its capability to reintegrate into the genome. By combining phylogenetic data with DNA methylation- , transcriptomics-, and Chromatin immunoprecipitation data, we will quantify the influence of sequence variation on these causes.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_c9.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      <h2>Past projects</h2>
      
      	<p class="projectHeader"><span class="projectName"><span class="projectNumber"><a name="c6">C6</span> Control and evolution of somite pattern formation
during early development (2006-2009)</span></p>
      <div class="projectBox"><table class="projectLeaders" cellpadding="0" cellspacing="0"><tr><td class="projectLeaderHeader">Project leader:</td><td class="projectLeaderCell"><a href="people.html#ugerland">Ulrich Gerland</a>, Ludwig-Maximilians-Universität München</td></tr></table><div class="projectImage"><img src="projects/c6/c6_s.jpg" /></div><p class="projectDesc">Vertebrate somitogenesis is being studied in a diverse set of animals, and its understanding has reached a level where theoretical modeling can provide important insights. Here, we focus on an evolutionary transition in the oscillatory expression pattern of the genes involved in the ‘somitogenesis clock’. Such a transition appears to have occurred between Zebrafish and Medaka. We will use theoretical modeling to study the phenotypic variability of the regulatory network controlling somitogenesis, and will perform experiments to characterize the effects of the regulatory evolution in key clock elements.</p><div class="projectLinkBox"><a class="projectLink"  href="publications_c6.html">publications&nbsp;<img class="menuItemPrefix" src="images/arrow_r_01.png" /></a></div><div style="clear:both;"></div></div>
      
      </div>
    </div>
  </div>
</body>
</html>
