<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div  id="contentPane"><h2>C7: Publications</h2>

	
	<div id="publication218" class="publication01">
	  <div class="publication01Title">
	    Plant and animal glycolate oxidases have a common eukaryotic ancestor and convergently duplicated to evolve long-chain 2-hydroxy acid oxidases.
	  </div>
	  <div class="publication01Info">
	    Esser C, Kuhn A, Groth G, Lercher MJ, Maurino VG. Mol Biol Evol. 2014 May;31(5):1089-101. doi: 10.1093/molbev/msu041. Epub 2014 Jan 9. PMID: 24408912 
	  </div>
	  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(218);">abstract <img id="abstractI218" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/24408912">paper [pdf]</a> &nbsp; 
	  </div>
	  <div id="abstractP218" class="publication01Abstract" style="display:none;">
	    <p>
	      Glycolate oxidase (GOX) is a crucial enzyme of plant photorespiration. The encoding gene is thought to have originated from endosymbiotic gene transfer between the eukaryotic host and the cyanobacterial endosymbiont at the base of plantae. However, animals also possess GOX activities. Plant and animal GOX belong to the gene family of (L)-2-hydroxyacid-oxidases ((L)-2-HAOX). We find that all (L)-2-HAOX proteins in animals and archaeplastida go back to one ancestral eukaryotic sequence; the sole exceptions are green algae of the chlorophyta lineage. Chlorophyta replaced the ancestral eukaryotic (L)-2-HAOX with a bacterial ortholog, a lactate oxidase that may have been obtained through the primary endosymbiosis at the base of plantae; independent losses of this gene may explain its absence in other algal lineages (glaucophyta, rhodophyta, and charophyta). We also show that in addition to GOX, plants possess (L)-2-HAOX proteins with different specificities for medium- and long-chain hydroxyacids (lHAOX), likely involved in fatty acid and protein catabolism. Vertebrates possess lHAOX proteins acting on similar substrates as plant lHAOX; however, the existence of GOX and lHAOX subfamilies in both plants and animals is not due to shared ancestry but is the result of convergent evolution in the two most complex eukaryotic lineages. On the basis of targeting sequences and predicted substrate specificities, we conclude that the biological role of plantae (L)-2-HAOX in photorespiration evolved by co-opting an existing peroxisomal protein.
	    </p>
	    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(218);">hide</a>
	    </div>
	  </div>
	</div>

	
	
	    <div id="publication8" class="publication01"><div 
  class="publication01Title">PopGenome: an efficient Swiss army knife for population genomic analyses in R</div><div 
  class="publication01Info">Pfeifer B, Wittelsbürger U, Ramos-Onsins SE,  Lercher MJ, Mol Biol Evol, 31(7):1929-36 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(8);">abstract <img id="abstractI8" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://mbe.oxfordjournals.org/content/31/7/1929.long">paper [pdf]</a> &nbsp; </div><div id="abstractP8" 
  class="publication01Abstract" style="display:none;">Although many computer programs can perform population genetics calculations, they are typically limited in the analyses and data input formats they offer; few applications can process the large data sets produced by whole-genome resequencing projects. Furthermore, there is no coherent framework for the easy integration of new statistics into existing pipelines, hindering the development and application of new population genetics and genomics approaches. Here, we present PopGenome, a population genomics package for the R software environment (a de facto standard for statistical analyses). PopGenome can efficiently process genome-scale data as well as large sets of individual loci. It reads DNA alignments and single-nucleotide polymorphism (SNP) data sets in most common formats, including those used by the HapMap, 1000 human genomes, and 1001 Arabidopsis genomes projects. PopGenome also reads associated annotation files in GFF format, enabling users to easily define regions or classify SNPs based on their annotation; all analyses can also be applied to sliding windows. PopGenome offers a wide range of diverse population genetics analyses, including neutrality tests as well as statistics for population differentiation, linkage disequilibrium, and recombination. PopGenome is linked to Hudson's MS and Ewing's MSMS programs to assess statistical significance based on coalescent simulations. PopGenome's integration in R facilitates effortless and reproducible downstream analyses as well as the production of publication-quality graphics. Developers can easily incorporate new analyses methods into the PopGenome framework. PopGenome and R are freely available from CRAN (http://cran.r-project.org/) for all major operating systems under the GNU General Public License. <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(8);">hide</a></div></div></div><div id="publication9" class="publication01"><div 
  class="publication01Title">The role of photorespiration during the evolution of C4 photosynthesis in the genus Flaveria</div><div 
  class="publication01Info">Mallmann J, Heckmann D, Bräutigam A,  Lercher MJ, Weber AP, Westhoff P, Gowik U, Elife, doi:10.7554/eLife.02478 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(9);">abstract <img id="abstractI9" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://elifesciences.org/content/3/e02478">paper [pdf]</a> &nbsp; </div><div id="abstractP9" 
  class="publication01Abstract" style="display:none;">C4 photosynthesis represents a most remarkable case of convergent evolution of a complex trait, which includes the reprogramming of the expression patterns of thousands of genes. Anatomical, physiological, and phylogenetic and analyses as well as computational modeling indicate that the establishment of a photorespiratory carbon pump (termed C2 photosynthesis) is a prerequisite for the evolution of C4. However, a mechanistic model explaining the tight connection between the evolution of C4 and C2 photosynthesis is currently lacking. Here we address this question through comparative transcriptomic and biochemical analyses of closely related C3, C3–C4, and C4 species, combined with Flux Balance Analysis constrained through a mechanistic model of carbon fixation. We show that C2 photosynthesis creates a misbalance in nitrogen metabolism between bundle sheath and mesophyll cells. Rebalancing nitrogen metabolism requires anaplerotic reactions that resemble at least parts of a basic C4 cycle. Our findings thus show how C2 photosynthesis represents a pre-adaptation for the C4 system, where the evolution of the C2 system establishes important C4 components as a side effect.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(9);">hide</a></div></div></div><div id="publication19" class="publication01"><div 
  class="publication01Title">Horizontal gene acquisitions by eukaryotes as drivers of adaptive evolution</div><div 
  class="publication01Info">Schönknecht G, Weber AP,  Lercher MJ, Bioessays, 36(1):9-20 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(19);">abstract <img id="abstractI19" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1478-3975/11/5/056003">paper [pdf]</a> &nbsp; </div><div id="abstractP19" 
  class="publication01Abstract" style="display:none;">In contrast to vertical gene transfer from parent to offspring, horizontal (or lateral) gene transfer moves genetic information between different species. Bacteria and archaea often adapt through horizontal gene transfer. Recent analyses indicate that eukaryotic genomes, too, have acquired numerous genes via horizontal transfer from prokaryotes and other lineages. Based on this we raise the hypothesis that horizontally acquired genes may have contributed more to adaptive evolution of eukaryotes than previously assumed. Current candidate sets of horizontally acquired eukaryotic genes may just be the tip of an iceberg. We have recently shown that adaptation of the thermoacidophilic red alga Galdieria sulphuraria to its hot, acid, toxic-metal laden, volcanic environment was facilitated by the acquisition of numerous genes from extremophile bacteria and archaea. Other recently published examples of horizontal acquisitions involved in adaptation include ice-binding proteins in marine algae, enzymes for carotenoid biosynthesis in aphids, and genes involved in fungal metabolism. Editor's suggested further reading in BioEssays Jumping the fine LINE between species: Horizontal transfer of transposable elements in animals catalyses genome evolution Abstract.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(19);">hide</a></div></div></div><div id="publication21" class="publication01"><div 
  class="publication01Title">Sybil--efficient constraint-based modelling in R</div><div 
  class="publication01Info">Gelius-Dietrich G, Desouki AA, Fritzemeier CJ,  Lercher MJ, BMC Syst Biol, 7:125 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(21);">abstract <img id="abstractI21" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.biomedcentral.com/1752-0509/7/125">paper [pdf]</a> &nbsp; </div><div id="abstractP21" 
  class="publication01Abstract" style="display:none;"><strong>BACKGROUND:</strong> Constraint-based analyses of metabolic networks are widely used to simulate the properties of genome-scale metabolic networks. Publicly available implementations tend to be slow, impeding large scale analyses such as the genome-wide computation of pairwise gene knock-outs, or the automated search for model improvements. Furthermore, available implementations cannot easily be extended or adapted by users.<br /><strong>RESULTS:</strong>Here, we present sybil, an open source software library for constraint-based analyses in R; R is a free, platform-independent environment for statistical computing and graphics that is widely used in bioinformatics. Among other functions, sybil currently provides efficient methods for flux-balance analysis (FBA), MOMA, and ROOM that are about ten times faster than previous implementations when calculating the effect of whole-genome single gene deletions in silico on a complete E. coli metabolic model.<strong>CONCLUSIONS:</strong><br />Due to the object-oriented architecture of sybil, users can easily build analysis pipelines in R or even implement their own constraint-based algorithms. Based on its highly efficient communication with different mathematical optimisation programs, sybil facilitates the exploration of high-dimensional optimisation problems on small time scales. Sybil and all its dependencies are open source. Sybil and its documentation are available for download from the comprehensive R archive network (CRAN).<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(21);">hide</a></div></div></div><div id="publication30" class="publication01"><div 
  class="publication01Title">Predicting C4 Photosynthesis Evolution: Modular, Individually Adaptive Steps on a Mount Fuji Fitness Landscape</div><div 
  class="publication01Info">Heckmann D, Schulze S, Denton A, Gowik U, Westhoff P, Weber APM, Lercher MJ, Cell 153, No. 6 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(30);">abstract <img id="abstractI30" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://download.cell.com/pdf/PIIS0092867413005874.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP30" 
  class="publication01Abstract" style="display:none;">An ultimate goal of evolutionary biology is the prediction and experimental verification of adaptive trajectories on macroevolutionary timescales. This aim has rarely been achieved for complex biological systems, as models usually lack clear correlates of organismal fitness. Here, we simulate the fitness landscape connecting two carbon fixation systems: C3 photosynthesis, used by most plant species, and the C4 system, which is more efficient at ambient CO2 levels and elevated temperatures and which repeatedly evolved from C3. Despite extensive sign epistasis, C4 photosynthesis is evolutionarily accessible through individually adaptive steps from any intermediate state. Simulations show that biochemical subtraits evolve in modules; the order and constitution of modules confirm and extend previous hypotheses based on species comparisons. Plant-species-designated C3-C4 intermediates lie on predicted evolutionary trajectories, indicating that they indeed represent transitory states. Contrary to expectations, we find no slowdown of adaptation and no diminishing fitness gains along evolutionary trajectories.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(30);">hide</a></div></div></div><div id="publication46" class="publication01"><div 
  class="publication01Title">Gene Transfer from Bacteria and Archaea Facilitated Evolution of an Extremophilic Eukaryote</div><div 
  class="publication01Info">Schönknecht G, Chen  WH, Ternes CM, Barbier GG, Shrestha RP, Stanke M, Bräutigam A, Baker BJ, Banfield JF, Garavito RM, Carr K, Wilkerson C, Rensing SA, Gagneul D, Dickenson NE, Oesterhelt C, Lercher MJ, Weber APM, Science, Vol. 339 no. 6124 pp. 1207-1210 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(46);">abstract <img id="abstractI46" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencemag.org/content/339/6124/1207.full.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP46" 
  class="publication01Abstract" style="display:none;">Some microbial eukaryotes, such as the extremophilic red alga <em>Galdieria sulphuraria</em>, live in hot, toxic metal-rich, acidic environments. To elucidate the underlying molecular mechanisms of adaptation, we sequenced the 13.7-megabase genome of <em>G. sulphuraria</em>. This alga shows an enormous metabolic flexibility, growing either photoautotrophically or heterotrophically on more than 50 carbon sources. Environmental adaptation seems to have been facilitated by horizontal gene transfer from various bacteria and archaea, often followed by gene family expansion. At least 5% of protein-coding genes of <em>G. sulphuraria</em> were probably acquired horizontally. These proteins are involved in ecologically important processes ranging from heavy-metal detoxification to glycerol uptake and metabolism. Thus, our findings show that a pan-domain gene pool has facilitated environmental adaptation in this unicellular eukaryote.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(46);">hide</a></div></div></div><div id="publication75" class="publication01"><div 
  class="publication01Title">Horizontal gene transfers as metagenomic gene duplications.</div><div 
  class="publication01Info">Grassi L, Caselle M, Lercher MJ, Lagomarsino MC, Molecular BioSystems 8, 790-795 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(75);">abstract <img id="abstractI75" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 22218456&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP75" 
  class="publication01Abstract" style="display:none;">While it is well accepted that horizontal gene transfer plays an important role in the evolution and the diversification of prokaryotic genomes, many questions remain open regarding its functional mechanisms of action and its interplay with the extant genome. This study addresses the relationship between proteome innovation by horizontal gene transfer and genome content in Proteobacteria. We characterize the transferred genes, focusing on the protein domain compositions and their relationships with the existing protein domain superfamilies in the genome. In agreement with previous observations, we find that the protein domain architectures of horizontally transferred genes are significantly shorter than the genomic average. Furthermore, protein domains that are more common in the total pool of genomes appear to have a proportionally higher chance to be transferred. This suggests that transfer events behave as if they were drawn randomly from a cross-genomic community gene pool, much like gene duplicates are drawn from a genomic gene pool. Finally, horizontally transferred genes carry domains of exogenous families less frequently for larger genomes, although they might do it more than expected by chance.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(75);">hide</a></div></div></div><div id="publication79" class="publication01"><div 
  class="publication01Title">OGEE: an online gene essentiality database.</div><div 
  class="publication01Info">Chen W, Minguez P, Lercher MJ, Bork P, Nucleic Acids Research 40, D901-6 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(79);">abstract <img id="abstractI79" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 22075992&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP79" 
  class="publication01Abstract" style="display:none;">OGEE is an Online GEne Essentiality database. Its main purpose is to enhance our understanding of the essentiality of genes. This is achieved by collecting not only experimentally tested essential and non-essential genes, but also associated gene features such as expression profiles, duplication status, conservation across species, evolutionary origins and involvement in embryonic development. We focus on large-scale experiments and complement our data with text-mining results. Genes are organized into data sets according to their sources. Genes with variable essentiality status across data sets are tagged as conditionally essential, highlighting the complex interplay between gene functions and environments. Linked tools allow the user to compare gene essentiality among different gene groups, or compare features of essential genes to non-essential genes, and visualize the results. OGEE is freely available at http://ogeedb.embl.de.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(79);">hide</a></div></div></div><div id="publication95" class="publication01"><div 
  class="publication01Title">An integrated approach to characterize genetic interaction networks in yeast metabolism.</div><div 
  class="publication01Info">Szappanos B, Kovacs K, Szamecz B, Honti F, Costanzo M, Baryshnikova A, Gelius-Dietrich G, Lercher MJ, Jelasity M, Myers CL, Andrews BJ et al., Nature Genetics 43, 656-662 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(95);">abstract <img id="abstractI95" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 21623372&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP95" 
  class="publication01Abstract" style="display:none;">Although experimental and theoretical efforts have been applied to globally map genetic interactions, we still do not understand how gene-gene interactions arise from the operation of biomolecular networks. To bridge the gap between empirical and computational studies, we i, quantitatively measured genetic interactions between approximately 185,000 metabolic gene pairs in Saccharomyces cerevisiae, ii, superposed the data on a detailed systems biology model of metabolism and iii, introduced a machine-learning method to reconcile empirical interaction data with model predictions. We systematically investigated the relative impacts of functional modularity and metabolic flux coupling on the distribution of negative and positive genetic interactions. We also provide a mechanistic explanation for the link between the degree of genetic interaction, pleiotropy and gene dispensability. Last, we show the feasibility of automated metabolic model refinement by correcting misannotations in NAD biosynthesis and confirming them by in vivo experiments.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(95);">hide</a></div></div></div><div id="publication106" class="publication01"><div 
  class="publication01Title">A gene&apos;s ability to buffer variation is predicted by its fitness contribution and genetic interactions.</div><div 
  class="publication01Info">Wang G, Liu J, Wang W, Zhang H, Lercher MJ, PloS One 6, e17650 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(106);">abstract <img id="abstractI106" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 21407817&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP106" 
  class="publication01Abstract" style="display:none;">BACKGROUND: Many single-gene knockouts result in increased phenotypic (e.g., morphological) variability among the mutant&apos;s offspring. This has been interpreted as an intrinsic ability of genes to buffer genetic and environmental variation. A phenotypic capacitor is a gene that appears to mask phenotypic variation: when knocked out, the offspring shows more variability than the wild type. Theory predicts that this phenotypic potential should be correlated with a gene&apos;s knockout fitness and its number of negative genetic interactions. Based on experimentally measured phenotypic capacity, it was suggested that knockout fitness was unimportant, but that phenotypic capacitors tend to be hubs in genetic and physical interaction networks. METHODOLOGY/PRINCIPAL FINDINGS: We re-analyse the available experimental data in a combined model, which includes knockout fitness and network parameters as well as expression level and protein length as predictors of phenotypic potential. Contrary to previous conclusions, we find that the strongest predictor is in fact haploid knockout fitness (responsible for 9% of the variation in phenotypic potential), with an additional contribution from the genetic interaction network (5%); once these two factors are taken into account, protein-protein interactions do not make any additional contribution to the variation in phenotypic potential. CONCLUSIONS/SIGNIFICANCE: We conclude that phenotypic potential is not a mysterious &quot;emergent&quot; property of cellular networks. Instead, it is very simply determined by the overall fitness reduction of the organism (which in its compromised state can no longer compensate for multiple factors that contribute to phenotypic variation), and by the number (and presumably nature) of genetic interactions of the knocked-out gene. In this light, Hsp90, the prototypical phenotypic capacitor, may not be representative: typical phenotypic capacitors are not direct &quot;buffers&quot; of variation, but are simply genes encoding central cellular functions.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(106);">hide</a></div></div></div><div id="publication108" class="publication01"><div 
  class="publication01Title">The effects of network neighbours on protein evolution.</div><div 
  class="publication01Info">Wang G, Lercher MJ, PloS One 6, e18288 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(108);">abstract <img id="abstractI108" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21532755&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP108" 
  class="publication01Abstract" style="display:none;">Interacting proteins may often experience similar selection pressures. Thus, we may expect that neighbouring proteins in biological interaction networks evolve at similar rates. This has been previously shown for protein-protein interaction networks. Similarly, we find correlated rates of evolution of neighbours in networks based on co-expression, metabolism, and synthetic lethal genetic interactions. While the correlations are statistically significant, their magnitude is small, with network effects explaining only between 2% and 7% of the variation. The strongest known predictor of the rate of protein evolution remains expression level. We confirmed the previous observation that similar expression levels of neighbours indeed explain their similar evolution rates in protein-protein networks, and showed that the same is true for metabolic networks. In co-expression and synthetic lethal genetic interaction networks, however, neighbouring genes still show somewhat similar evolutionary rates even after simultaneously controlling for expression level, gene essentiality and gene length. Thus, similar expression levels and related functions (as inferred from co-expression and synthetic lethal interactions) seem to explain correlated evolutionary rates of network neighbours across all currently available types of biological networks.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(108);">hide</a></div></div></div><div id="publication119" class="publication01"><div 
  class="publication01Title">Assessing the influence of adjacent gene orientation on the evolution of gene upstream regions in Arabidopsis thaliana.</div><div 
  class="publication01Info">He F, Chen W, Collins S, Acquisti C, Goebel U, Ramos-Onsins S, Lercher MJ, de Meaux J, Genetics 185, 695-701 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(119);">abstract <img id="abstractI119" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20233855&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP119" 
  class="publication01Abstract" style="display:none;">The orientation of flanking genes may influence the evolution of intergenic regions in which cis-regulatory elements are likely to be located: divergently transcribed genes share their 5&apos; regions, resulting either in smaller &quot;private&quot; spaces or in overlapping regulatory elements. Thus, upstream sequences of divergently transcribed genes (bi-directional upstream regions, or URs) may be more constrained than those of uni-directional gene pairs. We investigated this effect by analyzing nucleotide variation segregating within and between Arabidopsis species. Compared to uni-directional URs, bi-directional URs indeed display lower population mutation rate, as well as more low-frequency polymorphisms. Furthermore, we find that bi-directional regions undergo selection for the maintenance of intergenic distance. Altogether, however, we observe considerable variation in evolutionary rates, with putative signatures of selection on two uni-directional upstream regions.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(119);">hide</a></div></div></div><div id="publication128" class="publication01"><div 
  class="publication01Title">Co-expression of neighbouring genes in Arabidopsis: separating chromatin effects from direct interactions.</div><div 
  class="publication01Info">Chen W, de Meaux J, Lercher MJ, BMC Genomics 11, 178 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(128);">abstract <img id="abstractI128" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20233415&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP128" 
  class="publication01Abstract" style="display:none;">BACKGROUND:In all eukaryotic species examined, genes that are chromosomal neighbours are more similar in their expression than random gene pairs. Currently, it is still unclear how much of this local co-expression is caused by direct transcriptional interactions, and how much is due to shared chromatin environments.RESULTS:We analysed neighbouring genes in Arabidopsis thaliana. At large intergenic distances (&gt;400 bp), divergently and convergently transcribed gene pairs show very similar levels of co-expression, mediated most likely by shared chromatin environments. At gene distances below 400 bp, co-expression is strongly enhanced only for divergently transcribed gene pairs, indicating bi-directional transcription from a single promoter. Conversely, co-expression is suppressed for short convergently or uni-directionally transcribed pairs. This suppression points to transcriptional interference concentrated at the 3&apos; end, e.g., in the context of transcription termination.CONCLUSIONS:Classifying linked gene pairs by their orientation, we are able to partially tease apart the different levels of regional expression modulation. (i) Regional chromatin characteristics modulate the accessibility for regulation and transcription, regardless of gene orientation; the strength of this chromatin effect can be assessed from divergently or convergently transcribed distant neighbours. (ii) Shared promoter regions up to 400 bp in length enhance the co-expression of close bi-directional neighbours. (iii) Transcriptional interference of close neighbours is concentrated at the 3&apos; ends of genes, and reduces co-expression on average by 40%.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(128);">hide</a></div></div></div><div id="publication175" class="publication01"><div 
  class="publication01Title">Integration of horizontally transferred genes into regulatory interaction networks takes many million years.</div><div 
  class="publication01Info">Lercher MJ, Pal C, Molecular Biology And Evolution 25, 559-567 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(175);">abstract <img id="abstractI175" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 18158322&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP175" 
  class="publication01Abstract" style="display:none;">Adaptation of bacteria to new or changing environments is often associated with the uptake of foreign genes through horizontal gene transfer. However, it has remained unclear how (and how fast) new genes are integrated into their host&apos;s cellular networks. Combining the regulatory and protein interaction networks of Escherichia coli with comparative genomics tools, we provide the first systematic analysis of this issue. Genes transferred recently have fewer interaction partners compared to nontransferred genes in both regulatory and protein interaction networks. Thus, horizontally transferred genes involved in complex regulatory and protein-protein interactions are rarely favored by selection. Only few protein-protein interactions are gained after the initial integration of genes following the transfer event. In contrast, transferred genes are gradually integrated into the regulatory network of their host over evolutionary time. During adaptation to the host cellular environment, horizontally transferred genes recruit existing transcription factors of the host, reflected in the fast evolutionary rates of the cis-regulatory regions of transferred genes. Further, genes resulting from increasingly ancient transfer events show increasing numbers of transcriptional regulators as well as improved coregulation with interacting proteins. Fine-tuned integration of horizontally transferred genes into the regulatory network spans more than 8-22 million years and encompasses accelerated evolution of regulatory regions, stabilization of protein-protein interactions, and changes in codon usage.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(175);">hide</a></div></div></div><div id="publication208" class="publication01"><div 
  class="publication01Title">An integrated view of protein evolution.</div><div 
  class="publication01Info">Pal C, Papp B, Lercher MJ, Nature Reviews. Genetics 7, 337-348 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(208);">abstract <img id="abstractI208" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 16619049&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP208" 
  class="publication01Abstract" style="display:none;">Why do proteins evolve at different rates? Advances in systems biology and genomics have facilitated a move from studying individual proteins to characterizing global cellular factors. Systematic surveys indicate that protein evolution is not determined exclusively by selection on protein structure and function, but is also affected by the genomic position of the encoding genes, their expression patterns, their position in biological networks and possibly their robustness to mistranslation. Recent work has allowed insights into the relative importance of these factors. We discuss the status of a much-needed coherent view that integrates studies on protein evolution with biochemistry and functional and structural genomics.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(208);">hide</a></div></div></div><div id="publication210" class="publication01"><div 
  class="publication01Title">Chance and necessity in the evolution of minimal metabolic networks.</div><div 
  class="publication01Info">Pal C, Papp B, Lercher MJ, Csermely P, Oliver SG, Hurst LD, Nature 440, 667-670 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(210);">abstract <img id="abstractI210" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 16572170&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP210" 
  class="publication01Abstract" style="display:none;">It is possible to infer aspects of an organism&apos;s lifestyle from its gene content. Can the reverse also be done? Here we consider this issue by modelling evolution of the reduced genomes of endosymbiotic bacteria. The diversity of gene content in these bacteria may reflect both variation in selective forces and contingency-dependent loss of alternative pathways. Using an in silico representation of the metabolic network of Escherichia coli, we examine the role of contingency by repeatedly simulating the successive loss of genes while controlling for the environment. The minimal networks that result are variable in both gene content and number. Partially different metabolisms can thus evolve owing to contingency alone. The simulation outcomes do preserve a core metabolism, however, which is over-represented in strict intracellular bacteria. Moreover, differences between minimal networks based on lifestyle are predictable: by simulating their respective environmental conditions, we can model evolution of the gene content in Buchnera aphidicola and Wigglesworthia glossinidia with over 80% accuracy. We conclude that, at least for the particular cases considered here, gene content of an organism can be predicted with knowledge of its distant ancestors and its current lifestyle.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(210);">hide</a></div></div></div><div id="publication214" class="publication01"><div 
  class="publication01Title">Adaptive evolution of bacterial metabolic networks by horizontal gene transfer.</div><div 
  class="publication01Info">Pal C, Papp B, Lercher MJ, Nature Genetics 37, 1372-1375 (2005)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(214);">abstract <img id="abstractI214" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 16311593&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP214" 
  class="publication01Abstract" style="display:none;">Numerous studies have considered the emergence of metabolic pathways, but the modes of recent evolution of metabolic networks are poorly understood. Here, we integrate comparative genomics with flux balance analysis to examine (i) the contribution of different genetic mechanisms to network growth in bacteria, (ii) the selective forces driving network evolution and (iii) the integration of new nodes into the network. Most changes to the metabolic network of Escherichia coli in the past 100 million years are due to horizontal gene transfer, with little contribution from gene duplicates. Networks grow by acquiring genes involved in the transport and catalysis of external nutrients, driven by adaptations to changing environments. Accordingly, horizontally transferred genes are integrated at the periphery of the network, whereas central parts remain evolutionarily stable. Genes encoding physiologically coupled reactions are often transferred together, frequently in operons. Thus, bacterial metabolic networks evolve by direct uptake of peripheral reactions in response to changed environments.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(214);">hide</a></div></div></div>
	    <p><a href="project_area_c.html">back to project area C</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
