<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>B4: Publications</h2>
	    <div class="publication01"><div 
  class="publication01Title">Demographic history of Oceania inferred from genome-wide data.</div><div 
  class="publication01Info">Wollstein A, Lao O, Becker C, Brauer S, Trent RJ, Nürnberg P, Stoneking M, Kayser M, Current Biology 20, 1983-1992 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(82);">abstract <img id="abstractI82" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21074440&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP82" 
  class="publication01Abstract" style="display:none;">BACKGROUND:The human history of Oceania comprises two extremes: the initial colonizations of Near Oceania, one of the oldest out-of-Africa migrations, and of Remote Oceania, the most recent expansion into unoccupied territories. Genetic studies, mostly using uniparentally inherited DNA, have shed some light on human origins in Oceania, particularly indicating that Polynesians are of mixed East Asian and Near Oceanian ancestry. Here, we use ∼1 million single nucleotide polymorphisms (SNPs) to investigate the demographic history of Oceania in a more detailed manner.RESULTS:We developed a new approach to account for SNP ascertainment bias, used approximate Bayesian computation simulations to choose the best-fitting model of population history, and estimated demographic parameters. We find that the ancestors of Near Oceanians diverged from ancestral Eurasians ∼27 thousand years ago (kya), suggesting separate initial occupations of both territories. The genetic admixture in Polynesian history between East Asians (∼87%) and Near Oceanians (∼13%) occurred ∼3 kya, prior to the colonization of Polynesia. Fijians are of Polynesian (∼65%) and additional Near Oceanian (∼35%) ancestry not found in Polynesians, with this admixture occurring considerably after the initial settlement of Remote Oceania. Our data support a greater contribution of East Asian women than men in the admixture history of Remote Oceania and highlight population substructure in Polynesia and New Guinea.CONCLUSIONS:Despite the inherent ascertainment bias, genome-wide SNP data provide new insights into the genetic history of Oceana. Our approach to correct for ascertainment bias and obtain reliable inferences concerning demographic history should prove useful in other such studies.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(82);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Human Relationships Inferred from Genetic Variation</div><div 
  class="publication01Info">Lao O, Kayser M, Wiley Online Library (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(111);">abstract <img id="abstractI111" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sfb680.uni-koeln.de/docs/Lao_Kayser_encyclopedia.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP111" 
  class="publication01Abstract" style="display:none;">We humans are a diverse species, both at the phenotypic and the genetic levels. The genetic diversity of current human populations has been shaped by various demographic and evolutionary as well as some cultural factors. Thus, studying how genetic variation is distributed through individuals around the world can provide insights into (i) when and where our human ancestors first appeared on the planet, (ii) the dynamics of admixture with other Homo species not existing anymore, (iii) migration waves that brought humans across the world and (iv) processes of adaptations towards environmental and other factors that shaped human genomes and phenotypic traits. Additionally, the quantification of genetic variation between human populations provides data evidence for elucidating whether humans can be classified according to genetically homogeneous groups or not, with implications for medical and forensic studies. Hence, studying human genetic diversity is important for better understanding of our past, presence and future<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(111);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Genome-wide analysis indicates more Asian than Melanesian ancestry of Polynesians.</div><div 
  class="publication01Info">Kayser M, Lao O, Saar K, Brauer S, Wang X, Nürnberg P, Trent RJ, Stoneking M, American Journal Of Human Genetics 82, 194-198 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(147);">abstract <img id="abstractI147" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18179899&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP147" 
  class="publication01Abstract" style="display:none;">Analyses of mitochondrial DNA (mtDNA) and nonrecombining Y chromosome (NRY) variation in the same populations are sometimes concordant but sometimes discordant. Perhaps the most dramatic example known of the latter concerns Polynesians, in which about 94% of Polynesian mtDNAs are of East Asian origin, while about 66% of Polynesian Y chromosomes are of Melanesian origin. Here we analyze on a genome-wide scale, to our knowledge for the first time, the origins of the autosomal gene pool of Polynesians by screening 377 autosomal short tandem repeat (STR) loci in 47 Pacific Islanders and compare the results with those obtained from 44 Chinese and 24 individuals from Papua New Guinea. Our data indicate that on average about 79% of the Polynesian autosomal gene pool is of East Asian origin and 21% is of Melanesian origin. The genetic data thus suggest a dual origin of Polynesians with a high East Asian but also considerable Melanesian component, reflecting sex-biased admixture in Polynesian history in agreement with the Slow Boat model. More generally, these results also demonstrate that conclusions based solely on uniparental markers, which are frequently used in population history studies, may not accurately reflect the history of the autosomal gene pool of a population.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(147);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Identification of a candidate genetic variant for the high prevalence of type II diabetes in Polynesians.</div><div 
  class="publication01Info">Myles S, Hradetzky E, Engelken J, Lao O, Nürnberg P, Trent RJ, Wang X, Kayser M, Stoneking M, European Journal Of Human Genetics 15, 584-589 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(159);">abstract <img id="abstractI159" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17327880&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP159" 
  class="publication01Abstract" style="display:none;">The prevalence of non-insulin-dependent diabetes mellitus (type II diabetes) in Polynesia is among the highest recorded worldwide and is substantially higher than in neighboring human populations. Such large differences in the frequency of a phenotype between populations may be explained by large allele frequency differences between populations in genes associated with the phenotype. To identify genes that may explain the high between-population variation in type II diabetes prevalence in the Pacific, we determined the frequency of 10 type II diabetes-associated alleles in 23 Polynesians, 23 highland New Guineans and 19 Han Chinese, calculated population-pairwise Fst values for each allele and compared these values to the distribution of Fst values from approximately 100,000 SNPs from the same populations. The susceptibility allele in the PPARGC1A gene is at a frequency of 0.717 in Polynesians, 0.368 in Chinese but is absent in the New Guineans. The striking frequency difference between Polynesians and New Guineans is highly unusual (Fst=0.703, P=0.007) and we therefore suggest that this allele may play a role in the large difference in type II diabetes prevalence between Polynesians and neighboring populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(159);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Melanesian and Asian origins of Polynesians: mtDNA and Y chromosome gradients across the Pacific.</div><div 
  class="publication01Info">Kayser M, Brauer S, Cordaux R, Casto A, Lao O, Zhivotovsky LA, Moyse-Faurie C, Rutledge RB, Schiefenhoevel W, Gil D, Lin AA et al., Molecular Biology And Evolution 23, 2234-2244 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(172);">abstract <img id="abstractI172" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16923821&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP172" 
  class="publication01Abstract" style="display:none;">The human settlement of the Pacific Islands represents one of the most recent major migration events of mankind. Polynesians originated in Asia according to linguistic evidence or in Melanesia according to archaeological evidence. To shed light on the genetic origins of Polynesians, we investigated over 400 Polynesians from 8 island groups, in comparison with over 900 individuals from potential parental populations of Melanesia, Southeast and East Asia, and Australia, by means of Y chromosome (NRY) and mitochondrial DNA (mtDNA) markers. Overall, we classified 94.1% of Polynesian Y chromosomes and 99.8% of Polynesian mtDNAs as of either Melanesian (NRY-DNA: 65.8%, mtDNA: 6%) or Asian (NRY-DNA: 28.3%, mtDNA: 93.8%) origin, suggesting a dual genetic origin of Polynesians in agreement with the &quot;Slow Boat&quot; hypothesis. Our data suggest a pronounced admixture bias in Polynesians toward more Melanesian men than women, perhaps as a result of matrilocal residence in the ancestral Polynesian society. Although dating methods are consistent with somewhat similar entries of NRY/mtDNA haplogroups into Polynesia, haplotype sharing suggests an earlier appearance of Melanesian haplogroups than those from Asia. Surprisingly, we identified gradients in the frequency distribution of some NRY/mtDNA haplogroups across Polynesia and a gradual west-to-east decrease of overall NRY/mtDNA diversity, not only providing evidence for a west-to-east direction of Polynesian settlements but also suggesting that Pacific voyaging was regular rather than haphazard. We also demonstrate that Fiji played a pivotal role in the history of Polynesia: humans probably first migrated to Fiji, and subsequent settlement of Polynesia probably came from Fiji.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(172);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">A genome scan to detect candidate regions influenced by local natural selection in human populations.</div><div 
  class="publication01Info">Kayser M, Brauer S, Stoneking M, Molecular Biology And Evolution 20, 893-900 (2003)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(193);">abstract <img id="abstractI193" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=12717000&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP193" 
  class="publication01Abstract" style="display:none;">As human populations dispersed throughout the world, they were subjected to new selective forces, which must have led to local adaptation via natural selection and hence altered patterns of genetic variation. Yet, there are very few examples known in which such local selection has clearly influenced human genetic variation. A potential approach for detecting local selection is to screen random loci across the genome; those loci that exhibit unusually large genetic distances between human populations are then potential markers of genomic regions under local selection. We investigated this approach by genotyping 332 short tandem repeat (STR) loci in Africans and Europeans and calculating the genetic differentiation for each locus. Patterns of genetic diversity at these loci were consistent with greater variation in Africa and with local selection operating on populations as they moved out of Africa. For 11 loci exhibiting the largest genetic differences, we genotyped an additional STR locus located nearby; the genetic distances for these nearby loci were significantly larger than average. These genomic regions therefore reproducibly exhibit larger genetic distances between populations than the &quot;average&quot; genomic region, consistent with local selection. Our results demonstrate that genome scans are a promising means of identifying candidate regions that have been subjected to local selection.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(193);">hide</a></div></div></div>
	    <p><a href="project_area_b.html">back to project area B</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
