<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>B5: Publications</h2>
	    <div class="publication01"><div 
  class="publication01Title">How epigenetic mutations can affect genetic evolution: Model and mechanism</div><div 
  class="publication01Info">Klironomos F, Berg J, Collins S, Bioessays, 35 (6), 571-578 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(5);">abstract <img id="abstractI5" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1002/bies.201200169/pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP5" 
  class="publication01Abstract" style="display:none;">We hypothesize that heritable epigenetic changes can affect rates of fitness increase as well as patterns of genotypic and phenotypic change during adaptation. In particular, we suggest that when natural selection acts on pure epigenetic variation in addition to genetic variation, populations adapt faster, and adaptive phenotypes can arise before any genetic changes. This may make it difficult to reconcile the timing of adaptive events detected using conventional population genetics tools based on DNA sequence data with environmental drivers of adaptation, such as changes in climate. Epigenetic modifications are frequently associated with somatic cell differentiation, but recently epigenetic changes have been found that can be transmitted over many generations. Here, we show how the interplay of these heritable epigenetic changes with genetic changes can affect adaptive evolution, and how epigenetic changes affect the signature of selection in the genetic record.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(5);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Co-variation of germination and growth related traits across the European latitudinal range of Arabidopsis thaliana</div><div 
  class="publication01Info">Debieu M, Tang C, Stich B, Effgen S, Josephs E, Schmitt J, Nordborg M, Koornneef M, de Meaux J, PlosONE 8(5): e61075 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(6);">abstract <img id="abstractI6" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plosone.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0061075&representation=PDF">paper [pdf]</a> &nbsp; </div><div id="abstractP6" 
  class="publication01Abstract" style="display:none;">Life-history traits controlling the duration and timing of developmental phases in the life cycle jointly determine fitness. Therefore, life-history traits studied in isolation provide an incomplete view on the relevance of life-cycle variation for adaptation. In this study, we examine genetic variation in traits covering the major life history events of the annual species Arabidopsis thaliana: seed dormancy, vegetative growth rate and flowering time. In a sample of 112 genotypes collected throughout the European range of the species, both seed dormancy and flowering time follow a latitudinal gradient independent of the major population structure gradient. This finding confirms previous studies reporting the adaptive evolution of these two traits. Here, however, we further analyze patterns of co-variation among traits. We observe that co-variation between primary dormancy, vegetative growth rate and flowering time also follows a latitudinal cline. At higher latitudes, vegetative growth rate is positively correlated with primary dormancy and negatively with flowering time. In the South, this trend disappears. Patterns of trait co-variation change, presumably because major environmental gradients shift with latitude. This pattern appears unrelated to population structure, suggesting that changes in the coordinated evolution of major life history traits is adaptive. Our data suggest that A. thaliana provides a good model for the evolution of trade-offs and their genetic basis.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(6);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Flagellin perception varies quantitatively in Arabidopsis thaliana and its relatives.</div><div 
  class="publication01Info">Vetter MM, Kronholm I, He F, Häweker H, Reymond M, Bergelson J, Robatzek S, de Meaux J, Molecular Biology And Evolution 29, 1655-1667 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(36);">abstract <img id="abstractI36" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22319159&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP36" 
  class="publication01Abstract" style="display:none;">Much is known about the evolution of plant immunity components directed against specific pathogen strains: They show pervasive functional variation and have the potential to coevolve with pathogen populations. However, plants are effectively protected against most microbes by generalist immunity components that detect conserved pathogen-associated molecular patterns (PAMPs) and control the onset of PAMP-triggered immunity. In Arabidopsis thaliana, the receptor kinase flagellin sensing 2 (FLS2) confers recognition of bacterial flagellin (flg22) and activates a manifold defense response. To decipher the evolution of this system, we performed functional assays across a large set of A. thaliana genotypes and Brassicaceae relatives. We reveal extensive variation in flg22 perception, most of which results from changes in protein abundance. The observed variation correlates with both the severity of elicited defense responses and bacterial proliferation. We analyzed nucleotide variation segregating at FLS2 in A. thaliana and detected a pattern of variation suggestive of the rapid fixation of a novel adaptive allele. However, our study also shows that evolution at the receptor locus alone does not explain the evolution of flagellin perception; instead, components common to pathways downstream of PAMP perception likely contribute to the observed quantitative variation. Within and among close relatives, PAMP perception evolves quantitatively, which contrasts with the changes in recognition typically associated with the evolution of R genes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(36);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Genetic basis of adaptation in Arabidopsis thaliana: local adaptation at the seed dormancy QTL DOG1.</div><div 
  class="publication01Info">Kronholm I, Picó FX, Alonso-Blanco C, Goudet J, de Meaux J, Evolution 66, 2287-2302 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(37);">abstract <img id="abstractI37" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22759302&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP37" 
  class="publication01Abstract" style="display:none;">Local adaptation provides an opportunity to study the genetic basis of adaptation and investigate the allelic architecture of adaptive genes. We study delay of germination 1 (DOG1), a gene controlling natural variation in seed dormancy in Arabidopsis thaliana and investigate evolution of dormancy in 41 populations distributed in four regions separated by natural barriers. Using F(ST) and Q(ST) comparisons, we compare variation at DOG1 with neutral markers and quantitative variation in seed dormancy. Patterns of genetic differentiation among populations suggest that the gene DOG1 contributes to local adaptation. Although Q(ST) for seed dormancy is not different from F(ST) for neutral markers, a correlation with variation in summer precipitation supports that seed dormancy is adaptive. We characterize dormancy variation in several F(2) -populations and show that a series of functionally distinct alleles segregate at the DOG1 locus. Theoretical models have shown that the number and effect of alleles segregatin at quantitative trait loci (QTL) have important consequences for adaptation. Our results provide support to models postulating a large number of alleles at quantitative trait loci involved in adaptation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(37);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Widespread interspecific divergence in cis-regulation of transposable elements in the Arabidopsis genus.</div><div 
  class="publication01Info">He F, Zhang X, Hu J, Turck F, Dong X, Goebel U, Borevitz JO, de Meaux J, Molecular Biology And Evolution 29, 1081-1091 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(46);">abstract <img id="abstractI46" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22086904&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP46" 
  class="publication01Abstract" style="display:none;">Transposable elements (TEs) are so abundant and variable that they count among the most important mutational sources in genomes. Nonetheless, little is known about the genetics of their variation in activity or silencing across closely related species. Here, we demonstrate that regulation of TE genes can differ dramatically between the two closely related Arabidopsis species A. thaliana and A. lyrata. In leaf and floral tissues of F1 interspecific hybrids, about 47% of TEs show allele-specific expression, with the A. lyrata copy being generally expressed at higher level. We confirm that TEs are generally expressed in A. lyrata but not in A. thaliana. Allele-specific differences in TE expression are associated with divergence in epigenetic modifications like DNA and histone methylation between species as well as with sequence divergence. Our data demonstrate that A. thaliana silences TEs much better than A. lyrata. For long terminal repeat retrotransposons, these differences are more pronounced for younger insertions. Interspecific differences in TE silencing may have a great impact on genome size changes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(46);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">DOG1 expression is predicted by the seed-maturation environment and contributes to geographical variation in germination in <em>Arabidopsis thaliana</em></div><div 
  class="publication01Info">Chiang GCK, Bartsch M, Barua D, Nakayabashi K, Debieu M, Kronholm I, Koornneef M, Soppe WJJ, Donohue K, de Meaux J, Molecular Ecology, 20: 3336-3349 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(60);">abstract <img id="abstractI60" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05181.x/pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP60" 
  class="publication01Abstract" style="display:none;">Seasonal germination timing of Arabidopsis thaliana strongly influences overall life history expression and is the target of intense natural selection. This seasonal germination timing depends strongly on the interaction between genetics and seasonal environments both before and after seed dispersal. DELAY OF GERMINATION 1 (DOG1) is the first gene that has been identified to be associated with natural variation in primary dormancy in A. thaliana. Here, we report interaccession variation in DOG1 expression and document that DOG1 expression is associated with seed-maturation temperature effects on germination; DOG1 expression increased when seeds were matured at low temperature, and this increased expression was associated with increased dormancy of those seeds. Variation in DOG1 expression suggests a geographical structure such that southern accessions, which are more dormant, tend to initiate DOG1 expression earlier during seed maturation and achieved higher expression levels at the end of silique development than did northern accessions. Although elimination of the synthesis of phytohormone abscisic acid (ABA) results in the elimination of maternal temperature effects on dormancy, DOG1 expression predicted dormancy better than expression of genes involved in ABA metabolism.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(60);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Genetic and evolutionary perspectives on the interplay between plant immunity and development.</div><div 
  class="publication01Info">Alcázar R, Reymond M, Schmitz G, de Meaux J, Current Opinion In Plant Biology 14, 378-384 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(62);">abstract <img id="abstractI62" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21561797&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP62" 
  class="publication01Abstract" style="display:none;">There is now ample evidence that plant development, responses to abiotic environments, and immune responses are tightly intertwined in their physiology. Thus optimization of the immune system during evolution will occur in coordination with that of plant development. Two alternative and possibly complementary forces are at play: genetic constraints due to the pleiotropic action of players in both systems, and coevolution, if developmental changes modulate the cost-benefit balance of immunity. A current challenge is to elucidate the ecological forces driving evolution of quantitative variation for defense at molecular level. The analysis of natural co-variation for developmental and immunity traits in Arabidopsis thaliana promises to bring important insights.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(62);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Natural variation at Strubbelig Receptor Kinase 3 drives immune-triggered incompatibilities between Arabidopsis thaliana accessions.</div><div 
  class="publication01Info">Alcázar R, García AV, Kronholm I, de Meaux J, Koornneef M, Parker JE, Reymond M, Nature Genetics 42, 1135-1139 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(80);">abstract <img id="abstractI80" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21037570&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP80" 
  class="publication01Abstract" style="display:none;">Accumulation of genetic incompatibilities within species can lead to reproductive isolation and, potentially, speciation. In this study, we show that allelic variation at SRF3 (Strubbelig Receptor Family 3), encoding a receptor-like kinase, conditions the occurrence of incompatibility between Arabidopsis thaliana accessions. The geographical distribution of SRF3 alleles reveals that allelic forms causing epistatic incompatibility with a Landsberg erecta allele at the RPP1 resistance locus are present in A. thaliana accessions in central Asia. Incompatible SRF3 alleles condition for an enhanced early immune response to pathogens as compared to the resistance-dampening effect of compatible SRF3 forms in isogenic backgrounds. Variation in disease susceptibility suggests a basis for the molecular patterns of a recent selective sweep detected at the SRF3 locus in central Asian populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(80);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.</div><div 
  class="publication01Info">Atwell S, Huang YS, Vilhjálmsson BJ, Willems G, Horton M, Li Y, Meng D, Platt A, Tarone AM, Hu TT, Jiang R et al., Nature 465, 627-631 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(88);">abstract <img id="abstractI88" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20336072&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP88" 
  class="publication01Abstract" style="display:none;">Although pioneered by human geneticists as a potential solution to the challenging problem of finding the genetic basis of common human diseases, genome-wide association (GWA) studies have, owing to advances in genotyping and sequencing technology, become an obvious general approach for studying the genetics of natural variation and traits of agricultural importance. They are particularly useful when inbred lines are available, because once these lines have been genotyped they can be phenotyped multiple times, making it possible (as well as extremely cost effective) to study many different traits in many different environments, while replicating the phenotypic measurements to reduce environmental noise. Here we demonstrate the power of this approach by carrying out a GWA study of 107 phenotypes in Arabidopsis thaliana, a widely distributed, predominantly self-fertilizing model plant known to harbour considerable genetic variation for many adaptively important traits. Our results are dramatically different from those of human GWA studies, in that we identify many common alleles of major effect, but they are also, in many cases, harder to interpret because confounding by complex genetics and population structure make it difficult to distinguish true associations from false. However, a-priori candidates are significantly over-represented among these associations as well, making many of them excellent candidates for follow-up experiments. Our study demonstrates the feasibility of GWA studies in A. thaliana and suggests that the approach will be appropriate for many other organisms.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(88);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Assessing the influence of adjacent gene orientation on the evolution of gene upstream regions in Arabidopsis thaliana.</div><div 
  class="publication01Info">He F, Chen W, Collins S, Acquisti C, Goebel U, Ramos-Onsins S, Lercher MJ, de Meaux J, Genetics 185, 695-701 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(89);">abstract <img id="abstractI89" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20233855&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP89" 
  class="publication01Abstract" style="display:none;">The orientation of flanking genes may influence the evolution of intergenic regions in which cis-regulatory elements are likely to be located: divergently transcribed genes share their 5&apos; regions, resulting either in smaller &quot;private&quot; spaces or in overlapping regulatory elements. Thus, upstream sequences of divergently transcribed genes (bi-directional upstream regions, or URs) may be more constrained than those of uni-directional gene pairs. We investigated this effect by analyzing nucleotide variation segregating within and between Arabidopsis species. Compared to uni-directional URs, bi-directional URs indeed display lower population mutation rate, as well as more low-frequency polymorphisms. Furthermore, we find that bi-directional regions undergo selection for the maintenance of intergenic distance. Altogether, however, we observe considerable variation in evolutionary rates, with putative signatures of selection on two uni-directional upstream regions.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(89);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">The earliest stages of adaptation in an experimental plant population: strong selection on QTLS for seed dormancy.</div><div 
  class="publication01Info">Huang X, Schmitt J, Dorn L, Griffith C, Effgen S, Takao S, Koornneef M, Donohue K, Molecular Ecology 19, 1335-1351 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(93);">abstract <img id="abstractI93" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20149097&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP93" 
  class="publication01Abstract" style="display:none;">Colonizing species may often encounter strong selection during the initial stages of adaptation to novel environments. Such selection is particularly likely to act on traits expressed early in development since early survival is necessary for the expression of adaptive phenotypes later in life. Genetic studies of fitness under field conditions, however, seldom include the earliest developmental stages. Using a new set of recombinant inbred lines, we present a study of the genetic basis of fitness variation in Arabidopsis thaliana in which genotypes, environments, and geographic location were manipulated to study total lifetime fitness, beginning with the seed stage. Large-effect quantitative trait loci (QTLs) for fitness changed allele frequency and closely approached 90% in some treatments within a single generation. These QTLs colocated with QTLs for germination phenology when seeds were dispersed following a schedule of a typical winter annual, and they were detected in two geographic locations at different latitudes. Epistatically interacting loci affected both fitness and germination in many cases. QTLs for field germination phenology colocated with known QTLs for primary dormancy induction as assessed in laboratory tests, including the candidate genes DOG1 and DOG6. Therefore fitness, germination phenology, and primary dormancy are genetically associated at the level of specific chromosomal regions and candidate loci. Genes associated with the ability to arrest development at early life stages and assess environmental conditions are thereby likely targets of intense natural selection early in the colonization process.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(93);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Co-expression of neighbouring genes in Arabidopsis: separating chromatin effects from direct interactions.</div><div 
  class="publication01Info">Chen W, de Meaux J, Lercher MJ, BMC Genomics 11, 178 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(98);">abstract <img id="abstractI98" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20233415&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP98" 
  class="publication01Abstract" style="display:none;">BACKGROUND:In all eukaryotic species examined, genes that are chromosomal neighbours are more similar in their expression than random gene pairs. Currently, it is still unclear how much of this local co-expression is caused by direct transcriptional interactions, and how much is due to shared chromatin environments.RESULTS:We analysed neighbouring genes in Arabidopsis thaliana. At large intergenic distances (&gt;400 bp), divergently and convergently transcribed gene pairs show very similar levels of co-expression, mediated most likely by shared chromatin environments. At gene distances below 400 bp, co-expression is strongly enhanced only for divergently transcribed gene pairs, indicating bi-directional transcription from a single promoter. Conversely, co-expression is suppressed for short convergently or uni-directionally transcribed pairs. This suppression points to transcriptional interference concentrated at the 3&apos; end, e.g., in the context of transcription termination.CONCLUSIONS:Classifying linked gene pairs by their orientation, we are able to partially tease apart the different levels of regional expression modulation. (i) Regional chromatin characteristics modulate the accessibility for regulation and transcription, regardless of gene orientation; the strength of this chromatin effect can be assessed from divergently or convergently transcribed distant neighbours. (ii) Shared promoter regions up to 400 bp in length enhance the co-expression of close bi-directional neighbours. (iii) Transcriptional interference of close neighbours is concentrated at the 3&apos; ends of genes, and reduces co-expression on average by 40%.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(98);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Influence of mutation rate on estimators of genetic differentiation--lessons from Arabidopsis thaliana.</div><div 
  class="publication01Info">Kronholm I, Loudet O, de Meaux J, BMC Genetics 11, 33 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(102);">abstract <img id="abstractI102" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20433762&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP102" 
  class="publication01Abstract" style="display:none;">BACKGROUND:The analysis of molecular variation within and between populations is crucial to establish strategies for conservation as well as to detect the footprint of spatially heterogeneous selection. The traditional estimator of genetic differentiation (F(ST)) has been shown to be misleading if genetic diversity is high. Alternative estimators of F(ST) have been proposed, but their robustness to variation in mutation rate is not clearly established. We first investigated the effect of mutation and migration rate using computer simulations and examined their joint influence on Q(ST), a measure of genetic differentiation for quantitative traits. We further used experimental data in natural populations of Arabidopsis thaliana to characterize the effect of mutation rate on various estimates of population differentiation. Since natural species exhibit various degrees of self-fertilisation, we also investigated the effect of mating system on the different estimators.RESULTS:If mutation rate is high and migration rate low, classical measures of genetic differentiation are misleading. Only Phi(ST), an estimator that takes the mutational distances between alleles into account, is independent of mutation rate, for all migration rates. However, the performance of Phi(ST) depends on the underlying mutation model and departures from this model cause its performance to degrade. We further show that Q(ST) has the same bias. We provide evidence that, in A. thaliana, microsatellite variation correlates with mutation rate. We thereby demonstrate that our results on estimators of genetic differentiation have important implications, even for species that are well established models in population genetics and molecular biology.CONCLUSIONS:We find that alternative measures of differentiation like F&apos;(ST) and D are not suitable for estimating effective migration rate and should not be used in studies of local adaptation. Genetic differentiation should instead be measured using an estimator that takes mutation rate into account, such as Phi(ST). Furthermore, in systems where migration between populations is low, such as A. thaliana, Q(ST) &lt; F(ST) cannot be taken as evidence for homogenising selection as has been traditionally thought.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(102);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Structurally different alleles of the ath-MIR824 microRNA precursor are maintained at high frequency in Arabidopsis thaliana.</div><div 
  class="publication01Info">de Meaux J, Hu J, Tartler U, Goebel U, Proceedings Of The National Academy Of Sciences Of The United States Of America 105, 8994-8999 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(138);">abstract <img id="abstractI138" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18579782&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP138" 
  class="publication01Abstract" style="display:none;">In plants and animals, gene expression can be down-regulated at the posttranscriptional level by microRNAs (miRNAs), a class of small endogenous RNA. Comparative analysis of miRNA content across species indicates continuous birth and death of these loci in the course of evolution. However, little is known about the microevolutionary dynamics of these genetic elements, especially in plants. In this article we examine polymorphism at two miRNA-encoding loci in Arabidopsis thaliana, miR856 and miR824, which are not found in rice or poplar. We compare their diversity to other miRNA-encoding loci conserved across distant taxa. We find that levels of variation vary significantly across loci and that the two recently derived loci harbor patterns of diversity deviating from neutrality. miRNA miR856 shows a weak signature of a selective sweep whereas miR824 displays signs of balancing selection. A detailed examination of structural variation among alleles found at the miR824-encoding locus suggests nonrandom evolution of a thermoresistant substructure in the precursor. Expression analysis of pre-miR824 and its target, AGL16, indicates that these structural differences likely impact the processing of mature miR824. Our work highlights the relevance of RNA structure in precursor sequence evolution, suggesting that the evolutionary dynamics of miRNA-encoding loci is more complex than suggested by the constraints exerted on the interaction between mature miRNA fragments and their target exon.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(138);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Cloning of DOG1, a quantitative trait locus controlling seed dormancy in Arabidopsis.</div><div 
  class="publication01Info">Bentsink L, Jowett J, Hanhart CJ, Koornneef M, Proceedings Of The National Academy Of Sciences Of The United States Of America 103, 17042-17047 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(171);">abstract <img id="abstractI171" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17065317&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP171" 
  class="publication01Abstract" style="display:none;">Genetic variation for seed dormancy in nature is a typical quantitative trait controlled by multiple loci on which environmental factors have a strong effect. Finding the genes underlying dormancy quantitative trait loci is a major scientific challenge, which also has relevance for agriculture and ecology. In this study we describe the identification of the DELAY OF GERMINATION 1 (DOG1) gene previously identified as a quantitative trait locus involved in the control of seed dormancy. This gene was isolated by a combination of positional cloning and mutant analysis and is absolutely required for the induction of seed dormancy. DOG1 is a member of a small gene family of unknown molecular function, with five members in Arabidopsis. The functional natural allelic variation present in Arabidopsis is caused by polymorphisms in the cis-regulatory region of the DOG1 gene and results in considerable expression differences between the DOG1 alleles of the accessions analyzed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(171);">hide</a></div></div></div>
	    <p><a href="project_area_b.html">back to project area B</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
