<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>C4: Publications</h2>
	    <div id="publication1" class="publication01"><div 
  class="publication01Title">On the sub-permutations of pattern avoiding permutations</div><div 
  class="publication01Info">Disanto F, Wiehe T, Discrete mathematics, Volume 337, 28 December 2014, Pages 127–141 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(1);">abstract <img id="abstractI1" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S0012365X14002635">paper [pdf]</a> &nbsp; </div><div id="abstractP1" 
  class="publication01Abstract" style="display:none;">There is a deep connection between permutations and trees. Certain sub-structures of permutations, called sub-permutations, bijectively map to sub-trees of binary increasing trees. This opens a powerful tool set to study enumerative and probabilistic properties of sub-permutations and to investigate the relationships between 'local' and 'global' features using the concept of pattern avoidance. First, given a pattern {\mu}, we study how the avoidance of {\mu} in a permutation {\pi} affects the presence of other patterns in the sub-permutations of {\pi}. More precisely, considering patterns of length 3, we solve instances of the following problem: given a class of permutations K and a pattern {\mu}, we ask for the number of permutations π∈Avn(μ) whose sub-permutations in K satisfy certain additional constraints on their size. Second, we study the probability for a generic pattern to be contained in a random permutation {\pi} of size n without being present in the sub-permutations of {\pi} generated by the entry 1≤k≤n. These theoretical results can be useful to define efficient randomized pattern-search procedures based on classical algorithms of pattern-recognition, while the general problem of pattern-search is NP-complete.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(1);">hide</a></div></div></div><div id="publication4" class="publication01"><div 
  class="publication01Title">Demography-adjusted tests of neutrality based on genome-wide SNP data</div><div 
  class="publication01Info">Rafajlović M, Klassmann A, Eriksson A, Wiehe T, Mehlig B, Theoretical Population Biology, Vol. 95, pp. 1–12 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(4);">abstract <img id="abstractI4" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S0040580914000367">paper [pdf]</a> &nbsp; </div><div id="abstractP4" 
  class="publication01Abstract" style="display:none;">Tests of the neutral evolution hypothesis are usually built on the standard null model which assumes that mutations are neutral and the population size remains constant over time. However, it is unclear how such tests are affected if the last assumption is dropped. Here, we extend the unifying framework for tests based on the site frequency spectrum, introduced by Achaz and Ferretti, to populations of varying size. Key ingredients are the first two moments of the site frequency spectrum. We show how these moments can be computed analytically if a population has experienced two instantaneous size changes in the past. We apply our method to data from ten human populations gathered in the 1000 genomes project, estimate their demographies and define demography-adjusted versions of Tajima’s DD, Fay & Wu’s HH, and Zeng’s EE. Our results show that demography-adjusted test statistics facilitate the direct comparison between populations and that most of the differences among populations seen in the original unadjusted tests can be explained by their underlying demographies. Upon carrying out whole-genome screens for deviations from neutrality, we identify candidate regions of recent positive selection. We provide track files with values of the adjusted and unadjusted tests for upload to the UCSC genome browser.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(4);">hide</a></div></div></div><div id="publication5" class="publication01"><div 
  class="publication01Title">New tools in the box: An evolutionary synopsis of chromatin insulators</div><div 
  class="publication01Info">Heger P, Wiehe T, Trends In Genetics, Vol. 30, Issue 5, p161–171 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(5);">abstract <img id="abstractI5" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.cell.com/trends/genetics/abstract/S0168-9525(14)00053-5#">paper [pdf]</a> &nbsp; </div><div id="abstractP5" 
  class="publication01Abstract" style="display:none;">Despite progress in understanding genome organization and gene expression during the last decade, the evolutionary pathways that led to the intricate patterns of gene expression in different cells of an organism are still poorly understood. Important steps in this regulation take place at the level of chromatin, where the (epi)genomic environment of a gene determines its expression in time and space. Although the basic mechanisms of gene expression apply to all eukaryotes, multicellular organisms face the additional challenge of coordinating gene expression during development. In this review we summarize and put into evolutionary context current knowledge about chromatin insulators, an important class of regulatory factors mediating these tasks. Our interpretation of historical and recent findings points to a dynamic and ongoing evolution of insulator proteins characterized by multiple instances of convergent evolution, gene loss, and binding site changes in different organisms. The idea of two autonomously evolving insulator functions (as a barrier element and an enhancer blocker) further suggests that the evolution of metazoans and their enhancer-rich gene regulatory repertoire might be connected to the radiation of enhancer blocking insulators. Although speculative at the moment, such coevolution might create tools for complex gene regulation and therefore influence the evolutionary roadmaps of metazoans.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(5);">hide</a></div></div></div><div id="publication16" class="publication01"><div 
  class="publication01Title">Yule-generated trees constrained by node imbalance</div><div 
  class="publication01Info">Disanto F, Schlizio A, Wiehe T, Mathematical Biosciences (online access) August 13 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(16);">abstract <img id="abstractI16" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S0025556413002101">paper [pdf]</a> &nbsp; </div><div id="abstractP16" 
  class="publication01Abstract" style="display:none;">The <em>Yule</em> process generates a class of binary trees which is fundamental to population genetic models and other applications in evolutionary biology. In this paper, we introduce a family of sub-classes of ranked trees, called Ω-trees, which are characterized by imbalance of internal nodes. The degree of imbalance is defined by an integer 0⩽ω. For caterpillars, the extreme case of unbalanced trees, ω=0. Under models of neutral evolution, for instance the Yule model, trees with small ω are unlikely to occur by chance. Indeed, imbalance can be a signature of permanent selection pressure, such as observable in the genealogies of certain pathogens. From a mathematical point of view it is interesting to observe that the space of Ω-trees maintains several statistical invariants although it is drastically reduced in size compared to the space of unconstrained Yule trees. Using generating functions, we study here some basic combinatorial properties of Ω-trees. We focus on the distribution of the number of subtrees with two leaves. We show that expectation and variance of this distribution match those for unconstrained trees already for very small values of ω.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(16);">hide</a></div></div></div><div id="publication20" class="publication01"><div 
  class="publication01Title">Successive gain of insulator proteins in arthropod evolution</div><div 
  class="publication01Info">Heger P, George R, Wiehe T, Evolution (online access) June 4 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(20);">abstract <img id="abstractI20" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/evo.12155/abstract">paper [pdf]</a> &nbsp; </div><div id="abstractP20" 
  class="publication01Abstract" style="display:none;">Alteration of regulatory DNA elements or their binding proteins may have drastic consequences for morphological evolution. Chromatin insulators are one example of such proteins and play a fundamental role in organizing gene expression. While a single insulator protein, CTCF (CCCTC-binding factor), is known in vertebrates, <em>Drosophila melanogaster</em> utilizes six additional factors. We studied the evolution of these proteins and show here that—in contrast to the bilaterian-wide distribution of CTCF—all other <em>D. melanogaster</em> insulators are restricted to arthropods. The full set is present exclusively in the genus <em>Drosophila</em> whereas only two insulators, Su(Hw) and CTCF, existed at the base of the arthropod clade and all additional factors have been acquired successively at later stages. Secondary loss of factors in some lineages further led to the presence of different insulator subsets in arthropods. Thus, the evolution of insulator proteins within arthropods is an ongoing and dynamic process that reshapes and supplements the ancient CTCF-based system common to bilaterians. Expansion of insulator systems may therefore be a general strategy to increase an organism's gene regulatory repertoire and its potential for morphological plasticity.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(20);">hide</a></div></div></div><div id="publication26" class="publication01"><div 
  class="publication01Title">Coalescent tree imbalance and a simple test for selective sweeps based on microsatellite variation</div><div 
  class="publication01Info">Li H, Wiehe T, Plos Computational Biology, 9(5): e1003060 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(26);">abstract <img id="abstractI26" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003060">paper [pdf]</a> &nbsp; </div><div id="abstractP26" 
  class="publication01Abstract" style="display:none;">It is one of the major interests in population genetics to contrast the properties and consequences of neutral and non-neutral modes of evolution. As is well-known, positive Darwinian selection and genetic hitchhiking drastically change the profile of genetic diversity compared to neutral expectations. The present-day observable genetic diversity in a sample of DNA sequences depends on events in their evolutionary history, and in particular on the shape of the underlying genealogical tree. In this paper we study how the shape of coalescent trees is affected by the presence of positively selected mutations. We define a measure of tree topology and study its properties under scenarios of neutrality and positive selection. We show that this measure can reliably be estimated from experimental data, and define an easy-to-compute statistical test of the neutral evolution hypothesis. We apply this test to data from a population of the malaria parasite <em>Plasmodium falciparum</em> and confirm the signature of recent positive selection in the vicinity of a drug resistance locus.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(26);">hide</a></div></div></div><div id="publication31" class="publication01"><div 
  class="publication01Title">Exact enumeration of cherries and pitchforks in ranked trees under the coalescent model.</div><div 
  class="publication01Info">Disanto F, Wiehe T, Mathematical Biosciences 242, 195-200 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(31);">abstract <img id="abstractI31" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=23396093&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP31" 
  class="publication01Abstract" style="display:none;">We consider exact enumerations and probabilistic properties of ranked trees when generated under the random coalescent process. Using a new approach, based on generating functions, we derive several statistics such as the exact probability of finding k cherries in a ranked tree of fixed size n. We then extend our method to consider also the number of pitchforks. We find a recursive formula to calculate the joint and conditional probabilities of cherries and pitchforks when the size of the tree is fixed. These results provide insights into structural properties of coalescent trees under the model of neutral evolution.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(31);">hide</a></div></div></div><div id="publication33" class="publication01"><div 
  class="publication01Title">The effect of single recombination events on coalescent tree height and shape</div><div 
  class="publication01Info">Ferretti L, Disanto F, Wiehe T, PLoS One, 8(4):e60123 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(33);">abstract <img id="abstractI33" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plosone.org/article/fetchObjectAttachment.action;?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0060123&representation=PDF">paper [pdf]</a> &nbsp; </div><div id="abstractP33" 
  class="publication01Abstract" style="display:none;">The coalescent with recombination is a fundamental model to describe the genealogical history of DNA sequence samples from recombining organisms. Considering recombination as a process which acts along genomes and which creates sequence segments with shared ancestry, we study the influence of single recombination events upon tree characteristics of the coalescent. We focus on properties such as tree height and tree balance and quantify analytically the changes in these quantities incurred by recombination in terms of probability distributions. We find that changes in tree topology are often relatively mild under conditions of neutral evolution, while changes in tree height are on average quite large. Our results add to a quantitative understanding of the spatial coalescent and provide the neutral reference to which the impact by other evolutionary scenarios, for instance tree distortion by selective sweeps, can be compared.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(33);">hide</a></div></div></div><div id="publication46" class="publication01"><div 
  class="publication01Title">The chromatin insulator CTCF and the emergence of metazoan diversity.</div><div 
  class="publication01Info">Heger P, Marin B, Bartkuhn M, Schierenberg E, Wiehe T, Proceedings Of The National Academy Of Sciences Of The United States Of America 109, 17507-17512 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(46);">abstract <img id="abstractI46" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=23045651&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP46" 
  class="publication01Abstract" style="display:none;">The great majority of metazoans belong to bilaterian phyla. They diversified during a short interval in Earth&apos;s history known as the Cambrian explosion, ~540 million years ago. However, the genetic basis of these events is poorly understood. Here we argue that the vertebrate genome organizer CTCF (CCCTC-binding factor) played an important role for the evolution of bilaterian animals. We provide evidence that the CTCF protein and a genome-wide abundance of CTCF-specific binding motifs are unique to bilaterian phyla, but absent in other eukaryotes. We demonstrate that CTCF-binding sites within vertebrate and Drosophila Hox gene clusters have been maintained for several hundred million years, suggesting an ancient origin of the previously known interaction between Hox gene regulation and CTCF. In addition, a close correlation between the presence of CTCF and Hox gene clusters throughout the animal kingdom suggests conservation of the Hox-CTCF link across the Bilateria. On the basis of these findings, we propose the existence of a Hox-CTCF kernel as principal organizer of bilaterian body plans. Such a kernel could explain (i) the formation of Hox clusters in Bilateria, (ii) the diversity of bilaterian body plans, and (iii) the uniqueness and time of onset of the Cambrian explosion.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(46);">hide</a></div></div></div><div id="publication128" class="publication01"><div 
  class="publication01Title">Estimating mutation distances from unaligned genomes.</div><div 
  class="publication01Info">Haubold B, Pfaffelhuber P, Domazet-Loso M, Wiehe T, Journal Of Computational Biology 16, 1487-1500 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(128);">abstract <img id="abstractI128" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19803738&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP128" 
  class="publication01Abstract" style="display:none;">Abstract Alignment-free distance measures are generally less accurate but more efficient than traditional alignment-based metrics. In the context of genome sequence analysis, the efficiency gain is often so substantial that it outweights the loss in accuracy. However, a further disadvantage of alignment-free distances is that their relationship to evolutionary events such as substitutions is generally unknown. We have therefore derived an estimator of the number of substitutions per site between two unaligned DNA sequences, K(r). Simulations show that this estimator works well with &quot;ideal&quot; data. We compare K(r) to two alternative alignment-free distances: a k-tuple distance and a measure of relative entropy based on average common substring length. All three measures are applied to 27 primate mitochondrial genomes, eight whole genomes of Streptococcus agalactiae strains, and 12 whole genomes of Drosophila species. In each case, the cluster diagrams based on K(r) are equivalent to or significantly better than those based on the two alternative measures. This is due to the fact that in contrast to the alternative measures K(r) is derived from an explicit model of evolution. The computation of K(r) is efficiently implemented in the program kr, which can be downloaded freely from the internet.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(128);">hide</a></div></div></div><div id="publication142" class="publication01"><div 
  class="publication01Title">Simulation of DNA sequence evolution under models of recent directional selection.</div><div 
  class="publication01Info">Kim Y, Wiehe T, Briefings In Bioinformatics 10, 84-96 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(142);">abstract <img id="abstractI142" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19109303&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP142" 
  class="publication01Abstract" style="display:none;">Computer simulation is an essential tool in the analysis of DNA sequence variation for mapping events of recent adaptive evolution in the genome. Various simulation methods are employed to predict the signature of selection in sequence variation. The most informative and efficient method currently in use is coalescent simulation. However, this method is limited to simple models of directional selection. Whole-population forward-in-time simulations are the alternative to coalescent simulations for more complex models. The notorious problem of excessive computational cost in forward-in-time simulations can be overcome by various simplifying amendments. Overall, the success of simulations depends on the creative application of some population genetic theory to the simulation algorithm.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(142);">hide</a></div></div></div><div id="publication147" class="publication01"><div 
  class="publication01Title">Properties of Sequence Conservation in Upstream Regulatory and Protein Coding Sequences among Paralogs in Arabidopsis thaliana</div><div 
  class="publication01Info">Richardson DN, Wiehe T, Comparative Genomics 5817, 217 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(147);">abstract <img id="abstractI147" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://portal.acm.org/citation.cfm?id=1690891.1690909&amp;coll=DL&amp;dl=GUIDE&amp;CFID=87515263&amp;CFTOKEN=37311518">paper [pdf]</a> &nbsp; </div><div id="abstractP147" 
  class="publication01Abstract" style="display:none;">Whole genome duplication (WGD) has catalyzed the formation of new species, genes with novel functions, altered expression patterns, complexified signaling pathways and has provided organisms a level of genetic robustness. We studied the long-term evolution and interrelationships of 5’ upstream regulatory sequences (URSs), protein coding sequences (CDSs) and expression correlations (EC) of duplicated gene pairs in Arabidopsis. Three distinct methods revealed significant evolutionary conservation between paralogous URSs and were highly correlated with microarray-based expression correlation of the respective gene pairs. Positional information on exact matches between sequences unveiled the contribution of micro-chromosomal rearrangements on expression divergence. A three-way rank analysis of URS similarity, CDS divergence and EC uncovered specific gene functional biases. Transcription factor activity was associated with gene pairs exhibiting conserved URSs and divergent CDSs, whereas a broad array of metabolic enzymes was found to be associated with gene pairs showing diverged URSs but conserved CDSs.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(147);">hide</a></div></div></div><div id="publication152" class="publication01"><div 
  class="publication01Title">Identification of selective sweeps in closely related populations of the house mouse based on microsatellite scans.</div><div 
  class="publication01Info">Teschke M, Mukabayire O, Wiehe T, Tautz D, Genetics 180, 1537-1545 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(152);">abstract <img id="abstractI152" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18791245&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP152" 
  class="publication01Abstract" style="display:none;">Genome scans of polymorphisms promise to provide insights into the patterns and frequencies of positive selection under natural conditions. The use of microsatellites as markers has the potential to focus on very recent events, since in contrast to SNPs, their high mutation rates should remove signatures of older events. We assess this concept here in a large-scale study. We have analyzed two population pairs of the house mouse, one pair of the subspecies Mus musculus domesticus and the other of M. m. musculus. A total of 915 microsatellite loci chosen to cover the whole genome were assessed in a prescreening procedure, followed by individual typing of candidate loci. Schlötterer&apos;s ratio statistics (lnRH) were applied to detect loci with significant deviations from patterns of neutral expectation. For eight loci from each population pair we have determined the size of the potential sweep window and applied a second statistical procedure (linked locus statistics). For the two population pairs, we find five and four significant sweep loci, respectively, with an average estimated window size of 120 kb. On the basis of the analysis of individual allele frequencies, it is possible to identify the most recent sweep, for which we estimate an onset of 400-600 years ago. Given the known population history for the French-German population pair, we infer that the average frequency of selective sweeps in these populations is higher than 1 in 100 generations across the whole genome. We discuss the implications for adaptation processes in natural populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(152);">hide</a></div></div></div><div id="publication154" class="publication01"><div 
  class="publication01Title">Second-order moments of segregating sites under variable population size.</div><div 
  class="publication01Info">Zivković D, Wiehe T, Genetics 180, 341-357 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(154);">abstract <img id="abstractI154" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18716326&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP154" 
  class="publication01Abstract" style="display:none;">The identification of genomic regions that have been exposed to positive selection is a major challenge in population genetics. Since selective sweeps are expected to occur during environmental changes or when populations are colonizing a new habitat, statistical tests constructed on the assumption of constant population size are biased by the co-occurrence of population size changes and selection. To delimit this problem and gain better insights into demographic factors, theoretical results regarding the second-order moments of segregating sites, such as the variance of segregating sites, have been derived. Driven by emerging genomewide surveys, which allow the estimation of demographic parameters, a generalized version of Tajima&apos;s D has been derived that takes into account a previously estimated demographic scenario to test single loci for traces of selection against the null hypothesis of neutral evolution under variable population size.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(154);">hide</a></div></div></div><div id="publication179" class="publication01"><div 
  class="publication01Title">A pooling approach to detect signatures of selective sweeps in genome scans using microsatellites</div><div 
  class="publication01Info">Thomas M, Möller F, Wiehe T, Tautz D, Molecular Ecology Notes 7, 400-403 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(179);">abstract <img id="abstractI179" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2007.01697.x/abstract">paper [pdf]</a> &nbsp; </div><div id="abstractP179" 
  class="publication01Abstract" style="display:none;">We have evaluated a pooling approach that can reduce the number of polymerase chain reactions in a screen for selective sweeps by more than an order of magnitude. We show that the complex peak pattern that results from pooling of all samples from a given population is a faithful reflection of the composite pattern of the individual alleles, although with an under-representation of the larger alleles. Candidate loci for selective sweeps can be identified by visual inspection of the pool patterns. We have also implemented a software tool, which can find suitable microsatellite loci in the vicinity of annotated genes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(179);">hide</a></div></div></div><div id="publication184" class="publication01"><div 
  class="publication01Title">Identification of selective sweeps using a dynamically adjusted number of linked microsatellites.</div><div 
  class="publication01Info">Wiehe T, Nolte V, Zivković D, Schlötterer C, Genetics 175, 207-218 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(184);">abstract <img id="abstractI184" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17057237&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP184" 
  class="publication01Abstract" style="display:none;">There is currently large interest in distinguishing the signatures of genetic variation produced by demographic events from those produced by natural selection. We propose a simple multilocus statistical test to identify candidate sites of selective sweeps with high power. The test is based on the variability profile measured in an array of linked microsatellites. We also show that the analysis of flanking markers drastically reduces the number of false positives among the candidates that are identified in a genomewide survey of unlinked loci and find that this property is maintained in many population-bottleneck scenarios. However, for a certain range of intermediately severe population bottlenecks we find genomic signatures that are very similar to those produced by a selective sweep. While in these worst-case scenarios the power of the proposed test remains high, the false-positive rate reaches values close to 50%. Hence, selective sweeps may be hard to identify even if multiple linked loci are analyzed. Nevertheless, the integration of information from multiple linked loci always leads to a considerable reduction of the false-positive rate compared to a genome scan of unlinked loci. We discuss the application of this test to experimental data from Drosophila melanogaster.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(184);">hide</a></div></div></div>
	    <p><a href="project_area_c.html">back to project area C</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
