<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>A9: Publications</h2>

<div id="publication219" class="publication01">
  <div class="publication01Title">
New paper on the inference of transcriptional regulation appeared in Molecular Systems Biology
  </div>
  <div class="publication01Info">
Zacher B, Lidschreiber M, Cramer P, Gagneur J, Tresch A, DOI 10.15252/msb.20145654 | Published online 19.12.2014 Molecular Systems Biology(2014)10:768
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(219);">abstract <img id="abstractI219" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://msb.embopress.org/content/10/12/768">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP219" class="publication01Abstract" style="display:none;">
    <p>DNA replication, transcription and repair involve the recruitment of protein complexes that change their composition as they progress along the genome in a directed or strand‐specific manner. Chromatin immunoprecipitation in conjunction with hidden Markov models (HMMs) has been instrumental in understanding these processes, as they segment the genome into discrete states that can be related to DNA‐associated protein complexes. However, current HMM‐based approaches are not able to assign forward or reverse direction to states or properly integrate strand‐specific (e.g., RNA expression) with non‐strand‐specific (e.g., ChIP) data, which is indispensable to accurately characterize directed processes. To overcome these limitations, we introduce bidirectional HMMs which infer directed genomic states from occupancy profiles de novo. Application to RNA polymerase II‐associated factors in yeast and chromatin modifications in human T cells recovers the majority of transcribed loci, reveals gene‐specific variations in the yeast transcription cycle and indicates the existence of directed chromatin state patterns at transcribed, but not at repressed, regions in the human genome. In yeast, we identify 32 new transcribed loci, a regulated initiation–elongation transition, the absence of elongation factors Ctk1 and Paf1 from a class of genes, a distinct transcription mechanism for highly expressed genes and novel DNA sequence motifs associated with transcription termination. We anticipate bidirectional HMMs to significantly improve the analyses of genome‐associated directed processes.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(219);">hide</a>
    </div>
  </div>
</div>
	
<div id="publication215" class="publication01">
  <div class="publication01Title">
    Rapid identification of a natural knockout allele of ARMADILLO REPEAT-CONTAINING KINESIN1 that causes root hair branching by mapping-by-sequencing.
  </div>
  <div class="publication01Info">
    Rishmawi L, Sun H, Schneeberger K, Hülskamp M, Schrader A, Plant Physiol. 2014 Nov;166(3):1280-7. doi: 10.1104/pp.114.244046. Epub 2014 Sep 23.
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(215);">abstract <img id="abstractI215" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/25248719">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP215" class="publication01Abstract" style="display:none;">
    <p>In Arabidopsis (Arabidopsis thaliana), branched root hairs are an indicator of defects in root hair tip growth. Among 62 accessions, one accession (Heiligkreuztal2 [HKT2.4]) displayed branched root hairs, suggesting that this accession carries a mutation in a gene of importance for tip growth. We determined 200- to 300-kb mapping intervals using a mapping-by-sequencing approach of F2 pools from crossings of HKT2.4 with three different accessions. The intersection of these mapping intervals was 80 kb in size featuring not more than 36 HKT2.4-specific single nucleotide polymorphisms, only two of which changed the coding potential of genes. Among them, we identified the causative single nucleotide polymorphism changing a splicing site in ARMADILLO REPEAT-CONTAINING KINESIN1. The applied strategies have the potential to complement statistical methods in high-throughput phenotyping studies using different natural accessions to identify causative genes for distinct phenotypes represented by only one or a few accessions. 
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(215);">hide</a>
    </div>
  </div>
</div>

	    <div class="publication01"><div 
  class="publication01Title">Analysis of TTG1 function in Arabis alpina</div><div 
  class="publication01Info">Chopra D, Wolff H, Span J, Schellmann S, Coupland G, Albani MC, Schrader A, Hülskamp M, BMC Plant Biology 2014; 14:16 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(4);">abstract <img id="abstractI4" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.biomedcentral.com/1471-2229/14/16">paper [pdf]</a> &nbsp; </div><div id="abstractP4" 
  class="publication01Abstract" style="display:none;">
<p><strong>Background</strong></p>
<p>In Arabidopsis thaliana (A. thaliana) the WD40 protein TRANSPARENT TESTA GLABRA1 (TTG1) controls five traits relevant for the adaptation of plants to environmental changes including the production of proanthocyanidin, anthocyanidin, seed coat mucilage, trichomes and root hairs. The analysis of different Brassicaceae species suggests that the function of TTG1 is conserved within the family.</p>
<p><strong>Results</strong></p>
<p>In this work, we studied the function of TTG1 in Arabis alpina (A. alpina). A comparison of wild type and two Aattg1 alleles revealed that AaTTG1 is involved in the regulation of all five traits. A detailed analysis of the five traits showed striking phenotypic differences between A. alpina and A. thaliana such that trichome formation occurs also at later stages of leaf development and that root hairs form at non-root hair positions.</p>
<p><strong>Conclusions</strong></p>
<p>The evolutionary conservation of the regulation of the five traits by TTG1 on the one hand and the striking phenotypic differences make A. alpina a very interesting genetic model system to study the evolution of TTG1-dependent gene regulatory networks at a functional level.</p>
<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(4);">hide</a></div></div></div>
	    <p><a href="project_area_a.html">back to project area A</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
