<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/events.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
  <script id="randomimage" type="text/javascript" charset="utf-8" 
    src="scripts/randomimage.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02Active"><div 
        class="mainMenuItem02ActiveText">publication list</div></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>Cumulative publication list</h2>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2015</div>

			<div id="publication237" class="publication01">
				<div class="publication01Title">
					Differential interaction forces govern bacterial sorting in early biofilms
				</div>
				<div class="publication01Info">
					Enno R Oldewurtel, Nadzeya Kouzel, Lena Dewenter, Katja Henseler, Berenike Maier, eLife 2015;4:e10811
				</div>
				<div class="publication01Links"><a href="javascript:Publications.toggleAbstract(237);">abstract <img id="abstractI224" class="publication01Arrow" src="images/arrow_r_02.png"></img>
				</a> &nbsp; <a href="http://elifesciences.org/content/4/e10811">paper [pdf]</a> &nbsp;
				</div>
				<div id="abstractP237" class="publication01Abstract" style="display:none;">
					<p>
						Bacterial biofilms can generate micro-heterogeneity in terms of surface structures. However, little is known about the associated changes in the physics of cell–cell interaction and its impact on the architecture of biofilms. In this study, we used the type IV pilus of Neisseria gonorrhoeae to test whether variation of surface structures induces cell-sorting. We show that the rupture forces between pili are fine-tuned by post-translational modification. Bacterial sorting was dependent on pilus post-translational modification and pilus density. Active force generation was necessary for defined morphologies of mixed microcolonies. The observed morphotypes were in remarkable agreement with the differential strength of adhesion hypothesis proposing that a tug-of-war among surface structures of different cells governs cell sorting. We conclude that in early biofilms the density and rupture force of bacterial surface structures can trigger cell sorting based on similar physical principles as in developing embryos.
					</p>
					<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(237);">hide</a>
					</div>
				</div>
			</div>

	    <div id="publication233" class="publication01"><div 
  class="publication01Title">The Toxoplasma gondii rhoptry protein ROP18 is an Irga6-specific kinase and regulated by the dense granule protein GRA7.</div><div 
  class="publication01Info">Hermanns T., Müller UB., Könen-Waisman S., Howard JC., Steinfeldt T. , Cell Microbiol. 2015 Aug 6. doi: 10.1111/cmi.12499 (2015)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(233);">abstract <img id="abstractI233" class="publication01Arrow" src="images/arrow_r_02.png"></img>
		</a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/26247512">paper [pdf]</a> &nbsp; </div><div id="abstractP233"
  class="publication01Abstract" style="display:none;">In mice, avirulent strains (e.g. types II and III) of the protozoan parasite Toxoplasma gondii are restricted by the immunity-related GTPase (IRG) resistance system. Loading of IRG proteins onto the PVM is required for vacuolar rupture resulting in parasite clearance. In virulent strain (e.g. type I) infections polymorphic effector proteins ROP5 and ROP18 cooperate to phosphorylate and thereby inactivate mouse IRG proteins to preserve PVM integrity. In this study we confirmed the dense granule protein GRA7 as an additional component of the ROP5/ROP18 kinase complex and identified GRA7 association with the PVM by direct binding to ROP5. The absence of GRA7 results in reduced phosphorylation of Irga6 correlated with increased vacuolar IRG protein amounts and attenuated virulence. Earlier work identified additional IRG proteins as targets of T. gondii ROP18 kinase. We show that the only specific target of ROP18 among IRG proteins is in fact Irga6. Similarily, we demonstrate that GRA7 is strictly an Irga6-specific virulence effector. This identifies T. gondii GRA7 as a regulator for ROP18-specific inactivation of Irga6. The structural diversity of the IRG proteins implies that certain family members constitute additional specific targets for other yet unknown T. gondii virulence effectors.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(233);">hide</a></div></div></div>



	    <div id="publication232" class="publication01"><div 
  class="publication01Title">Multiple-Line Inference of Selection on Quantitative Traits</div><div 
  class="publication01Info"> N. Riedel, B. S. Khatri, M. Lässig, and J. Berg; Genetics September 2015 201:305-322; Early online July 2, 2015</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(232);">abstract <img id="abstractI232" class="publication01Arrow" src="images/arrow_r_02.png"></img>
		</a> &nbsp; <a href="http://www.genetics.org/content/201/1/305.full">paper [pdf]</a> &nbsp; </div><div id="abstractP232"
  class="publication01Abstract" style="display:none;">To Be Added<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(232);">hide</a></div></div></div>





	<div id="publication230" class="publication01">
	  <div class="publication01Title">
	    Phase transition in random adaptive walks on correlated fitness landscapes
	  </div>
	  <div class="publication01Info">
	    Su-Chan Park, Ivan G. Szendro, Johannes Neidhart and Joachim Krug, Physical Revivew E 91, 042707 (2015)
	  </div>
	  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(230);">abstract <img id="abstractI230" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://journals.aps.org/pre/abstract/10.1103/PhysRevE.91.042707">paper [pdf]</a> &nbsp; 
	  </div>
	  <div id="abstractP230" class="publication01Abstract" style="display:none;">
	    <p>
	      We study biological evolution on a random fitness landscape where correlations are introduced through a linear fitness gradient of strength c. When selection is strong and mutations rare the dynamics is a directed uphill walk that terminates at a local fitness maximum. We analytically calculate the dependence of the walk length on the genome size L. When the distribution of the random fitness component has an exponential tail, we find a phase transition of the walk length D between a phase at small c, where walks are short (D∼lnL), and a phase at large c, where walks are long (D∼L). For all other distributions only a single phase exists for any c>0. The considered process is equivalent to a zero temperature Metropolis dynamics for the random energy model in an external magnetic field, thus also providing insight into the aging dynamics of spin glasses.
	    </p>
	    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(230);">hide</a>
	    </div>
	  </div>
	</div>


	<div id="publication229" class="publication01">
	  <div class="publication01Title">
	    Analysis of adaptive walks on NK fitness landscapes with different interaction schemes
	  </div>
	  <div class="publication01Info">
	    Stefan Nowak and Joachim Krug, Preprint  arXiv:1503.07796, to appear in Journal of Statistical Mechanics
	  </div>
	  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(229);">abstract <img id="abstractI229" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://arxiv.org/abs/1503.07796">paper [pdf]</a> &nbsp; 
	  </div>
	  <div id="abstractP229" class="publication01Abstract" style="display:none;">
	    <p>
	      Fitness landscapes are genotype to fitness mappings commonly used in evolutionary biology and computer science which are closely related to spin glass models. In this paper, we study the NK model for fitness landscapes where the interaction scheme between genes can be explicitly defined. The focus is on how this scheme influences the overall shape of the landscape. Our main tool for the analysis are adaptive walks, an idealized dynamics by which the population moves uphill in fitness and terminates at a local fitness maximum. We use three different types of walks and investigate how their length (the number of steps required to reach a local peak) and height (the fitness at the endpoint of the walk) depend on the dimensionality and structure of the landscape. We find that the distribution of local maxima over the landscape is particularly sensitive to the choice of interaction pattern. Most quantities that we measure are simply correlated to the rank of the scheme, which is equal to the number of nonzero coefficients in the expansion of the fitness landscape in terms of Walsh functions.
	    </p>
	    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(229);">hide</a>
	    </div>
	  </div>
	</div>

	<div id="publication228" class="publication01">
  <div class="publication01Title">
    Negative Epistasis and Evolvability in TEM-1 β-Lactamase—The Thin Line between an Enzyme's Conformational Freedom and Disorder
  </div>
  <div class="publication01Info">
Dellus-Gur E, Elias M, Caselli E, Prati F, Salverda MLM, de Visser JAGM, Fraser JS, Tawfik DS, Journal of Molecular Biology, in press (2015)
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(228);">abstract <img id="abstractI228" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://fraserlab.com/static/pdf/gur_2015.pdf">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP228" class="publication01Abstract" style="display:none;">
    <p>
Epistasis is a key factor in evolution since it determines which combinations of mutations provide adaptive solutions and which mutational pathways toward these solutions are accessible by natural selection. There is growing evidence for the pervasiveness of sign epistasis—a complete reversion of mutational effects, particularly in protein evolution—yet its molecular basis remains poorly understood. We describe the structural basis of sign epistasis between G238S and R164S, two adaptive mutations in TEM-1 β-lactamase— an enzyme that endows antibiotics resistance. Separated by 10 Å, these mutations initiate two separate trajectories toward increased hydrolysis rates and resistance toward second and third-generation cephalosporins antibiotics. Both mutations allow the enzyme's active site to adopt alternative conformations and accommodate the new antibiotics. By solving the corresponding set of crystal structures, we found that R164S causes local disorder whereas G238S induces discrete conformations. When combined, the mutations in 238 and 164 induce local disorder whereby nonproductive conformations that perturb the enzyme's catalytic preorganization dominate. Specifically, Asn170 that coordinates the deacylating water molecule is misaligned, in both the free form and the inhibitor-bound double mutant. This local disorder is not restored by stabilizing global suppressor mutations and thus leads to an evolutionary cul-de-sac. Conformational dynamism therefore underlines the reshaping potential of protein's structures and functions but also limits protein evolvability because of the fragility of the interactions networks that maintain protein structures.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(228);">hide</a>
    </div>
  </div>
</div>




	<div id="publication227" class="publication01">
  <div class="publication01Title">
    Role of pleiotropy during adaptation of TEM-1 β-lactamase to two novel antibiotics.
  </div>
  <div class="publication01Info">
    Schenk MF, Witte S, Salverda MLM, Koopmanschap B, Krug J, de Visser JAGM, Evolutionary Applications 8: 248-260 (2015)
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(227);">abstract <img id="abstractI227" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/25861383">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP227" class="publication01Abstract" style="display:none;">
    <p>
Pleiotropy is a key feature of the genotype-phenotype map, and its form and extent have many evolutionary implications, including for the dynamics of adaptation and the evolution of specialization. Similarly, pleiotropic effects of antibiotic resistance mutations may affect the evolution of antibiotic resistance in the simultaneous or fluctuating presence of different antibiotics. Here, we study the role of pleiotropy during the in vitro adaptation of the enzyme TEM-1 β-lactamase to two novel antibiotics, cefotaxime (CTX) and ceftazidime (CAZ). We subject replicate lines for four rounds of evolution to selection with CTX and CAZ alone, and in their combined and fluctuating presence. Evolved alleles show positive correlated responses when selecting with single antibiotics. Nevertheless, pleiotropic constraints are apparent from the effects of single mutations and from selected alleles showing smaller correlated than direct responses and smaller responses after simultaneous and fluctuating selection with both than with single antibiotics. We speculate that these constraints result from structural changes in the oxyanion pocket surrounding the active site, where accommodation of CTX and the larger CAZ is balanced against their positioning with respect to the active site. Our findings suggest limited benefits from the combined or fluctuating application of these related cephalosporins for containing antibiotic resistance.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(227);">hide</a>
    </div>
  </div>
</div>


	    <div id="publication221" class="publication01">
	      <div class="publication01Title">
		Gene transfer efficiency in gonococcal biofilms: role of biofilm age, and architecture, and pilin antigenic variation
	      </div>
	      <div class="publication01Info">
		Kouzel, N., Oldewurtel, E.R., Maier, B., J. Bacteriol., JB.00171-15 in press (2015)
	      </div>
	      <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(221);">abstract <img id="abstractI221" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://jb.asm.org/content/early/2015/05/06/JB.00171-15.long">paper [pdf]</a> &nbsp; 
	      </div>
	      <div id="abstractP221" class="publication01Abstract" style="display:none;">
		<p>
		  
		</p>
		<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(221);">hide</a>
		</div>
	      </div>
	    </div>


	    <div id="publication220" class="publication01">
	      <div class="publication01Title">Evolution of <em>CONSTANTS</em> Regulation and Function after Gene Duplication produced a Photoperiodic Flowering Switch in the Brassicaceae</div>
	      <div class="publication01Info">Simon S, Rühl M, de Montaigu A, Wötzel St, Coupland G, Molecular Biology and Evolution, DOI: 10.1093/molbev/msv110 (2015)</div>
	      <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(220);">abstract <img id="abstractI220" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://mbe.oxfordjournals.org/content/early/2015/05/12/molbev.msv110.abstract">paper [pdf]</a> &nbsp; 
	      </div>
	      <div id="abstractP220" class="publication01Abstract" style="display:none;">
		<p>
		  Environmental control of flowering allows plant reproduction to occur under optimal conditions and facilitates adaptation to different locations. At high latitude, flowering of many plants is controlled by seasonal changes in day length. The photoperiodic flowering pathway confers this response in the Brassicaceae, which colonized temperate latitudes after divergence from the Cleomaceae, their subtropical sister family. The CONSTANS (CO) transcription factor of Arabidopsis thaliana, a member of the Brassicaceae, is central to the photoperiodic flowering response and shows characteristic patterns of transcription required for day-length sensing. CO is believed to be widely conserved among flowering plants, however we show that it arose after gene duplication at the root of the Brassicaceae followed by divergence of transcriptional regulation and protein function. CO has two close homologs, CONSTANS-LIKE1 (COL1) and COL2, which are related to CO by tandem duplication and whole genome duplication, respectively. The single CO homolog present in the Cleomaceae shows transcriptional and functional features similar to those of COL1 and COL2, suggesting these were ancestral. We detect cis-regulatory and codon changes characteristic of CO and use transgenic assays to demonstrate their significance in the day-length dependent activation of the CO target gene FLOWERING LOCUS T. Thus, the function of CO as a potent photoperiodic flowering switch evolved in the Brassicaceae after gene duplication. The origin of CO may have contributed to the range expansion of the Brassicaceae and suggests that in other families CO genes involved in photoperiodic flowering arose by convergent evolution.</p>
		<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(220);">hide</a>
		</div>
	      </div>
	    </div>


	    <div id="publication217" class="publication01">
	      <div class="publication01Title">
		Dynamic BMP signaling polarized by Toll patterns the dorsoventral axis in a hemimetabolous insect
	      </div>
	      <div class="publication01Info">
		Lena Sachs, Yen-Ta Chen, Axel Drechsler, Jeremy A Lynch, Kristen A Panfilio, Michael Lässig, Johannes Berg, Siegfried Roth, DOI: http://dx.doi.org/10.7554/eLife.05502, Published May 12, 2015, Cite as eLife 2015;4:e05502
	      </div>
	      <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(217);">abstract <img id="abstractI217" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://elifesciences.org/content/4/e05502">paper [pdf]</a> &nbsp; 
	      </div>

	      <div id="abstractP217" class="publication01Abstract" style="display:none;">
		<p>
		  Toll-dependent patterning of the dorsoventral axis in Drosophila represents one of the best understood gene regulatory networks. However, its evolutionary origin has remained elusive. Outside the insects Toll is not known for a patterning function, but rather for a role in pathogen defense. Here, we show that in the milkweed bug Oncopeltus fasciatus, whose lineage split from Drosophila's more than 350 million years ago, Toll is only required to polarize a dynamic BMP signaling network. A theoretical model reveals that this network has self-regulatory properties and that shallow Toll signaling gradients are sufficient to initiate axis formation. Such gradients can account for the experimentally observed twinning of insect embryos upon egg fragmentation and might have evolved from a state of uniform Toll activity associated with protecting insect eggs against pathogens.
		</p>
		<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(217);">hide</a>
		</div>
	      </div>
	    </div>
	  </div>

          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2014</div>

          <div id="publication1" class="publication01"><div 
  class="publication01Title">On the sub-permutations of pattern avoiding permutations</div><div 
  class="publication01Info">Disanto F, Wiehe T, Discrete mathematics, Volume 337, 28 December 2014, Pages 127–141 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(1);">abstract <img id="abstractI1" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S0012365X14002635">paper [pdf]</a> &nbsp; </div><div id="abstractP1" 
  class="publication01Abstract" style="display:none;">There is a deep connection between permutations and trees. Certain sub-structures of permutations, called sub-permutations, bijectively map to sub-trees of binary increasing trees. This opens a powerful tool set to study enumerative and probabilistic properties of sub-permutations and to investigate the relationships between 'local' and 'global' features using the concept of pattern avoidance. First, given a pattern {\mu}, we study how the avoidance of {\mu} in a permutation {\pi} affects the presence of other patterns in the sub-permutations of {\pi}. More precisely, considering patterns of length 3, we solve instances of the following problem: given a class of permutations K and a pattern {\mu}, we ask for the number of permutations π∈Avn(μ) whose sub-permutations in K satisfy certain additional constraints on their size. Second, we study the probability for a generic pattern to be contained in a random permutation {\pi} of size n without being present in the sub-permutations of {\pi} generated by the entry 1≤k≤n. These theoretical results can be useful to define efficient randomized pattern-search procedures based on classical algorithms of pattern-recognition, while the general problem of pattern-search is NP-complete.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(1);">hide</a></div></div></div>

<div id="publication219" class="publication01">
  <div class="publication01Title">
New paper on the inference of transcriptional regulation appeared in Molecular Systems Biology
  </div>
  <div class="publication01Info">
Zacher B, Lidschreiber M, Cramer P, Gagneur J, Tresch A, DOI 10.15252/msb.20145654 | Published online 19.12.2014 Molecular Systems Biology(2014)10:768
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(219);">abstract <img id="abstractI219" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://msb.embopress.org/content/10/12/768">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP219" class="publication01Abstract" style="display:none;">
    <p>DNA replication, transcription and repair involve the recruitment of protein complexes that change their composition as they progress along the genome in a directed or strand‐specific manner. Chromatin immunoprecipitation in conjunction with hidden Markov models (HMMs) has been instrumental in understanding these processes, as they segment the genome into discrete states that can be related to DNA‐associated protein complexes. However, current HMM‐based approaches are not able to assign forward or reverse direction to states or properly integrate strand‐specific (e.g., RNA expression) with non‐strand‐specific (e.g., ChIP) data, which is indispensable to accurately characterize directed processes. To overcome these limitations, we introduce bidirectional HMMs which infer directed genomic states from occupancy profiles de novo. Application to RNA polymerase II‐associated factors in yeast and chromatin modifications in human T cells recovers the majority of transcribed loci, reveals gene‐specific variations in the yeast transcription cycle and indicates the existence of directed chromatin state patterns at transcribed, but not at repressed, regions in the human genome. In yeast, we identify 32 new transcribed loci, a regulated initiation–elongation transition, the absence of elongation factors Ctk1 and Paf1 from a class of genes, a distinct transcription mechanism for highly expressed genes and novel DNA sequence motifs associated with transcription termination. We anticipate bidirectional HMMs to significantly improve the analyses of genome‐associated directed processes.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(219);">hide</a>
    </div>
  </div>
</div>



<div id="publication226" class="publication01">
  <div class="publication01Title">
    Adaptation in tunably rugged fitness landscapes: The Rough Mount Fuji Model
  </div>
  <div class="publication01Info">
    Johannes Neidhart, Ivan G. Szendro and Joachim Krug Genetics 198, 699-721 (2014)
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(226);">abstract <img id="abstractI226" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.genetics.org/content/198/2/699.short">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP226" class="publication01Abstract" style="display:none;">
    <p>
Much of the current theory of adaptation is based on Gillespie’s mutational landscape model (MLM), which assumes that the fitness values of genotypes linked by single mutational steps are independent random variables. On the other hand, a growing body of empirical evidence shows that real fitness landscapes, while possessing a considerable amount of ruggedness, are smoother than predicted by the MLM. In the present article we propose and analyze a simple fitness landscape model with tunable ruggedness based on the rough Mount Fuji (RMF) model originally introduced by Aita et al. in the context of protein evolution. We provide a comprehensive collection of results pertaining to the topographical structure of RMF landscapes, including explicit formulas for the expected number of local fitness maxima, the location of the global peak, and the fitness correlation function. The statistics of single and multiple adaptive steps on the RMF landscape are explored mainly through simulations, and the results are compared to the known behavior in the MLM model. Finally, we show that the RMF model can explain the large number of second-step mutations observed on a highly fit first-step background in a recent evolution experiment with a microvirid bacteriophage.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(226);">hide</a>
    </div>
  </div>
</div>


<div id="publication225" class="publication01">
  <div class="publication01Title">
    Empirical fitness landscapes and the predictability of evolution
  </div>
  <div class="publication01Info">
    J. Arjan G.M. de Visser and Joachim Krug, Nature Reviews Genetics 15, 480-490 (2014)
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(225);">abstract <img id="abstractI225" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.nature.com/nrg/journal/v15/n7/full/nrg3744.html">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP225" class="publication01Abstract" style="display:none;">
    <p>
      The genotype–fitness map (that is, the fitness landscape) is a key determinant of evolution, yet it has mostly been used as a superficial metaphor because we know little about its structure. This is now changing, as real fitness landscapes are being analysed by constructing genotypes with all possible combinations of small sets of mutations observed in phylogenies or in evolution experiments. In turn, these first glimpses of empirical fitness landscapes inspire theoretical analyses of the predictability of evolution. Here, we review these recent empirical and theoretical developments, identify methodological issues and organizing principles, and discuss possibilities to develop more realistic fitness landscape models.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(225);">hide</a>
    </div>
  </div>
</div>





	<div id="publication214" class="publication01">
	  <div class="publication01Title">
	    Analysis of a Plant Complex Resistance Gene Locus Underlying Immune-Related Hybrid Incompatibility and Its Occurrence in Nature
	  </div>
	  <div class="publication01Info">
	    Alcázar R, von Reth M, Bauto J, Chae E, Weigel D, Koornneef M, Parker JE, PLOS Published: December 11, 2014, DOI: 10.1371/journal.pgen.1004848
	  </div>
	  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(214);">abstract <img id="abstractI214" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004848">paper [pdf]</a> &nbsp; </div>
	  <div id="abstractP214" class="publication01Abstract" style="display:none;">
	    Mechanisms underlying speciation in plants include detrimental (incompatible) genetic interactions between parental alleles that incur a fitness cost in hybrids. We reported on recessive hybrid incompatibility between an Arabidopsis thaliana strain from Poland, Landsberg erecta (Ler), and many Central Asian A. thaliana strains. The incompatible interaction is determined by a polymorphic cluster of Toll/interleukin-1 receptor-nucleotide binding-leucine rich repeat (TNL) RPP1 (Recognition of Peronospora parasitica1)-like genes in Ler and alleles of the receptor-like kinase Strubbelig Receptor Family 3 (SRF3) in Central Asian strains Kas-2 or Kond, causing temperature-dependent autoimmunity and loss of growth and reproductive fitness. Here, we genetically dissected the RPP1-like Ler locus to determine contributions of individual RPP1-like Ler (R1–R8) genes to the incompatibility. In a neutral background, expression of most RPP1-like Ler genes, except R3, has no effect on growth or pathogen resistance. Incompatibility involves increased R3 expression and engineered R3 overexpression in a neutral background induces dwarfism and sterility. However, no individual RPP1-like Ler gene is sufficient for incompatibility between Ler and Kas-2 or Kond, suggesting that co-action of at least two RPP1-like members underlies this epistatic interaction. We find that the RPP1-like Ler haplotype is frequent and occurs with other Ler RPP1-like alleles in a local population in Gorzów Wielkopolski (Poland). Only Gorzów individuals carrying the RPP1-like Ler haplotype are incompatible with Kas-2 and Kond, whereas other RPP1-like alleles in the population are compatible. Therefore, the RPP1-like Ler haplotype has been maintained in genetically different individuals at a single site, allowing exploration of forces shaping the evolution of RPP1-like genes at local and regional population scales.
	    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(214);">hide</a></div>
	  </div>
      </div>


<div id="publication217" class="publication01">
  <div class="publication01Title">
    Ancient and diverged TGF-β signaling components in Nasonia vitripennis.
  </div>
  <div class="publication01Info">
    Özüak O, Buchta T, Roth S, Lynch JA, Dev Genes Evol. 2014 Dec;224(4-6):223-33. doi: 10.1007/s00427-014-0481-0. Epub 2014 Oct 11.
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(217);">abstract <img id="abstractI217" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/25304164">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP217" class="publication01Abstract" style="display:none;">
    <p>
The transforming growth factor beta (TGF)-β signaling pathway and its modulators are involved in many aspects of cellular growth and differentiation in all metazoa. Although most of the core components of the pathway are highly conserved, many lineage-specific adaptations have been observed including changes regarding paralog number, presence and absence of modulators, and functional relevance for particular processes. In the parasitic jewel wasp Nasonia vitripennis, the bone morphogenetic proteins (BMPs), one of the major subgroups of the TGF-β superfamily, play a more fundamental role in dorsoventral (DV) patterning than in all other insects studied so far. However, Nasonia lacks the BMP antagonist Short gastrulation (Sog)/chordin, which is essential for polarizing the BMP gradient along the DV axis in most bilaterian animals. Here, we present a broad survey of TGF-β signaling in Nasonia with the aim to detect other lineage-specific peculiarities and to identify potential mechanisms, which explain how BMP-dependent DV pattering occurs in the early Nasonia embryo in the absence of Sog. 
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(217);">hide</a>
    </div>
  </div>
</div>



<div id="publication216" class="publication01">
  <div class="publication01Title">
    Dorsoventral polarity of the Nasonia embryo primarily relies on a BMP gradient formed without input from Toll.
  </div>
  <div class="publication01Info">
    Özüak O, Buchta T, Roth S, Lynch JA, Curr Biol. 2014 Oct 20;24(20):2393-8. doi: 10.1016/j.cub.2014.08.035. Epub 2014 Oct 9.
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(216);">abstract <img id="abstractI216" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/25308075">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP216" class="publication01Abstract" style="display:none;">
    <p>
      In Drosophila, Toll signaling leads to a gradient of nuclear uptake of Dorsal with a peak at the ventral egg pole and is the source for dorsoventral (DV) patterning and polarity of the embryo. In contrast, Toll signaling plays no role in embryonic patterning in most animals, while BMP signaling plays the major role. In order to understand the origin of the novelty of the Drosophila system, we have examined DV patterning in Nasonia vitripennis (Nv), a representative of the Hymenoptera and thus the most ancient branch points within the Holometabola. We have previously shown that while the expression of several conserved DV patterning genes is almost identical in Nasonia and Drosophila embryos at the onset of gastrulation, the ways these patterns evolve in early embryogenesis are very different from what is seen in Drosophila or the beetle Tribolium. In contrast to Drosophila or Tribolium, we find that wasp Toll has a very limited ventral role, whereas BMP is required for almost all DV polarity of the embryo, and these two signaling systems act independently of each other to generate DV polarity. This result gives insights into how the Toll pathway could have usurped a BMP-based DV patterning system in insects. In addition, our work strongly suggests that a novel system for BMP activity gradient formation must be employed in the wasp, since orthologs of crucial components of the fly system are either missing entirely or lack function in the embryo.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(216);">hide</a>
    </div>
  </div>
</div>



<div id="publication215" class="publication01">
  <div class="publication01Title">
    Rapid identification of a natural knockout allele of ARMADILLO REPEAT-CONTAINING KINESIN1 that causes root hair branching by mapping-by-sequencing.
  </div>
  <div class="publication01Info">
    Rishmawi L, Sun H, Schneeberger K, Hülskamp M, Schrader A, Plant Physiol. 2014 Nov;166(3):1280-7. doi: 10.1104/pp.114.244046. Epub 2014 Sep 23.
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(215);">abstract <img id="abstractI215" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/25248719">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP215" class="publication01Abstract" style="display:none;">
    <p>In Arabidopsis (Arabidopsis thaliana), branched root hairs are an indicator of defects in root hair tip growth. Among 62 accessions, one accession (Heiligkreuztal2 [HKT2.4]) displayed branched root hairs, suggesting that this accession carries a mutation in a gene of importance for tip growth. We determined 200- to 300-kb mapping intervals using a mapping-by-sequencing approach of F2 pools from crossings of HKT2.4 with three different accessions. The intersection of these mapping intervals was 80 kb in size featuring not more than 36 HKT2.4-specific single nucleotide polymorphisms, only two of which changed the coding potential of genes. Among them, we identified the causative single nucleotide polymorphism changing a splicing site in ARMADILLO REPEAT-CONTAINING KINESIN1. The applied strategies have the potential to complement statistical methods in high-throughput phenotyping studies using different natural accessions to identify causative genes for distinct phenotypes represented by only one or a few accessions. 
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(215);">hide</a>
    </div>
  </div>
</div>

<div id="publication2" class="publication01"><div 
  class="publication01Title">Multidimensional epistasis and the transitory advantage of sex</div><div 
  class="publication01Info">Nowak S, Neidhart J, Szendro IG, Krug J, PLOS Computational Biology 10(9): e1003836 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(2);">abstract <img id="abstractI2" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1003836">paper [pdf]</a> &nbsp; </div><div id="abstractP2" 
  class="publication01Abstract" style="display:none;">Identifying and quantifying the benefits of sex and recombination is a long-standing problem in evolutionary theory. In particular, contradictory claims have been made about the existence of a benefit of recombination on high dimensional fitness landscapes in the presence of sign epistasis. Here we present a comparative numerical study of sexual and asexual evolutionary dynamics of haploids on tunably rugged model landscapes under strong selection, paying special attention to the temporal development of the evolutionary advantage of recombination and the link between population diversity and the rate of adaptation. We show that the adaptive advantage of recombination on static rugged landscapes is strictly transitory. At early times, an advantage of recombination arises through the possibility to combine individually occurring beneficial mutations, but this effect is reversed at longer times by the much more efficient trapping of recombining populations at local fitness peaks. These findings are explained by means of well-established results for a setup with only two loci. In accordance with the Red Queen hypothesis the transitory advantage can be prolonged indefinitely in fluctuating environments, and it is maximal when the environment fluctuates on the same time scale on which trapping at local optima typically occurs.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(2);">hide</a></div></div></div><div id="publication3" class="publication01"><div 
  class="publication01Title">Role of pleiotropy during adaptation of TEM-1 β-lactamase to two novel antibiotics</div><div 
  class="publication01Info">Schenk MF, Witte S, Salverda MLM, Koopmanschap B, Krug J, de Visser JAGM, Evolutionary Applications, early view (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(3);">abstract <img id="abstractI3" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/eva.12200/abstract">paper [pdf]</a> &nbsp; </div><div id="abstractP3" 
  class="publication01Abstract" style="display:none;">Pleiotropy is a key feature of the genotype–phenotype map, and its form and extent have many evolutionary implications, including for the dynamics of adaptation and the evolution of specialization. Similarly, pleiotropic effects of antibiotic resistance mutations may affect the evolution of antibiotic resistance in the simultaneous or fluctuating presence of different antibiotics. Here, we study the role of pleiotropy during the in vitro adaptation of the enzyme TEM-1 β-lactamase to two novel antibiotics, cefotaxime (CTX) and ceftazidime (CAZ). We subject replicate lines for four rounds of evolution to selection with CTX and CAZ alone, and in their combined and fluctuating presence. Evolved alleles show positive correlated responses when selecting with single antibiotics. Nevertheless, pleiotropic constraints are apparent from the effects of single mutations and from selected alleles showing smaller correlated than direct responses and smaller responses after simultaneous and fluctuating selection with both than with single antibiotics. We speculate that these constraints result from structural changes in the oxyanion pocket surrounding the active site, where accommodation of CTX and the larger CAZ is balanced against their positioning with respect to the active site. Our findings suggest limited benefits from the combined or fluctuating application of these related cephalosporins for containing antibiotic resistance.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(3);">hide</a></div></div></div><div id="publication4" class="publication01"><div 
  class="publication01Title">Nonsense-mediated mRNA decay modulates immune receptor levels to regulate plant antibacterial defense</div><div 
  class="publication01Info">Gloggnitzer J, Akimcheva S, Srinivasan A, Kusenda B, Riehs N, Stampfl H, Bautor J, Dekrout B, Jonak C, Jiménez-Gómez JM, Parker JE, Riha K, Cell Host Microbe, 16(3):376-90 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(4);">abstract <img id="abstractI4" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S1931312814003023">paper [pdf]</a> &nbsp; </div><div id="abstractP4" 
  class="publication01Abstract" style="display:none;">Nonsense-mediated mRNA decay (NMD) is a conserved eukaryotic RNA surveillance mechanism that degrades aberrant mRNAs. NMD impairment in Arabidopsis is linked to constitutive immune response activation and enhanced antibacterial resistance, but the underlying mechanisms are unknown. Here we show that NMD contributes to innate immunity in Arabidopsis by controlling the turnover of numerous TIR domain-containing, nucleotide-binding, leucine-rich repeat (TNL) immune receptor-encoding mRNAs. Autoimmunity resulting from NMD impairment depends on TNL signaling pathway components and can be triggered through deregulation of a single TNL gene, RPS6. Bacterial infection of plants causes host-programmed inhibition of NMD, leading to stabilization of NMD-regulated TNL transcripts. Conversely, constitutive NMD activity prevents TNL stabilization and impairs plant defense, demonstrating that host-regulated NMD contributes to disease resistance. Thus, NMD shapes plant innate immunity by controlling the threshold for activation of TNL resistance pathways.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(4);">hide</a></div></div></div><div id="publication5" class="publication01"><div 
  class="publication01Title">Clonal interference and Muller's ratchet in spatial habitats</div><div 
  class="publication01Info">Otwinowski J, Krug J, Physical Biology 11:056003 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(5);">abstract <img id="abstractI5" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1478-3975/11/5/056003">paper [pdf]</a> &nbsp; </div><div id="abstractP5" 
  class="publication01Abstract" style="display:none;">Competition between independently arising beneficial mutations is enhanced in spatial populations due to the linear rather than exponential growth of clones. Recent theoretical studies have pointed out that the resulting fitness dynamics is analogous to a surface growth process, where new layers nucleate and spread stochastically, leading to the build up of scale-invariant roughness. This scenario differs qualitatively from the standard view of adaptation in that the speed of adaptation becomes independent of population size while the fitness variance does not. Here we exploit recent progress in the understanding of surface growth processes to obtain precise predictions for the universal, non-Gaussian shape of the fitness distribution for one-dimensional habitats, which are verified by simulations. When the mutations are deleterious rather than beneficial the problem becomes a spatial version of Muller's ratchet. In contrast to the case of well-mixed populations, the rate of fitness decline remains finite even in the limit of an infinite habitat, provided the ratio [Formula: see text] between the deleterious mutation rate and the square of the (negative) selection coefficient is sufficiently large. Using, again, an analogy to surface growth models we show that the transition between the stationary and the moving state of the ratchet is governed by directed percolation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(5);">hide</a></div></div></div><div id="publication6" class="publication01">

	  <div id="publication218" class="publication01">
	    <div class="publication01Title">
	      Adaptation in tunably rugged fitness landscapes: The Rough Mount Fuji Model
	    </div>
	    <div class="publication01Info">
	      Neidhart J, Szendro IG, Krug J, (Submitted on 13 Feb 2014 (v1), last revised 1 Aug 2014 (this version, v2))
	    </div>
	    <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(218);">abstract <img id="abstractI218" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://arxiv.org/abs/1402.3065v1">paper [pdf]</a> &nbsp; 
	    </div>
	    <div id="abstractP218" class="publication01Abstract" style="display:none;">
	      <p>
		Much of the current theory of adaptation is based on Gillespie's mutational landscape model (MLM), which assumes that the fitness values of genotypes linked by single mutational steps are independent random variables. On the other hand, a growing body of empirical evidence shows that real fitness landscapes, while possessing a considerable amount of ruggedness, are smoother than predicted by the MLM. In the present article we propose and analyse a simple fitness landscape model with tunable ruggedness based on the Rough Mount Fuji (RMF) model originally introduced by Aita et al. [Biopolymers 54:64-79 (2000)] in the context of protein evolution. We provide a comprehensive collection of results pertaining to the topographical structure of RMF landscapes, including explicit formulae for the expected number of local fitness maxima, the location of the global peak, and the fitness correlation function. The statistics of single and multiple adaptive steps on the RMF landscape are explored mainly through simulations, and the results are compared to the known behavior in the MLM model. Finally, we show that the RMF model can explain the large number of second-step mutations observed on a highly-fit first step backgound in a recent evolution experiment with a microvirid bacteriophage [Miller et al., Genetics 187:185-202 (2011)]. 
	      </p>
	      <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(218);">hide</a>
	      </div>
	    </div>
	  </div>

	<div id="publication231" class="publication01">
	  <div class="publication01Title">
	    Plant and animal glycolate oxidases have a common eukaryotic ancestor and convergently duplicated to evolve long-chain 2-hydroxy acid oxidases.
	  </div>
	  <div class="publication01Info">
	    Esser C, Kuhn A, Groth G, Lercher MJ, Maurino VG. Mol Biol Evol. 2014 May;31(5):1089-101. doi: 10.1093/molbev/msu041. Epub 2014 Jan 9. PMID: 24408912 
	  </div>
	  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(231);">abstract <img id="abstractI231" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/24408912">paper [pdf]</a> &nbsp; 
	  </div>
	  <div id="abstractP231" class="publication01Abstract" style="display:none;">
	    <p>
	      Glycolate oxidase (GOX) is a crucial enzyme of plant photorespiration. The encoding gene is thought to have originated from endosymbiotic gene transfer between the eukaryotic host and the cyanobacterial endosymbiont at the base of plantae. However, animals also possess GOX activities. Plant and animal GOX belong to the gene family of (L)-2-hydroxyacid-oxidases ((L)-2-HAOX). We find that all (L)-2-HAOX proteins in animals and archaeplastida go back to one ancestral eukaryotic sequence; the sole exceptions are green algae of the chlorophyta lineage. Chlorophyta replaced the ancestral eukaryotic (L)-2-HAOX with a bacterial ortholog, a lactate oxidase that may have been obtained through the primary endosymbiosis at the base of plantae; independent losses of this gene may explain its absence in other algal lineages (glaucophyta, rhodophyta, and charophyta). We also show that in addition to GOX, plants possess (L)-2-HAOX proteins with different specificities for medium- and long-chain hydroxyacids (lHAOX), likely involved in fatty acid and protein catabolism. Vertebrates possess lHAOX proteins acting on similar substrates as plant lHAOX; however, the existence of GOX and lHAOX subfamilies in both plants and animals is not due to shared ancestry but is the result of convergent evolution in the two most complex eukaryotic lineages. On the basis of targeting sequences and predicted substrate specificities, we conclude that the biological role of plantae (L)-2-HAOX in photorespiration evolved by co-opting an existing peroxisomal protein.
	    </p>
	    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(231);">hide</a>
	    </div>
	  </div>
	</div>



<div 
  class="publication01Title">Evolutionary conservation of cold-induced antisense RNAs of FLOWERING LOCUS C in <em>Arabidopsis thaliana</em> perennial relatives</div><div 
  class="publication01Info">Castaings L, Bergonzi S, Albani MC, Kemi U, Savolainen O, Coupland G, Nature communications, DOI: 10.1038/ncomms5457 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(6);">abstract <img id="abstractI6" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4109010/pdf/ncomms5457.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP6" 
  class="publication01Abstract" style="display:none;">Antisense RNA (asRNA) COOLAIR is expressed at A. thaliana FLOWERING LOCUS C (FLC) in response to winter temperatures. Its contribution to cold-induced silencing of FLC was proposed but its functional and evolutionary significance remain unclear. Here we identify a highly conserved block containing the COOLAIR first exon and core promoter at the 3' end of several FLC orthologues. Furthermore, asRNAs related to COOLAIR are expressed at FLC loci in the perennials A. alpina and A. lyrata, although some splicing variants differ from A. thaliana. Study of the A. alpina orthologue, PERPETUAL FLOWERING 1 (PEP1), demonstrates that AaCOOLAIR is induced each winter of the perennial life cycle. Introduction of PEP1 into A. thaliana reveals that AaCOOLAIR cis-elements confer cold-inducibility in this heterologous species while the difference between PEP1 and FLC mRNA patterns depends on both cis-elements and species-specific trans-acting factors. Thus, expression of COOLAIR is highly conserved, supporting its importance in FLC regulation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(6);">hide</a></div></div></div><div id="publication7" class="publication01"><div 
  class="publication01Title">Genome-wide quantitative analysis of DNA methylation from bisulfite sequencing data</div><div 
  class="publication01Info">Akman K, Haaf T, Gravina S, Vijg J, Tresch A, Bioinformatics, 30(13) (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(7);">abstract <img id="abstractI7" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://bioinformatics.oxfordjournals.org/content/30/13/1933.long">paper [pdf]</a> &nbsp; </div><div id="abstractP7" 
  class="publication01Abstract" style="display:none;">Although many computer programs can perform population genetics calculations, they are typically limited in the analyses and data input formats they offer; few applications can process the large data sets produced by whole-genome resequencing projects. Furthermore, there is no coherent framework for the easy integration of new statistics into existing pipelines, hindering the development and application of new population genetics and genomics approaches. Here, we present PopGenome, a population genomics package for the R software environment (a de facto standard for statistical analyses). PopGenome can efficiently process genome-scale data as well as large sets of individual loci. It reads DNA alignments and single-nucleotide polymorphism (SNP) data sets in most common formats, including those used by the HapMap, 1000 human genomes, and 1001 Arabidopsis genomes projects. PopGenome also reads associated annotation files in GFF format, enabling users to easily define regions or classify SNPs based on their annotation; all analyses can also be applied to sliding windows. PopGenome offers a wide range of diverse population genetics analyses, including neutrality tests as well as statistics for population differentiation, linkage disequilibrium, and recombination. PopGenome is linked to Hudson's MS and Ewing's MSMS programs to assess statistical significance based on coalescent simulations. PopGenome's integration in R facilitates effortless and reproducible downstream analyses as well as the production of publication-quality graphics. Developers can easily incorporate new analyses methods into the PopGenome framework. PopGenome and R are freely available from CRAN (http://cran.r-project.org/) for all major operating systems under the GNU General Public License. <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(7);">hide</a></div></div></div><div id="publication8" class="publication01"><div 
  class="publication01Title">PopGenome: an efficient Swiss army knife for population genomic analyses in R</div><div 
  class="publication01Info">Pfeifer B, Wittelsbürger U, Ramos-Onsins SE,  Lercher MJ, Mol Biol Evol, 31(7):1929-36 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(8);">abstract <img id="abstractI8" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://mbe.oxfordjournals.org/content/31/7/1929.long">paper [pdf]</a> &nbsp; </div><div id="abstractP8" 
  class="publication01Abstract" style="display:none;">Although many computer programs can perform population genetics calculations, they are typically limited in the analyses and data input formats they offer; few applications can process the large data sets produced by whole-genome resequencing projects. Furthermore, there is no coherent framework for the easy integration of new statistics into existing pipelines, hindering the development and application of new population genetics and genomics approaches. Here, we present PopGenome, a population genomics package for the R software environment (a de facto standard for statistical analyses). PopGenome can efficiently process genome-scale data as well as large sets of individual loci. It reads DNA alignments and single-nucleotide polymorphism (SNP) data sets in most common formats, including those used by the HapMap, 1000 human genomes, and 1001 Arabidopsis genomes projects. PopGenome also reads associated annotation files in GFF format, enabling users to easily define regions or classify SNPs based on their annotation; all analyses can also be applied to sliding windows. PopGenome offers a wide range of diverse population genetics analyses, including neutrality tests as well as statistics for population differentiation, linkage disequilibrium, and recombination. PopGenome is linked to Hudson's MS and Ewing's MSMS programs to assess statistical significance based on coalescent simulations. PopGenome's integration in R facilitates effortless and reproducible downstream analyses as well as the production of publication-quality graphics. Developers can easily incorporate new analyses methods into the PopGenome framework. PopGenome and R are freely available from CRAN (http://cran.r-project.org/) for all major operating systems under the GNU General Public License. <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(8);">hide</a></div></div></div><div id="publication9" class="publication01"><div 
  class="publication01Title">The role of photorespiration during the evolution of C4 photosynthesis in the genus Flaveria</div><div 
  class="publication01Info">Mallmann J, Heckmann D, Bräutigam A,  Lercher MJ, Weber AP, Westhoff P, Gowik U, Elife, doi:10.7554/eLife.02478 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(9);">abstract <img id="abstractI9" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://elifesciences.org/content/3/e02478">paper [pdf]</a> &nbsp; </div><div id="abstractP9" 
  class="publication01Abstract" style="display:none;">C4 photosynthesis represents a most remarkable case of convergent evolution of a complex trait, which includes the reprogramming of the expression patterns of thousands of genes. Anatomical, physiological, and phylogenetic and analyses as well as computational modeling indicate that the establishment of a photorespiratory carbon pump (termed C2 photosynthesis) is a prerequisite for the evolution of C4. However, a mechanistic model explaining the tight connection between the evolution of C4 and C2 photosynthesis is currently lacking. Here we address this question through comparative transcriptomic and biochemical analyses of closely related C3, C3–C4, and C4 species, combined with Flux Balance Analysis constrained through a mechanistic model of carbon fixation. We show that C2 photosynthesis creates a misbalance in nitrogen metabolism between bundle sheath and mesophyll cells. Rebalancing nitrogen metabolism requires anaplerotic reactions that resemble at least parts of a basic C4 cycle. Our findings thus show how C2 photosynthesis represents a pre-adaptation for the C4 system, where the evolution of the C2 system establishes important C4 components as a side effect.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(9);">hide</a></div></div></div><div id="publication10" class="publication01"><div 
  class="publication01Title">Empirical fitness landscapes and the predictability of evolution</div><div 
  class="publication01Info">de Visser JAGM, Krug J, Nature Reviews Genetics, published online (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(10);">abstract <img id="abstractI10" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.nature.com/nrg/journal/vaop/ncurrent/abs/nrg3744.html">paper [pdf]</a> &nbsp; </div><div id="abstractP10" 
  class="publication01Abstract" style="display:none;">The genotype–fitness map (that is, the fitness landscape) is a key determinant of evolution, yet it has mostly been used as a superficial metaphor because we know little about its structure. This is now changing, as real fitness landscapes are being analysed by constructing genotypes with all possible combinations of small sets of mutations observed in phylogenies or in evolution experiments. In turn, these first glimpses of empirical fitness landscapes inspire theoretical analyses of the predictability of evolution. Here, we review these recent empirical and theoretical developments, identify methodological issues and organizing principles, and discuss possibilities to develop more realistic fitness landscape models.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(10);">hide</a></div></div></div><div id="publication11" class="publication01"><div 
  class="publication01Title">Coiled-Coil Proteins Facilitated the Functional Expansion of the Centrosome </div><div 
  class="publication01Info">Kuhn M, Hyman AA, Beyer A, PloS Computational Biology, published online (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(11);">abstract <img id="abstractI11" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003657">paper [pdf]</a> &nbsp; </div><div id="abstractP11" 
  class="publication01Abstract" style="display:none;">Repurposing existing proteins for new cellular functions is recognized as a main mechanism of evolutionary innovation, but its role in organelle evolution is unclear. Here, we explore the mechanisms that led to the evolution of the centrosome, an ancestral eukaryotic organelle that expanded its functional repertoire through the course of evolution. We developed a refined sequence alignment technique that is more sensitive to coiled coil proteins, which are abundant in the centrosome. For proteins with high coiled-coil content, our algorithm identified 17% more reciprocal best hits than BLAST. Analyzing 108 eukaryotic genomes, we traced the evolutionary history of centrosome proteins. In order to assess how these proteins formed the centrosome and adopted new functions, we computationally emulated evolution by iteratively removing the most recently evolved proteins from the centrosomal protein interaction network. Coiled-coil proteins that first appeared in the animal–fungi ancestor act as scaffolds and recruit ancestral eukaryotic proteins such as kinases and phosphatases to the centrosome. This process created a signaling hub that is crucial for multicellular development. Our results demonstrate how ancient proteins can be co-opted to different cellular localizations, thereby becoming involved in novel functions.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(11);">hide</a></div></div></div><div id="publication12" class="publication01"><div 
  class="publication01Title">Demography-adjusted tests of neutrality based on genome-wide SNP data</div><div 
  class="publication01Info">Rafajlović M, Klassmann A, Eriksson A, Wiehe T, Mehlig B, Theoretical Population Biology, Vol. 95, pp. 1–12 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(12);">abstract <img id="abstractI12" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S0040580914000367">paper [pdf]</a> &nbsp; </div><div id="abstractP12" 
  class="publication01Abstract" style="display:none;">Tests of the neutral evolution hypothesis are usually built on the standard null model which assumes that mutations are neutral and the population size remains constant over time. However, it is unclear how such tests are affected if the last assumption is dropped. Here, we extend the unifying framework for tests based on the site frequency spectrum, introduced by Achaz and Ferretti, to populations of varying size. Key ingredients are the first two moments of the site frequency spectrum. We show how these moments can be computed analytically if a population has experienced two instantaneous size changes in the past. We apply our method to data from ten human populations gathered in the 1000 genomes project, estimate their demographies and define demography-adjusted versions of Tajima’s DD, Fay & Wu’s HH, and Zeng’s EE. Our results show that demography-adjusted test statistics facilitate the direct comparison between populations and that most of the differences among populations seen in the original unadjusted tests can be explained by their underlying demographies. Upon carrying out whole-genome screens for deviations from neutrality, we identify candidate regions of recent positive selection. We provide track files with values of the adjusted and unadjusted tests for upload to the UCSC genome browser.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(12);">hide</a></div></div></div><div id="publication13" class="publication01"><div 
  class="publication01Title">New tools in the box: An evolutionary synopsis of chromatin insulators</div><div 
  class="publication01Info">Heger P, Wiehe T, Trends In Genetics, Vol. 30, Issue 5, p161–171 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(13);">abstract <img id="abstractI13" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.cell.com/trends/genetics/abstract/S0168-9525(14)00053-5#">paper [pdf]</a> &nbsp; </div><div id="abstractP13" 
  class="publication01Abstract" style="display:none;">Despite progress in understanding genome organization and gene expression during the last decade, the evolutionary pathways that led to the intricate patterns of gene expression in different cells of an organism are still poorly understood. Important steps in this regulation take place at the level of chromatin, where the (epi)genomic environment of a gene determines its expression in time and space. Although the basic mechanisms of gene expression apply to all eukaryotes, multicellular organisms face the additional challenge of coordinating gene expression during development. In this review we summarize and put into evolutionary context current knowledge about chromatin insulators, an important class of regulatory factors mediating these tasks. Our interpretation of historical and recent findings points to a dynamic and ongoing evolution of insulator proteins characterized by multiple instances of convergent evolution, gene loss, and binding site changes in different organisms. The idea of two autonomously evolving insulator functions (as a barrier element and an enhancer blocker) further suggests that the evolution of metazoans and their enhancer-rich gene regulatory repertoire might be connected to the radiation of enhancer blocking insulators. Although speculative at the moment, such coevolution might create tools for complex gene regulation and therefore influence the evolutionary roadmaps of metazoans.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(13);">hide</a></div></div></div><div id="publication14" class="publication01"><div 
  class="publication01Title">A predictive fitness model for influenza</div><div 
  class="publication01Info">Łuksza M, Lässig M, Nature, 507, 57-61 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(14);">abstract <img id="abstractI14" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.nature.com/nature/journal/vaop/ncurrent/abs/nature13087.html">paper [pdf]</a> &nbsp; </div><div id="abstractP14" 
  class="publication01Abstract" style="display:none;">The seasonal human influenza A (H3N2) virus undergoes rapid evolution, which produces significant year-to-year sequence turnover in the population of circulating strains. Adaptive mutations respond to human immune challenge and occur primarily in antigenic epitopes, the antibody-binding domains of the viral surface protein haemagglutinin. Here we develop a fitness model for haemagglutinin that predicts the evolution of the viral population from one year to the next. Two factors are shown to determine the fitness of a strain: adaptive epitope changes and deleterious mutations outside the epitopes. We infer both fitness components for the strains circulating in a given year, using population-genetic data of all previous strains. From fitness and frequency of each strain, we predict the frequency of its descendent strains in the following year. This fitness model maps the adaptive history of influenza A and suggests a principled method for vaccine selection. Our results call for a more comprehensive  epidemiology of influenza and other fast-evolving pathogens that integrates antigenic phenotypes with other viral functions coupled by genetic linkage.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(14);">hide</a></div></div></div><div id="publication15" class="publication01"><div 
  class="publication01Title">Leaf Shape Evolution Through Duplication, Regulatory Diversification, and Loss of a Homeobox Gene</div><div 
  class="publication01Info">Vlad D, Kierzkowski D, Rast MI, Vuolo F, Dello Ioio R, Galinha C, Gan X, Hajheidari M, Hay A, Smith RS, Huijser P, Bailey CD, Tsiantis M, Science Vol. 343 no. 6172 pp. 780-783 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(15);">abstract <img id="abstractI15" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencemag.org/content/343/6172/780.abstract">paper [pdf]</a> &nbsp; </div><div id="abstractP15" 
  class="publication01Abstract" style="display:none;">In this work, we investigate morphological differences between Arabidopsis thaliana, which has simple leaves, and its relative Cardamine hirsuta, which has dissected leaves comprising distinct leaflets. With the use of genetics, interspecific gene transfers, and time-lapse imaging, we show that leaflet development requires the REDUCED COMPLEXITY (RCO) homeodomain protein. RCO functions specifically in leaves, where it sculpts developing leaflets by repressing growth at their flanks. RCO evolved in the Brassicaceae family through gene duplication and was lost in A. thaliana, contributing to leaf simplification in this species. Species-specific RCO action with respect to its paralog results from its distinct gene expression pattern in the leaf base. Thus, regulatory evolution coupled with gene duplication and loss generated leaf shape diversity by modifying local growth patterns during organogenesis.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(15);">hide</a></div></div></div><div id="publication16" class="publication01"><div 
  class="publication01Title">Innate resistance against Toxoplasma gondii: an evolutionary tale of mice, cats, and men</div><div 
  class="publication01Info">Gazzinelli RT, Mendonça-Neto R, Lilue J, Howard J, Sher A, Cell Host Microbe. 15(2):132-8 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(16);">abstract <img id="abstractI16" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S1931312814000274">paper [pdf]</a> &nbsp; </div><div id="abstractP16" 
  class="publication01Abstract" style="display:none;">Recent studies have revealed remarkable species specificity of the Toll-like receptors (TLRs) TLR11 and TLR12 and the immunity-related GTPase (IRG) proteins that are essential elements for detection and immune control of Toxoplasma gondii in mice, but not in humans. The biological and evolutionary implications of these findings for the T. gondii host-pathogen relationship and for human disease are discussed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(16);">hide</a></div></div></div><div id="publication17" class="publication01"><div 
  class="publication01Title">Periodic mRNA synthesis and degradation co‐operate during cell cycle gene expression</div><div 
  class="publication01Info">Eser P, Demel C, Maier KC, Schwalb B, Pirkl N, Martin DE, Cramer P, Tresch A, Mol Syst Biol. 10:717 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(17);">abstract <img id="abstractI17" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://msb.embopress.org/content/10/1/717">paper [pdf]</a> &nbsp; </div><div id="abstractP17" 
  class="publication01Abstract" style="display:none;">During the cell cycle, the levels of hundreds of mRNAs change in a periodic manner, but how this is achieved by alterations in the rates of mRNA synthesis and degradation has not been studied systematically. Here, we used metabolic RNA labeling and comparative dynamic transcriptome analysis (cDTA) to derive mRNA synthesis and degradation rates every 5 min during three cell cycle periods of the yeast <em>Saccharomyces cerevisiae</em>. A novel statistical model identified 479 genes that show periodic changes in mRNA synthesis and generally also periodic changes in their mRNA degradation rates. Peaks of mRNA degradation generally follow peaks of mRNA synthesis, resulting in sharp and high peaks of mRNA levels at defined times during the cell cycle. Whereas the timing of mRNA synthesis is set by upstream DNA motifs and their associated transcription factors (TFs), the synthesis rate of a periodically expressed gene is apparently set by its core promoter.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(17);">hide</a></div></div></div><div id="publication18" class="publication01"><div 
  class="publication01Title">Analysis of TTG1 function in Arabis alpina</div><div 
  class="publication01Info">Chopra D, Wolff H, Span J, Schellmann S, Coupland G, Albani MC, Schrader A, Hülskamp M, BMC Plant Biology 2014; 14:16 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(18);">abstract <img id="abstractI18" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.biomedcentral.com/1471-2229/14/16">paper [pdf]</a> &nbsp; </div><div id="abstractP18" 
  class="publication01Abstract" style="display:none;">
<p><strong>Background</strong></p>
<p>In Arabidopsis thaliana (A. thaliana) the WD40 protein TRANSPARENT TESTA GLABRA1 (TTG1) controls five traits relevant for the adaptation of plants to environmental changes including the production of proanthocyanidin, anthocyanidin, seed coat mucilage, trichomes and root hairs. The analysis of different Brassicaceae species suggests that the function of TTG1 is conserved within the family.</p>
<p><strong>Results</strong></p>
<p>In this work, we studied the function of TTG1 in Arabis alpina (A. alpina). A comparison of wild type and two Aattg1 alleles revealed that AaTTG1 is involved in the regulation of all five traits. A detailed analysis of the five traits showed striking phenotypic differences between A. alpina and A. thaliana such that trichome formation occurs also at later stages of leaf development and that root hairs form at non-root hair positions.</p>
<p><strong>Conclusions</strong></p>
<p>The evolutionary conservation of the regulation of the five traits by TTG1 on the one hand and the striking phenotypic differences make A. alpina a very interesting genetic model system to study the evolution of TTG1-dependent gene regulatory networks at a functional level.</p>
<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(18);">hide</a></div></div></div><div id="publication19" class="publication01"><div 
  class="publication01Title">Horizontal gene acquisitions by eukaryotes as drivers of adaptive evolution</div><div 
  class="publication01Info">Schönknecht G, Weber AP,  Lercher MJ, Bioessays, 36(1):9-20 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(19);">abstract <img id="abstractI19" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1478-3975/11/5/056003">paper [pdf]</a> &nbsp; </div><div id="abstractP19" 
  class="publication01Abstract" style="display:none;">In contrast to vertical gene transfer from parent to offspring, horizontal (or lateral) gene transfer moves genetic information between different species. Bacteria and archaea often adapt through horizontal gene transfer. Recent analyses indicate that eukaryotic genomes, too, have acquired numerous genes via horizontal transfer from prokaryotes and other lineages. Based on this we raise the hypothesis that horizontally acquired genes may have contributed more to adaptive evolution of eukaryotes than previously assumed. Current candidate sets of horizontally acquired eukaryotic genes may just be the tip of an iceberg. We have recently shown that adaptation of the thermoacidophilic red alga Galdieria sulphuraria to its hot, acid, toxic-metal laden, volcanic environment was facilitated by the acquisition of numerous genes from extremophile bacteria and archaea. Other recently published examples of horizontal acquisitions involved in adaptation include ice-binding proteins in marine algae, enzymes for carotenoid biosynthesis in aphids, and genes involved in fungal metabolism. Editor's suggested further reading in BioEssays Jumping the fine LINE between species: Horizontal transfer of transposable elements in animals catalyses genome evolution Abstract.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(19);">hide</a></div></div></div>
          </div>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2013</div>


	    <div id="publication222" class="publication01">
	      <div class="publication01Title">
		Speed Switching of Gonococcal Surface Motility Correlates with Proton Motive Force
	      </div>
	      <div class="publication01Info">
		Kurre, R., Kouzel, N., Ramakrishnan, K., Oldewurtel, E.R., Maier, B., PLoS ONE, 8(6), e67718 (2013)
	      </div>
	      <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(222);">abstract <img id="abstractI222" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0067718">paper [pdf]</a> &nbsp; 
	      </div>
	      <div id="abstractP222" class="publication01Abstract" style="display:none;">
		<p>
		  Bacterial type IV pili are essential for adhesion to surfaces, motility, microcolony formation, and horizontal gene transfer in many bacterial species. These polymers are strong molecular motors that can retract at two different speeds. In the human pathogen Neisseria gonorrhoeae speed switching of single pili from 2 µm/s to 1 µm/s can be triggered by oxygen depletion. Here, we address the question how proton motive force (PMF) influences motor speed. Using pHluorin expression in combination with dyes that are sensitive to transmembrane ΔpH gradient or transmembrane potential ΔΨ, we measured both components of the PMF at varying external pH. Depletion of PMF using uncouplers reversibly triggered switching into the low speed mode. Reduction of the PMF by ≈ 35 mV was enough to trigger speed switching. Reducing ATP levels by inhibition of the ATP synthase did not induce speed switching. Furthermore, we showed that the strictly aerobic Myxococcus xanthus failed to move upon depletion of PMF or oxygen, indicating that although the mechanical properties of the motor are conserved, its regulatory inputs have evolved differently. We conclude that depletion of PMF triggers speed switching of gonococcal pili. Although ATP is required for gonococcal pilus retraction, our data indicate that PMF is an independent additional energy source driving the high speed mode.
		</p>
		<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(222);">hide</a>
		</div>
	      </div>
	    </div>



          <div id="publication20" class="publication01"><div 
  class="publication01Title">Universality and predictability in molecular quantitative genetics</div><div 
  class="publication01Info">Nourmohammad A, Held T, Lässig M, Current Opinion in Genetics and Development, Volume 23, Issue 6, Pages 684–693 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(20);">abstract <img id="abstractI20" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.thp.uni-koeln.de/~lassig/docs/2013d.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP20" 
  class="publication01Abstract" style="display:none;">Molecular traits, such as gene expression levels or protein binding affinities, are increasingly accessible to quantitative measurement by modern high-throughput techniques. Such traits measure molecular functions and, from an evolutionary point of view, are important as targets of natural selection. We review recent developments in evolutionary theory and experiments that are expected to become building blocks of a quantitative genetics of molecular traits. We focus on universal evolutionary characteristics: these are largely independent of a trait's genetic basis, which is often at least partially unknown. We show that universal measurements can be used to infer selection on a quantitative trait, which determines its evolutionary mode of conservation or adaptation. Furthermore, universality is closely linked to predictability of trait evolution across lineages. We argue that universal trait statistics extends over a range of cellular scales and opens new avenues of quantitative evolutionary systems biology.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(20);">hide</a></div></div></div><div id="publication21" class="publication01"><div 
  class="publication01Title">Sybil--efficient constraint-based modelling in R</div><div 
  class="publication01Info">Gelius-Dietrich G, Desouki AA, Fritzemeier CJ,  Lercher MJ, BMC Syst Biol, 7:125 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(21);">abstract <img id="abstractI21" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.biomedcentral.com/1752-0509/7/125">paper [pdf]</a> &nbsp; </div><div id="abstractP21" 
  class="publication01Abstract" style="display:none;"><strong>BACKGROUND:</strong> Constraint-based analyses of metabolic networks are widely used to simulate the properties of genome-scale metabolic networks. Publicly available implementations tend to be slow, impeding large scale analyses such as the genome-wide computation of pairwise gene knock-outs, or the automated search for model improvements. Furthermore, available implementations cannot easily be extended or adapted by users.<br /><strong>RESULTS:</strong>Here, we present sybil, an open source software library for constraint-based analyses in R; R is a free, platform-independent environment for statistical computing and graphics that is widely used in bioinformatics. Among other functions, sybil currently provides efficient methods for flux-balance analysis (FBA), MOMA, and ROOM that are about ten times faster than previous implementations when calculating the effect of whole-genome single gene deletions in silico on a complete E. coli metabolic model.<strong>CONCLUSIONS:</strong><br />Due to the object-oriented architecture of sybil, users can easily build analysis pipelines in R or even implement their own constraint-based algorithms. Based on its highly efficient communication with different mathematical optimisation programs, sybil facilitates the exploration of high-dimensional optimisation problems on small time scales. Sybil and all its dependencies are open source. Sybil and its documentation are available for download from the comprehensive R archive network (CRAN).<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(21);">hide</a></div></div></div><div id="publication22" class="publication01"><div 
  class="publication01Title">A Genomics-Based Classification of Human Lung Tumors</div><div 
  class="publication01Info">Seidel D et al., Science Translational Medicine, 5 (209) p. 209ra153 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(22);">abstract <img id="abstractI22" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://stm.sciencemag.org/content/5/209/209ra153.full.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP22" 
  class="publication01Abstract" style="display:none;">We characterized genome alterations in 1255 clinically annotated lung tumors of all histological subgroups to identify genetically defined and clinically relevant subtypes. More than 55% of all cases had at least one oncogenic genome alteration potentially amenable to specific therapeutic intervention, including several personalized treatment approaches that are already in clinical evaluation. Marked differences in the pattern of genomic alterations existed between and within histological subtypes, thus challenging the original histomorphological diagnosis. Immunohistochemical studies confirmed many of these reassigned subtypes. The reassignment eliminated almost all cases of large cell carcinomas, some of which had therapeutically relevant alterations. Prospective testing of our genomics-based diagnostic algorithm in 5145 lung cancer patients enabled a genome-based diagnosis in 3863 (75%) patients, confirmed the feasibility of rational reassignments of large cell lung cancer, and led to improvement in overall survival in patients with EGFR-mutant or ALK-rearranged cancers. Thus, our findings provide support for broad implementation of genome-based diagnosis of lung cancer.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(22);">hide</a></div></div></div><div id="publication23" class="publication01"><div 
  class="publication01Title">Evolutionary Accessibility of Modular Fitness Landscapes</div><div 
  class="publication01Info">Schmiegelt B, Krug J, Journal of Statistical Physics, October 2013, 1-22 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(23);">abstract <img id="abstractI23" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://link.springer.com/article/10.1007%2Fs10955-013-0868-8">paper [pdf]</a> &nbsp; </div><div id="abstractP23" 
  class="publication01Abstract" style="display:none;">A fitness landscape is a mapping from the space of genetic sequences, which is modeled here as a binary hypercube of dimension L, to the real numbers. We consider random models of fitness landscapes, where fitness values are assigned according to some probabilistic rule, and study the statistical properties of pathways to the global fitness maximum along which fitness increases monotonically. Such paths are important for evolution because they are the only ones that are accessible to an adapting population when mutations occur at a low rate. The focus of this work is on the block model introduced by A.S. Perelson and C.A. Macken (Proc. Natl. Acad. Sci. USA 92:9657, 1995) where the genome is decomposed into disjoint sets of loci (‘modules’) that contribute independently to fitness, and fitness values within blocks are assigned at random. We show that the number of accessible paths can be written as a product of the path numbers within the blocks, which provides a detailed analytic description of the path statistics. The block model can be viewed as a special case of Kauffman’s NK-model, and we compare the analytic results to simulations of the NK-model with different genetic architectures. We find that the mean number of accessible paths in the different versions of the model are quite similar, but the distribution of the path number is qualitatively different in the block model due to its multiplicative structure. A similar statement applies to the number of local fitness maxima in the NK-models, which has been studied extensively in previous works. The overall evolutionary accessibility of the landscape, as quantified by the probability to find at least one accessible path to the global maximum, is dramatically lowered by the modular structure.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(23);">hide</a></div></div></div><div id="publication24" class="publication01"><div 
  class="publication01Title">Reciprocal virulence and resistance polymorphism in the relationship between Toxoplasma gondii and the house mouse</div><div 
  class="publication01Info">Lilue J, Müller UB, Steinfeldt T, Howard JC, Elife. 2013 Oct 29;2:e01298 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(24);">abstract <img id="abstractI24" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://elife.elifesciences.org/content/2/e01298.full.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP24" 
  class="publication01Abstract" style="display:none;">Virulence in the ubiquitous intracellular protozoon <em>Toxoplasma gondii</em> for its natural intermediate host, the mouse, appears paradoxical from an evolutionary standpoint because death of the mouse before encystment interrupts the parasite life cycle. Virulent <em>T. gondii</em> strains secrete kinases and pseudokinases that inactivate the immunity-related GTPases (IRG proteins) responsible for mouse resistance to avirulent strains. Such considerations stimulated a search for IRG alleles unknown in laboratory mice that might confer resistance to virulent strains of <em>T. gondii</em>. We report that the mouse IRG system shows extraordinary polymorphic complexity in the wild. We describe an IRG haplotype from a wild-derived mouse strain that confers resistance against virulent parasites by interference with the virulent kinase complex. In such hosts virulent strains can encyst, hinting at an explanation for the evolution of virulence polymorphism in <em>T. gondii</em>.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(24);">hide</a></div></div></div><div id="publication25" class="publication01"><div 
  class="publication01Title">Rate of Adaptation in Sexuals and Asexuals: A Solvable Model of the Fisher–Muller Effect</div><div 
  class="publication01Info">Park SC, Krug J, Genetics November 1, 2013 vol. 195 no. 3 941-955 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(25);">abstract <img id="abstractI25" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.genetics.org/content/195/3/941.full.pdf+html">paper [pdf]</a> &nbsp; </div><div id="abstractP25" 
  class="publication01Abstract" style="display:none;">The adaptation of large asexual populations is hampered by the competition between independently arising beneficial mutations in different individuals, which is known as clonal interference. In classic work, Fisher and Muller proposed that recombination provides an evolutionary advantage in large populations by alleviating this competition. Based on recent progress in quantifying the speed of adaptation in asexual populations undergoing clonal interference, we present a detailed analysis of the Fisher–Muller mechanism for a model genome consisting of two loci with an infinite number of beneficial alleles each and multiplicative (nonepistatic) fitness effects. We solve the deterministic, infinite population dynamics exactly and show that, for a particular, natural mutation scheme, the speed of adaptation in sexuals is twice as large as in asexuals. This result is argued to hold for any nonzero value of the rate of recombination. Guided by the infinite population result and by previous work on asexual adaptation, we postulate an expression for the speed of adaptation in finite sexual populations that agrees with numerical simulations over a wide range of population sizes and recombination rates. The ratio of the sexual to asexual adaptation speed is a function of population size that increases in the clonal interference regime and approaches 2 for extremely large populations. The simulations also show that the imbalance between the numbers of accumulated mutations at the two loci is strongly suppressed even by a small amount of recombination. The generalization of the model to an arbitrary number L of loci is briefly discussed. If each offspring samples the alleles at each locus from the gene pool of the whole population rather than from two parents, the ratio of the sexual to asexual adaptation speed is approximately equal to L in large populations. A possible realization of this scenario is the reassortment of genetic material in RNA viruses with L genomic segments.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(25);">hide</a></div></div></div><div id="publication26" class="publication01"><div 
  class="publication01Title">Arabidopsis semidwarfs evolved from independent mutations in GA20ox1, ortholog to green revolution dwarf alleles in rice and barley</div><div 
  class="publication01Info">Barboza L, Effgen S, Alonso-Blanco C, Kooke R, Keurentjes JJ, Koornneef M, Alcázar R, Proc Natl Acad Sci USA. 2013 Sep 24;110(39):15818-23 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(26);">abstract <img id="abstractI26" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.pnas.org/content/110/39/15818.long">paper [pdf]</a> &nbsp; </div><div id="abstractP26" 
  class="publication01Abstract" style="display:none;">Understanding the genetic bases of natural variation for developmental and stress-related traits is a major goal of current plant biology. Variation in plant hormone levels and signaling might underlie such phenotypic variation occurring even within the same species. Here we report the genetic and molecular basis of semidwarf individuals found in natural Arabidopsis thaliana populations. Allelism tests demonstrate that independent loss-of-function mutations at GA locus 5 (GA5), which encodes gibberellin 20-oxidase 1 (GA20ox1) involved in the last steps of gibberellin biosynthesis, are found in different populations from southern, western, and northern Europe; central Asia; and Japan. Sequencing of GA5 identified 21 different loss-of-function alleles causing semidwarfness without any obvious general tradeoff affecting plant performance traits. GA5 shows signatures of purifying selection, whereas GA5 loss-of-function alleles can also exhibit patterns of positive selection in specific populations as shown by Fay and Wu’s H statistics. These results suggest that antagonistic pleiotropy might underlie the occurrence of GA5 loss-of-function mutations in nature. Furthermore, because GA5 is the ortholog of rice SD1 and barley Sdw1/Denso green revolution genes, this study illustrates the occurrence of conserved adaptive evolution between wild A.thaliana and domesticated plants.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(26);">hide</a></div></div></div><div id="publication27" class="publication01"><div 
  class="publication01Title">Yule-generated trees constrained by node imbalance</div><div 
  class="publication01Info">Disanto F, Schlizio A, Wiehe T, Mathematical Biosciences (online access) August 13 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(27);">abstract <img id="abstractI27" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S0025556413002101">paper [pdf]</a> &nbsp; </div><div id="abstractP27" 
  class="publication01Abstract" style="display:none;">The <em>Yule</em> process generates a class of binary trees which is fundamental to population genetic models and other applications in evolutionary biology. In this paper, we introduce a family of sub-classes of ranked trees, called Ω-trees, which are characterized by imbalance of internal nodes. The degree of imbalance is defined by an integer 0⩽ω. For caterpillars, the extreme case of unbalanced trees, ω=0. Under models of neutral evolution, for instance the Yule model, trees with small ω are unlikely to occur by chance. Indeed, imbalance can be a signature of permanent selection pressure, such as observable in the genealogies of certain pathogens. From a mathematical point of view it is interesting to observe that the space of Ω-trees maintains several statistical invariants although it is drastically reduced in size compared to the space of unconstrained Yule trees. Using generating functions, we study here some basic combinatorial properties of Ω-trees. We focus on the distribution of the number of subtrees with two leaves. We show that expectation and variance of this distribution match those for unconstrained trees already for very small values of ω.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(27);">hide</a></div></div></div><div id="publication28" class="publication01"><div 
  class="publication01Title">Irgm1 (LRG-47), a regulator of cell-autonomous immunity, does not localize to mycobacterial or listerial phagosomes in IFN-γ-induced mouse cells</div><div 
  class="publication01Info">Springer H, Schramm M, Taylor GA, Howard JC, J. Immunol. Epub July 10 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(28);">abstract <img id="abstractI28" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.jimmunol.org/content/191/4/1765.long">paper [pdf]</a> &nbsp; </div><div id="abstractP28" 
  class="publication01Abstract" style="display:none;">The IFN-inducible protein Irgm1 (LRG-47) belongs to the family of immunity-related GTPases that function in cell-autonomous resistance against intracellular pathogens in mice. Irgm1 deficiency is associated with a severe immunodeficiency syndrome. The protein has been variously interpreted as a direct effector molecule on bacterial phagosomes or on other organelles or as an inducer of autophagy. In this study, we re-examined one of these claims, namely that Irgm1 targets mycobacterial and listerial phagosomes. We found no colocalization of endogenous Irgm1, using two immunofluorescent staining techniques, either in fibroblasts or in macrophages. We demonstrated the predicted existence of two protein isoforms of Irgm1 derived from differential splicing and described immunological reagents for their detection. Both Irgm1 isoforms localize to the Golgi apparatus and weakly to mitochondria; however, only the long Irgm1 isoforms can be detected on endolysosomal membranes. Together with the previous observation that the general immunodeficiency phenotype of Irgm1(-/-) mice is reversed in Irgm1/Irgm3 double-deficient mice, our results argue against a direct effector function of Irgm1 at the bacterial phagosome. We discuss these findings in the context of evidence that Irgm1 functions as a negative regulator of other members of the immunity-related GTPase protein family.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(28);">hide</a></div></div></div><div id="publication29" class="publication01"><div 
  class="publication01Title">Fitness landscape of nucleosome positioning</div><div 
  class="publication01Info">Weghorn D, Lässig M, Proc. Natl. Acad. Sci., vol. 110 no. 27 10988-10993 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(29);">abstract <img id="abstractI29" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.pnas.org/content/110/27/10988.full.pdf+html">paper [pdf]</a> &nbsp; </div><div id="abstractP29" 
  class="publication01Abstract" style="display:none;">Histone–DNA complexes, so-called nucleosomes, are the building blocks of DNA packaging in eukaryotic cells. The histone-binding affinity of a local DNA segment depends on its elastic properties and determines its accessibility within the nucleus, which plays an important role in the regulation of gene expression. Here, we derive a fitness landscape for intergenic DNA segments in yeast as a function of two molecular phenotypes: their elasticity-dependent histone affinity and their coverage with transcription factor binding sites. This landscape reveals substantial selection against nucleosome formation over a wide range of both phenotypes. We use it as the core component of a quantitative evolutionary model for intergenic DNA segments. This model consistently predicts the observed diversity of histone affinities within wild Saccharomyces paradoxus populations, as well as the affinity divergence between neighboring Saccharomyces species. Our analysis establishes histone binding and transcription factor binding as two separable modes of sequence evolution, each of which is a direct target of natural selection.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(29);">hide</a></div></div></div><div id="publication30" class="publication01"><div 
  class="publication01Title">Predicting C4 Photosynthesis Evolution: Modular, Individually Adaptive Steps on a Mount Fuji Fitness Landscape</div><div 
  class="publication01Info">Heckmann D, Schulze S, Denton A, Gowik U, Westhoff P, Weber APM, Lercher MJ, Cell 153, No. 6 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(30);">abstract <img id="abstractI30" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://download.cell.com/pdf/PIIS0092867413005874.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP30" 
  class="publication01Abstract" style="display:none;">An ultimate goal of evolutionary biology is the prediction and experimental verification of adaptive trajectories on macroevolutionary timescales. This aim has rarely been achieved for complex biological systems, as models usually lack clear correlates of organismal fitness. Here, we simulate the fitness landscape connecting two carbon fixation systems: C3 photosynthesis, used by most plant species, and the C4 system, which is more efficient at ambient CO2 levels and elevated temperatures and which repeatedly evolved from C3. Despite extensive sign epistasis, C4 photosynthesis is evolutionarily accessible through individually adaptive steps from any intermediate state. Simulations show that biochemical subtraits evolve in modules; the order and constitution of modules confirm and extend previous hypotheses based on species comparisons. Plant-species-designated C3-C4 intermediates lie on predicted evolutionary trajectories, indicating that they indeed represent transitory states. Contrary to expectations, we find no slowdown of adaptation and no diminishing fitness gains along evolutionary trajectories.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(30);">hide</a></div></div></div><div id="publication31" class="publication01"><div 
  class="publication01Title">Successive gain of insulator proteins in arthropod evolution</div><div 
  class="publication01Info">Heger P, George R, Wiehe T, Evolution (online access) June 4 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(31);">abstract <img id="abstractI31" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/evo.12155/abstract">paper [pdf]</a> &nbsp; </div><div id="abstractP31" 
  class="publication01Abstract" style="display:none;">Alteration of regulatory DNA elements or their binding proteins may have drastic consequences for morphological evolution. Chromatin insulators are one example of such proteins and play a fundamental role in organizing gene expression. While a single insulator protein, CTCF (CCCTC-binding factor), is known in vertebrates, <em>Drosophila melanogaster</em> utilizes six additional factors. We studied the evolution of these proteins and show here that—in contrast to the bilaterian-wide distribution of CTCF—all other <em>D. melanogaster</em> insulators are restricted to arthropods. The full set is present exclusively in the genus <em>Drosophila</em> whereas only two insulators, Su(Hw) and CTCF, existed at the base of the arthropod clade and all additional factors have been acquired successively at later stages. Secondary loss of factors in some lineages further led to the presence of different insulator subsets in arthropods. Thus, the evolution of insulator proteins within arthropods is an ongoing and dynamic process that reshapes and supplements the ancient CTCF-based system common to bilaterians. Expansion of insulator systems may therefore be a general strategy to increase an organism's gene regulatory repertoire and its potential for morphological plasticity.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(31);">hide</a></div></div></div><div id="publication32" class="publication01"><div 
  class="publication01Title">Patterning the dorsal-ventral axis of the wasp Nasonia vitripennis: Novelty and Convergence</div><div 
  class="publication01Info">Buchta T, Özüak O, Stappert D, Roth S, Lynch JA, Dev Biol, Epub ahead of print (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(32);">abstract <img id="abstractI32" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S0012160613002984">paper [pdf]</a> &nbsp; </div><div id="abstractP32" 
  class="publication01Abstract" style="display:none;">Regulatory networks composed of interacting genes are responsible for pattern formation and cell type specification in a wide variety of developmental contexts. Evolution must act on these regulatory networks in order to change the proportions, distribution, and characteristics of specified cells. Thus, understanding how these networks operate in homologous systems across multiple levels of phylogenetic divergence is critical for understanding the evolution of developmental systems. Among the most thoroughly characterized regulatory networks is the dorsal-ventral patterning system of the fly Drosophila melanogaster. Due to the thorough understanding of this system, it is an ideal starting point for comparative analyses. Here we report an analysis of the DV patterning system of the wasp, Nasonia vitripennis. This wasp undergoes a mode of long germ embryogenesis that is superficially nearly identical to that of Drosophila, but one that was likely independently derived. We have found that while the expression of genes just prior to the onset of gastrulation is almost identical in Nasonia and Drosophila, both the upstream network responsible for generating this pattern, and the downstream morphogenetic movements that it sets in motion, are significantly diverged. From this we conclude that many network structures are available to evolution to achieve particular developmental ends.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(32);">hide</a></div></div></div><div id="publication33" class="publication01"><div 
  class="publication01Title">How epigenetic mutations can affect genetic evolution: Model and mechanism</div><div 
  class="publication01Info">Klironomos F, Berg J, Collins S, Bioessays, 35 (6), 571-578 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(33);">abstract <img id="abstractI33" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1002/bies.201200169/pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP33" 
  class="publication01Abstract" style="display:none;">We hypothesize that heritable epigenetic changes can affect rates of fitness increase as well as patterns of genotypic and phenotypic change during adaptation. In particular, we suggest that when natural selection acts on pure epigenetic variation in addition to genetic variation, populations adapt faster, and adaptive phenotypes can arise before any genetic changes. This may make it difficult to reconcile the timing of adaptive events detected using conventional population genetics tools based on DNA sequence data with environmental drivers of adaptation, such as changes in climate. Epigenetic modifications are frequently associated with somatic cell differentiation, but recently epigenetic changes have been found that can be transmitted over many generations. Here, we show how the interplay of these heritable epigenetic changes with genetic changes can affect adaptive evolution, and how epigenetic changes affect the signature of selection in the genetic record.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(33);">hide</a></div></div></div><div id="publication34" class="publication01"><div 
  class="publication01Title">Symplesiomorphies in the WUSCHEL clade suggest that the last common ancestor of seed plants contained at least four independent stem cell niches</div><div 
  class="publication01Info">Nardmann J, Werr W, New Phytologist, 199: 1081–1092 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(34);">abstract <img id="abstractI34" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/nph.12343/pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP34" 
  class="publication01Abstract" style="display:none;">Evolutionary studies addressing plant architecture have uncovered several significant dichotomies between lower and higher land plant radiations, which are based on differences in meristem histology and function. Here, we assess the establishment of different stem cell niches during land plant evolution based on genes of the stem cell-promoting WUSCHEL (WUS) clade of the WOX (WUSCHEL-related homeobox) gene family. WOX gene orthology was addressed by phylogenetic analyses of full-length WOX protein sequences and cellular expression pattern studies indicate process homology. Gene amplifications in the WUS clade were present in the last common ancestor (LCA) of extant gymnosperms and angiosperms. Whereas the evolution of complex multicellular shoot and root meristems relates to members in the WUS/WOX5 sub-branch, the evolution of marginal and plate meristems or the vascular cambium is associated with gene duplications that gave rise to WOX3 and WOX4, respectively. A fourth WUS clade member, WOX2, was apparently recruited for apical cell fate specification during early embryogenesis. The evolution and functional interplay of WOX3 and WOX4 possibly promoted a novel mode of leaf development, and evolutionary adaptations in their activities have contributed to the great diversity in shape and architecture of leaves in seed plants.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(34);">hide</a></div></div></div><div id="publication35" class="publication01"><div 
  class="publication01Title">Co-variation of germination and growth related traits across the European latitudinal range of Arabidopsis thaliana</div><div 
  class="publication01Info">Debieu M, Tang C, Stich B, Effgen S, Josephs E, Schmitt J, Nordborg M, Koornneef M, de Meaux J, PlosONE 8(5): e61075 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(35);">abstract <img id="abstractI35" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plosone.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0061075&representation=PDF">paper [pdf]</a> &nbsp; </div><div id="abstractP35" 
  class="publication01Abstract" style="display:none;">Life-history traits controlling the duration and timing of developmental phases in the life cycle jointly determine fitness. Therefore, life-history traits studied in isolation provide an incomplete view on the relevance of life-cycle variation for adaptation. In this study, we examine genetic variation in traits covering the major life history events of the annual species Arabidopsis thaliana: seed dormancy, vegetative growth rate and flowering time. In a sample of 112 genotypes collected throughout the European range of the species, both seed dormancy and flowering time follow a latitudinal gradient independent of the major population structure gradient. This finding confirms previous studies reporting the adaptive evolution of these two traits. Here, however, we further analyze patterns of co-variation among traits. We observe that co-variation between primary dormancy, vegetative growth rate and flowering time also follows a latitudinal cline. At higher latitudes, vegetative growth rate is positively correlated with primary dormancy and negatively with flowering time. In the South, this trend disappears. Patterns of trait co-variation change, presumably because major environmental gradients shift with latitude. This pattern appears unrelated to population structure, suggesting that changes in the coordinated evolution of major life history traits is adaptive. Our data suggest that A. thaliana provides a good model for the evolution of trade-offs and their genetic basis.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(35);">hide</a></div></div></div><div id="publication36" class="publication01"><div 
  class="publication01Title">A TIR-NBS protein encoded by Arabidopsis Chilling Sensitive 1 (CHS1) limits chloroplast damage and cell death at low temperature</div><div 
  class="publication01Info">Zbierzak AM, Porfirova S, Griebel T, Melzer M, Parker JE, Dörmann P, Plant J. 74: online (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(36);">abstract <img id="abstractI36" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.12219/pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP36" 
  class="publication01Abstract" style="display:none;">Survival of plants at low temperature depends on mechanisms for limiting physiological damage and maintaining growth. We mapped the chs1-1 (chilling sensitive1-1) mutation in Arabidopsis accession Columbia to the TIR-NBS gene At1g17610. In chs1-1, a single amino acid exchange at the CHS1 N-terminus close to the conserved TIR domain creates a stable mutant protein that fails to protect leaves against chilling stress. The sequence of another TIR-NBS gene (At5g40090) named CHL1 (CHS1-like 1) is related to that of CHS1. Over-expression of CHS1 or CHL1 alleviates chilling damage and enhances plant growth at moderate (24°C) and chilling (13°C) temperatures, suggesting a role for both proteins in growth homeostasis. chs1-1 mutants show induced salicylic acid production and defense gene expression at 13°C, indicative of autoimmunity. Genetic analysis of chs1-1 in combination with defense pathway mutants shows that chs1-1 chilling sensitivity requires the TIR-NBS-LRR and basal resistance regulators encoded by EDS1 and PAD4 but not salicylic acid. By following the timing of metabolic, physiological and chloroplast ultrastructural changes in chs1-1 leaves during chilling, we have established that alterations in photosynthetic complexes and thylakoid membrane integrity precede leaf cell death measured by ion leakage. At 24°C, the chs1-1 mutant appears normal but produces a massive necrotic response to virulent Pseudomonas syringae pv. tomato infection, although this does not affect bacterial proliferation. Our results suggest that CHS1 acts at an intersection between temperature sensing and biotic stress pathway activation to maintain plant performance over a range of conditions.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(36);">hide</a></div></div></div><div id="publication37" class="publication01"><div 
  class="publication01Title">Coalescent tree imbalance and a simple test for selective sweeps based on microsatellite variation</div><div 
  class="publication01Info">Li H, Wiehe T, Plos Computational Biology, 9(5): e1003060 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(37);">abstract <img id="abstractI37" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003060">paper [pdf]</a> &nbsp; </div><div id="abstractP37" 
  class="publication01Abstract" style="display:none;">It is one of the major interests in population genetics to contrast the properties and consequences of neutral and non-neutral modes of evolution. As is well-known, positive Darwinian selection and genetic hitchhiking drastically change the profile of genetic diversity compared to neutral expectations. The present-day observable genetic diversity in a sample of DNA sequences depends on events in their evolutionary history, and in particular on the shape of the underlying genealogical tree. In this paper we study how the shape of coalescent trees is affected by the presence of positively selected mutations. We define a measure of tree topology and study its properties under scenarios of neutrality and positive selection. We show that this measure can reliably be estimated from experimental data, and define an easy-to-compute statistical test of the neutral evolution hypothesis. We apply this test to data from a population of the malaria parasite <em>Plasmodium falciparum</em> and confirm the signature of recent positive selection in the vicinity of a drug resistance locus.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(37);">hide</a></div></div></div><div id="publication38" class="publication01"><div 
  class="publication01Title">Patterns of Epistasis between Beneficial Mutations in an Antibiotic Resistance Gene</div><div 
  class="publication01Info">Schenk MF, Szendro IG, Salverda ML, Krug J, de Visser JA, Mol Biol Evol 30 (8): 1779-1787 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(38);">abstract <img id="abstractI38" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://mbe.oxfordjournals.org/content/30/8/1779.full.pdf+html">paper [pdf]</a> &nbsp; </div><div id="abstractP38" 
  class="publication01Abstract" style="display:none;">Understanding epistasis is central to biology. For instance, epistatic interactions determine the topography of the fitness landscape and affect the dynamics and determinism of adaptation. However, few empirical data are available, and comparing results is complicated by confounding variation in the system and the type of mutations used. Here, we take a systematic approach by quantifying epistasis in two sets of four beneficial mutations in the antibiotic resistance enzyme TEM-1 β-lactamase. Mutations in these sets have either large or small effects on cefotaxime resistance when present as single mutations. By quantifying the epistasis and ruggedness in both landscapes, we find two general patterns. First, resistance is maximal for combinations of two mutations in both fitness landscapes and declines when more mutations are added due to abundant sign epistasis and a pattern of diminishing returns with genotype resistance. Second, large-effect mutations interact more strongly than small-effect mutations, suggesting that the effect size of mutations may be an organizing principle in understanding patterns of epistasis. By fitting the data to simple phenotype resistance models, we show that this pattern may be explained by the nonlinear dependence of resistance on enzyme stability and an unknown phenotype when mutations have antagonistically pleiotropic effects. The comparison to a previously published set of mutations in the same gene with a joint benefit further shows that the enzyme's fitness landscape is locally rugged but does contain adaptive pathways that lead to high resistance. <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(38);">hide</a></div></div></div><div id="publication39" class="publication01"><div 
  class="publication01Title">Exact results for the amplitude spectra of fitness landscapes</div><div 
  class="publication01Info">Neidhart J, Szendro I G, Krug J, J. Theor. Biol., Volume 332, 7 September 2013, Pages 218–227 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(39);">abstract <img id="abstractI39" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencedirect.com/science/article/pii/S0022519313002178">paper [pdf]</a> &nbsp; </div><div id="abstractP39" 
  class="publication01Abstract" style="display:none;">Starting from fitness correlation functions, we calculate exact expressions for the amplitude spectra of fitness landscapes as defined by P.F. Stadler [J. Math. Chem. 20, 1 (1996)] for common landscape models, including Kauffman's NK-model, rough Mt. Fuji landscapes and general linear superpositions of such landscapes. We further show that correlations decaying exponentially with Hamming distance yield exponentially decaying spectra similar to those reported recently for a model of molecular signal transduction. Finally, we compare our results for the model systems to the spectra of various experimentally measured fitness landscapes. We claim that our analytical results should be helpful when trying to interpret empirical data and guide the search for improved fitness landscape models.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(39);">hide</a></div></div></div><div id="publication40" class="publication01"><div 
  class="publication01Title">Developmental gene discovery in a hemimetabolous insect: de novo assembly and annotation of a transcriptome for the cricket Gryllus bimaculatus</div><div 
  class="publication01Info">Zeng V, Ewen-Campen B, Horch HW, Roth S, Mito T, Extavour CG, PLoS ONE 8(5): e61479 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(40);">abstract <img id="abstractI40" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0061479">paper [pdf]</a> &nbsp; </div><div id="abstractP40" 
  class="publication01Abstract" style="display:none;">Most genomic resources available for insects represent the Holometabola, which are insects that undergo complete metamorphosis like beetles and flies. In contrast, the Hemimetabola (direct developing insects), representing the basal branches of the insect tree, have very few genomic resources. We have therefore created a large and publicly available transcriptome for the hemimetabolous insect Gryllus bimaculatus (cricket), a well-developed laboratory model organism whose potential for functional genetic experiments is currently limited by the absence of genomic resources. cDNA was prepared using mRNA obtained from adult ovaries containing all stages of oogenesis, and from embryo samples on each day of embryogenesis. Using 454 Titanium pyrosequencing, we sequenced over four million raw reads, and assembled them into 21,512 isotigs (predicted transcripts) and 120,805 singletons with an average coverage per base pair of 51.3. We annotated the transcriptome manually for over 400 conserved genes involved in embryonic patterning, gametogenesis, and signaling pathways. BLAST comparison of the transcriptome against the NCBI non-redundant protein database (nr) identified significant similarity to nr sequences for 55.5% of transcriptome sequences, and suggested that the transcriptome may contain 19,874 unique transcripts. For predicted transcripts without significant similarity to known sequences, we assessed their similarity to other orthopteran sequences, and determined that these transcripts contain recognizable protein domains, largely of unknown function. We created a searchable, web-based database to allow public access to all raw, assembled and annotated data. This database is to our knowledge the largest de novo assembled and annotated transcriptome resource available for any hemimetabolous insect. We therefore anticipate that these data will contribute significantly to more effective and higher-throughput deployment of molecular analysis tools in Gryllus.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(40);">hide</a></div></div></div><div id="publication41" class="publication01"><div 
  class="publication01Title">Cyanobacterial protease inhibitors lead to maternal transfer of increased protease gene expression in Daphnia.</div><div 
  class="publication01Info">Schwarzenberger A, Von Elert E, Oecologia, Volume 172, Issue 1, pp 11-20 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(41);">abstract <img id="abstractI41" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://link.springer.com/article/10.1007%2Fs00442-012-2479-5">paper [pdf]</a> &nbsp; </div><div id="abstractP41" 
  class="publication01Abstract" style="display:none;">Protease inhibitors (PIs) have frequently been found in cyanobacterial blooms and have been shown to affect the major herbivore <em>Daphnia</em> by decreasing growth and inhibiting gut protease activity. However, it has been shown that a clone of <em>Daphnia</em> is able to respond to dietary PIs by increasing its protease gene expression. Such an inducible response might be maternally transferred to the next generation. Therefore, we tested a tolerant clone for maternal transfer of protease gene expression. When exposed to the trypsin inhibitor-producing cyanobacterium Microcystis aeruginosa PCC7806 Mut, <em>Daphnia</em> mothers and their untreated newborns showed an increase in trypsin gene expression compared to naïve mothers grown on control food and their offspring. The maternally transferred increase in gene expression was accompanied by a higher somatic growth rate of the offspring generation from exposed mothers compared to offspring from naïve mothers. This higher growth rate compensated for the lower dry mass of newborns from exposed mothers and led to the same fitness as observed in the offspring of naïve mothers. In nature, clones that can maternally transfer increased protease gene expression should have an advantage over clones that cannot. The selection for such more tolerant clones by naturally occurring PIs might lead to microevolution of natural <em>Daphnia</em> populations, and to local adaptation in the long term. This is the first study to show an adaptive maternal transfer of increased target gene expression in an ecological context.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(41);">hide</a></div></div></div><div id="publication42" class="publication01"><div 
  class="publication01Title">Exact enumeration of cherries and pitchforks in ranked trees under the coalescent model.</div><div 
  class="publication01Info">Disanto F, Wiehe T, Mathematical Biosciences 242, 195-200 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(42);">abstract <img id="abstractI42" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=23396093&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP42" 
  class="publication01Abstract" style="display:none;">We consider exact enumerations and probabilistic properties of ranked trees when generated under the random coalescent process. Using a new approach, based on generating functions, we derive several statistics such as the exact probability of finding k cherries in a ranked tree of fixed size n. We then extend our method to consider also the number of pitchforks. We find a recursive formula to calculate the joint and conditional probabilities of cherries and pitchforks when the size of the tree is fixed. These results provide insights into structural properties of coalescent trees under the model of neutral evolution.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(42);">hide</a></div></div></div><div id="publication43" class="publication01"><div 
  class="publication01Title">Interspecific differences between D. pulex and D. magna in sensitivity to cyanobacteria with protease inhibitors</div><div 
  class="publication01Info">Kuster CJ, Von Elert E, Plos One 8 (5): e62658 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(43);">abstract <img id="abstractI43" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plosone.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0062658&representation=PDF">paper [pdf]</a> &nbsp; </div><div id="abstractP43" 
  class="publication01Abstract" style="display:none;">It is known that cyanobacteria negatively affect herbivores due to their production of toxins such as protease inhibitors. In the present study we investigated potential interspecific differences between two major herbivores, Daphnia magna and Daphnia pulex, in terms of their tolerance to cyanobacteria with protease inhibitors. Seven clones each of D. magna and of D. pulex were isolated from different habitats in Europe and North America. To test for interspecific differences in the daphnids’ tolerance to cyanobacteria, their somatic and population growth rates were determined for each D. magna and D. pulex clone after exposure to varying concentrations of two Microcystis aeruginosa strains. The M. aeruginosa strains NIVA and PCC− contained either chymotrypsin or trypsin inhibitors, but no microcystins. Mean somatic and population growth rates on a diet with 20% NIVA were significantly more reduced in D. pulex than in D. magna. On a diet with 10% PCC−, the population growth of D. pulex was significantly more reduced than that of D. magna. This indicates that D. magna is more tolerant to cyanobacteria with protease inhibitors than D. pulex. The reduction of growth rates was possibly caused by an interference of cyanobacterial inhibitors with proteases in the gut of Daphnia, as many other conceivable factors, which might have been able to explain the reduced growth, could be excluded as causal factors. Protease assays revealed that the sensitivities of chymotrypsins and trypsins to cyanobacterial protease inhibitors did not differ between D. magna and D. pulex. However, D. magna exhibited a 2.3-fold higher specific chymotrypsin activity than D. pulex, which explains the observed higher tolerance to cyanobacterial protease inhibitors of D. magna. The present study suggests that D. magna may control the development of cyanobacterial blooms more efficiently than D. pulex due to differences in their tolerance to cyanobacteria with protease inhibitors.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(43);">hide</a></div></div></div><div id="publication44" class="publication01"><div 
  class="publication01Title">The effect of single recombination events on coalescent tree height and shape</div><div 
  class="publication01Info">Ferretti L, Disanto F, Wiehe T, PLoS One, 8(4):e60123 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(44);">abstract <img id="abstractI44" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plosone.org/article/fetchObjectAttachment.action;?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0060123&representation=PDF">paper [pdf]</a> &nbsp; </div><div id="abstractP44" 
  class="publication01Abstract" style="display:none;">The coalescent with recombination is a fundamental model to describe the genealogical history of DNA sequence samples from recombining organisms. Considering recombination as a process which acts along genomes and which creates sequence segments with shared ancestry, we study the influence of single recombination events upon tree characteristics of the coalescent. We focus on properties such as tree height and tree balance and quantify analytically the changes in these quantities incurred by recombination in terms of probability distributions. We find that changes in tree topology are often relatively mild under conditions of neutral evolution, while changes in tree height are on average quite large. Our results add to a quantitative understanding of the spatial coalescent and provide the neutral reference to which the impact by other evolutionary scenarios, for instance tree distortion by selective sweeps, can be compared.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(44);">hide</a></div></div></div><div id="publication45" class="publication01"><div 
  class="publication01Title">Seasonal succession of cyanobacterial protease inhibitors and Daphnia magna genotypes in a eutrophic Swedish lake</div><div 
  class="publication01Info">Schwarzenberger A, D'hondt S, Vyverman W, Von Elert E, Aquatic Sciences, Volume 75, Issue 3, pp 433-445 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(45);">abstract <img id="abstractI45" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://link.springer.com/article/10.1007%2Fs00027-013-0290-y">paper [pdf]</a> &nbsp; </div><div id="abstractP45" 
  class="publication01Abstract" style="display:none;">Lakes are well known for having a pattern of seasonal succession of phytoplankton and zooplankton. The succession of different taxa of phytoplankton results in a succession of zooplankton taxa, and within the genus <em>Daphnia</em>, into a succession of different genotypes (clones). One cause for this succession of <em>Daphnia</em> clones might be the production of digestive protease inhibitors by cyanobacteria, which usually bloom in summer. Here we report seasonal changes in the frequency and the abundance of <em>Daphnia magna</em> haplotypes in a eutrophic lake, which developed a chymotrypsin-inhibitor-producing cyanobacterial bloom in May. These seasonal changes were not related to changes of biotic and abiotic lake parameters. However, a very high content of chymotrypsin inhibitors was observed in May (but not in other months). This was assumed to have exerted a strong punctual selection pressure on the <em>Daphnia</em> population and on the direct targets of the protease inhibitors, i.e. the digestive chymotrypsins of Daphnia. Actually, D. magna from before and during the cyanobacterial bloom showed a different protease pattern on activity stained SDS-PAGE in comparison to clones from the month after the bloom. However, no difference in tolerance, measured as IC50 values, to inhibition by natural lake seston from May was found between the clones from before and after the bloom. Thus, the hypothesis that a seasonal adaptation of D. magna subpopulations from either April/May or June might have occurred could not be proven. This suggests that the <em>Daphnia</em> population investigated here is locally adapted to cyanobacterial protease inhibitors.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(45);">hide</a></div></div></div><div id="publication46" class="publication01"><div 
  class="publication01Title">Gene Transfer from Bacteria and Archaea Facilitated Evolution of an Extremophilic Eukaryote</div><div 
  class="publication01Info">Schönknecht G, Chen  WH, Ternes CM, Barbier GG, Shrestha RP, Stanke M, Bräutigam A, Baker BJ, Banfield JF, Garavito RM, Carr K, Wilkerson C, Rensing SA, Gagneul D, Dickenson NE, Oesterhelt C, Lercher MJ, Weber APM, Science, Vol. 339 no. 6124 pp. 1207-1210 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(46);">abstract <img id="abstractI46" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sciencemag.org/content/339/6124/1207.full.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP46" 
  class="publication01Abstract" style="display:none;">Some microbial eukaryotes, such as the extremophilic red alga <em>Galdieria sulphuraria</em>, live in hot, toxic metal-rich, acidic environments. To elucidate the underlying molecular mechanisms of adaptation, we sequenced the 13.7-megabase genome of <em>G. sulphuraria</em>. This alga shows an enormous metabolic flexibility, growing either photoautotrophically or heterotrophically on more than 50 carbon sources. Environmental adaptation seems to have been facilitated by horizontal gene transfer from various bacteria and archaea, often followed by gene family expansion. At least 5% of protein-coding genes of <em>G. sulphuraria</em> were probably acquired horizontally. These proteins are involved in ecologically important processes ranging from heavy-metal detoxification to glycerol uptake and metabolism. Thus, our findings show that a pan-domain gene pool has facilitated environmental adaptation in this unicellular eukaryote.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(46);">hide</a></div></div></div><div id="publication47" class="publication01"><div 
  class="publication01Title">Co-option of a coordinate system defined by the EGFr and Dpp pathways in the evolution of a morphological novelty</div><div 
  class="publication01Info">Vreede BM, Lynch JA, Roth S, Sucena E, Evodevo 4:7 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(47);">abstract <img id="abstractI47" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.evodevojournal.com/content/4/1/7">paper [pdf]</a> &nbsp; </div><div id="abstractP47" 
  class="publication01Abstract" style="display:none;"><p style="font-weight:bold;">Background</p>
<p>Morphological innovation is an elusive and fascinating concept in evolutionary biology. A novel structure may open up an array of possibilities for adaptation, and thus is fundamental to the evolution of complex multicellular life. We use the respiratory appendages on the dorsal-anterior side of the Drosophila eggshell as a model system for morphological novelty. To study the co-option of genetic pathways in the evolution of this novelty we have compared oogenesis and eggshell patterning in Drosophila melanogaster with Ceratitis capitata, a dipteran whose eggs do not bear dorsal appendages.</p>
<p style="font-weight:bold;">Results</p>
<p>During the final stages of oogenesis, the appendages are formed by specific groups of cells in the follicular epithelium of the egg chamber. These cells are defined via signaling activity of the Dpp and EGFr pathways, and we find that both pathways are active in C. capitata oogenesis. The transcription factor gene mirror is expressed downstream of EGFr activation in a dorsolateral domain in the D. melanogaster egg chamber, but could not be detected during C. capitata oogenesis. In D. melanogaster, mirror regulates the expression of two important genes: broad, which defines the appendage primordia, and pipe, involved in embryonic dorsoventral polarity. In C. capitata, broad remains expressed ubiquitously throughout the follicular epithelium, and is not restricted to the appendage primordia. Interestingly pipe expression did not differ between the two species.</p>
<p style="font-weight:bold;">Conclusions
<p>Our analysis identifies both broad and mirror as important nodes that have been redeployed in the Drosophila egg chamber patterning network in the evolution of a morphologically novel feature. Further, our results show how pre-existing signals can provide an epithelium with a spatial coordinate system, which can be co-opted for novel patterns.</p><div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(47);">hide</a></div></div></div><div id="publication48" class="publication01"><div 
  class="publication01Title">Evolutionary dynamics and statistical physics</div><div 
  class="publication01Info">Fisher DS, Lässig M, Shraiman B, J. Stat. Mech. N01001 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(48);">abstract <img id="abstractI48" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1742-5468/2013/01/N01001">paper [pdf]</a> &nbsp; </div><div id="abstractP48" 
  class="publication01Abstract" style="display:none;">This introductory article provides the background to and motivation for this special issue and the relationship between evolutionary dynamics and statistical physics.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(48);">hide</a></div></div></div><div id="publication49" class="publication01"><div 
  class="publication01Title">Predictability of evolution depends nonmonotonically on population size.</div><div 
  class="publication01Info">Szendro IG, Franke J, de Visser JAGM, Krug J, Proceedings Of The National Academy Of Sciences Of The United States Of America 110, 571-576 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(49);">abstract <img id="abstractI49" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=23267075&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP49" 
  class="publication01Abstract" style="display:none;">To gauge the relative importance of contingency and determinism in evolution is a fundamental problem that continues to motivate much theoretical and empirical research. In recent evolution experiments with microbes, this question has been explored by monitoring the repeatability of adaptive changes in replicate populations. Here, we present the results of an extensive computational study of evolutionary predictability based on an experimentally measured eight-locus fitness landscape for the filamentous fungus Aspergillus niger. To quantify predictability, we define entropy measures on observed mutational trajectories and endpoints. In contrast to the common expectation of increasingly deterministic evolution in large populations, we find that these entropies display an initial decrease and a subsequent increase with population size N, governed, respectively, by the scales Nμ and Nμ(2), corresponding to the supply rates of single and double mutations, where μ denotes the mutation rate. The amplitude of this pattern is determined by μ. We show that these observations are generic by comparing our findings for the experimental fitness landscape to simulations on simple model landscapes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(49);">hide</a></div></div></div><div id="publication50" class="publication01"><div 
  class="publication01Title">Evolution of molecular phenotypes under stabilizing selection</div><div 
  class="publication01Info">Nourmohammad A, Schiffels S, Lässig M, Journal Of Statistical Mechanics, P01012 (34 pages) (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(50);">abstract <img id="abstractI50" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1742-5468/2013/01/P01012/pdf/1742-5468_2013_01_P01012.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP50" 
  class="publication01Abstract" style="display:none;">Molecular phenotypes are important links between genomic information and organismic functions, fitness, and evolution. Complex phenotypes, which are also called quantitative traits, often depend on multiple genomic loci. Their evolution builds on genome evolution in a complicated way, which involves selection, genetic drift, mutations and recombination. Here we develop a coarse-grained evolutionary statistics for phenotypes, which decouples from details of the underlying genotypes. We derive approximate evolution equations for the distribution of phenotype values within and across populations. This dynamics covers evolutionary processes at high and low recombination rates, that is, it applies to sexual and asexual populations. In a fitness landscape with a single optimal phenotype value, the phenotypic diversity within populations and the divergence between populations reach evolutionary equilibria, which describe stabilizing selection. We compute the equilibrium distributions of both quantities analytically and we show that the ratio of mean divergence and diversity depends on the strength of selection in a universal way: it is largely independent of the phenotype's genomic encoding and of the recombination rate. This establishes a new method for the inference of selection on molecular phenotypes beyond the genome level. We discuss the implications of our findings for the predictability of evolutionary processes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(50);">hide</a></div></div></div><div id="publication51" class="publication01"><div 
  class="publication01Title">Quantitative analyses of empirical fitness landscapes</div><div 
  class="publication01Info">Szendro IG, Schenk MF, Franke J, Krug J, de Visser JAGM, Journal Of Statistical Mechanics, P01005 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(51);">abstract <img id="abstractI51" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1742-5468/2013/01/P01005/pdf/1742-5468_2013_01_P01005.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP51" 
  class="publication01Abstract" style="display:none;">The concept of a fitness landscape is a powerful metaphor that offers insight into various aspects of evolutionary processes and guidance for the study of evolution. Until recently, empirical evidence on the ruggedness of these landscapes was lacking, but since it became feasible to construct all possible genotypes containing combinations of a limited set of mutations, the number of studies has grown to a point where a classification of landscapes becomes possible. The aim of this review is to identify measures of epistasis that allow a meaningful comparison of fitness landscapes and then apply them to the empirical landscapes to discern factors that affect ruggedness. The various measures of epistasis that have been proposed in the literature appear to be equivalent. Our comparison shows that the ruggedness of the empirical landscape is affected by whether the included mutations are beneficial or deleterious and by whether intra- or intergenic epistasis is involved. Finally, the empirical landscapes are compared to landscapes generated with the Rough Mt.\ Fuji model. Despite the simplicity of this model, it captures the features of the experimental landscapes remarkably well.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(51);">hide</a></div></div></div>
          </div>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2012</div>
          <div id="publication52" class="publication01"><div 
  class="publication01Title">Antimicrobial effects of murine mesenchymal stromal cells directed against Toxoplasma gondii and Neospora caninum: role of immunity-related GTPases (IRGs) and guanylate-binding proteins (GBPs).</div><div 
  class="publication01Info">Spekker  K, Leineweber  M, Degrandi  D, Ince  V, Brunder  S, Schmidt  S K, Stuhlsatz  S, Howard  J C, Schares  G, Degistirici  O, Meisel R et al., Medical Microbiology and Immunology (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(52);">abstract <img id="abstractI52" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=23269418&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP52" 
  class="publication01Abstract" style="display:none;">Mesenchymal stromal cells (MSCs) have a multilineage differentiation potential and provide immunosuppressive and antimicrobial functions. Murine as well as human MSCs restrict the proliferation of T cells. However, species-specific differences in the underlying molecular mechanisms have been described. Here, we analyzed the antiparasitic effector mechanisms active in murine MSCs. Murine MSCs, in contrast to human MSCs, could not restrict the growth of a highly virulent strain of Toxoplasma gondii (BK) after stimulation with IFN-γ. However, the growth of a type II strain of T. gondii (ME49) was strongly inhibited by IFN-γ-activated murine MSCs. Immunity-related GTPases (IRGs) as well as guanylate-binding proteins (GBPs) contributed to this antiparasitic effect. Further analysis showed that IFN-γ-activated mMSCs also inhibit the growth of Neospora caninum, a parasite belonging to the apicomplexan group as well. Detailed studies with murine IFN-γ-activated MSC indicated an involvement in IRGs like Irga6, Irgb6 and Irgd in the inhibition of N. caninum. Additional data showed that, furthermore, GBPs like mGBP1 and mGBP2 could have played a role in the anti-N. caninum effect of murine MSCs. These data underline that MSCs, in addition to their regenerative and immunosuppressive activity, function as antiparasitic effector cells as well. However, IRGs are not present in the human genome, indicating a species-specific difference in anti-T. gondii and anti-N. caninum effect between human and murine MSCs.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(52);">hide</a></div></div></div><div id="publication53" class="publication01"><div 
  class="publication01Title">Natural variation in small molecule-induced TIR-NB-LRR signaling induces root growth arrest via EDS1- and PAD4-complexed R protein VICTR in Arabidopsis</div><div 
  class="publication01Info">Kim TH, Kunz HH, Bhattacharjee S, Hauser F, Park J, Engineer C, Liu A, Ha T, Parker JE, Gassmann W, Schroeder JI, Plant Cell 24:5177-5192 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(53);">abstract <img id="abstractI53" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plantcell.org/content/early/2012/12/26/tpc.112.107235.full.pdf+html">paper [pdf]</a> &nbsp; </div><div id="abstractP53" 
  class="publication01Abstract" style="display:none;">In a chemical genetics screen we identified the small-molecule [5-(3,4-dichlorophenyl)furan-2-yl]-piperidine-1-ylmethanethione (DFPM) that triggers rapid inhibition of early abscisic acid signal transduction via PHYTOALEXIN DEFICIENT4 (PAD4)- and ENHANCED DISEASE SUSCEPTIBILITY1 (EDS1)-dependent immune signaling mechanisms. However, mechanisms upstream of EDS1 and PAD4 in DFPM-mediated signaling remain unknown. Here, we report that DFPM generates an Arabidopsis thaliana accession-specific root growth arrest in Columbia-0 (Col-0) plants. The genetic locus responsible for this natural variant, VICTR (VARIATION IN COMPOUND TRIGGERED ROOT growth response), encodes a TIR-NB-LRR (for Toll-Interleukin1 Receptor–nucleotide binding–Leucine-rich repeat) protein. Analyses of T-DNA insertion victr alleles showed that VICTR is necessary for DFPM-induced root growth arrest and inhibition of abscisic acid–induced stomatal closing. Transgenic expression of the Col-0 VICTR allele in DFPM-insensitive Arabidopsis accessions recapitulated the DFPM-induced root growth arrest. EDS1 and PAD4, both central regulators of basal resistance and effector-triggered immunity, as well as HSP90 chaperones and their cochaperones RAR1 and SGT1B, are required for the DFPM-induced root growth arrest. Salicylic acid and jasmonic acid signaling pathway components are dispensable. We further demonstrate that VICTR associates with EDS1 and PAD4 in a nuclear protein complex. These findings show a previously unexplored association between a TIR-NB-LRR protein and PAD4 and identify functions of plant immune signaling components in the regulation of root meristematic zone-targeted growth arrest.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(53);">hide</a></div></div></div><div id="publication54" class="publication01"><div 
  class="publication01Title">PEP1 of Arabis alpina Is Encoded by Two Overlapping Genes That Contribute to Natural Genetic Variation in Perennial Flowering.</div><div 
  class="publication01Info">Albani MC, Castaings L, Wötzel S, Mateos JL, Wunder J, Wang R, Reymond M, Coupland G, PLoS Genetics 8, e1003130 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(54);">abstract <img id="abstractI54" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=23284298&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP54" 
  class="publication01Abstract" style="display:none;">Higher plants exhibit a variety of different life histories. Annual plants live for less than a year and after flowering produce seeds and senesce. By contrast perennials live for many years, dividing their life cycle into episodes of vegetative growth and flowering. Environmental cues control key check points in both life histories. Genes controlling responses to these cues exhibit natural genetic variation that has been studied most in short-lived annuals. We characterize natural genetic variation conferring differences in the perennial life cycle of Arabis alpina. Previously the accession Pajares was shown to flower after prolonged exposure to cold (vernalization) and only for a limited period before returning to vegetative growth. We describe five accessions of A. alpina that do not require vernalization to flower and flower continuously. Genetic complementation showed that these accessions carry mutant alleles at PERPETUAL FLOWERING 1 (PEP1), which encodes a MADS box transcription factor orthologous to FLOWERING LOCUS C in the annual Arabidopsis thaliana. Each accession carries a different mutation at PEP1, suggesting that such variation has arisen independently many times. Characterization of these alleles demonstrated that in most accessions, including Pajares, the PEP1 locus contains a tandem arrangement of a full length and a partial PEP1 copy, which give rise to two full-length transcripts that are differentially expressed. This complexity contrasts with the single gene present in A. thaliana and might contribute to the more complex expression pattern of PEP1 that is associated with the perennial life-cycle. Our work demonstrates that natural accessions of A. alpina exhibit distinct life histories conferred by differences in PEP1 activity, and that continuous flowering forms have arisen multiple times by inactivation of the floral repressor PEP1. Similar phenotypic variation is found in other herbaceous perennial species, and our results provide a paradigm for how characteristic perennial phenotypes might arise.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(54);">hide</a></div></div></div><div id="publication55" class="publication01"><div 
  class="publication01Title">The arginine-rich N-terminal domain of ROP18 is necessary for vacuole targeting and virulence of Toxoplasma gondii.</div><div 
  class="publication01Info">Fentress SJ, Steinfeldt T, Howard JC, Sibley LD, Cellular Microbiology 14, 1921-1933 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(55);">abstract <img id="abstractI55" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22906355&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP55" 
  class="publication01Abstract" style="display:none;">Toxoplasma gondii uses specialized secretory organelles called rhoptries to deliver virulence determinants into the host cell during parasite invasion. One such determinant called rhoptry protein 18 (ROP18) is a polymorphic serine/threonine kinase that phosphorylates host targets to modulate acute virulence. Following secretion into the host cell, ROP18 traffics to the parasitophorous vacuole membrane (PVM) where it is tethered to the cytosolic face of this host-pathogen interface. However, the functional consequences of PVM association are not known. In this report, we show that ROP18 mutants altered in an arginine-rich domain upstream of the kinase domain fail to associate to the PVM following secretion from rhoptries. During infection, host cells upregulate immunity-related GTPases that localize to and destroy the PVM surrounding the parasites. ROP18 disarms this host innate immune pathway by phosphorylating IRGs in a critical GTPase domain and preventing loading on the PVM. Vacuole-targeting mutants of ROP18 failed to phosphorylate Irga6 and were unable to divert IRGs from the PVM, despite retaining intrinsic kinase activity. As a consequence, these mutants were avirulent in a mouse model of acute toxoplasmosis. Thus, the association of ROP18 with the PVM, mediated by its N-terminal arginine-rich domain, is critical to its function as a virulence determinant.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(55);">hide</a></div></div></div><div id="publication56" class="publication01"><div 
  class="publication01Title">GraphAlignment: Bayesian pairwise alignment of biological networks</div><div 
  class="publication01Info">Kolář M, Meier J, Mustonen V, Lässig M, Berg J, BMC Systems Biology, 6:144 doi:10.1186/1752-0509-6-144 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(56);">abstract <img id="abstractI56" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.biomedcentral.com/content/pdf/1752-0509-6-144.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP56" 
  class="publication01Abstract" style="display:none;"><p><strong>Background</strong></p>
<p>With increased experimental availability and accuracy of bio-molecular networks, tools for their comparative and evolutionary analysis are needed. A key component for such studies is the alignment of networks.<p>
<p><strong>Results</strong></p>
<p>We introduce the Bioconductor package GraphAlignment for pairwise alignment of bio-molecular networks. The alignment incorporates information both from network vertices and network edges and is based on an explicit evolutionary model, allowing inference of all scoring parameters directly from empirical data. We compare the performance of our algorithm to an alternative algorithm, Græmlin 2.0.</p>
<p>On simulated data, GraphAlignment outperforms Græmlin 2.0 in several benchmarks except for computational complexity. When there is little or no noise in the data, GraphAlignment is slower than Græmlin 2.0. It is faster than Græmlin 2.0 when processing noisy data containing spurious vertex associations. Its typical case complexity grows approximately as 𝒪(N2.6).</p>
<p>On empirical bacterial protein-protein interaction networks (PIN) and gene co-expression networks, GraphAlignment outperforms Græmlin 2.0 with respect to coverage and specificity, albeit by a small margin. On large eukaryotic PIN, Græmlin 2.0 outperforms GraphAlignment.</p>
<p><strong>Conclusions</strong></p>
<p>The GraphAlignment algorithm is robust to spurious vertex associations, correctly resolves paralogs, and shows very good performance in identification of homologous vertices defined by high vertex and/or interaction similarity. The simplicity and generality of GraphAlignment edge scoring makes the algorithm an appropriate choice for global alignment of networks.</p><div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(56);">hide</a></div></div></div><div id="publication57" class="publication01"><div 
  class="publication01Title">The chromatin insulator CTCF and the emergence of metazoan diversity.</div><div 
  class="publication01Info">Heger P, Marin B, Bartkuhn M, Schierenberg E, Wiehe T, Proceedings Of The National Academy Of Sciences Of The United States Of America 109, 17507-17512 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(57);">abstract <img id="abstractI57" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=23045651&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP57" 
  class="publication01Abstract" style="display:none;">The great majority of metazoans belong to bilaterian phyla. They diversified during a short interval in Earth&apos;s history known as the Cambrian explosion, ~540 million years ago. However, the genetic basis of these events is poorly understood. Here we argue that the vertebrate genome organizer CTCF (CCCTC-binding factor) played an important role for the evolution of bilaterian animals. We provide evidence that the CTCF protein and a genome-wide abundance of CTCF-specific binding motifs are unique to bilaterian phyla, but absent in other eukaryotes. We demonstrate that CTCF-binding sites within vertebrate and Drosophila Hox gene clusters have been maintained for several hundred million years, suggesting an ancient origin of the previously known interaction between Hox gene regulation and CTCF. In addition, a close correlation between the presence of CTCF and Hox gene clusters throughout the animal kingdom suggests conservation of the Hox-CTCF link across the Bilateria. On the basis of these findings, we propose the existence of a Hox-CTCF kernel as principal organizer of bilaterian body plans. Such a kernel could explain (i) the formation of Hox clusters in Bilateria, (ii) the diversity of bilaterian body plans, and (iii) the uniqueness and time of onset of the Cambrian explosion.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(57);">hide</a></div></div></div><div id="publication58" class="publication01"><div 
  class="publication01Title">Clonal interference in the evolution of influenza.</div><div 
  class="publication01Info">Strelkowa N, Lässig M, Genetics 192, 671-682 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(58);">abstract <img id="abstractI58" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22851649&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP58" 
  class="publication01Abstract" style="display:none;">The seasonal influenza A virus undergoes rapid evolution to escape human immune response. Adaptive changes occur primarily in antigenic epitopes, the antibody-binding domains of the viral hemagglutinin. This process involves recurrent selective sweeps, in which clusters of simultaneous nucleotide fixations in the hemagglutinin coding sequence are observed about every 4 years. Here, we show that influenza A (H3N2) evolves by strong clonal interference. This mode of evolution is a red queen race between viral strains with different beneficial mutations. Clonal interference explains and quantifies the observed sweep pattern: we find an average of at least one strongly beneficial amino acid substitution per year, and a given selective sweep has three to four driving mutations on average. The inference of selection and clonal interference is based on frequency time series of single-nucleotide polymorphisms, which are obtained from a sample of influenza genome sequences over 39 years. Our results imply that mode and speed of influenza evolution are governed not only by positive selection within, but also by background selection outside antigenic epitopes: immune adaptation and conservation of other viral functions interfere with each other. Hence, adapting viral proteins are predicted to be particularly brittle. We conclude that a quantitative understanding of influenza&apos;s evolutionary and epidemiological dynamics must be based on all genomic domains and functions coupled by clonal interference.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(58);">hide</a></div></div></div><div id="publication59" class="publication01"><div 
  class="publication01Title">Molecular mechanisms of tolerance to cyanobacterial protease inhibitors revealed by clonal differences in Daphnia magna.</div><div 
  class="publication01Info">Schwarzenberger A, Kuster CJ, Von Elert E, Molecular Ecology 21, 4898-4911 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(59);">abstract <img id="abstractI59" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22943151&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP59" 
  class="publication01Abstract" style="display:none;">Protease inhibitors of primary producers are a major food quality constraint for herbivores. In nutrient-rich freshwater ecosystems, the interaction between primary producers and herbivores is mainly represented by Daphnia and cyanobacteria. Protease inhibitors have been found in many cyanobacterial blooms. These inhibitors have been shown (both in vitro and in situ) to inhibit the most important group of digestive proteases in the daphnid&apos;s gut, that is, trypsins and chymotrypsins. In this study, we fed four different Daphnia magna genotypes with the trypsin-inhibitor-containing cyanobacterial strain Microcystis aeruginosa PCC 7806 Mut. Upon exposure to dietary trypsin inhibitors, all D. magna genotypes showed increased gene expression of digestive trypsins and chymotrypsins. Exposure to dietary trypsin inhibitors resulted in increased activity of chymotrypsins and reduced activity of trypsin. Strong intraspecific differences in tolerance of the four D. magna genotypes to the dietary trypsin inhibitors were found. The degree of tolerance depended on the D. magna genotype. The genotypes&apos; tolerance was positively correlated with the residual trypsin activity and the different IC(50) values of the trypsins. On the genetic level, the different trypsin loci varied between the D. magna genotypes. The two tolerant Daphnia genotypes that both originate from the same lake, which frequently produces cyanobacterial blooms, clustered in a neighbour-joining phylogenetic tree based on the three trypsin loci. This suggests that the genetic variability of trypsin loci was an important cause for the observed intraspecific variability in tolerance to cyanobacterial trypsin inhibitors. Based on these findings, it is reasonable to assume that such genetic variability can also be found in natural populations and thus constitutes the basis for local adaptation of natural populations to dietary protease inhibitors.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(59);">hide</a></div></div></div><div id="publication60" class="publication01"><div 
  class="publication01Title">Evolutionary Accessibility in Tunably Rugged Fitness Landscapes</div><div 
  class="publication01Info">Franke J, Krug J, Journal of Statistical Physics, Volume 148, Issue 4, pp 705-722 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(60);">abstract <img id="abstractI60" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.springerlink.com/content/p3x7512078512n66/">paper [pdf]</a> &nbsp; </div><div id="abstractP60" 
  class="publication01Abstract" style="display:none;">The adaptive evolution of a population under the influence of mutation and selection is strongly influenced by the structure of the underlying fitness landscape, which encodes the interactions between mutations at different genetic loci. Theoretical studies of such landscapes have been carried out for several decades, but only recently experimental fitness measurements encompassing all possible combinations of small sets of mutations have become available. The empirical studies have spawned new questions about the accessibility of optimal genotypes under natural selection. Depending on population dynamic parameters such as mutation rate and population size, evolutionary accessibility can be quantified through the statistics of accessible mutational pathways (along which fitness increases monotonically), or through the study of the basin of attraction of the optimal genotype under greedy (steepest ascent) dynamics. Here we investigate these two measures of accessibility in the framework of Kauffman's LK-model, a paradigmatic family of random fitness landscapes with tunable ruggedness. The key parameter governing the strength of genetic interactions is the number K of interaction partners of each of the L sites in the genotype sequence. In general, accessibility increases with increasing genotype dimensionality L and decreases with increasing number of interactions K. Remarkably, however, we find that some measures of accessibility behave non-monotonically as a function of K, indicating a special role of the most sparsely connected, non-trivial cases K=1 and 2. The relation between models for fitness landscapes and spin glasses is also addressed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(60);">hide</a></div></div></div><div id="publication61" class="publication01"><div 
  class="publication01Title">Seasonal dynamics of sestonic protease inhibition: Impact on <em>Daphnia</em> populations</div><div 
  class="publication01Info">Kuster CJ, Schwarzenberger A, Von Elert E, Hydrobiologia, Volume 172, Issue 1, pp 11-20 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(61);">abstract <img id="abstractI61" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://link.springer.com/article/10.1007/s10750-012-1303-x">paper [pdf]</a> &nbsp; </div><div id="abstractP61" 
  class="publication01Abstract" style="display:none;"><em>Daphnia</em> populations often show rapid microevolutionary adaptation to environmental changes. Here, we investigated the possibility that microevolution of <em>Daphnia</em> populations could be driven by natural sestonic Protease Inhibition (PI). We hypothesized that PI changes seasonally, which might lead to concomitant changes in tolerance to PI in a co-occurring <em>Daphnia</em> magna population. In order to test this, seston from a eutrophic pond was sampled regularly over two successive years. Extracts of these freeze-dried samples were used to determine their Inhibitory Potential (IP) on D. magna gut proteases. In the summer seston the IP against chymotrypsins exceeded that of spring seston 200-fold. In order to test for possible impacts on the co-existing D. magna population, we isolated clones before (spring) and after (fall) the peak of the IP. Microsatellite analyses revealed that the two subpopulations were genetically distinct. Individual exposure of three clones from each population to varying concentrations of a cyanobacterium that contains chymotrypsin inhibitors revealed a decrease in population and somatic growth rate for each clone, but no seasonal effects on <em>Daphnia</em>’s tolerance. In order to include maternal effects, we conducted a multi-clonal competition experiment on various cyanobacterial concentrations. However, no evidence for seasonally increased tolerance of D. magna to dietary protease inhibitors could be found.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(61);">hide</a></div></div></div><div id="publication62" class="publication01"><div 
  class="publication01Title">The genetic basis of flowering responses to seasonal cues.</div><div 
  class="publication01Info">Andrés F, Coupland G, Nature Reviews. Genetics 13, 627-639 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(62);">abstract <img id="abstractI62" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22898651&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP62" 
  class="publication01Abstract" style="display:none;">Plants respond to the changing seasons to initiate developmental programmes precisely at particular times of year. Flowering is the best characterized of these seasonal responses, and in temperate climates it often occurs in spring. Genetic approaches in Arabidopsis thaliana have shown how the underlying responses to changes in day length (photoperiod) or winter temperature (vernalization) are conferred and how these converge to create a robust seasonal response. Recent advances in plant genome analysis have demonstrated the diversity in these regulatory systems in many plant species, including several crops and perennials, such as poplar trees. Here, we report progress in defining the diverse genetic mechanisms that enable plants to recognize winter, spring and autumn to initiate flower development.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(62);">hide</a></div></div></div><div id="publication63" class="publication01"><div 
  class="publication01Title">Mean-field theory for the inverse Ising problem at low temperatures</div><div 
  class="publication01Info">Nguyen HC, Berg J, Phys. Rev. Lett. 109, 050602 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(63);">abstract <img id="abstractI63" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://prl.aps.org/abstract/PRL/v109/i5/e050602">paper [pdf]</a> &nbsp; </div><div id="abstractP63" 
  class="publication01Abstract" style="display:none;">The large amounts of data from molecular biology and neuroscience have lead to a renewed interest in the inverse Ising problem: how to reconstruct parameters of the Ising model (couplings between spins and external fields) from a number of spin configurations sampled from the Boltzmann measure. To invert the relationship between model parameters and observables (magnetizations and correlations), mean-field approximations are often used, allowing the determination of model parameters from data. However, all known mean-field methods fail at low temperatures with the emergence of multiple thermodynamic states. Here, we show how clustering spin configurations can approximate these thermodynamic states and how mean-field methods applied to thermodynamic states allow an efficient reconstruction of Ising models also at low temperatures.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(63);">hide</a></div></div></div><div id="publication64" class="publication01"><div 
  class="publication01Title">High-resolution melting analysis: a genotyping tool for population studies on <em>Daphnia</em></div><div 
  class="publication01Info">Kuster CJ, Von Elert E, Molecular Ecology Resources 12, 1048-1057 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(64);">abstract <img id="abstractI64" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1755-0998.2012.03177.x/abstract">paper [pdf]</a> &nbsp; </div><div id="abstractP64" 
  class="publication01Abstract" style="display:none;">Determining genetic variation at the DNA level within and between natural populations is important for understanding the role of natural selection on phenotypic traits, but many techniques of screening for genetic variation are either cost intensive, not sensitive enough or too labour- and time-consuming. Here, we demonstrate high-resolution melting analysis (HRMA) as a cost-effective and powerful tool for screening variable target genes in natural populations. HRMA is based on monitoring the melting of PCR amplicons. Owing to saturating concentrations of a dye that binds at high concentrations to double-stranded DNA, it is possible to genotype high numbers of samples rapidly and accurately. We analysed digestive trypsins of two <em>Daphnia magna</em> populations as an application example for HRMA. One population originated from a pond containing toxic cyanobacteria that possibly produce protease inhibitors and the other from a pond without such cyanobacteria. The hypothesis was that D. magna clones from ponds with cyanobacteria have undergone selection by these inhibitors, which has led to different trypsin alleles. We first sequenced pooled genomic PCR products of trypsins from both populations to identify variable DNA sequences of active trypsins. Second, we screened variable DNA sequences of each D. magna clone from both populations for single nucleotide polymorphisms via HRMA. The HRMA results revealed that both populations exhibited phenotypic differences in the analysed trypsins. Our results indicate that HRMA is a powerful genotyping tool for studying the variation of target genes in response to selection within and between natural <em>Daphnia</em> populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(64);">hide</a></div></div></div><div id="publication65" class="publication01"><div 
  class="publication01Title">Inducible tolerance to dietary protease inhibitors in Daphnia magna.</div><div 
  class="publication01Info">Von Elert E, Zitt A, Schwarzenberger A, The Journal Of Experimental Biology 215, 2051-2059 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(65);">abstract <img id="abstractI65" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22623193&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP65" 
  class="publication01Abstract" style="display:none;">Daphnia has been shown to acquire tolerance to cyanobacterial toxins within an animals&apos; lifetime and to transfer this tolerance to the next generation. Here we used a strain of the cyanobacterium Microcystis aeruginosa, which contained two chymotrypsin inhibitors (BN920 and CP954), the green alga Scenedesmus obliquus as reference food and a clone of D. magna to investigate the physiological mechanism of acquired tolerance to these cyanobacterial toxins. The intracellular concentrations of CP954 and BN920 were 1550 and 120 μmol l(-1), respectively. When food suspensions of the green alga contained &gt;60% M. aeruginosa, growth rates of D. magna were reduced. When grown on the green alga, three chymotrypsins ranging in mass from 16 to 22 kDa were distinguished in D. magna. Purified BN920 and CP954 specifically inhibited D. magna chymotrypsins. Feeding with encapsulated BN920 resulted in growth depression in D. magna and replacement of the chymotrypsins by three chymotrypsins with smaller molecular mass. With just 20% M. aeruginosa, the same changes in the chymotrypsin pattern as with the pure inhibitor were observed. IC(50) values for inhibition of chymotrypsins of D. magna growing on the green alga were 5.4 nmol l(-1) (BN920) and 7.4 nmol l(-1) (CP954). When D. magna was grown on 20% M. aeruginosa, 2.2-fold higher IC(50) values were observed. This indicated that increased tolerance to these dietary inhibitors was acquired within an animal&apos;s lifetime by remodelling the digestive chymotrypsins, which in turn serves as an intra-generational defence against these cyanobacterial inhibitors. This mechanism might be relevant for the transfer of tolerance to the next generation through maternal effects.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(65);">hide</a></div></div></div><div id="publication66" class="publication01"><div 
  class="publication01Title">Flagellin perception varies quantitatively in Arabidopsis thaliana and its relatives.</div><div 
  class="publication01Info">Vetter MM, Kronholm I, He F, Häweker H, Reymond M, Bergelson J, Robatzek S, de Meaux J, Molecular Biology And Evolution 29, 1655-1667 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(66);">abstract <img id="abstractI66" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22319159&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP66" 
  class="publication01Abstract" style="display:none;">Much is known about the evolution of plant immunity components directed against specific pathogen strains: They show pervasive functional variation and have the potential to coevolve with pathogen populations. However, plants are effectively protected against most microbes by generalist immunity components that detect conserved pathogen-associated molecular patterns (PAMPs) and control the onset of PAMP-triggered immunity. In Arabidopsis thaliana, the receptor kinase flagellin sensing 2 (FLS2) confers recognition of bacterial flagellin (flg22) and activates a manifold defense response. To decipher the evolution of this system, we performed functional assays across a large set of A. thaliana genotypes and Brassicaceae relatives. We reveal extensive variation in flg22 perception, most of which results from changes in protein abundance. The observed variation correlates with both the severity of elicited defense responses and bacterial proliferation. We analyzed nucleotide variation segregating at FLS2 in A. thaliana and detected a pattern of variation suggestive of the rapid fixation of a novel adaptive allele. However, our study also shows that evolution at the receptor locus alone does not explain the evolution of flagellin perception; instead, components common to pathways downstream of PAMP perception likely contribute to the observed quantitative variation. Within and among close relatives, PAMP perception evolves quantitatively, which contrasts with the changes in recognition typically associated with the evolution of R genes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(66);">hide</a></div></div></div><div id="publication67" class="publication01"><div 
  class="publication01Title">Genetic basis of adaptation in Arabidopsis thaliana: local adaptation at the seed dormancy QTL DOG1.</div><div 
  class="publication01Info">Kronholm I, Picó FX, Alonso-Blanco C, Goudet J, de Meaux J, Evolution 66, 2287-2302 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(67);">abstract <img id="abstractI67" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22759302&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP67" 
  class="publication01Abstract" style="display:none;">Local adaptation provides an opportunity to study the genetic basis of adaptation and investigate the allelic architecture of adaptive genes. We study delay of germination 1 (DOG1), a gene controlling natural variation in seed dormancy in Arabidopsis thaliana and investigate evolution of dormancy in 41 populations distributed in four regions separated by natural barriers. Using F(ST) and Q(ST) comparisons, we compare variation at DOG1 with neutral markers and quantitative variation in seed dormancy. Patterns of genetic differentiation among populations suggest that the gene DOG1 contributes to local adaptation. Although Q(ST) for seed dormancy is not different from F(ST) for neutral markers, a correlation with variation in summer precipitation supports that seed dormancy is adaptive. We characterize dormancy variation in several F(2) -populations and show that a series of functionally distinct alleles segregate at the DOG1 locus. Theoretical models have shown that the number and effect of alleles segregatin at quantitative trait loci (QTL) have important consequences for adaptation. Our results provide support to models postulating a large number of alleles at quantitative trait loci involved in adaptation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(67);">hide</a></div></div></div><div id="publication68" class="publication01"><div 
  class="publication01Title">Mutation in TERMINAL FLOWER1 Reverses the Photoperiodic Requirement for Flowering in the Wild Strawberry Fragaria vesca</div><div 
  class="publication01Info">Koskela EA, Mouhu K, Albani MC, Kurokura T, Rantanen M, Sargent DJ, Battey NH, Coupland G, Elomaa P, Hytonen T, Plant Physiol 159, 1043-105 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(68);">abstract <img id="abstractI68" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plantphysiol.org/content/159/3/1043.full.pdf+html">paper [pdf]</a> &nbsp; </div><div id="abstractP68" 
  class="publication01Abstract" style="display:none;">Photoperiodic flowering has been extensively studied in the annual short-day and long-day plants rice (Oryza sativa) and Arabidopsis (Arabidopsis thaliana), whereas less is known about the control of flowering in perennials. In the perennial wild strawberry, Fragaria vesca (Rosaceae), short-day and perpetual flowering long-day accessions occur. Genetic analyses showed that differences in their flowering responses are caused by a single gene, SEASONAL FLOWERING LOCUS, which may encode the F. vesca homolog of TERMINAL FLOWER1 (FvTFL1). We show through high-resolution mapping and transgenic approaches that FvTFL1 is the basis of this change in flowering behavior and demonstrate that FvTFL1 acts as a photoperiodically regulated repressor. In short-day F. vesca, long photoperiods activate FvTFL1 mRNA expression and short days suppress it, promoting flower induction. These seasonal cycles in FvTFL1 mRNA level confer seasonal cycling of vegetative and reproductive development. Mutations in FvTFL1 prevent long-day suppression of flowering, and the early flowering that then occurs under long days is dependent on the F. vesca homolog of FLOWERING LOCUS T. This photoperiodic response mechanism differs from those described in model annual plants. We suggest that this mechanism controls flowering within the perennial growth cycle in F. vesca and demonstrate that a change in a single gene reverses the photoperiodic requirements for flowering.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(68);">hide</a></div></div></div><div id="publication69" class="publication01"><div 
  class="publication01Title">Quantifying the adaptive potential of an antibiotic resistance enzyme.</div><div 
  class="publication01Info">Schenk MF, Szendro IG, Krug J, de Visser JAGM, PLoS Genetics 8, e1002783 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(69);">abstract <img id="abstractI69" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22761587&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP69" 
  class="publication01Abstract" style="display:none;">For a quantitative understanding of the process of adaptation, we need to understand its &quot;raw material,&quot; that is, the frequency and fitness effects of beneficial mutations. At present, most empirical evidence suggests an exponential distribution of fitness effects of beneficial mutations, as predicted for Gumbel-domain distributions by extreme value theory. Here, we study the distribution of mutation effects on cefotaxime (Ctx) resistance and fitness of 48 unique beneficial mutations in the bacterial enzyme TEM-1 β-lactamase, which were obtained by screening the products of random mutagenesis for increased Ctx resistance. Our contributions are threefold. First, based on the frequency of unique mutations among more than 300 sequenced isolates and correcting for mutation bias, we conservatively estimate that the total number of first-step mutations that increase Ctx resistance in this enzyme is 87 [95% CI 75-189], or 3.4% of all 2,583 possible base-pair substitutions. Of the 48 mutations, 10 are synonymous and the majority of the 38 non-synonymous mutations occur in the pocket surrounding the catalytic site. Second, we estimate the effects of the mutations on Ctx resistance by determining survival at various Ctx concentrations, and we derive their fitness effects by modeling reproduction and survival as a branching process. Third, we find that the distribution of both measures follows a Fréchet-type distribution characterized by a broad tail of a few exceptionally fit mutants. Such distributions have fundamental evolutionary implications, including an increased predictability of evolution, and may provide a partial explanation for recent observations of striking parallel evolution of antibiotic resistance.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(69);">hide</a></div></div></div><div id="publication70" class="publication01"><div 
  class="publication01Title">Does the Bicoid gradient matter?</div><div 
  class="publication01Info">Roth S, Lynch J, Cell 149, 511-512 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(70);">abstract <img id="abstractI70" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22541424&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP70" 
  class="publication01Abstract" style="display:none;">The generation and interpretation of positional information are key processes in developmental systems. In this issue, Chen et al. report discoveries made in the Drosophila embryo that give new insights into how positional information can be produced by patterning gradients.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(70);">hide</a></div></div></div><div id="publication71" class="publication01"><div 
  class="publication01Title">Development. Making waves for segments.</div><div 
  class="publication01Info">Roth S, Panfilio KA, Science (New York, N.Y.) 336, 306-307 (2012)</div><div 
  class="publication01Links"><a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22517849&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div></div><div id="publication72" class="publication01"><div 
  class="publication01Title">Signals of speciation within Arabidopsis thaliana in comparison with its relatives.</div><div 
  class="publication01Info">Alcázar R, Pecinka A, Aarts MGM, Fransz PF, Koornneef M, Current Opinion In Plant Biology 15, 205-211 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(72);">abstract <img id="abstractI72" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22265228&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP72" 
  class="publication01Abstract" style="display:none;">The species within the now well-defined Arabidopsis genus provide biological materials suitable to investigate speciation and the development of reproductive isolation barriers between related species. Even within the model species A. thaliana, genetic differentiation between populations due to environmental adaptation or demographic history can lead to cases where hybrids between accessions are non-viable. Experimental evidence supports the importance of genome duplications and genetic epistatic interactions in the occurrence of reproductive isolation. Other examples of adaptation to specific environments can be found in Arabidopsis relatives where hybridization and chromosome doubling lead to new amphidiploid species. Molecular signals of speciation found in the Arabidopsis genus should provide a better understanding of speciation processes in plants from a genetic, molecular and evolutionary perspective.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(72);">hide</a></div></div></div><div id="publication73" class="publication01"><div 
  class="publication01Title">Chance and risk in adaptive evolution.</div><div 
  class="publication01Info">Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 109, 4719-4720 (2012)</div><div 
  class="publication01Links"><a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22411817&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div></div><div id="publication74" class="publication01"><div 
  class="publication01Title">Bethe-Peierls approximation and the inverse Ising problem</div><div 
  class="publication01Info">Chau Nguyen H, Berg J, Journal Of Statistical Mechanics 03, 004 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(74);">abstract <img id="abstractI74" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2012JSMTE..03..004C&amp;link_type=ABSTRACT">paper [pdf]</a> &nbsp; </div><div id="abstractP74" 
  class="publication01Abstract" style="display:none;">We apply the Bethe-Peierls approximation to the inverse Ising problem and show how the linear response relation leads to a simple method for reconstructing couplings and fields of the Ising model. This reconstruction is exact on tree graphs, yet its computational expense is comparable to those of other mean-field methods. We compare the performance of this method to the independent-pair, naive mean-field, and Thouless-Anderson-Palmer approximations, the Sessak-Monasson expansion, and susceptibility propagation on the Cayley tree, SK model and random graph with fixed connectivity. At low temperatures, Bethe reconstruction outperforms all of these methods, while at high temperatures it is comparable to the best method available so far (the Sessak-Monasson method). The relationship between Bethe reconstruction and other mean-field methods is discussed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(74);">hide</a></div></div></div><div id="publication75" class="publication01"><div 
  class="publication01Title">Horizontal gene transfers as metagenomic gene duplications.</div><div 
  class="publication01Info">Grassi L, Caselle M, Lercher MJ, Lagomarsino MC, Molecular BioSystems 8, 790-795 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(75);">abstract <img id="abstractI75" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 22218456&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP75" 
  class="publication01Abstract" style="display:none;">While it is well accepted that horizontal gene transfer plays an important role in the evolution and the diversification of prokaryotic genomes, many questions remain open regarding its functional mechanisms of action and its interplay with the extant genome. This study addresses the relationship between proteome innovation by horizontal gene transfer and genome content in Proteobacteria. We characterize the transferred genes, focusing on the protein domain compositions and their relationships with the existing protein domain superfamilies in the genome. In agreement with previous observations, we find that the protein domain architectures of horizontally transferred genes are significantly shorter than the genomic average. Furthermore, protein domains that are more common in the total pool of genomes appear to have a proportionally higher chance to be transferred. This suggests that transfer events behave as if they were drawn randomly from a cross-genomic community gene pool, much like gene duplicates are drawn from a genomic gene pool. Finally, horizontally transferred genes carry domains of exogenous families less frequently for larger genomes, although they might do it more than expected by chance.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(75);">hide</a></div></div></div><div id="publication76" class="publication01"><div 
  class="publication01Title">Widespread interspecific divergence in cis-regulation of transposable elements in the Arabidopsis genus.</div><div 
  class="publication01Info">He F, Zhang X, Hu J, Turck F, Dong X, Goebel U, Borevitz JO, de Meaux J, Molecular Biology And Evolution 29, 1081-1091 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(76);">abstract <img id="abstractI76" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22086904&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP76" 
  class="publication01Abstract" style="display:none;">Transposable elements (TEs) are so abundant and variable that they count among the most important mutational sources in genomes. Nonetheless, little is known about the genetics of their variation in activity or silencing across closely related species. Here, we demonstrate that regulation of TE genes can differ dramatically between the two closely related Arabidopsis species A. thaliana and A. lyrata. In leaf and floral tissues of F1 interspecific hybrids, about 47% of TEs show allele-specific expression, with the A. lyrata copy being generally expressed at higher level. We confirm that TEs are generally expressed in A. lyrata but not in A. thaliana. Allele-specific differences in TE expression are associated with divergence in epigenetic modifications like DNA and histone methylation between species as well as with sequence divergence. Our data demonstrate that A. thaliana silences TEs much better than A. lyrata. For long terminal repeat retrotransposons, these differences are more pronounced for younger insertions. Interspecific differences in TE silencing may have a great impact on genome size changes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(76);">hide</a></div></div></div><div id="publication77" class="publication01"><div 
  class="publication01Title">Comparative genomics of Coccidian parasites differing in host range and transmission strategy</div><div 
  class="publication01Info">Reid AJ, Vermont SJ, Cotton JA, Harris D, Hill-Cawthorne GA, Könen-Waisman S, Latham S, Mourier T, Norton R, Quail M, Sanders M, Shanmugam D, Sohal A, Wasmuth J, Brunk B, Grigg M, Howard JC, Parkinson J, Roos DS, Trees AJ, Berriman M, Pain A, Wastling JM, PLoS Pathogens (3): e1002567 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(77);">abstract <img id="abstractI77" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plospathogens.org/article/info%3Adoi%2F10.1371%2Fjournal.ppat.1002567">paper [pdf]</a> &nbsp; </div><div id="abstractP77" 
  class="publication01Abstract" style="display:none;"><em>Toxoplasma gondii</em> is a zoonotic protozoan parasite which infects nearly one third of the human population and is found in an extraordinary range of vertebrate hosts. Its epidemiology depends heavily on horizontal transmission, especially between rodents and its definitive host, the cat. Neospora caninum is a recently discovered close relative of <em>Toxoplasma</em>, whose definitive host is the dog. Both species are tissue-dwelling Coccidia and members of the phylum Apicomplexa; they share many common features, but Neospora neither infects humans nor shares the same wide host range as Toxoplasma, rather it shows a striking preference for highly efficient vertical transmission in cattle. These species therefore provide a remarkable opportunity to investigate mechanisms of host restriction, transmission strategies, virulence and zoonotic potential. We sequenced the genome of <em>N. caninum</em> and transcriptomes of the invasive stage of both species, undertaking an extensive comparative genomics and transcriptomics analysis. We estimate that these organisms diverged from their common ancestor around 28 million years ago and find that both genomes and gene expression are remarkably conserved. However, in N. caninum we identified an unexpected expansion of surface antigen gene families and the divergence of secreted virulence factors, including rhoptry kinases. Specifically we show that the rhoptry kinase ROP18 is pseudogenised in N. caninum and that, as a possible consequence, Neospora is unable to phosphorylate host immunity-related GTPases, as <em>Toxoplasma</em> does. This defense strategy is thought to be key to virulence in <em>Toxoplasma</em>. We conclude that the ecological niches occupied by these species are influenced by a relatively small number of gene products which operate at the host-parasite interface and that the dominance of vertical transmission in N. caninum may be associated with the evolution of reduced virulence in this species.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(77);">hide</a></div></div></div><div id="publication78" class="publication01"><div 
  class="publication01Title">Correlations of record events as a test for heavy-tailed distributions.</div><div 
  class="publication01Info">Franke  J, Wergen  G, Krug  J, Physical Review Letters 108, 064101 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(78);">abstract <img id="abstractI78" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22401074&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP78" 
  class="publication01Abstract" style="display:none;">A record is an entry in a time series that is larger or smaller than all previous entries. If the time series consists of independent, identically distributed random variables with a superimposed linear trend, record events are positively (negatively) correlated when the tail of the distribution is heavier (lighter) than exponential. Here we use these correlations to detect heavy-tailed behavior in small sets of independent random variables. The method consists of converting random subsets of the data into time series with a tunable linear drift and computing the resulting record correlations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(78);">hide</a></div></div></div><div id="publication79" class="publication01"><div 
  class="publication01Title">OGEE: an online gene essentiality database.</div><div 
  class="publication01Info">Chen W, Minguez P, Lercher MJ, Bork P, Nucleic Acids Research 40, D901-6 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(79);">abstract <img id="abstractI79" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 22075992&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP79" 
  class="publication01Abstract" style="display:none;">OGEE is an Online GEne Essentiality database. Its main purpose is to enhance our understanding of the essentiality of genes. This is achieved by collecting not only experimentally tested essential and non-essential genes, but also associated gene features such as expression profiles, duplication status, conservation across species, evolutionary origins and involvement in embryonic development. We focus on large-scale experiments and complement our data with text-mining results. Genes are organized into data sets according to their sources. Genes with variable essentiality status across data sets are tagged as conditionally essential, highlighting the complex interplay between gene functions and environments. Linked tools allow the user to compare gene essentiality among different gene groups, or compare features of essential genes to non-essential genes, and visualize the results. OGEE is freely available at http://ogeedb.embl.de.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(79);">hide</a></div></div></div><div id="publication80" class="publication01"><div 
  class="publication01Title">The invention of WUS-like stem cell-promoting functions in plants predates leptosporangiate ferns.</div><div 
  class="publication01Info">Nardmann J, Werr W, Plant Molecular Biology 78, 123-134 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(80);">abstract <img id="abstractI80" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22076631&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP80" 
  class="publication01Abstract" style="display:none;">The growth of land plants depends on stem cell-containing meristems which show major differences in their architecture from basal to higher plant species. In Arabidopsis, the stem cell niches in the shoot and root meristems are promoted by WUSCHEL (WUS) and WOX5, respectively. Both genes are members of a non-ancestral clade of the WUS-related homeobox (WOX) gene family, which is absent in extant bryophytes and lycophytes. Our analyses of five fern species suggest that a single WUS orthologue was present in the last common ancestor (LCA) of leptosporangiate ferns and seed plants. In the extant fern Ceratopteris richardii, the WUS pro-orthologue marks the pluripotent cell fate of immediate descendants of the root apical initial, so-called merophytes, which undergo a series of stereotypic cell divisions and give rise to all cell types of the root except the root cap. The invention of a WUS-like function within the WOX gene family in an ancestor of leptosporangiate ferns and seed plants and its amplification and sub-functionalisation to different stem cell niches might relate to the success of seed plants, especially angiosperms.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(80);">hide</a></div></div></div><div id="publication81" class="publication01"><div 
  class="publication01Title">A Toxoplasma gondii pseudokinase inhibits host IRG resistance proteins.</div><div 
  class="publication01Info">Fleckenstein MC, Reese ML, Könen-Waisman S, Boothroyd JC, Howard JC, Steinfeldt T, PLoS Biology 10, e1001358 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(81);">abstract <img id="abstractI81" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22802726&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP81" 
  class="publication01Abstract" style="display:none;">The ability of mice to resist infection with the protozoan parasite, Toxoplasma gondii, depends in large part on the function of members of a complex family of atypical large GTPases, the interferon-gamma-inducible immunity-related GTPases (IRG proteins). Nevertheless, some strains of T. gondii are highly virulent for mice because, as recently shown, they secrete a polymorphic protein kinase, ROP18, from the rhoptries into the host cell cytosol at the moment of cell invasion. Depending on the allele, ROP18 can act as a virulence factor for T. gondii by phosphorylating and thereby inactivating mouse IRG proteins. In this article we show that IRG proteins interact not only with ROP18, but also strongly with the products of another polymorphic locus, ROP5, already implicated as a major virulence factor from genetic crosses, but whose function has previously been a complete mystery. ROP5 proteins are members of the same protein family as ROP18 kinases but are pseudokinases by sequence, structure, and function. We show by a combination of genetic and biochemical approaches that ROP5 proteins act as essential co-factors for ROP18 and present evidence that they work by enforcing an inactive GDP-dependent conformation on the IRG target protein. By doing so they prevent GTP-dependent activation and simultaneously expose the target threonines on the switch I loop for phosphorylation by ROP18, resulting in permanent inactivation of the protein. This represents a novel mechanism in which a pseudokinase facilitates the phosphorylation of a target by a partner kinase by preparing the substrate for phosphorylation, rather than by upregulation of the activity of the kinase itself.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(81);">hide</a></div></div></div><div id="publication82" class="publication01"><div 
  class="publication01Title">Comparisons of the embryonic development of <em>Drosophila</em>, <em>Nasonia</em>, and <em>Tribolium</em></div><div 
  class="publication01Info">Lynch JA, El-Sherif E, Brown SJ, Wiley Interdisciplinary Reviews: Developmental Biology, 1: 16–39 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(82);">abstract <img id="abstractI82" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1002/wdev.3/pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP82" 
  class="publication01Abstract" style="display:none;">Studying the embryogenesis of diverse insect species is crucial to understanding insect evolution. Here, we review current advances in understanding the development of two emerging model organisms: the wasp Nasonia vitripennis and the beetle Tribolium castaneum in comparison with the well-studied fruit fly Drosophila melanogaster. Although Nasonia represents the most basally branching order of holometabolous insects, it employs a derived long germband mode of embryogenesis, more like that of Drosophila, whereas Tribolium undergoes an intermediate germband mode of embryogenesis, which is more similar to the ancestral mechanism. Comparing the embryonic development and genetic regulation of early patterning events in these three insects has given invaluable insights into insect evolution. The similar mode of embryogenesis of Drosophila and Nasonia is reflected in their reliance on maternal morphogenetic gradients. However, they employ different genes as maternal factors, reflecting the evolutionary distance separating them. Tribolium, on the other hand, relies heavily on self-regulatory mechanisms other than maternal cues, reflecting its sequential nature of segmentation and the need for reiterated patterning.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(82);">hide</a></div></div></div>
          </div>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2011</div>
          <div id="publication83" class="publication01"><div 
  class="publication01Title">Emergent neutrality in adaptive asexual evolution.</div><div 
  class="publication01Info">Schiffels S, Szöllosi GJ, Mustonen V, Lässig M, Genetics 189, 1361-1375 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(83);">abstract <img id="abstractI83" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21926305&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP83" 
  class="publication01Abstract" style="display:none;">In nonrecombining genomes, genetic linkage can be an important evolutionary force. Linkage generates interference interactions, by which simultaneously occurring mutations affect each other&apos;s chance of fixation. Here, we develop a comprehensive model of adaptive evolution in linked genomes, which integrates interference interactions between multiple beneficial and deleterious mutations into a unified framework. By an approximate analytical solution, we predict the fixation rates of these mutations, as well as the probabilities of beneficial and deleterious alleles at fixed genomic sites. We find that interference interactions generate a regime of emergent neutrality: all genomic sites with selection coefficients smaller in magnitude than a characteristic threshold have nearly random fixed alleles, and both beneficial and deleterious mutations at these sites have nearly neutral fixation rates. We show that this dynamic limits not only the speed of adaptation, but also a population&apos;s degree of adaptation in its current environment. We apply the model to different scenarios: stationary adaptation in a time-dependent environment and approach to equilibrium in a fixed environment. In both cases, the analytical predictions are in good agreement with numerical simulations. Our results suggest that interference can severely compromise biological functions in an adapting population, which sets viability limits on adaptive evolution under linkage.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(83);">hide</a></div></div></div><div id="publication84" class="publication01"><div 
  class="publication01Title">Mathematics and biology: a Kantian view on the history of pattern formation theory.</div><div 
  class="publication01Info">Roth S, Development Genes and Evolution 221, 255-279 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(84);">abstract <img id="abstractI84" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22086125&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP84" 
  class="publication01Abstract" style="display:none;">Driesch&apos;s statement, made around 1900, that the physics and chemistry of his day were unable to explain self-regulation during embryogenesis was correct and could be extended until the year 1972. The emergence of theories of self-organisation required progress in several areas including chemistry, physics, computing and cybernetics. Two parallel lines of development can be distinguished which both culminated in the early 1970s. Firstly, physicochemical theories of self-organisation arose from theoretical (Lotka 1910-1920) and experimental work (Bray 1920; Belousov 1951) on chemical oscillations. However, this research area gained broader acceptance only after thermodynamics was extended to systems far from equilibrium (1922-1967) and the mechanism of the prime example for a chemical oscillator, the Belousov-Zhabotinski reaction, was deciphered in the early 1970s. Secondly, biological theories of self-organisation were rooted in the intellectual environment of artificial intelligence and cybernetics. Turing wrote his The chemical basis of morphogenesis (1952) after working on the construction of one of the first electronic computers. Likewise, Gierer and Meinhardt&apos;s theory of local activation and lateral inhibition (1972) was influenced by ideas from cybernetics. The Gierer-Meinhardt theory provided an explanation for the first time of both spontaneous formation of spatial order and of self-regulation that proved to be extremely successful in elucidating a wide range of patterning processes. With the advent of developmental genetics in the 1980s, detailed molecular and functional data became available for complex developmental processes, allowing a new generation of data-driven theoretical approaches. Three examples of such approaches will be discussed. The successes and limitations of mathematical pattern formation theory throughout its history suggest a picture of the organism, which has structural similarity to views of the organic world held by the philosopher Immanuel Kant at the end of the eighteenth century.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(84);">hide</a></div></div></div><div id="publication85" class="publication01"><div 
  class="publication01Title">The impact of temperature on balancing immune responsiveness and growth in Arabidopsis.</div><div 
  class="publication01Info">Alcázar R, Parker JE, Trends In Plant Science 16, 666-675 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(85);">abstract <img id="abstractI85" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21963982&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP85" 
  class="publication01Abstract" style="display:none;">Plants have evolved polymorphic immune receptors to recognize pathogens causing disease. However, triggering of resistance needs to be tuned to the local environment to maintain a balance between defense and growth. We consider here the impact of temperature as a key environmental factor influencing immune pathway activation in Arabidopsis. Genetic compensatory and molecular buffering mechanisms affecting the diversification, functionality and subcellular dynamics of immune receptors, reveal multiple points at which temperature intersects with host resistance signaling systems, including a role of at least one receptor in sensing temperature change. Analysis of temperature-dependent autoimmunity caused by allelic mismatches in hybrids of evolutionary diverged Arabidopsis accessions is illuminating processes by which plants maintain &apos;poise&apos; between immune responsiveness and fitness in natural populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(85);">hide</a></div></div></div><div id="publication86" class="publication01"><div 
  class="publication01Title">Adaptive walks and extreme value theory.</div><div 
  class="publication01Info">Neidhart J, Krug J, Physical Review Letters 107, 178102 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(86);">abstract <img id="abstractI86" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22107587&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP86" 
  class="publication01Abstract" style="display:none;">We study biological evolution in a high-dimensional genotype space in the regime of rare mutations and strong selection. The population performs an uphill walk which terminates at local fitness maxima. Assigning fitness randomly to genotypes, we show that the mean walk length is logarithmic in the number of initially available beneficial mutations, with a prefactor determined by the tail of the fitness distribution. This result is derived analytically in a simplified setting where the mutational neighborhood is fixed during the adaptive process, and confirmed by numerical simulations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(86);">hide</a></div></div></div><div id="publication87" class="publication01"><div 
  class="publication01Title">The causes of epistasis (Review)</div><div 
  class="publication01Info">de Visser JAGM, Cooper TF, Elena SF, Proc. R. Soc. B vol. 278 3617-3624 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(87);">abstract <img id="abstractI87" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://rspb.royalsocietypublishing.org/content/278/1725/3617.full.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP87" 
  class="publication01Abstract" style="display:none;">Since Bateson's discovery that genes can suppress the phenotypic effects of other genes, gene interactions-called epistasis-have been the topic of a vast research effort. Systems and developmental biologists study epistasis to understand the genotype-phenotype map, whereas evolutionary biologists recognize the fundamental importance of epistasis for evolution. Depending on its form, epistasis may lead to divergence and speciation, provide evolutionary benefits to sex and affect the robustness and evolvability of organisms. That epistasis can itself be shaped by evolution has only recently been realized. Here, we review the empirical pattern of epistasis, and some of the factors that may affect the form and extent of epistasis. Based on their divergent consequences, we distinguish between interactions with or without mean effect, and those affecting the magnitude of fitness effects or their sign. Empirical work has begun to quantify epistasis in multiple dimensions in the context of metabolic and fitness landscape models. We discuss possible proximate causes (such as protein function and metabolic networks) and ultimate factors (including mutation, recombination, and the importance of natural selection and genetic drift). We conclude that, in general, pleiotropy is an important prerequisite for epistasis, and that epistasis may evolve as an adaptive or intrinsic consequence of changes in genetic robustness and evolvability.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(87);">hide</a></div></div></div><div id="publication88" class="publication01"><div 
  class="publication01Title">Formation of regulatory modules by local sequence duplication.</div><div 
  class="publication01Info">Nourmohammad A, Lässig M, PLoS Computational Biology 7, e1002167 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(88);">abstract <img id="abstractI88" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21998564&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP88" 
  class="publication01Abstract" style="display:none;">Turnover of regulatory sequence and function is an important part of molecular evolution. But what are the modes of sequence evolution leading to rapid formation and loss of regulatory sites? Here we show that a large fraction of neighboring transcription factor binding sites in the fly genome have formed from a common sequence origin by local duplications. This mode of evolution is found to produce regulatory information: duplications can seed new sites in the neighborhood of existing sites. Duplicate seeds evolve subsequently by point mutations, often towards binding a different factor than their ancestral neighbor sites. These results are based on a statistical analysis of 346 cis-regulatory modules in the Drosophila melanogaster genome, and a comparison set of intergenic regulatory sequence in Saccharomyces cerevisiae. In fly regulatory modules, pairs of binding sites show significantly enhanced sequence similarity up to distances of about 50 bp. We analyze these data in terms of an evolutionary model with two distinct modes of site formation: (i) evolution from independent sequence origin and (ii) divergent evolution following duplication of a common ancestor sequence. Our results suggest that pervasive formation of binding sites by local sequence duplications distinguishes the complex regulatory architecture of higher eukaryotes from the simpler architecture of unicellular organisms.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(88);">hide</a></div></div></div><div id="publication89" class="publication01"><div 
  class="publication01Title">Immunity-related GTPase M (IRGM) proteins influence the localization of guanylate-binding protein 2 (GBP2) by modulating macroautophagy.</div><div 
  class="publication01Info">Traver MK, Henry SC, Cantillana V, Oliver T, Hunn JP, Howard JC, Beer S, Pfeffer K, Coers J, Taylor GA, The Journal of Biological Chemistry 286, 30471-30480 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(89);">abstract <img id="abstractI89" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21757726&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP89" 
  class="publication01Abstract" style="display:none;">The immunity-related GTPases (IRGs) are a family of proteins induced by interferon-γ that play a crucial role in innate resistance to intracellular pathogens. The M subfamily of IRG proteins (IRGM) plays a profound role in this context, in part because of the ability of its members to regulate the localization and expression of other IRG proteins. We present here evidence that IRGM proteins affect the localization of the guanylate-binding proteins (GBPs), a second family of interferon-induced GTP-binding proteins that also function in innate immunity. Absence of Irgm1 or Irgm3 led to accumulation of Gbp2 in intracellular compartments that were positive for both the macroautophagy (hereafter referred to as autophagy) marker LC3 and the autophagic adapter molecule p62/Sqstm1. Gbp2 was similarly relocalized in cells in which autophagy was impaired because of the absence of Atg5. Both in Atg5- and IRGM-deficient cells, the IRG protein Irga6 relocalized to the same compartments as Gbp2, raising the possibility of a common regulatory mechanism. However, other data indicated that Irga6, but not Gbp2, was ubiquitinated in IRGM-deficient cells. Similarly, coimmunoprecipitation studies indicated that although Irgm3 did interact directly with Irgb6, it did not interact with Gbp2. Collectively, these data suggest that IRGM proteins indirectly modulate the localization of GBPs through a distinct mechanism from that through which they regulate IRG protein localization. Further, these results suggest that a core function of IRGM proteins is to regulate autophagic flux, which influences the localization of GBPs and possibly other factors that instruct cell-autonomous immune resistance.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(89);">hide</a></div></div></div><div id="publication90" class="publication01"><div 
  class="publication01Title">DOG1 expression is predicted by the seed-maturation environment and contributes to geographical variation in germination in <em>Arabidopsis thaliana</em></div><div 
  class="publication01Info">Chiang GCK, Bartsch M, Barua D, Nakayabashi K, Debieu M, Kronholm I, Koornneef M, Soppe WJJ, Donohue K, de Meaux J, Molecular Ecology, 20: 3336-3349 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(90);">abstract <img id="abstractI90" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05181.x/pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP90" 
  class="publication01Abstract" style="display:none;">Seasonal germination timing of Arabidopsis thaliana strongly influences overall life history expression and is the target of intense natural selection. This seasonal germination timing depends strongly on the interaction between genetics and seasonal environments both before and after seed dispersal. DELAY OF GERMINATION 1 (DOG1) is the first gene that has been identified to be associated with natural variation in primary dormancy in A. thaliana. Here, we report interaccession variation in DOG1 expression and document that DOG1 expression is associated with seed-maturation temperature effects on germination; DOG1 expression increased when seeds were matured at low temperature, and this increased expression was associated with increased dormancy of those seeds. Variation in DOG1 expression suggests a geographical structure such that southern accessions, which are more dormant, tend to initiate DOG1 expression earlier during seed maturation and achieved higher expression levels at the end of silique development than did northern accessions. Although elimination of the synthesis of phytohormone abscisic acid (ABA) results in the elimination of maternal temperature effects on dormancy, DOG1 expression predicted dormancy better than expression of genes involved in ABA metabolism.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(90);">hide</a></div></div></div><div id="publication91" class="publication01"><div 
  class="publication01Title">Evolutionary accessibility of mutational pathways.</div><div 
  class="publication01Info">Franke J, Klözer A, de Visser JAGM, Krug J, PLoS Computational Biology 7, e1002134 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(91);">abstract <img id="abstractI91" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21876664&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP91" 
  class="publication01Abstract" style="display:none;">Functional effects of different mutations are known to combine to the total effect in highly nontrivial ways. For the trait under evolutionary selection (&apos;fitness&apos;), measured values over all possible combinations of a set of mutations yield a fitness landscape that determines which mutational states can be reached from a given initial genotype. Understanding the accessibility properties of fitness landscapes is conceptually important in answering questions about the predictability and repeatability of evolutionary adaptation. Here we theoretically investigate accessibility of the globally optimal state on a wide variety of model landscapes, including landscapes with tunable ruggedness as well as neutral &apos;holey&apos; landscapes. We define a mutational pathway to be accessible if it contains the minimal number of mutations required to reach the target genotype, and if fitness increases in each mutational step. Under this definition accessibility is high, in the sense that at least one accessible pathway exists with a substantial probability that approaches unity as the dimensionality of the fitness landscape (set by the number of mutational loci) becomes large. At the same time the number of alternative accessible pathways grows without bounds. We test the model predictions against an empirical 8-locus fitness landscape obtained for the filamentous fungus Aspergillus niger. By analyzing subgraphs of the full landscape containing different subsets of mutations, we are able to probe the mutational distance scale in the empirical data. The predicted effect of high accessibility is supported by the empirical data and is very robust, which we argue reflects the generic topology of sequence spaces. Together with the restrictive assumptions that lie in our definition of accessibility, this implies that the globally optimal configuration should be accessible to genome wide evolution, but the repeatability of evolutionary trajectories is limited owing to the presence of a large number of alternative mutational pathways.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(91);">hide</a></div></div></div><div id="publication92" class="publication01"><div 
  class="publication01Title">Genetic and evolutionary perspectives on the interplay between plant immunity and development.</div><div 
  class="publication01Info">Alcázar R, Reymond M, Schmitz G, de Meaux J, Current Opinion In Plant Biology 14, 378-384 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(92);">abstract <img id="abstractI92" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21561797&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP92" 
  class="publication01Abstract" style="display:none;">There is now ample evidence that plant development, responses to abiotic environments, and immune responses are tightly intertwined in their physiology. Thus optimization of the immune system during evolution will occur in coordination with that of plant development. Two alternative and possibly complementary forces are at play: genetic constraints due to the pleiotropic action of players in both systems, and coevolution, if developmental changes modulate the cost-benefit balance of immunity. A current challenge is to elucidate the ecological forces driving evolution of quantitative variation for defense at molecular level. The analysis of natural co-variation for developmental and immunity traits in Arabidopsis thaliana promises to bring important insights.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(92);">hide</a></div></div></div><div id="publication93" class="publication01"><div 
  class="publication01Title">The IRG protein-based resistance mechanism in mice and its relation to virulence in Toxoplasma gondii.</div><div 
  class="publication01Info">Howard JC, Hunn JP, Steinfeldt T, Current Opinion in Microbiology 14, 414-421 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(93);">abstract <img id="abstractI93" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21783405&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP93" 
  class="publication01Abstract" style="display:none;">IRG proteins (immunity-related GTPases) provide an early defense mechanism in mice against the protozoal pathogen, Toxoplasma gondii. This is a particularly suitable time to provide a brief review of this host-pathogen interaction because the nature of the IRG resistance system, and to some extent its mode of action, have become known in the past few years. Likewise, forward genetic screens have recently drawn attention to a number of loci contributing to the differential virulence of T. gondii strains in mice. It is now clear that at least some important virulence mechanisms exert their action against components of the IRG resistance system. Thus these two mechanisms form the two poles of a dynamic host-pathogen virulence-resistance relationship with interesting and accessible properties.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(93);">hide</a></div></div></div><div id="publication94" class="publication01"><div 
  class="publication01Title">Nonlinear fitness landscape of a molecular pathway.</div><div 
  class="publication01Info">Perfeito L, Ghozzi S, Berg J, Schnetz K, Lässig M, PLoS Genetics 7(7): e1002160 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(94);">abstract <img id="abstractI94" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://link.springer.com/article/10.1007%2Fs00442-012-2479-5">paper [pdf]</a> &nbsp; </div><div id="abstractP94" 
  class="publication01Abstract" style="display:none;">Genes are regulated because their expression involves a fitness cost to the organism. The production of proteins by transcription and translation is a well-known cost factor, but the enzymatic activity of the proteins produced can also reduce fitness, depending on the internal state and the environment of the cell. Here, we map the fitness costs of a key metabolic network, the lactose utilization pathway in <em>Escherichia coli.</em> We measure the growth of several regulatory lac operon mutants in different environments inducing expression of the lac genes. We find a strikingly nonlinear fitness landscape, which depends on the production rate and on the activity rate of the lac proteins. A simple fitness model of the lac pathway, based on elementary biophysical processes, predicts the growth rate of all observed strains. The nonlinearity of fitness is explained by a feedback loop: production and activity of the lac proteins reduce growth, but growth also affects the density of these molecules. This nonlinearity has important consequences for molecular function and evolution. It generates a cliff in the fitness landscape, beyond which populations cannot maintain growth. In viable populations, there is an expression barrier of the lac genes, which cannot be exceeded in any stationary growth process. Furthermore, the nonlinearity determines how the fitness of operon mutants depends on the inducer environment. We argue that fitness nonlinearities, expression barriers, and gene–environment interactions are generic features of fitness landscapes for metabolic pathways, and we discuss their implications for the evolution of regulation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(94);">hide</a></div></div></div><div id="publication95" class="publication01"><div 
  class="publication01Title">An integrated approach to characterize genetic interaction networks in yeast metabolism.</div><div 
  class="publication01Info">Szappanos B, Kovacs K, Szamecz B, Honti F, Costanzo M, Baryshnikova A, Gelius-Dietrich G, Lercher MJ, Jelasity M, Myers CL, Andrews BJ et al., Nature Genetics 43, 656-662 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(95);">abstract <img id="abstractI95" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 21623372&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP95" 
  class="publication01Abstract" style="display:none;">Although experimental and theoretical efforts have been applied to globally map genetic interactions, we still do not understand how gene-gene interactions arise from the operation of biomolecular networks. To bridge the gap between empirical and computational studies, we i, quantitatively measured genetic interactions between approximately 185,000 metabolic gene pairs in Saccharomyces cerevisiae, ii, superposed the data on a detailed systems biology model of metabolism and iii, introduced a machine-learning method to reconcile empirical interaction data with model predictions. We systematically investigated the relative impacts of functional modularity and metabolic flux coupling on the distribution of negative and positive genetic interactions. We also provide a mechanistic explanation for the link between the degree of genetic interaction, pleiotropy and gene dispensability. Last, we show the feasibility of automated metabolic model refinement by correcting misannotations in NAD biosynthesis and confirming them by in vivo experiments.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(95);">hide</a></div></div></div><div id="publication96" class="publication01"><div 
  class="publication01Title">Evolutionary advantage of small populations on complex fitness landscapes.</div><div 
  class="publication01Info">Jain K, Krug J, Park S, Evolution 65, 1945-1955 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(96);">abstract <img id="abstractI96" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21729050&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP96" 
  class="publication01Abstract" style="display:none;">Recent experimental and theoretical studies have shown that small asexual populations evolving on complex fitness landscapes may achieve a higher fitness than large ones due to the increased heterogeneity of adaptive trajectories. Here, we introduce a class of haploid three-locus fitness landscapes that allow the investigation of this scenario in a precise and quantitative way. Our main result derived analytically shows how the probability of choosing the path of the largest initial fitness increase grows with the population size. This makes large populations more likely to get trapped at local fitness peaks and implies an advantage of small populations at intermediate time scales. The range of population sizes where this effect is operative coincides with the onset of clonal interference. Additional studies using ensembles of random fitness landscapes show that the results achieved for a particular choice of three-locus landscape parameters are robust and also persist as the number of loci increases. Our study indicates that an advantage for small populations is likely whenever the fitness landscape contains local maxima. The advantage appears at intermediate time scales, which are long enough for trapping at local fitness maxima to have occurred but too short for peak escape by the creation of multiple mutants.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(96);">hide</a></div></div></div><div id="publication97" class="publication01"><div 
  class="publication01Title">Evolutionary genetics: evolution with foresight.</div><div 
  class="publication01Info">Salverda MLM, de Visser JAGM, Current Biology 21, R398-400 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(97);">abstract <img id="abstractI97" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21601801&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP97" 
  class="publication01Abstract" style="display:none;">Evolution has no foresight, but produces ad hoc solutions by tinkering with available variation. A new study demonstrates how evolution nevertheless prepares organisms for the future by increasing their evolvability.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(97);">hide</a></div></div></div><div id="publication98" class="publication01"><div 
  class="publication01Title">The IFN-γ-inducible GTPase, Irga6, protects mice against Toxoplasma gondii but not against Plasmodium berghei and some other intracellular pathogens</div><div 
  class="publication01Info">Liesenfeld O, Parvanova I, Zerrahn J, Han SJ, Heinrich F, Muñoz M, Kaiser F, Aebischer T, Buch T, Waisman A, Reichmann G, Utermöhlen O, von Stebut E, von Loewenich FD, Bogdan C, Specht S, Saeftel M, Hoerauf A, Mota MM, Könen-Waisman S, Kaufmann SH, Howard JC, PLoS One, 6(6):e20568 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(98);">abstract <img id="abstractI98" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3117789/pdf/pone.0020568.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP98" 
  class="publication01Abstract" style="display:none;">Clearance of infection with intracellular pathogens in mice involves interferon-regulated GTPases of the IRG protein family. Experiments with mice genetically deficient in members of this family such as Irgm1(LRG-47), Irgm3(IGTP), and Irgd(IRG-47) has revealed a critical role in microbial clearance, especially for <em>Toxoplasma gondii</em>. The in vivo role of another member of this family, Irga6 (IIGP, IIGP1) has been studied in less detail. We investigated the susceptibility of two independently generated mouse strains deficient in Irga6 to in vivo infection with T. gondii, Mycobacterium tuberculosis, Leishmania mexicana, L. major, Listeria monocytogenes, Anaplasma phagocytophilum and Plasmodium berghei. Compared with wild-type mice, mice deficient in Irga6 showed increased susceptibility to oral and intraperitoneal infection with T. gondii but not to infection with the other organisms. Surprisingly, infection of Irga6-deficient mice with the related apicomplexan parasite, P. berghei, did not result in increased replication in the liver stage and no Irga6 (or any other IRG protein) was detected at the parasitophorous vacuole membrane in IFN-γ-induced wild-type cells infected with P. berghei in vitro. Susceptibility to infection with T. gondii was associated with increased mortality and reduced time to death, increased numbers of inflammatory foci in the brains and elevated parasite loads in brains of infected Irga6-deficient mice. In vitro, Irga6-deficient macrophages and fibroblasts stimulated with IFN-γ were defective in controlling parasite replication. Taken together, our results implicate Irga6 in the control of infection with T. gondii and further highlight the importance of the IRG system for resistance to this pathogen.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(98);">hide</a></div></div></div><div id="publication99" class="publication01"><div 
  class="publication01Title">Bistability in two-locus models with selection, mutation, and recombination.</div><div 
  class="publication01Info">Park S, Krug J, Journal Of Mathematical Biology 62, 763-788 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(99);">abstract <img id="abstractI99" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20617437&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP99" 
  class="publication01Abstract" style="display:none;">The evolutionary effect of recombination depends crucially on the epistatic interactions between linked loci. A paradigmatic case where recombination is known to be strongly disadvantageous is a two-locus fitness landscape displaying reciprocal sign epistasis with two fitness peaks of unequal height. Although this type of model has been studied since the 1960s, a full analytic understanding of the stationary states of mutation-selection balance was not achieved so far. Focusing on the bistability arising due to the recombination, we consider here the deterministic, haploid two-locus model with reversible mutations, selection and recombination. We find analytic formulae for the critical recombination probability r ( c ) above which two stable stationary solutions appear which are localized on each of the two fitness peaks. We also derive the stationary genotype frequencies in various parameter regimes. In particular, when the recombination rate is close to r ( c ) and the fitness difference between the two peaks is small, we obtain a compact description in terms of a cubic polynomial which is analogous to the Landau theory of physical phase transitions.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(99);">hide</a></div></div></div><div id="publication100" class="publication01"><div 
  class="publication01Title">Aa TFL1 confers an age-dependent response to vernalization in perennial Arabis alpina.</div><div 
  class="publication01Info">Wang R, Albani MC, Vincent C, Bergonzi S, Luan M, Bai Y, Kiefer C, Castillo R, Coupland G, The Plant Cell 23, 1307-1321 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(100);">abstract <img id="abstractI100" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21498681&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP100" 
  class="publication01Abstract" style="display:none;">Flowering of many plants is induced by environmental signals, but these responses can depend on the age of the plant. Exposure of Arabidopsis thaliana to vernalization (winter temperatures) at germination induces flowering, whereas a close perennial relative Arabis alpina only responds if exposed when at least 5 weeks old. We show that vernalization of these older A. alpina plants reduces expression of the floral repressor PEP1 and activates the orthologs of the Arabidopsis flowering genes SOC1 (Aa SOC1) and LFY (Aa LFY). By contrast, when younger plants are vernalized, PEP1 and Aa SOC1 mRNA levels change as in older plants, but Aa LFY is not expressed. We demonstrate that A. alpina TFL1 (Aa TFL1) blocks flowering and prevents Aa LFY expression when young plants are exposed to vernalization. In addition, in older plants, Aa TFL1 increases the duration of vernalization required for Aa LFY expression and flowering. Aa TFL1 has similar functions in axillary shoots, thus ensuring that following a flowering episode vegetative branches are maintained to continue the perennial life cycle. We propose that Aa TFL1 blocks flowering of young plants exposed to vernalization by setting a threshold for a flowering pathway that is increased in activity as the shoot ages, thus contributing to several perennial traits.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(100);">hide</a></div></div></div><div id="publication101" class="publication01"><div 
  class="publication01Title">The phylogenetic origin of oskar coincided with the origin of maternally provisioned germ plasm and pole cells at the base of the Holometabola.</div><div 
  class="publication01Info">Lynch JA, Ozüak O, Khila A, Abouheif E, Desplan C, Roth S, PLoS Genetics 7, e1002029 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(101);">abstract <img id="abstractI101" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21552321&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP101" 
  class="publication01Abstract" style="display:none;">The establishment of the germline is a critical, yet surprisingly evolutionarily labile, event in the development of sexually reproducing animals. In the fly Drosophila, germ cells acquire their fate early during development through the inheritance of the germ plasm, a specialized maternal cytoplasm localized at the posterior pole of the oocyte. The gene oskar (osk) is both necessary and sufficient for assembling this substance. Both maternal germ plasm and oskar are evolutionary novelties within the insects, as the germline is specified by zygotic induction in basally branching insects, and osk has until now only been detected in dipterans. In order to understand the origin of these evolutionary novelties, we used comparative genomics, parental RNAi, and gene expression analyses in multiple insect species. We have found that the origin of osk and its role in specifying the germline coincided with the innovation of maternal germ plasm and pole cells at the base of the holometabolous insects and that losses of osk are correlated with changes in germline determination strategies within the Holometabola. Our results indicate that the invention of the novel gene osk was a key innovation that allowed the transition from the ancestral late zygotic mode of germline induction to a maternally controlled establishment of the germline found in many holometabolous insect species. We propose that the ancestral role of osk was to connect an upstream network ancestrally involved in mRNA localization and translational control to a downstream regulatory network ancestrally involved in executing the germ cell program.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(101);">hide</a></div></div></div><div id="publication102" class="publication01"><div 
  class="publication01Title">Initial mutations direct alternative pathways of protein evolution.</div><div 
  class="publication01Info">Salverda MLM, Dellus E, Gorter FA, Debets AJM, van der Oost J, Hoekstra RF, Tawfik DS, de Visser JAGM, PLoS Genetics 7, e1001321 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(102);">abstract <img id="abstractI102" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21408208&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP102" 
  class="publication01Abstract" style="display:none;">Whether evolution is erratic due to random historical details, or is repeatedly directed along similar paths by certain constraints, remains unclear. Epistasis (i.e. non-additive interaction between mutations that affect fitness) is a mechanism that can contribute to both scenarios. Epistasis can constrain the type and order of selected mutations, but it can also make adaptive trajectories contingent upon the first random substitution. This effect is particularly strong under sign epistasis, when the sign of the fitness effects of a mutation depends on its genetic background. In the current study, we examine how epistatic interactions between mutations determine alternative evolutionary pathways, using in vitro evolution of the antibiotic resistance enzyme TEM-1 β-lactamase. First, we describe the diversity of adaptive pathways among replicate lines during evolution for resistance to a novel antibiotic (cefotaxime). Consistent with the prediction of epistatic constraints, most lines increased resistance by acquiring three mutations in a fixed order. However, a few lines deviated from this pattern. Next, to test whether negative interactions between alternative initial substitutions drive this divergence, alleles containing initial substitutions from the deviating lines were evolved under identical conditions. Indeed, these alternative initial substitutions consistently led to lower adaptive peaks, involving more and other substitutions than those observed in the common pathway. We found that a combination of decreased enzymatic activity and lower folding cooperativity underlies negative sign epistasis in the clash between key mutations in the common and deviating lines (Gly238Ser and Arg164Ser, respectively). Our results demonstrate that epistasis contributes to contingency in protein evolution by amplifying the selective consequences of random mutations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(102);">hide</a></div></div></div><div id="publication103" class="publication01"><div 
  class="publication01Title">Rare events in population genetics: stochastic tunneling in a two-locus model with recombination.</div><div 
  class="publication01Info">Altland A, Fischer A, Krug J, Szendro IG, Physical Review Letters 106, 088101 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(103);">abstract <img id="abstractI103" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21405603&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP103" 
  class="publication01Abstract" style="display:none;">We study the evolution of a population in a two-locus genotype space, in which the negative effects of two single mutations are overcompensated in a high-fitness double mutant. We discuss how the interplay of finite population size N and sexual recombination at rate r affects the escape times t(esc) to the double mutant. For small populations demographic noise generates massive fluctuations in t(esc). The mean escape time varies nonmonotonically with r, and grows exponentially as lnt(esc)∼N(r-r(*))(3/2) beyond a critical value r(*).<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(103);">hide</a></div></div></div><div id="publication104" class="publication01"><div 
  class="publication01Title">The immunity-related GTPases in mammals: a fast-evolving cell-autonomous resistance system against intracellular pathogens.</div><div 
  class="publication01Info">Hunn JP, Feng CG, Sher A, Howard JC, Mammalian Genome : Official Journal Of The International Mammalian Genome Society 22, 43-54 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(104);">abstract <img id="abstractI104" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21052678&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP104" 
  class="publication01Abstract" style="display:none;">The immunity-related GTPases (IRGs) belong to the family of large, interferon-inducible GTPases and constitute a cell-autonomous resistance system essential for the control of vacuolar pathogens like Toxoplasma gondii in mice. Recent results demonstrated that numerous IRG members accumulate collaboratively at the parasitophorous vacuole of invading T. gondii leading to the destruction of the vacuole and the parasite and subsequent necrotic host cell death. Complex regulatory interactions between different IRG proteins are necessary for these processes. Disturbance of this finely balanced system, e.g., by single genetic deficiency for the important negative regulator Irgm1 or the autophagic regulator Atg5, leads to spontaneous activation of the effector IRG proteins when induced by IFNγ. This activation has cytotoxic consequences resulting in a severe lymphopenia, macrophage defects, and failure of the adaptive immune system in Irgm1-deficient mice. However, alternative functions in phagosome maturation and induction of autophagy have been proposed for Irgm1. The IRG system has been studied primarily in mice, but IRG genes are present throughout the mammalian lineage. Interestingly, the number, type, and diversity of genes present differ greatly even between closely related species, probably reflecting intimate host-pathogen coevolution driven by an armed race between the IRG resistance proteins and pathogen virulence factors. IRG proteins are targets for polymorphic T. gondii virulence factors, and genetic variation in the IRG system between different mouse strains correlates with resistance and susceptibility to virulent T. gondii strains.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(104);">hide</a></div></div></div><div id="publication105" class="publication01"><div 
  class="publication01Title">The evolution of dorsal-ventral patterning mechanisms in insects.</div><div 
  class="publication01Info">Lynch JA, Roth S, Genes &amp; Development 25, 107-118 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(105);">abstract <img id="abstractI105" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21245164&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP105" 
  class="publication01Abstract" style="display:none;">The gene regulatory network (GRN) underpinning dorsal-ventral (DV) patterning of the Drosophila embryo is among the most thoroughly understood GRNs, making it an ideal system for comparative studies seeking to understand the evolution of development. With the emergence of widely applicable techniques for testing gene function, species with sequenced genomes, and multiple tractable species with diverse developmental modes, a phylogenetically broad and molecularly deep understanding of the evolution of DV axis formation in insects is feasible. Here, we review recent progress made in this field, compare our emerging molecular understanding to classical embryological experiments, and suggest future directions of inquiry.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(105);">hide</a></div></div></div><div id="publication106" class="publication01"><div 
  class="publication01Title">A gene&apos;s ability to buffer variation is predicted by its fitness contribution and genetic interactions.</div><div 
  class="publication01Info">Wang G, Liu J, Wang W, Zhang H, Lercher MJ, PloS One 6, e17650 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(106);">abstract <img id="abstractI106" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 21407817&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP106" 
  class="publication01Abstract" style="display:none;">BACKGROUND: Many single-gene knockouts result in increased phenotypic (e.g., morphological) variability among the mutant&apos;s offspring. This has been interpreted as an intrinsic ability of genes to buffer genetic and environmental variation. A phenotypic capacitor is a gene that appears to mask phenotypic variation: when knocked out, the offspring shows more variability than the wild type. Theory predicts that this phenotypic potential should be correlated with a gene&apos;s knockout fitness and its number of negative genetic interactions. Based on experimentally measured phenotypic capacity, it was suggested that knockout fitness was unimportant, but that phenotypic capacitors tend to be hubs in genetic and physical interaction networks. METHODOLOGY/PRINCIPAL FINDINGS: We re-analyse the available experimental data in a combined model, which includes knockout fitness and network parameters as well as expression level and protein length as predictors of phenotypic potential. Contrary to previous conclusions, we find that the strongest predictor is in fact haploid knockout fitness (responsible for 9% of the variation in phenotypic potential), with an additional contribution from the genetic interaction network (5%); once these two factors are taken into account, protein-protein interactions do not make any additional contribution to the variation in phenotypic potential. CONCLUSIONS/SIGNIFICANCE: We conclude that phenotypic potential is not a mysterious &quot;emergent&quot; property of cellular networks. Instead, it is very simply determined by the overall fitness reduction of the organism (which in its compromised state can no longer compensate for multiple factors that contribute to phenotypic variation), and by the number (and presumably nature) of genetic interactions of the knocked-out gene. In this light, Hsp90, the prototypical phenotypic capacitor, may not be representative: typical phenotypic capacitors are not direct &quot;buffers&quot; of variation, but are simply genes encoding central cellular functions.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(106);">hide</a></div></div></div><div id="publication107" class="publication01"><div 
  class="publication01Title">The activation mechanism of Irga6, an interferon-inducible GTPase contributing to mouse resistance against Toxoplasma gondii.</div><div 
  class="publication01Info">Pawlowski N, Khaminets A, Hunn JP, Papic N, Schmidt A, Uthaiah RC, Lange R, Vopper G, Martens S, Wolf E, Howard JC et al., BMC Biology 9, 7 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(107);">abstract <img id="abstractI107" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21276251&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP107" 
  class="publication01Abstract" style="display:none;">BACKGROUND:The interferon-inducible immunity-related GTPases (IRG proteins/p47 GTPases) are a distinctive family of GTPases that function as powerful cell-autonomous resistance factors. The IRG protein, Irga6 (IIGP1), participates in the disruption of the vacuolar membrane surrounding the intracellular parasite, Toxoplasma gondii, through which it communicates with its cellular hosts. Some aspects of the protein&apos;s behaviour have suggested a dynamin-like molecular mode of action, in that the energy released by GTP hydrolysis is transduced into mechanical work that results in deformation and ultimately rupture of the vacuolar membrane.RESULTS:Irga6 forms GTP-dependent oligomers in vitro and thereby activates hydrolysis of the GTP substrate. In this study we define the catalytic G-domain interface by mutagenesis and present a structural model, of how GTP hydrolysis is activated in Irga6 complexes, based on the substrate-twinning reaction mechanism of the signal recognition particle (SRP) and its receptor (SRα). In conformity with this model, we show that the bound nucleotide is part of the catalytic interface and that the 3&apos;hydroxyl of the GTP ribose bound to each subunit is essential for trans-activation of hydrolysis of the GTP bound to the other subunit. We show that both positive and negative regulatory interactions between IRG proteins occur via the catalytic interface. Furthermore, mutations that disrupt the catalytic interface also prevent Irga6 from accumulating on the parasitophorous vacuole membrane of T. gondii, showing that GTP-dependent Irga6 activation is an essential component of the resistance mechanism.CONCLUSIONS:The catalytic interface of Irga6 defined in the present experiments can probably be used as a paradigm for the nucleotide-dependent interactions of all members of the large family of IRG GTPases, both activating and regulatory. Understanding the activation mechanism of Irga6 will help to explain the mechanism by which IRG proteins exercise their resistance function. We find no support from sequence or G-domain structure for the idea that IRG proteins and the SRP GTPases have a common phylogenetic origin. It therefore seems probable, if surprising, that the substrate-assisted catalytic mechanism has been independently evolved in the two protein families.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(107);">hide</a></div></div></div><div id="publication108" class="publication01"><div 
  class="publication01Title">The effects of network neighbours on protein evolution.</div><div 
  class="publication01Info">Wang G, Lercher MJ, PloS One 6, e18288 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(108);">abstract <img id="abstractI108" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21532755&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP108" 
  class="publication01Abstract" style="display:none;">Interacting proteins may often experience similar selection pressures. Thus, we may expect that neighbouring proteins in biological interaction networks evolve at similar rates. This has been previously shown for protein-protein interaction networks. Similarly, we find correlated rates of evolution of neighbours in networks based on co-expression, metabolism, and synthetic lethal genetic interactions. While the correlations are statistically significant, their magnitude is small, with network effects explaining only between 2% and 7% of the variation. The strongest known predictor of the rate of protein evolution remains expression level. We confirmed the previous observation that similar expression levels of neighbours indeed explain their similar evolution rates in protein-protein networks, and showed that the same is true for metabolic networks. In co-expression and synthetic lethal genetic interaction networks, however, neighbouring genes still show somewhat similar evolutionary rates even after simultaneously controlling for expression level, gene essentiality and gene length. Thus, similar expression levels and related functions (as inferred from co-expression and synthetic lethal interactions) seem to explain correlated evolutionary rates of network neighbours across all currently available types of biological networks.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(108);">hide</a></div></div></div><div id="publication109" class="publication01"><div 
  class="publication01Title">The maternal and early embryonic transcriptome of the milkweed bug Oncopeltus fasciatus.</div><div 
  class="publication01Info">Ewen-Campen B, Shaner N, Panfilio KA, Suzuki Y, Roth S, Extavour CG, BMC Genomics 12, 61 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(109);">abstract <img id="abstractI109" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21266083&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP109" 
  class="publication01Abstract" style="display:none;">BACKGROUND:Most evolutionary developmental biology (&quot;evo-devo&quot;) studies of emerging model organisms focus on small numbers of candidate genes cloned individually using degenerate PCR. However, newly available sequencing technologies such as 454 pyrosequencing have recently begun to allow for massive gene discovery in animals without sequenced genomes. Within insects, although large volumes of sequence data are available for holometabolous insects, developmental studies of basally branching hemimetabolous insects typically suffer from low rates of gene discovery.RESULTS:We used 454 pyrosequencing to sequence over 500 million bases of cDNA from the ovaries and embryos of the milkweed bug Oncopeltus fasciatus, which lacks a sequenced genome. This indirectly developing insect occupies an important phylogenetic position, branching basal to Diptera (including fruit flies) and Hymenoptera (including honeybees), and is an experimentally tractable model for short-germ development. 2,087,410 reads from both normalized and non-normalized cDNA assembled into 21,097 sequences (isotigs) and 112,531 singletons. The assembled sequences fell into 16,617 unique gene models, and included predictions of splicing isoforms, which we examined experimentally. Discovery of new genes plateaued after assembly of ~1.5 million reads, suggesting that we have sequenced nearly all transcripts present in the cDNA sampled. Many transcripts have been assembled at close to full length, and there is a net gain of sequence data for over half of the pre-existing O. fasciatus accessions for developmental genes in GenBank. We identified 10,775 unique genes, including members of all major conserved metazoan signaling pathways and genes involved in several major categories of early developmental processes. We also specifically address the effects of cDNA normalization on gene discovery in de novo transcriptome analyses.CONCLUSIONS:Our sequencing, assembly and annotation framework provide a simple and effective way to achieve high-throughput gene discovery for organisms lacking a sequenced genome. These data will have applications to the study of the evolution of arthropod genes and genetic pathways, and to the wider evolution, development and genomics communities working with emerging model organisms.[The sequence data from this study have been submitted to GenBank under study accession number SRP002610 (http://www.ncbi.nlm.nih.gov/sra?term=SRP002610). Custom scripts generated are available at http://www.extavourlab.com/protocols/index.html. Seven Additional files are available.].<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(109);">hide</a></div></div></div>
          </div>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2010</div>
          <div id="publication110" class="publication01"><div 
  class="publication01Title">Natural variation at Strubbelig Receptor Kinase 3 drives immune-triggered incompatibilities between Arabidopsis thaliana accessions.</div><div 
  class="publication01Info">Alcázar R, García AV, Kronholm I, de Meaux J, Koornneef M, Parker JE, Reymond M, Nature Genetics 42, 1135-1139 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(110);">abstract <img id="abstractI110" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21037570&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP110" 
  class="publication01Abstract" style="display:none;">Accumulation of genetic incompatibilities within species can lead to reproductive isolation and, potentially, speciation. In this study, we show that allelic variation at SRF3 (Strubbelig Receptor Family 3), encoding a receptor-like kinase, conditions the occurrence of incompatibility between Arabidopsis thaliana accessions. The geographical distribution of SRF3 alleles reveals that allelic forms causing epistatic incompatibility with a Landsberg erecta allele at the RPP1 resistance locus are present in A. thaliana accessions in central Asia. Incompatible SRF3 alleles condition for an enhanced early immune response to pathogens as compared to the resistance-dampening effect of compatible SRF3 forms in isogenic backgrounds. Variation in disease susceptibility suggests a basis for the molecular patterns of a recent selective sweep detected at the SRF3 locus in central Asian populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(110);">hide</a></div></div></div><div id="publication111" class="publication01"><div 
  class="publication01Title">Significance analysis and statistical mechanics: an application to clustering.</div><div 
  class="publication01Info">Łuksza M, Lässig M, Berg J, Physical Review Letters 105, 220601 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(111);">abstract <img id="abstractI111" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2010PhRvL.105v0601L&amp;link_type=ABSTRACT">paper [pdf]</a> &nbsp; </div><div id="abstractP111" 
  class="publication01Abstract" style="display:none;">This Letter addresses the statistical significance of structures in random data: given a set of vectors and a measure of mutual similarity, how likely is it that a subset of these vectors forms a cluster with enhanced similarity among its elements? The computation of this cluster p value for randomly distributed vectors is mapped onto a well-defined problem of statistical mechanics. We solve this problem analytically, establishing a connection between the physics of quenched disorder and multiple-testing statistics in clustering and related problems. In an application to gene expression data, we find a remarkable link between the statistical significance of a cluster and the functional relationships between its genes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(111);">hide</a></div></div></div><div id="publication112" class="publication01"><div 
  class="publication01Title">Demographic history of Oceania inferred from genome-wide data.</div><div 
  class="publication01Info">Wollstein A, Lao O, Becker C, Brauer S, Trent RJ, Nürnberg P, Stoneking M, Kayser M, Current Biology 20, 1983-1992 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(112);">abstract <img id="abstractI112" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21074440&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP112" 
  class="publication01Abstract" style="display:none;">BACKGROUND:The human history of Oceania comprises two extremes: the initial colonizations of Near Oceania, one of the oldest out-of-Africa migrations, and of Remote Oceania, the most recent expansion into unoccupied territories. Genetic studies, mostly using uniparentally inherited DNA, have shed some light on human origins in Oceania, particularly indicating that Polynesians are of mixed East Asian and Near Oceanian ancestry. Here, we use ∼1 million single nucleotide polymorphisms (SNPs) to investigate the demographic history of Oceania in a more detailed manner.RESULTS:We developed a new approach to account for SNP ascertainment bias, used approximate Bayesian computation simulations to choose the best-fitting model of population history, and estimated demographic parameters. We find that the ancestors of Near Oceanians diverged from ancestral Eurasians ∼27 thousand years ago (kya), suggesting separate initial occupations of both territories. The genetic admixture in Polynesian history between East Asians (∼87%) and Near Oceanians (∼13%) occurred ∼3 kya, prior to the colonization of Polynesia. Fijians are of Polynesian (∼65%) and additional Near Oceanian (∼35%) ancestry not found in Polynesians, with this admixture occurring considerably after the initial settlement of Remote Oceania. Our data support a greater contribution of East Asian women than men in the admixture history of Remote Oceania and highlight population substructure in Polynesia and New Guinea.CONCLUSIONS:Despite the inherent ascertainment bias, genome-wide SNP data provide new insights into the genetic history of Oceana. Our approach to correct for ascertainment bias and obtain reliable inferences concerning demographic history should prove useful in other such studies.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(112);">hide</a></div></div></div><div id="publication113" class="publication01"><div 
  class="publication01Title">Novel modes of localization and function of nanos in the wasp Nasonia.</div><div 
  class="publication01Info">Lynch JA, Desplan C, Development (Cambridge, England) 137, 3813-3821 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(113);">abstract <img id="abstractI113" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20929949&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP113" 
  class="publication01Abstract" style="display:none;">Abdominal patterning in Drosophila requires the function of nanos (nos) to prevent translation of hunchback (hb) mRNA in the posterior of the embryo. nos function is restricted to the posterior by the translational repression of mRNA that is not incorporated into the posteriorly localized germ plasm during oogenesis. The wasp Nasonia vitripennis (Nv) undergoes a long germ mode of development very similar to Drosophila, although the molecular patterning mechanisms employed in these two organisms have diverged significantly, reflecting the independent evolution of this mode of development. Here, we report that although Nv nanos (Nv-nos) has a conserved function in embryonic patterning through translational repression of hb, the timing and mechanisms of this repression are significantly delayed in the wasp compared with the fly. This delay in Nv-nos function appears to be related to the dynamic behavior of the germ plasm in Nasonia, as well as to the maternal provision of Nv-Hb protein during oogenesis. Unlike in flies, there appears to be two functional populations of Nv-nos mRNA: one that is concentrated in the oosome and is taken up into the pole cells before evidence of Nv-hb repression is observed; another that forms a gradient at the posterior and plays a role in Nv-hb translational repression. Altogether, our results show that, although the embryonic patterning function of nos orthologs is broadly conserved, the mechanisms employed to achieve this function are distinct.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(113);">hide</a></div></div></div><div id="publication114" class="publication01"><div 
  class="publication01Title">Records and sequences of records from random variables with a linear trend</div><div 
  class="publication01Info">Franke J, Wergen G, Krug J, J. Stat. Mech. P10013 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(114);">abstract <img id="abstractI114" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1742-5468/2010/10/P10013">paper [pdf]</a> &nbsp; </div><div id="abstractP114" 
  class="publication01Abstract" style="display:none;">We consider records and sequences of records drawn from discrete time series of the form $$X_{n}=Y_{n}+cn$$, where the $$Y_{n}$$ are independent and identically distributed random variables and $$c$$ is a constant drift. For very small and very large drift velocities, we investigate the asymptotic behavior of the probability $$p_n(c)$$ of a record occurring in the $$n$$th step and the probability $$P_N(c)$$ that all $$N$$ entries are records, i.e. that $$X_1 < X_2 < ... < X_N$$. Our work is motivated by the analysis of temperature time series in climatology, and by the study of mutational pathways in evolutionary biology.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(114);">hide</a></div></div></div><div id="publication115" class="publication01"><div 
  class="publication01Title">Generation of distinct signaling modes via diversification of the Egfr ligand-processing cassette.</div><div 
  class="publication01Info">Rousso T, Lynch J, Yogev S, Roth S, Schejter ED, Shilo B, Development (Cambridge, England) 137, 3427-3437 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(115);">abstract <img id="abstractI115" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20826532&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP115" 
  class="publication01Abstract" style="display:none;">Egfr ligand processing in Drosophila involves trafficking of the ligand precursor by the chaperone Star from the endoplasmic reticulum (ER) to a secretory compartment, where the precursor is cleaved by the intramembrane protease Rhomboid. Some of the Drosophila Rhomboids also reside in the ER, where they attenuate signaling by premature cleavage of Star. The genome of the flour beetle Tribolium castaneum contains a single gene for each of the ligand-processing components, providing an opportunity to assess the regulation and impact of a simplified ligand-processing cassette. We find that the central features of ligand retention, trafficking by the chaperone and cleavage by Rhomboid have been conserved. The single Rhomboid is localized to both ER and secretory compartments. However, we show that Tribolium Star is refractive to Rhomboid cleavage. Consequently, this ligand-processing system effectively mediates long-range Egfr activation in the Tribolium embryonic ventral ectoderm, despite ER localization of Rhomboid. Diversification of the Egfr signaling pathway appears to have coupled gene duplication events with modulation of the biochemical properties and subcellular localization patterns of Rhomboid proteases and their substrates.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(115);">hide</a></div></div></div><div id="publication116" class="publication01"><div 
  class="publication01Title">Evolution of extracellular Dpp modulators in insects: The roles of tolloid and twisted-gastrulation in dorsoventral patterning of the Tribolium embryo.</div><div 
  class="publication01Info">Nunes da Fonseca R, van der Zee M, Roth S, Developmental Biology 345, 80-93 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(116);">abstract <img id="abstractI116" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20510683&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP116" 
  class="publication01Abstract" style="display:none;">The formation of the BMP gradient which patterns the DV axis in flies and vertebrates requires several extracellular modulators like the inhibitory protein Sog/Chordin, the metalloprotease Tolloid (Tld), which cleaves Sog/Chordin, and the CR domain protein Twisted gastrulation (Tsg). While flies and vertebrates have only one sog/chordin gene they possess several paralogues of tld and tsg. A simpler and probably ancestral situation is observed in the short-germ beetle Tribolium castaneum (Tc), which possesses only one tld and one tsg gene. Here we show that in T. castaneum tld is required for early BMP signalling except in the head region and Tc-tld function is, as expected, dependent on Tc-sog. In contrast, Tc-tsg is required for all aspects of early BMP signalling and acts in a Tc-sog-independent manner. For comparison with Drosophila melanogaster we constructed fly embryos lacking all early Tsg activity (tsg;;srw double mutants) and show that they still establish a BMP signalling gradient. Thus, our results suggest that the role of Tsg proteins for BMP gradient formation has changed during insect evolution.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(116);">hide</a></div></div></div><div id="publication117" class="publication01"><div 
  class="publication01Title">EGF signaling and the origin of axial polarity among the insects.</div><div 
  class="publication01Info">Lynch JA, Peel AD, Drechsler A, Averof M, Roth S, Current Biology 20, 1042-1047 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(117);">abstract <img id="abstractI117" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20471269&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP117" 
  class="publication01Abstract" style="display:none;">The eggs of insects are unusual in that they often have bilateral symmetry when they are laid, indicating that both anterior-posterior (AP) and dorsal-ventral (DV) symmetries are broken during oogenesis. The molecular basis of this process is well understood in Drosophila melanogaster, in which symmetry breaking events for both axes depend on the asymmetric position of the oocyte nucleus and on germline-soma signaling mediated by the Tgf alpha-like epidermal growth factor (EGF) ligand Gurken. Germline-soma signaling interactions centered around the oocyte nucleus have been proposed in other insect species, but the molecular nature of these interactions has not been elucidated. We have examined the behavior of the oocyte nucleus and the function of EGF signaling components in the ovaries of the wasp Nasonia vitripennis, the beetle Tribolium castaneum, and the cricket Gryllus bimaculatus. We have found that EGF signaling has broadly conserved roles in mediating the encapsulation of oocytes by the somatic follicle cell layer, in establishing polarity of the egg chambers, and in setting up the DV axis of the embryo. These results provide insights into the evolutionary origins of the unique strategy employed by insects to establish embryonic axial polarity during oogenesis.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(117);">hide</a></div></div></div><div id="publication118" class="publication01"><div 
  class="publication01Title">Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.</div><div 
  class="publication01Info">Atwell S, Huang YS, Vilhjálmsson BJ, Willems G, Horton M, Li Y, Meng D, Platt A, Tarone AM, Hu TT, Jiang R et al., Nature 465, 627-631 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(118);">abstract <img id="abstractI118" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20336072&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP118" 
  class="publication01Abstract" style="display:none;">Although pioneered by human geneticists as a potential solution to the challenging problem of finding the genetic basis of common human diseases, genome-wide association (GWA) studies have, owing to advances in genotyping and sequencing technology, become an obvious general approach for studying the genetics of natural variation and traits of agricultural importance. They are particularly useful when inbred lines are available, because once these lines have been genotyped they can be phenotyped multiple times, making it possible (as well as extremely cost effective) to study many different traits in many different environments, while replicating the phenotypic measurements to reduce environmental noise. Here we demonstrate the power of this approach by carrying out a GWA study of 107 phenotypes in Arabidopsis thaliana, a widely distributed, predominantly self-fertilizing model plant known to harbour considerable genetic variation for many adaptively important traits. Our results are dramatically different from those of human GWA studies, in that we identify many common alleles of major effect, but they are also, in many cases, harder to interpret because confounding by complex genetics and population structure make it difficult to distinguish true associations from false. However, a-priori candidates are significantly over-represented among these associations as well, making many of them excellent candidates for follow-up experiments. Our study demonstrates the feasibility of GWA studies in A. thaliana and suggests that the approach will be appropriate for many other organisms.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(118);">hide</a></div></div></div><div id="publication119" class="publication01"><div 
  class="publication01Title">Assessing the influence of adjacent gene orientation on the evolution of gene upstream regions in Arabidopsis thaliana.</div><div 
  class="publication01Info">He F, Chen W, Collins S, Acquisti C, Goebel U, Ramos-Onsins S, Lercher MJ, de Meaux J, Genetics 185, 695-701 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(119);">abstract <img id="abstractI119" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20233855&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP119" 
  class="publication01Abstract" style="display:none;">The orientation of flanking genes may influence the evolution of intergenic regions in which cis-regulatory elements are likely to be located: divergently transcribed genes share their 5&apos; regions, resulting either in smaller &quot;private&quot; spaces or in overlapping regulatory elements. Thus, upstream sequences of divergently transcribed genes (bi-directional upstream regions, or URs) may be more constrained than those of uni-directional gene pairs. We investigated this effect by analyzing nucleotide variation segregating within and between Arabidopsis species. Compared to uni-directional URs, bi-directional URs indeed display lower population mutation rate, as well as more low-frequency polymorphisms. Furthermore, we find that bi-directional regions undergo selection for the maintenance of intergenic distance. Altogether, however, we observe considerable variation in evolutionary rates, with putative signatures of selection on two uni-directional upstream regions.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(119);">hide</a></div></div></div><div id="publication120" class="publication01"><div 
  class="publication01Title">Balanced nuclear and cytoplasmic activities of EDS1 are required for a complete plant innate immune response</div><div 
  class="publication01Info">Garcia AV, Blanvillain-Baufumé S, Huibers R P, Wiermer M, Li G, Gobbato E, Rietz S, Parker JE, PLoS Path 6: e1000970 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(120);">abstract <img id="abstractI120" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.plospathogens.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.ppat.1000970&representation=PDF">paper [pdf]</a> &nbsp; </div><div id="abstractP120" 
  class="publication01Abstract" style="display:none;">An important layer of plant innate immunity to host-adapted pathogens is conferred by intracellular nucleotide-binding/oligomerization domain-leucine rich repeat (NB-LRR) receptors recognizing specific microbial effectors. Signaling from activated receptors of the TIR (Toll/Interleukin-1 Receptor)-NB-LRR class converges on the nucleo-cytoplasmic immune regulator EDS1 (Enhanced Disease Susceptibility1). In this report we show that a receptor-stimulated increase in accumulation of nuclear EDS1 precedes or coincides with the EDS1-dependent induction and repression of defense-related genes. EDS1 is capable of nuclear transport receptor-mediated shuttling between the cytoplasm and nucleus. By enhancing EDS1 export from inside nuclei (through attachment of an additional nuclear export sequence (NES)) or conditionally releasing EDS1 to the nucleus (by fusion to a glucocorticoid receptor (GR)) in transgenic Arabidopsis we establish that the EDS1 nuclear pool is essential for resistance to biotrophic and hemi-biotrophic pathogens and for transcriptional reprogramming. Evidence points to post-transcriptional processes regulating receptor-triggered accumulation of EDS1 in nuclei. Changes in nuclear EDS1 levels become equilibrated with the cytoplasmic EDS1 pool and cytoplasmic EDS1 is needed for complete resistance and restriction of host cell death at infection sites. We propose that coordinated nuclear and cytoplasmic activities of EDS1 enable the plant to mount an appropriately balanced immune response to pathogen attack.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(120);">hide</a></div></div></div><div id="publication121" class="publication01"><div 
  class="publication01Title">Coordinated loading of IRG resistance GTPases on to the Toxoplasma gondii parasitophorous vacuole.</div><div 
  class="publication01Info">Khaminets A, Hunn JP, Könen-Waisman S, Zhao YO, Preukschat D, Coers J, Boyle JP, Ong Y, Boothroyd JC, Reichmann G, Howard JC et al., Cellular Microbiology 12, 939-961 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(121);">abstract <img id="abstractI121" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20109161&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP121" 
  class="publication01Abstract" style="display:none;">The immunity-related GTPases (IRGs) constitute an interferon-induced intracellular resistance mechanism in mice against Toxoplasma gondii. IRG proteins accumulate on the parasitophorous vacuole membrane (PVM), leading to its disruption and to death of the parasite. How IRGs target the PVM is unknown. We show that accumulation of IRGs on the PVM begins minutes after parasite invasion and increases for about 1 h. Targeting occurs independently of several signalling pathways and the microtubule network, suggesting that IRG transport is diffusion-driven. The intensity of IRG accumulation on the PVM, however, is reduced in absence of the autophagy regulator, Atg5. In wild-type cells IRG proteins accumulate cooperatively on PVMs in a definite order reflecting a temporal hierarchy, with Irgb6 and Irgb10 apparently acting as pioneers. Loading of IRG proteins onto the vacuoles of virulent Toxoplasma strains is attenuated and the two pioneer IRGs are the most affected. The polymorphic rhoptry kinases, ROP16, ROP18 and the catalytically inactive proteins, ROP5A-D, are not individually responsible for this effect. Thus IRG proteins protect mice against avirulent strains of Toxoplasma but fail against virulent strains. The complex cooperative behaviour of IRG proteins in resisting Toxoplasma may hint at undiscovered complexity also in virulence mechanisms.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(121);">hide</a></div></div></div><div id="publication122" class="publication01"><div 
  class="publication01Title">Epithelial reorganization events during late extraembryonic development in a hemimetabolous insect.</div><div 
  class="publication01Info">Panfilio KA, Roth S, Developmental Biology 340, 100-115 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(122);">abstract <img id="abstractI122" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20045678&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP122" 
  class="publication01Abstract" style="display:none;">As extra-embryonic tissues, the amnion and serosa are not considered to contribute materially to the insect embryo, yet they must execute an array of morphogenetic movements before they are dispensable. In hemimetabolous insects, these movements have been known for over a century, but they have remained virtually unexamined. This study addresses late extraembryonic morphogenesis in the milkweed bug, Oncopeltus fasciatus. Cell shape changes and apoptosis profiles are used to characterize the membranes as they undergo a large repertoire of final reorganizational events that reposition the embryo (katatrepsis), and eliminate the membranes themselves in an ordered fashion (dorsal closure). A number of key features were identified. First, amnion-serosa &quot;fusion&quot; involves localized apoptosis in the amnion and the formation of a supracellular actin purse string at the amnion-serosa border. During katatrepsis, a &apos;focus&apos; of serosal cells undergoes precocious columnarization and may serve as an anchor for contraction. Lastly, dorsal closure involves novel modifications of the amnion and embryonic flank that are without counterpart during the well-known process of dorsal closure in the fruit fly Drosophila melanogaster. These data also address the long-standing question of the final fate of the amnion: it undergoes apoptosis during dorsal closure and thus is likely to be solely extraembryonic.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(122);">hide</a></div></div></div><div id="publication123" class="publication01"><div 
  class="publication01Title">The earliest stages of adaptation in an experimental plant population: strong selection on QTLS for seed dormancy.</div><div 
  class="publication01Info">Huang X, Schmitt J, Dorn L, Griffith C, Effgen S, Takao S, Koornneef M, Donohue K, Molecular Ecology 19, 1335-1351 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(123);">abstract <img id="abstractI123" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20149097&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP123" 
  class="publication01Abstract" style="display:none;">Colonizing species may often encounter strong selection during the initial stages of adaptation to novel environments. Such selection is particularly likely to act on traits expressed early in development since early survival is necessary for the expression of adaptive phenotypes later in life. Genetic studies of fitness under field conditions, however, seldom include the earliest developmental stages. Using a new set of recombinant inbred lines, we present a study of the genetic basis of fitness variation in Arabidopsis thaliana in which genotypes, environments, and geographic location were manipulated to study total lifetime fitness, beginning with the seed stage. Large-effect quantitative trait loci (QTLs) for fitness changed allele frequency and closely approached 90% in some treatments within a single generation. These QTLs colocated with QTLs for germination phenology when seeds were dispersed following a schedule of a typical winter annual, and they were detected in two geographic locations at different latitudes. Epistatically interacting loci affected both fitness and germination in many cases. QTLs for field germination phenology colocated with known QTLs for primary dormancy induction as assessed in laboratory tests, including the candidate genes DOG1 and DOG6. Therefore fitness, germination phenology, and primary dormancy are genetically associated at the level of specific chromosomal regions and candidate loci. Genes associated with the ability to arrest development at early life stages and assess environmental conditions are thereby likely targets of intense natural selection early in the colonization process.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(123);">hide</a></div></div></div><div id="publication124" class="publication01"><div 
  class="publication01Title">Fitness flux and ubiquity of adaptive evolution.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 107, 4248-4253 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(124);">abstract <img id="abstractI124" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20145113&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP124" 
  class="publication01Abstract" style="display:none;">Natural selection favors fitter variants in a population, but actual evolutionary processes may decrease fitness by mutations and genetic drift. How is the stochastic evolution of molecular biological systems shaped by natural selection? Here, we derive a theorem on the fitness flux in a population, defined as the selective effect of its genotype frequency changes. The fitness-flux theorem generalizes Fisher&apos;s fundamental theorem of natural selection to evolutionary processes including mutations, genetic drift, and time-dependent selection. It shows that a generic state of populations is adaptive evolution: there is a positive fitness flux resulting from a surplus of beneficial over deleterious changes. In particular, stationary nonequilibrium evolution processes are predicted to be adaptive. Under specific nonstationary conditions, notably during a decrease in population size, the average fitness flux can become negative. We show that these predictions are in accordance with experiments in bacteria and bacteriophages and with genomic data in Drosophila. Our analysis establishes fitness flux as a universal measure of adaptation in molecular evolution.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(124);">hide</a></div></div></div><div id="publication125" class="publication01"><div 
  class="publication01Title">A test of the neutral model of expression change in natural populations of house mouse subspecies.</div><div 
  class="publication01Info">Staubach F, Teschke M, Voolstra CR, Wolf JBW, Tautz D, Evolution 64, 549-560 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(125);">abstract <img id="abstractI125" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19674092&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP125" 
  class="publication01Abstract" style="display:none;">Changes in expression of genes are thought to contribute significantly to evolutionary divergence. To study the relative role of selection and neutrality in shaping expression changes, we analyzed 24 genes in three different tissues of the house mouse (Mus musculus). Samples from two natural populations of the subspecies M. m. domesticus and M. m. musculus were investigated using quantitative PCR assays and sequencing of the upstream region. We have developed an approach to quantify expression polymorphism within such populations and to disentangle technical from biological variation in the data. We found a correlation between expression polymorphism within populations and divergence between populations. Furthermore, we found a correlation between expression polymorphism and sequence polymorphism of the respective genes. These data are most easily interpreted within a framework of a predominantly neutral model of gene expression change, where only a fraction of the changes may have been driven by positive selection. Although most genes investigated were expressed in all three tissues analyzed, significant changes of expression levels occurred predominantly in a single tissue only. This adds to the notion that enhancer-specific effects or transregulatory effects can modulate the evolution of gene expression in a tissue-specific way.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(125);">hide</a></div></div></div><div id="publication126" class="publication01"><div 
  class="publication01Title">The speed of evolution in large asexual populations</div><div 
  class="publication01Info">Park S, Simon D, Krug J, J. Stat. Phys. 138, 381-410 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(126);">abstract <img id="abstractI126" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.springerlink.com/content/w443g7543065l8h5/">paper [pdf]</a> &nbsp; </div><div id="abstractP126" 
  class="publication01Abstract" style="display:none;">We consider an asexual biological population of constant size N evolving in discrete time under the influence of selection and mutation. Beneficial mutations appear at rate U and their selective effects s are drawn from a distribution g(s). After introducing the required models and concepts of mathematical population genetics, we review different approaches to computing the speed of logarithmic fitness increase as a function of N, U and g(s). We present an exact solution of the infinite population size limit and provide an estimate of the population size beyond which it is valid. We then discuss approximate approaches to the finite population problem, distinguishing between the case of a single selection coefficient, g(s)=δ(s−s b ), and a continuous distribution of selection coefficients. Analytic estimates for the speed are compared to numerical simulations up to population sizes of order 10^300.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(126);">hide</a></div></div></div><div id="publication127" class="publication01"><div 
  class="publication01Title">Bayesian analysis of biological networks: clusters, motifs, cross-species correlations</div><div 
  class="publication01Info">Berg J, Lässig M, in Statistical And Evolutionary Analysis Of Biological Network Data, M. Stumpf and C. Wiuf (Eds.), Imperial College Press (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(127);">abstract <img id="abstractI127" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.thp.uni-koeln.de/~lassig/docs/2009a.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP127" 
  class="publication01Abstract" style="display:none;">An important part of the analysis of bio-molecular networks is to detect different functional units. Different functions are reflected in a different evolutionary dynamics, and hence in different statistical characteristics of network parts. In this sense, the {\em global statistics} of a biological network, e.g., its connectivity distribution, provides a background, and {\em local deviations} from this background signal functional units. In the computational analysis of biological networks, we thus typically have to discriminate between different statistical models governing different parts of the dataset. The nature of these models depends on the biological question asked. We illustrate this rationale here with three examples: identification of functional parts as highly connected 	extit{network clusters}, finding 	extit{network motifs}, which occur in a similar form at different places in the network, and the analysis of 	extit{cross-species network correlations}, which reflect evolutionary dynamics between species.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(127);">hide</a></div></div></div><div id="publication128" class="publication01"><div 
  class="publication01Title">Co-expression of neighbouring genes in Arabidopsis: separating chromatin effects from direct interactions.</div><div 
  class="publication01Info">Chen W, de Meaux J, Lercher MJ, BMC Genomics 11, 178 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(128);">abstract <img id="abstractI128" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20233415&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP128" 
  class="publication01Abstract" style="display:none;">BACKGROUND:In all eukaryotic species examined, genes that are chromosomal neighbours are more similar in their expression than random gene pairs. Currently, it is still unclear how much of this local co-expression is caused by direct transcriptional interactions, and how much is due to shared chromatin environments.RESULTS:We analysed neighbouring genes in Arabidopsis thaliana. At large intergenic distances (&gt;400 bp), divergently and convergently transcribed gene pairs show very similar levels of co-expression, mediated most likely by shared chromatin environments. At gene distances below 400 bp, co-expression is strongly enhanced only for divergently transcribed gene pairs, indicating bi-directional transcription from a single promoter. Conversely, co-expression is suppressed for short convergently or uni-directionally transcribed pairs. This suppression points to transcriptional interference concentrated at the 3&apos; end, e.g., in the context of transcription termination.CONCLUSIONS:Classifying linked gene pairs by their orientation, we are able to partially tease apart the different levels of regional expression modulation. (i) Regional chromatin characteristics modulate the accessibility for regulation and transcription, regardless of gene orientation; the strength of this chromatin effect can be assessed from divergently or convergently transcribed distant neighbours. (ii) Shared promoter regions up to 400 bp in length enhance the co-expression of close bi-directional neighbours. (iii) Transcriptional interference of close neighbours is concentrated at the 3&apos; ends of genes, and reduces co-expression on average by 40%.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(128);">hide</a></div></div></div><div id="publication129" class="publication01"><div 
  class="publication01Title">Comparative analysis of flowering in annual and perennial plants.</div><div 
  class="publication01Info">Albani MC, Coupland G, Current topics in developmental biology 91, 323-348 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(129);">abstract <img id="abstractI129" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20705187&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP129" 
  class="publication01Abstract" style="display:none;">In plants the switch from vegetative growth to flowering involves a major transition in the development of the shoot apex. This transition can occur once, in annual species, or repeatedly, in perennial plants. In annuals, flowering is associated with senescence and death of the whole plant, whereas perennials flower in consecutive years and maintain vegetative development after flowering. The perennial life strategy depends on differential behavior of meristems on a single plant so that some remain in the vegetative state while others undergo the floral transition. A. thaliana provides a powerful model system for understanding the mechanisms of flowering in annuals. Here we review the events that occur in the meristem of A. thaliana during the floral transition and compare these with our understanding of flowering in perennial systems.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(129);">hide</a></div></div></div><div id="publication130" class="publication01"><div 
  class="publication01Title">Conservation of MAP kinase activity and MSP genes in parthenogenetic nematodes.</div><div 
  class="publication01Info">Heger P, Kroiher M, Ndifon N, Schierenberg E, BMC developmental biology 10, 51 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(130);">abstract <img id="abstractI130" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20478028&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP130" 
  class="publication01Abstract" style="display:none;">BACKGROUND:MAP (mitogen-activated protein) kinase activation is a prerequisite for oocyte maturation, ovulation and fertilisation in many animals. In the hermaphroditic nematode Caenorhabditis elegans, an MSP (major sperm protein) dependent pathway is utilised for MAP kinase activation and successive oocyte maturation with extracellular MSP released from sperm acting as activator. How oocyte-to-embryo transition is triggered in parthenogenetic nematode species that lack sperm, is not known.RESULTS:We investigated two key elements of oocyte-to-embryo transition, MSP expression and MAP kinase signaling, in two parthenogenetic nematodes and their close hermaphroditic relatives. While activated MAP kinase is present in all analysed nematodes irrespective of the reproductive mode, MSP expression differs. In contrast to hermaphroditic or bisexual species, we do not find MSP expression at the protein level in parthenogenetic nematodes. However, genomic sequence analysis indicates that functional MSP genes are present in several parthenogenetic species.CONCLUSIONS:We present three alternative interpretations to explain our findings. (1) MSP has lost its function as a trigger of MAP kinase activation and is not expressed in parthenogenetic nematodes. Activation of the MAP kinase pathway is achieved by another, unknown mechanism. Functional MSP genes are required for occasionally emerging males found in some parthenogenetic species. (2) Because of long-term disadvantages, parthenogenesis is of recent origin. MSP genes remained intact during this short interval although they are useless. As in the first scenario, an unknown mechanism is responsible for MAP kinase activation. (3) The molecular machinery regulating oocyte-to-embryo transition in parthenogenetic nematodes is conserved with respect to C. elegans, thus requiring intact MSP genes. However, MSP expression has been shifted to non-sperm cells and is reduced below the detection limits, but is still sufficient to trigger MAP kinase activation and embryogenesis.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(130);">hide</a></div></div></div><div id="publication131" class="publication01"><div 
  class="publication01Title">Gene expression and activity of digestive proteases in Daphnia: effects of cyanobacterial protease inhibitors.</div><div 
  class="publication01Info">Schwarzenberger A, Zitt A, Kroth P, Mueller S, Von Elert E, BMC Physiology 10, 6 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(131);">abstract <img id="abstractI131" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20441581&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP131" 
  class="publication01Abstract" style="display:none;">BACKGROUND:The frequency of cyanobacterial blooms has increased worldwide, and these blooms have been claimed to be a major factor leading to the decline of the most important freshwater herbivores, i.e. representatives of the genus Daphnia. This suppression of Daphnia is partly attributed to the presence of biologically active secondary metabolites in cyanobacteria. Among these metabolites, protease inhibitors are found in almost every natural cyanobacterial bloom and have been shown to specifically inhibit Daphnia&apos;s digestive proteases in vitro, but to date no physiological responses of these serine proteases to cyanobacterial protease inhibitors in Daphnia have been reported in situ at the protein and genetic levels.RESULTS:Nine digestive proteases were detected in D. magna using activity-stained SDS-PAGE. Subsequent analyses by LC-MS/MS and database search led to the identification of respective protease genes. D. magna responded to dietary protease inhibitors by up-regulation of the expression of these respective proteases at the RNA-level and by the induction of new and less sensitive protease isoforms at the protein level. The up-regulation in response to dietary trypsin- and chymotrypsin-inhibitors ranged from 1.4-fold to 25.6-fold. These physiological responses of Daphnia, i.e. up-regulation of protease expression and the induction of isoforms, took place even after feeding on 20% cyanobacterial food for only 24 h. These physiological responses proved to be independent from microcystin effects.CONCLUSION:Here for the first time it was shown in situ that a D. magna clone responds physiologically to dietary cyanobacterial protease inhibitors by phenotypic plasticity of the targets of these specific inhibitors, i.e. Daphnia gut proteases. These regulatory responses are adaptive for D. magna, as they increase the capacity for protein digestion in the presence of dietary protease inhibitors. The type and extent of these responses in protease expression might determine the degree of growth reduction in D. magna in the presence of cyanobacterial protease inhibitors. The rapid response of Daphnia to cyanobacterial protease inhibitors supports the assumption that dietary cyanobacterial protease inhibitors exert a strong selection pressure on Daphnia proteases themselves.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(131);">hide</a></div></div></div><div id="publication132" class="publication01"><div 
  class="publication01Title">Influence of mutation rate on estimators of genetic differentiation--lessons from Arabidopsis thaliana.</div><div 
  class="publication01Info">Kronholm I, Loudet O, de Meaux J, BMC Genetics 11, 33 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(132);">abstract <img id="abstractI132" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20433762&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP132" 
  class="publication01Abstract" style="display:none;">BACKGROUND:The analysis of molecular variation within and between populations is crucial to establish strategies for conservation as well as to detect the footprint of spatially heterogeneous selection. The traditional estimator of genetic differentiation (F(ST)) has been shown to be misleading if genetic diversity is high. Alternative estimators of F(ST) have been proposed, but their robustness to variation in mutation rate is not clearly established. We first investigated the effect of mutation and migration rate using computer simulations and examined their joint influence on Q(ST), a measure of genetic differentiation for quantitative traits. We further used experimental data in natural populations of Arabidopsis thaliana to characterize the effect of mutation rate on various estimates of population differentiation. Since natural species exhibit various degrees of self-fertilisation, we also investigated the effect of mating system on the different estimators.RESULTS:If mutation rate is high and migration rate low, classical measures of genetic differentiation are misleading. Only Phi(ST), an estimator that takes the mutational distances between alleles into account, is independent of mutation rate, for all migration rates. However, the performance of Phi(ST) depends on the underlying mutation model and departures from this model cause its performance to degrade. We further show that Q(ST) has the same bias. We provide evidence that, in A. thaliana, microsatellite variation correlates with mutation rate. We thereby demonstrate that our results on estimators of genetic differentiation have important implications, even for species that are well established models in population genetics and molecular biology.CONCLUSIONS:We find that alternative measures of differentiation like F&apos;(ST) and D are not suitable for estimating effective migration rate and should not be used in studies of local adaptation. Genetic differentiation should instead be measured using an estimator that takes mutation rate into account, such as Phi(ST). Furthermore, in systems where migration between populations is low, such as A. thaliana, Q(ST) &lt; F(ST) cannot be taken as evidence for homogenising selection as has been traditionally thought.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(132);">hide</a></div></div></div><div id="publication133" class="publication01"><div 
  class="publication01Title">Localisation and mislocalisation of the interferon-inducible immunity-related GTPase, Irgm1 (LRG-47) in mouse cells.</div><div 
  class="publication01Info">Zhao YO, Könen-Waisman S, Taylor GA, Martens S, Howard JC, PloS One 5, e8648 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(133);">abstract <img id="abstractI133" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20072621&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP133" 
  class="publication01Abstract" style="display:none;">Irgm1 (LRG-47) is an interferon-inducible Golgi membrane associated GTPase of the mouse whose disruption causes susceptibility to many different intracellular pathogens. Irgm1 has been variously interpreted as a regulator of homologous effector GTPases of the IRG family, a regulator of phagosome maturation and as an initiator of autophagy in interferon-induced cells. We find that endogenous Irgm1 localises to late endosomal and lysosomal compartments in addition to the Golgi membranes. The targeting motif known to be required for Golgi localisation is surprisingly also required for endolysosomal localisation. However, unlike Golgi localisation, localisation to the endolysosomal system also requires the functional integrity of the nucleotide binding site, and thus probably reflects transient activation. Golgi localisation is lost when Irgm1 is tagged at either N- or C-termini with EGFP, while localisation to the endolysosomal system is relatively favoured. N-terminally tagged Irgm1 localises predominantly to early endosomes, while C-terminally tagged Irgm1 localises to late endosomes and lysosomes. Both these anomalous distributions are reversed by inactivation of the nucleotide binding site, and the tagged proteins both revert to Golgi membrane localisation. Irgm1 is the first IRG protein to be found associated with the endolysosomal membrane system in addition to either Golgi (Irgm1 and Irgm2) or ER (Irgm3) membranes, and we interpret the result to be in favour of a regulatory function of IRGM proteins at cellular membrane systems. In future analyses it should be borne in mind that tagging of Irgm1 leads to loss of Golgi localisation and enhanced localisation on endolysosomal membranes, probably as a result of constitutive activation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(133);">hide</a></div></div></div><div id="publication134" class="publication01"><div 
  class="publication01Title">Phosphorylation of mouse immunity-related GTPase (IRG) resistance proteins is an evasion strategy for virulent Toxoplasma gondii.</div><div 
  class="publication01Info">Steinfeldt T, Könen-Waisman S, Tong L, Pawlowski N, Lamkemeyer T, Sibley LD, Hunn JP, Howard JC, PLoS Biology 8, e1000576 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(134);">abstract <img id="abstractI134" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21203588&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP134" 
  class="publication01Abstract" style="display:none;">Virulence of complex pathogens in mammals is generally determined by multiple components of the pathogen interacting with the functional complexity and multiple layering of the mammalian immune system. It is most unusual for the resistance of a mammalian host to be overcome by the defeat of a single defence mechanism. In this study we uncover and analyse just such a case at the molecular level, involving the widespread intracellular protozoan pathogen Toxoplasma gondii and one of its most important natural hosts, the house mouse (Mus musculus). Natural polymorphism in virulence of Eurasian T. gondii strains for mice has been correlated in genetic screens with the expression of polymorphic rhoptry kinases (ROP kinases) secreted into the host cell during infection. We show that the molecular targets of the virulent allelic form of ROP18 kinase are members of a family of cellular GTPases, the interferon-inducible IRG (immunity-related GTPase) proteins, known from earlier work to be essential resistance factors in mice against avirulent strains of T. gondii. Virulent T. gondii strain ROP18 kinase phosphorylates several mouse IRG proteins. We show that the parasite kinase phosphorylates host Irga6 at two threonines in the nucleotide-binding domain, biochemically inactivating the GTPase and inhibiting its accumulation and action at the T. gondii parasitophorous vacuole membrane. Our analysis identifies the conformationally active switch I region of the GTP-binding site as an Achilles&apos; heel of the IRG protein pathogen-resistance mechanism. The polymorphism of ROP18 in natural T. gondii populations indicates the existence of a dynamic, rapidly evolving ecological relationship between parasite virulence factors and host resistance factors. This system should be unusually fruitful for analysis at both ecological and molecular levels since both T. gondii and the mouse are widespread and abundant in the wild and are well-established model species with excellent analytical tools available.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(134);">hide</a></div></div></div><div id="publication135" class="publication01"><div 
  class="publication01Title">Spontaneous focal activation of invariant natural killer T (iNKT) cells in mouse liver and kidney.</div><div 
  class="publication01Info">Zeng J, Howard JC, BMC Biology 8, 142 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(135);">abstract <img id="abstractI135" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21118540&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP135" 
  class="publication01Abstract" style="display:none;">BACKGROUND:Invariant natural killer T (iNKT) cells differ from other T cells by their hyperactive effector T-cell status, in addition to the expression of NK lineage receptors and semi-invariant T-cell receptors. It is generally agreed that the immune phenotype of iNKT cells is maintained by repeated activation in peripheral tissues although no explicit evidence for such iNKT cell activity in vivo has so far been reported.RESULTS:We used an interferon (IFN)-γ-inducible cytoplasmic protein, Irga6, as a histological marker for local IFN-γ production. Irga6 was intensely expressed in small foci of liver parenchymal cells and kidney tubular epithelium. Focal Irga6 expression was unaffected by germ-free status or loss of TLR signalling and was totally dependent on IFN-γ secreted by T cells in the centres of expression foci. These were shown to be iNKT cells by diagnostic T cell receptor usage and their activity was lost in both CD1 d and Jα-deficient mice.CONCLUSIONS:This is the first report that supplies direct evidence for explicit activation events of NKT cells in vivo and raises issues about the triggering mechanism and consequences for immune functions in liver and kidney.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(135);">hide</a></div></div></div><div id="publication136" class="publication01"><div 
  class="publication01Title">The mouse resistance protein Irgm1 (LRG-47): a regulator or an effector of pathogen defense?</div><div 
  class="publication01Info">Hunn JP, Howard JC, PLoS Pathogens 6, e1001008 (2010)</div><div 
  class="publication01Links"><a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20664789&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div></div>
          </div>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2009</div>
          <div id="publication137" class="publication01"><div 
  class="publication01Title">Adaptive gene regulatory networks</div><div 
  class="publication01Info">Stauffer F, Berg J, EPL 88, 48004 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(137);">abstract <img id="abstractI137" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/0295-5075/88/4/48004">paper [pdf]</a> &nbsp; </div><div id="abstractP137" 
  class="publication01Abstract" style="display:none;">Regulatory interactions between genes show a large amount of cross-species variability, even when the underlying functions are conserved: there are many ways to achieve the same function. Here we investigate the ability of regulatory networks to reproduce given expression levels within a simple model of gene regulation. We find an exponentially large space of regulatory networks compatible with a given set of expression levels, giving rise to an extensive entropy of networks. Typical realisations of regulatory networks are found to share a bias towards symmetric interactions, in line with empirical evidence.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(137);">hide</a></div></div></div><div id="publication138" class="publication01"><div 
  class="publication01Title">Brucella abortus induces Irgm3 and Irga6 expression via type-I IFN by a MyD88-dependent pathway, without the requirement of TLR2, TLR4, TLR5 and TLR9.</div><div 
  class="publication01Info">Lapaque N, Muller A, Alexopoulou L, Howard JC, Gorvel J, Microbial Pathogenesis 47, 299-304 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(138);">abstract <img id="abstractI138" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19747534&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP138" 
  class="publication01Abstract" style="display:none;">The innate immune system senses bacterial pathogens by pattern recognition receptors, such as the well-characterised Toll-like Receptors (TLR). The activation of TLR signalling cascades depends on several adaptor proteins, among which MyD88 plays a key role in triggering innate immune responses. Here, we show in murine macrophages that Brucella abortus triggers expression of the interferon-inducible resistance proteins (IRGs, p47 GTPases) via type-I IFN secretion at late time points, when Brucella has reached its replication niche. This induction requires the adaptor molecule MyD88 but does not involve the TLRs normally implicated in sensing Gram-negative bacteria, namely TLR2, TLR4, TLR5 and TLR9. Brucella mutants lacking the functional VirB type-IV secretion system were not capable of inducing Irgm3 and Irga6 expression, suggesting that the type-IV secretion system is part of the triggering of the activation process. Our data suggest that Brucella is recognized intracellularly by an unknown receptor, different from the conventional ones used for Gram-negative sensing, but one that nevertheless signals through MyD88.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(138);">hide</a></div></div></div><div id="publication139" class="publication01"><div 
  class="publication01Title">Estimating mutation distances from unaligned genomes.</div><div 
  class="publication01Info">Haubold B, Pfaffelhuber P, Domazet-Loso M, Wiehe T, Journal Of Computational Biology 16, 1487-1500 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(139);">abstract <img id="abstractI139" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19803738&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP139" 
  class="publication01Abstract" style="display:none;">Abstract Alignment-free distance measures are generally less accurate but more efficient than traditional alignment-based metrics. In the context of genome sequence analysis, the efficiency gain is often so substantial that it outweights the loss in accuracy. However, a further disadvantage of alignment-free distances is that their relationship to evolutionary events such as substitutions is generally unknown. We have therefore derived an estimator of the number of substitutions per site between two unaligned DNA sequences, K(r). Simulations show that this estimator works well with &quot;ideal&quot; data. We compare K(r) to two alternative alignment-free distances: a k-tuple distance and a measure of relative entropy based on average common substring length. All three measures are applied to 27 primate mitochondrial genomes, eight whole genomes of Streptococcus agalactiae strains, and 12 whole genomes of Drosophila species. In each case, the cluster diagrams based on K(r) are equivalent to or significantly better than those based on the two alternative measures. This is due to the fact that in contrast to the alternative measures K(r) is derived from an explicit model of evolution. The computation of K(r) is efficiently implemented in the program kr, which can be downloaded freely from the internet.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(139);">hide</a></div></div></div><div id="publication140" class="publication01"><div 
  class="publication01Title">Emergence of a new gene from an intergenic region.</div><div 
  class="publication01Info">Heinen TJAJ, Staubach F, Häming D, Tautz D, Current Biology 19, 1527-1531 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(140);">abstract <img id="abstractI140" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19733073&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP140" 
  class="publication01Abstract" style="display:none;">It is generally assumed that new genes would arise by gene duplication mechanisms, because the signals for regulation and transcript processing would be unlikely to evolve in parallel with a new gene function. We have identified here a transcript in the house mouse (Mus musculus) that has arisen within the past 2.5-3.5 million years in a large intergenic region. The region is present in many mammals, including humans, allowing us to exclude the involvement of gene duplication, transposable elements, or other genome rearrangements, which are typically found for other cases of newly evolved genes. The gene has three exons, shows alternative splicing, and is specifically expressed in postmeiotic cells of the testis. The transcript is restricted to species within the genus Mus and its emergence correlates with indel mutations in the 5&apos; regulatory region of the transcript. A recent selective sweep is associated with the transcript region in M. m. musculus populations. A knockout in the laboratory strain BL6 results in reduced sperm motility and reduced testis weight. Our results show that cryptic signals for transcript regulation and processing exist in intergenic regions and can become the basis for the evolution of a new functional gene.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(140);">hide</a></div></div></div><div id="publication141" class="publication01"><div 
  class="publication01Title">Human Relationships Inferred from Genetic Variation</div><div 
  class="publication01Info">Lao O, Kayser M, Wiley Online Library (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(141);">abstract <img id="abstractI141" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.sfb680.uni-koeln.de/docs/Lao_Kayser_encyclopedia.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP141" 
  class="publication01Abstract" style="display:none;">We humans are a diverse species, both at the phenotypic and the genetic levels. The genetic diversity of current human populations has been shaped by various demographic and evolutionary as well as some cultural factors. Thus, studying how genetic variation is distributed through individuals around the world can provide insights into (i) when and where our human ancestors first appeared on the planet, (ii) the dynamics of admixture with other Homo species not existing anymore, (iii) migration waves that brought humans across the world and (iv) processes of adaptations towards environmental and other factors that shaped human genomes and phenotypic traits. Additionally, the quantification of genetic variation between human populations provides data evidence for elucidating whether humans can be classified according to genetically homogeneous groups or not, with implications for medical and forensic studies. Hence, studying human genetic diversity is important for better understanding of our past, presence and future<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(141);">hide</a></div></div></div><div id="publication142" class="publication01"><div 
  class="publication01Title">Discrete shoot and root stem cell-promoting WUS/WOX5 functions are an evolutionary innovation of angiosperms.</div><div 
  class="publication01Info">Nardmann J, Reisewitz P, Werr W, Molecular Biology And Evolution 26, 1745-1755 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(142);">abstract <img id="abstractI142" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19387013&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP142" 
  class="publication01Abstract" style="display:none;">The morphologically diverse bodies of seed plants comprising gymnosperms and angiosperms, which separated some 350 Ma, grow by the activity of meristems containing stem cell niches. In the dicot model Arabidopsis thaliana, these are maintained by the stem cell-promoting functions of WUS and WUSCHEL-related homeobox 5 (WOX5) in the shoot and the root, respectively. Both genes are members of the WOX gene family, which has a monophyletic origin in green algae. The establishment of the WOX gene phylogeny from basal land plants through gymnosperms to basal and higher angiosperms reveals three major branches: a basal clade consisting of WOX13-related genes present in some green algae and throughout all land plant genomes, a second clade containing WOX8/9/11/12 homologues, and a modern clade restricted to seed plants. The analysis of the origin of the modern branch in two basal angiosperms (Amborella trichopoda and Nymphaea jamesoniana) and three gymnosperms (Pinus sylvestris, Ginkgo biloba, and Gnetum gnemon) shows that all members of the modern clade consistently found in monocots and dicots exist at the base of the angiosperm lineage, including WUS and WOX5 orthologues. In contrast, our analyses identify a single WUS/WOX5 homologue in all three gymnosperm genomes, consistent with a monophyletic origin in the last common ancestor of gymnosperms and angiosperms. Phylogenetic data, WUS- and WOX5-specific evolutionary signatures, as well as the expression pattern and stem cell-promoting function of the single gymnosperm WUS/WOX5 pro-orthologue in Arabidopsis indicate a gene duplication event followed by subfunctionalization at the base of angiosperms.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(142);">hide</a></div></div></div><div id="publication143" class="publication01"><div 
  class="publication01Title">Evolution of axis formation: mRNA localization, regulatory circuits and posterior specification in non-model arthropods.</div><div 
  class="publication01Info">Nunes da Fonseca R, Lynch JA, Roth S, Current Opinion In Genetics &amp; Development 19, 404-411 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(143);">abstract <img id="abstractI143" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19493672&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP143" 
  class="publication01Abstract" style="display:none;">Drosophila melanogaster has been used as an arthropod model to understand the establishment of embryonic axes. However, the long germ type of embryogenesis in Drosophila is highly divergent from the ancestral, short germ mode seen in most other arthropods. Therefore, broader sampling of other arthropod taxa is required to understand the evolution of axial patterning mechanisms within this phylum. Here we describe recent progress toward this end. First, we describe differing strategies for providing patterning information along the anterior-posterior axis of arthropod embryos. Second, we discuss the recent findings on dorso-ventral axis of beetles and spiders that highlight the importance of regulatory interactions at the zygotic level. Third, recent discoveries of the roles of Wnt signalling, the Notch/Delta pathway and caudal in establishing and patterning the posterior embryonic region in spiders and beetles are described. In the end, we try to integrate these new findings with non-model arthropods in a common framework and discuss the major gaps of knowledge that should be addressed by future studies.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(143);">hide</a></div></div></div><div id="publication144" class="publication01"><div 
  class="publication01Title">Exploring the effect of sex on empirical fitness landscapes.</div><div 
  class="publication01Info">de Visser JAGM, Park S, Krug J, The American Naturalist 174 Suppl 1, S15-30 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(144);">abstract <img id="abstractI144" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19456267&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP144" 
  class="publication01Abstract" style="display:none;">The nature of epistasis has important consequences for the evolutionary significance of sex and recombination. Recent efforts to find negative epistasis as a source of negative linkage disequilibrium and associated long-term advantage to sex have yielded little support. Sign epistasis, where the sign of the fitness effects of alleles varies across genetic backgrounds, is responsible for the ruggedness of the fitness landscape, with several unexplored implications for the evolution of sex. Here, we describe fitness landscapes for two sets of strains of the asexual fungus Aspergillus niger involving all combinations of five mutations. We find that approximately 30% of the single-mutation fitness effects are positive despite their negative effect in the wild-type strain and that several local fitness maxima and minima are present. We then compare adaptation of sexual and asexual populations on these empirical fitness landscapes by using simulations. The results show a general disadvantage of sex on these rugged landscapes, caused by the breakdown by recombination of genotypes on fitness peaks. Sex facilitates movement to the global peak only for some parameter values on one landscape, indicating its dependence on the landscape&apos;s topography. We discuss possible reasons for the discrepancy between our results and the reports of faster adaptation of sexual populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(144);">hide</a></div></div></div><div id="publication145" class="publication01"><div 
  class="publication01Title">Balance of Irgm protein activities determines IFN-gamma-induced host defense.</div><div 
  class="publication01Info">Henry SC, Daniell XG, Burroughs AR, Indaram M, Howell DN, Coers J, Starnbach MN, Hunn JP, Howard JC, Feng CG, Sher AEA, Journal Of Leukocyte Biology 85, 877-885 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(145);">abstract <img id="abstractI145" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19176402&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP145" 
  class="publication01Abstract" style="display:none;">The immunity-related GTPases (IRG), also known as p47 GTPases, are a family of proteins that are tightly regulated by IFNs at the transcriptional level and serve as key mediators of IFN-regulated resistance to intracellular bacteria and protozoa. Among the IRG proteins, loss of Irgm1 has the most profound impact on IFN-gamma-induced host resistance at the physiological level. Surprisingly, the losses of host resistance seen in the absence of Irgm1 are sometimes more striking than those seen in the absence of IFN-gamma. In the current work, we address the underlying mechanism. We find that in several contexts, another protein in the IRG family, Irgm3, functions to counter the effects of Irgm1. By creating mice that lack Irgm1 and Irgm3, we show that several phenotypes important to host resistance that are caused by Irgm1 deficiency are reversed by coincident Irgm3 deficiency; these include resistance to Salmonella typhimurium in vivo, the ability to affect IFN-gamma-induced Salmonella killing in isolated macrophages, and the ability to regulate macrophage adhesion and motility in vitro. Other phenotypes that are caused by Irgm1 deficiency, including susceptibility to Toxoplasma gondii and the regulation of GKS IRG protein expression and localization, are not reversed but exacerbated when Irgm3 is also absent. These data suggest that members of the Irgm subfamily within the larger IRG family possess activities that can be opposing or cooperative depending on the context, and it is the balance of these activities that is pivotal in mediating IFN-gamma-regulated host resistance.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(145);">hide</a></div></div></div><div id="publication146" class="publication01"><div 
  class="publication01Title">Modeling infectious disease in mice: co-adaptation and the role of host-specific IFNgamma responses.</div><div 
  class="publication01Info">Coers J, Starnbach MN, Howard JC, PLoS Pathogens 5, e1000333 (2009)</div><div 
  class="publication01Links"><a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19478881&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div></div><div id="publication147" class="publication01"><div 
  class="publication01Title">Reentrant phase transition in a predator-prey model.</div><div 
  class="publication01Info">Sung-Guk H, Park S, Jun KB, Physical Review. E 79, 066114 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(147);">abstract <img id="abstractI147" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=19658571&retmode=ref&cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP147" 
  class="publication01Abstract" style="display:none;">We numerically investigate the six-species predator-prey game in complex networks as well as in d -dimensional regular hypercubic lattices with d=1,2,...,6 . The food-web topology of the six species contains two directed loops, each of which is composed of cyclically predating three species. As the mutation rate is lowered below the well-defined phase transition point, the Z2 symmetry related with the interchange in the two loops is spontaneously broken, and it has been known that the system develops the defensive alliance in which three cyclically predating species defend each other against the invasion of other species. In the Watts-Strogatz small-world network structure characterized by the rewiring probability alpha , the phase diagram shows the reentrant behavior as alpha is varied, indicating a twofold role of the shortcuts. In d -dimensional regular hypercubic lattices, the system also exhibits the reentrant phase transition as d is increased. We identify universality class of the phase transition and discuss the proper mean-field limit of the system.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(147);">hide</a></div></div></div><div id="publication148" class="publication01"><div 
  class="publication01Title">Death and resurrection of the human IRGM gene.</div><div 
  class="publication01Info">Bekpen C, Marques-Bonet T, Alkan C, Antonacci F, Leogrande MB, Ventura M, Kidd JM, Siswara P, Howard JC, Eichler EE, PLoS Genetics 5, e1000403 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(148);">abstract <img id="abstractI148" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19266026&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP148" 
  class="publication01Abstract" style="display:none;">Immunity-related GTPases (IRG) play an important role in defense against intracellular pathogens. One member of this gene family in humans, IRGM, has been recently implicated as a risk factor for Crohn&apos;s disease. We analyzed the detailed structure of this gene family among primates and showed that most of the IRG gene cluster was deleted early in primate evolution, after the divergence of the anthropoids from prosimians ( about 50 million years ago). Comparative sequence analysis of New World and Old World monkey species shows that the single-copy IRGM gene became pseudogenized as a result of an Alu retrotransposition event in the anthropoid common ancestor that disrupted the open reading frame (ORF). We find that the ORF was reestablished as a part of a polymorphic stop codon in the common ancestor of humans and great apes. Expression analysis suggests that this change occurred in conjunction with the insertion of an endogenous retrovirus, which altered the transcription initiation, splicing, and expression profile of IRGM. These data argue that the gene became pseudogenized and was then resurrected through a series of complex structural events and suggest remarkable functional plasticity where alleles experience diverse evolutionary pressures over time. Such dynamism in structure and evolution may be critical for a gene family locked in an arms race with an ever-changing repertoire of intracellular parasites.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(148);">hide</a></div></div></div><div id="publication149" class="publication01"><div 
  class="publication01Title">From fitness landscapes to seascapes: non-equilibrium dynamics of selection and adaptation.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Trends In Genetics 25, 111-119 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(149);">abstract <img id="abstractI149" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19232770&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP149" 
  class="publication01Abstract" style="display:none;">Evolution is a quest for innovation. Organisms adapt to changing natural selection by evolving new phenotypes. Can we read this dynamics in their genomes? Not every mutation under positive selection responds to a change in selection: beneficial changes also occur at evolutionary equilibrium, repairing previous deleterious changes and restoring existing functions. Adaptation, by contrast, is viewed here as a non-equilibrium phenomenon: the genomic response to time-dependent selection. Our approach extends the static concept of fitness landscapes to dynamic fitness seascapes. It shows that adaptation requires a surplus of beneficial substitutions over deleterious ones. Here, we focus on the evolution of yeast and Drosophila genomes, providing examples where adaptive evolution can and cannot be inferred, despite the presence of positive selection.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(149);">hide</a></div></div></div><div id="publication150" class="publication01"><div 
  class="publication01Title">Selection on cis-regulatory variation at B4galnt2 and its influence on von Willebrand factor in house mice.</div><div 
  class="publication01Info">Johnsen JM, Teschke M, Pavlidis P, McGee BM, Tautz D, Ginsburg D, Baines JF, Molecular Biology And Evolution 26, 567-578 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(150);">abstract <img id="abstractI150" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19088380&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP150" 
  class="publication01Abstract" style="display:none;">The RIIIS/J inbred mouse strain is a model for type 1 von Willebrand disease (VWD), a common human bleeding disorder. Low von Willebrand factor (VWF) levels in RIIIS/J are due to a regulatory mutation, Mvwf1, which directs a tissue-specific switch in expression of a glycosyltransferase, B4GALNT2, from intestine to blood vessel. We recently found that Mvwf1 lies on a founder allele common among laboratory mouse strains. To investigate the evolutionary forces operating at B4galnt2, we conducted a survey of DNA sequence polymorphism and microsatellite variation spanning the B4galnt2 gene region in natural Mus musculus domesticus populations. Two divergent haplotypes segregate in these natural populations, one of which corresponds to the RIIIS/J sequence. Different local populations display dramatic differences in the frequency of these haplotypes, and reduced microsatellite variability near B4galnt2 within the RIIIS/J haplotype is consistent with the recent action of natural selection. The level and pattern of DNA sequence polymorphism in the 5&apos; flanking region of the gene significantly deviates from the neutral expectation and suggests that variation in B4galnt2 expression may be under balancing selection and/or arose from a recently introgressed allele that subsequently increased in frequency due to natural selection. However, coalescent simulations indicate that the heterogeneity in divergence between haplotypes is greater than expected under an introgression model. Analysis of a population where the RIIIS/J haplotype is in high frequency reveals an association between this haplotype, the B4galnt2 tissue-specific switch, and a significant decrease in plasma VWF levels. Given these observations, we propose that low VWF levels may represent a fitness cost that is offset by a yet unknown benefit of the B4galnt2 tissue-specific switch. Similar mechanisms may account for the variability in VWF levels and high prevalence of VWD in other mammals, including humans.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(150);">hide</a></div></div></div><div id="publication151" class="publication01"><div 
  class="publication01Title">Toxoplasma gondii and the Immunity-Related GTPase (IRG) resistance system in mice: a review.</div><div 
  class="publication01Info">Zhao YO, Rohde C, Lilue JT, Könen-Waisman S, Khaminets A, Hunn JP, Howard JC, Memórias Do Instituto Oswaldo Cruz 104, 234-240 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(151);">abstract <img id="abstractI151" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19430648&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP151" 
  class="publication01Abstract" style="display:none;">The Immunity Related GTPases (IRG proteins) constitute a large family of interferon-inducible proteins that mediate early resistance to Toxoplasma gondii infection in mice. At least six members of this family are required for resistance of mice to virulent T. gondii strains. Recent results have shown that the complexity of the resistance arises from complex regulatory interactions between different family members. The mode of action against T. gondii depends on the ability of IRG proteins to accumulate on the parasitophorous vacuole of invading tachyzoites and to induce local damage to the vacuole resulting in disruption of the vacuolar membrane. Virulent strains of T. gondii overcome the IRG resistance system, probably by interfering with the loading of IRG proteins onto the parasitophorous vacuole membrane. It may be assumed that T. gondii strains highly virulent for mice will be disadvantaged in the wild due to the rapid extinction of the infected host, while it is self-evident that susceptibility to virulent strains is disadvantageous to the mouse host. We consider the possibility that this double disadvantage is compensated in wild populations by segregating alleles with different resistance and susceptibility properties in the IRG system.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(151);">hide</a></div></div></div><div id="publication152" class="publication01"><div 
  class="publication01Title">Incremental steps toward incompatibility revealed by Arabidopsis epistatic interactions modulating salicylic acid pathway activation.</div><div 
  class="publication01Info">Alcázar R, García AV, Parker JE, Reymond M, Proceedings Of The National Academy Of Sciences Of The United States Of America 106, 334-339 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(152);">abstract <img id="abstractI152" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19106299&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP152" 
  class="publication01Abstract" style="display:none;">Plant growth is influenced by genetic factors and environmental cues. Genotype-by-environment interactions are governed by complex genetic epistatic networks that are subject to natural selection. Here we describe a novel epistatic interaction modulating growth in response to temperature common to 2 Arabidopsis recombinant inbred line (RIL) populations (Ler x Kas-2 and Ler x Kond). At 14 degrees C, lines with specific allele combinations at interacting loci (incompatible interactions) have severe growth defects. These lines exhibit deregulated cell death programs and enhanced disease resistance. At 20 degrees C, growth defects are suppressed, but a positive trait of enhanced resistance is retained. Mapping of 1 interacting QTL to a cluster of RPP1-like TIR-NB-LRR genes on chromosome 3 is consistent with our finding that environmentally conditioned epistasis depends on activation of the salicylic acid (SA) stress signaling pathway. The nature of the epistatic interaction conforms to the Dobzhansky-Muller model of genetic incompatibility with incomplete penetrance for reproductive isolation. Variation in fitness of different incompatible lines reveals the presence of additional modifiers in the genetic background. We propose that certain interacting loci lead to an optimal balance between growth and resistance to pathogens by modulating SA signaling under specific environments. This could allow the accumulation of additional incompatibilities before reaching complete reproductive isolation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(152);">hide</a></div></div></div><div id="publication153" class="publication01"><div 
  class="publication01Title">Simulation of DNA sequence evolution under models of recent directional selection.</div><div 
  class="publication01Info">Kim Y, Wiehe T, Briefings In Bioinformatics 10, 84-96 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(153);">abstract <img id="abstractI153" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19109303&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP153" 
  class="publication01Abstract" style="display:none;">Computer simulation is an essential tool in the analysis of DNA sequence variation for mapping events of recent adaptive evolution in the genome. Various simulation methods are employed to predict the signature of selection in sequence variation. The most informative and efficient method currently in use is coalescent simulation. However, this method is limited to simple models of directional selection. Whole-population forward-in-time simulations are the alternative to coalescent simulations for more complex models. The notorious problem of excessive computational cost in forward-in-time simulations can be overcome by various simplifying amendments. Overall, the success of simulations depends on the creative application of some population genetic theory to the simulation algorithm.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(153);">hide</a></div></div></div><div id="publication154" class="publication01"><div 
  class="publication01Title">A common and unstable copy number variant is associated with differences in Glo1 expression and anxiety-like behavior.</div><div 
  class="publication01Info">Williams R, Lim JE, Harr B, Wing C, Walters R, Distler MG, Teschke M, Wu C, Wiltshire T, Su AI, Sokoloff GEA, PloS One 4, e4649 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(154);">abstract <img id="abstractI154" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19266052&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP154" 
  class="publication01Abstract" style="display:none;">Glyoxalase 1 (Glo1) has been implicated in anxiety-like behavior in mice and in multiple psychiatric diseases in humans. We used mouse Affymetrix exon arrays to detect copy number variants (CNV) among inbred mouse strains and thereby identified a approximately 475 kb tandem duplication on chromosome 17 that includes Glo1 (30,174,390-30,651,226 Mb; mouse genome build 36). We developed a PCR-based strategy and used it to detect this duplication in 23 of 71 inbred strains tested, and in various outbred and wild-caught mice. Presence of the duplication is associated with a cis-acting expression QTL for Glo1 (LOD&gt;30) in BXD recombinant inbred strains. However, evidence for an eQTL for Glo1 was not obtained when we analyzed single SNPs or 3-SNP haplotypes in a panel of 27 inbred strains. We conclude that association analysis in the inbred strain panel failed to detect an eQTL because the duplication was present on multiple highly divergent haplotypes. Furthermore, we suggest that non-allelic homologous recombination has led to multiple reversions to the non-duplicated state among inbred strains. We show associations between multiple duplication-containing haplotypes, Glo1 expression and anxiety-like behavior in both inbred strain panels and outbred CD-1 mice. Our findings provide a molecular basis for differential expression of Glo1 and further implicate Glo1 in anxiety-like behavior. More broadly, these results identify problems with commonly employed tests for association in inbred strains when CNVs are present. Finally, these data provide an example of biologically significant phenotypic variability in model organisms that can be attributed to CNVs.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(154);">hide</a></div></div></div><div id="publication155" class="publication01"><div 
  class="publication01Title">A dedicated promoter drives constitutive expression of the cell-autonomous immune resistance GTPase, Irga6 (IIGP1) in mouse liver.</div><div 
  class="publication01Info">Zeng J, Parvanova IA, Howard JC, PloS One 4, e6787 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(155);">abstract <img id="abstractI155" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20368812&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP155" 
  class="publication01Abstract" style="display:none;">BACKGROUND:In general, immune effector molecules are induced by infection.METHODOLOGY AND PRINCIPAL FINDINGS:However, strong constitutive expression of the cell-autonomous resistance GTPase, Irga6 (IIGP1), was found in mouse liver, contrasting with previous evidence that expression of this protein is exclusively dependent on induction by IFNgamma. Constitutive and IFNgamma-inducible expression of Irga6 in the liver were shown to be dependent on transcription initiated from two independent untranslated 5&apos; exons, which splice alternatively into the long exon encoding the full-length protein sequence. Irga6 is expressed constitutively in freshly isolated hepatocytes and is competent in these cells to accumulate on the parasitophorous vacuole membrane of infecting Toxoplasma gondii tachyzoites.CONCLUSIONS AND SIGNIFICANCE:The role of constitutive hepatocyte expression of Irga6 in resistance to parasites invading from the gut via the hepatic portal system is discussed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(155);">hide</a></div></div></div><div id="publication156" class="publication01"><div 
  class="publication01Title">Differences in embryonic pattern formation between Caenorhabditis elegans and its close parthenogenetic relative Diploscapter coronatus.</div><div 
  class="publication01Info">Lahl V, Schulze J, Schierenberg E, The International Journal Of Developmental Biology 53, 507-515 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(156);">abstract <img id="abstractI156" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19378252&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP156" 
  class="publication01Abstract" style="display:none;">In order to evaluate the evolutionary preservation of developmental programs during nematode embryogenesis, we searched for close relatives of the model system Caenorhabditis elegans with deviant patterns. The parthenogenetically reproducing species Diploscapter coronatus shows prominent differences to C. elegans. While in the 2-cell stage of C. elegans a rotation of the nuclear/centrosome complex is found only in the posterior P1 cell, in D. coronatus cell isolation indicates that rotation takes place in a cell-autonomous manner in both blastomeres, resulting in a linear 4-cell array. In C. elegans, the ABp cell becomes different from its ABa sister via a germline-induced induction. In D. coronatus, AB daughters do not touch the germline but nevertheless execute different fates, suggesting a cell-autonomous mechanism or signaling over distance. Laser ablation experiments revealed that active migration of the EMS cell is required to transform the linearly ordered blastomeres into a 3-dimensional embryo, and the difference can be most easily explained with a heterochronic shift with respect to cell mobility. In D. coronatus, reversal of cleavage polarity in the germline, typical for C. elegans, is absent. This results in four different transient variants of posterior blastomeres which eventually merge into a single pattern prior to the onset of gastrulation. This merging includes primordial germ cell migrations of variable extent toward the gut precursor cell and suggests a specific cell-cell recognition mechanism. Cell distribution in advanced embryos is essentially indistinguishable between both species.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(156);">hide</a></div></div></div><div id="publication157" class="publication01"><div 
  class="publication01Title">Loss of the insulator protein CTCF during nematode evolution.</div><div 
  class="publication01Info">Heger P, Marin B, Schierenberg E, BMC Molecular Biology 10, 84 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(157);">abstract <img id="abstractI157" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19712444&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP157" 
  class="publication01Abstract" style="display:none;">BACKGROUND:The zinc finger (ZF) protein CTCF (CCCTC-binding factor) is highly conserved in Drosophila and vertebrates where it has been shown to mediate chromatin insulation at a genomewide level. A mode of genetic regulation that involves insulators and insulator binding proteins to establish independent transcriptional units is currently not known in nematodes including Caenorhabditis elegans. We therefore searched in nematodes for orthologs of proteins that are involved in chromatin insulation.RESULTS:While orthologs for other insulator proteins were absent in all 35 analysed nematode species, we find orthologs of CTCF in a subset of nematodes. As an example for these we cloned the Trichinella spiralis CTCF-like gene and revealed a genomic structure very similar to the Drosophila counterpart. To investigate the pattern of CTCF occurrence in nematodes, we performed phylogenetic analysis with the ZF protein sets of completely sequenced nematodes. We show that three ZF proteins from three basal nematodes cluster together with known CTCF proteins whereas no zinc finger protein of C. elegans and other derived nematodes does so.CONCLUSION:Our findings show that CTCF and possibly chromatin insulation are present in basal nematodes. We suggest that the insulator protein CTCF has been secondarily lost in derived nematodes like C. elegans. We propose a switch in the regulation of gene expression during nematode evolution, from the common vertebrate and insect type involving distantly acting regulatory elements and chromatin insulation to a so far poorly characterised mode present in more derived nematodes. Here, all or some of these components are missing. Instead operons, polycistronic transcriptional units common in derived nematodes, seemingly adopted their function.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(157);">hide</a></div></div></div><div id="publication158" class="publication01"><div 
  class="publication01Title">Properties of Sequence Conservation in Upstream Regulatory and Protein Coding Sequences among Paralogs in Arabidopsis thaliana</div><div 
  class="publication01Info">Richardson DN, Wiehe T, Comparative Genomics 5817, 217 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(158);">abstract <img id="abstractI158" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://portal.acm.org/citation.cfm?id=1690891.1690909&amp;coll=DL&amp;dl=GUIDE&amp;CFID=87515263&amp;CFTOKEN=37311518">paper [pdf]</a> &nbsp; </div><div id="abstractP158" 
  class="publication01Abstract" style="display:none;">Whole genome duplication (WGD) has catalyzed the formation of new species, genes with novel functions, altered expression patterns, complexified signaling pathways and has provided organisms a level of genetic robustness. We studied the long-term evolution and interrelationships of 5’ upstream regulatory sequences (URSs), protein coding sequences (CDSs) and expression correlations (EC) of duplicated gene pairs in Arabidopsis. Three distinct methods revealed significant evolutionary conservation between paralogous URSs and were highly correlated with microarray-based expression correlation of the respective gene pairs. Positional information on exact matches between sequences unveiled the contribution of micro-chromosomal rearrangements on expression divergence. A three-way rank analysis of URS similarity, CDS divergence and EC uncovered specific gene functional biases. Transcription factor activity was associated with gene pairs exhibiting conserved URSs and divergent CDSs, whereas a broad array of metabolic enzymes was found to be associated with gene pairs showing diverged URSs but conserved CDSs.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(158);">hide</a></div></div></div><div id="publication159" class="publication01"><div 
  class="publication01Title">Robustness and epistasis in mutation-selection models.</div><div 
  class="publication01Info">Wolff A, Krug J, Physical Biology 6, 036007 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(159);">abstract <img id="abstractI159" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19411737&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP159" 
  class="publication01Abstract" style="display:none;">We investigate the fitness advantage associated with the robustness of a phenotype against deleterious mutations using deterministic mutation-selection models of a quasispecies type equipped with a mesa-shaped fitness landscape. We obtain analytic results for the robustness effect which become exact in the limit of infinite sequence length. Thereby, we are able to clarify a seeming contradiction between recent rigorous work and an earlier heuristic treatment based on mapping to a Schrödinger equation. We exploit the quantum mechanical analogy to calculate a correction term for finite sequence lengths and verify our analytic results by numerical studies. In addition, we investigate the occurrence of an error threshold for a general class of epistatic landscapes and show that diminishing epistasis is a necessary but not sufficient condition for error threshold behaviour.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(159);">hide</a></div></div></div><div id="publication160" class="publication01"><div 
  class="publication01Title">Why didn&apos;t Darwin discover Mendel&apos;s laws?</div><div 
  class="publication01Info">Howard JC, Journal Of Biology 8, 15 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(160);">abstract <img id="abstractI160" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19291260&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP160" 
  class="publication01Abstract" style="display:none;">Darwin&apos;s focus on small quantitative variations as the raw material of evolution may have prevented him from discovering the laws of inheritance.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(160);">hide</a></div></div></div>
          </div>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2008</div>
          <div id="publication161" class="publication01"><div 
  class="publication01Title">Dynamic correlation between intrahost HIV-1 quasispecies evolution and disease progression.</div><div 
  class="publication01Info">Lee HY, Perelson AS, Park S, Leitner T, PLoS Computational Biology 4, e1000240 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(161);">abstract <img id="abstractI161" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19079613&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP161" 
  class="publication01Abstract" style="display:none;">Quantifying the dynamics of intrahost HIV-1 sequence evolution is one means of uncovering information about the interaction between HIV-1 and the host immune system. In the chronic phase of infection, common dynamics of sequence divergence and diversity have been reported. We developed an HIV-1 sequence evolution model that simulated the effects of mutation and fitness of sequence variants. The amount of evolution was described by the distance from the founder strain, and fitness was described by the number of offspring a parent sequence produces. Analysis of the model suggested that the previously observed saturation of divergence and decrease of diversity in later stages of infection can be explained by a decrease in the proportion of offspring that are mutants as the distance from the founder strain increases rather than due to an increase of viral fitness. The prediction of the model was examined by performing phylogenetic analysis to estimate the change in the rate of evolution during infection. In agreement with our modeling, in 13 out of 15 patients (followed for 3-12 years) we found that the rate of intrahost HIV-1 evolution was not constant but rather slowed down at a rate correlated with the rate of CD4+ T-cell decline. The correlation between the dynamics of the evolutionary rate and the rate of CD4+ T-cell decline, coupled with our HIV-1 sequence evolution model, explains previously conflicting observations of the relationships between the rate of HIV-1 quasispecies evolution and disease progression.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(161);">hide</a></div></div></div><div id="publication162" class="publication01"><div 
  class="publication01Title">Inferring the history of speciation in house mice from autosomal, X-linked, Y-linked and mitochondrial genes.</div><div 
  class="publication01Info">Geraldes A, Basset P, Gibson B, Smith KL, Harr B, Yu H, Bulatova N, Ziv Y, Nachman MW, Molecular Ecology 17, 5349-5363 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(162);">abstract <img id="abstractI162" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19121002&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP162" 
  class="publication01Abstract" style="display:none;">Patterns of genetic differentiation among taxa at early stages of divergence provide an opportunity to make inferences about the history of speciation. Here, we conduct a survey of DNA-sequence polymorphism and divergence at loci on the autosomes, X chromosome, Y chromosome and mitochondrial DNA in samples of Mus domesticus, M. musculus and M. castaneus. We analyzed our data under a divergence with gene flow model and estimate that the effective population size of M. castaneus is 200,000-400,000, of M. domesticus is 100,000-200,000 and of M. musculus is 60,000-120,000. These data also suggest that these species started to diverge approximately 500,000 years ago. Consistent with this recent divergence, we observed considerable variation in the genealogical patterns among loci. For some loci, all alleles within each species formed a monophyletic group, while at other loci, species were intermingled on the phylogeny of alleles. This intermingling probably reflects both incomplete lineage sorting and gene flow after divergence. Likelihood ratio tests rejected a strict allopatric model with no gene flow in comparisons between each pair of species. Gene flow was asymmetric: no gene flow was detected into M. domesticus, while significant gene flow was detected into both M. castaneus and M. musculus. Finally, most of the gene flow occurred at autosomal loci, resulting in a significantly higher ratio of fixed differences to polymorphisms at the X and Y chromosomes relative to autosomes in some comparisons, or just the X chromosome in others, emphasizing the important role of the sex chromosomes in general and the X chromosome in particular in speciation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(162);">hide</a></div></div></div><div id="publication163" class="publication01"><div 
  class="publication01Title">Identification of selective sweeps in closely related populations of the house mouse based on microsatellite scans.</div><div 
  class="publication01Info">Teschke M, Mukabayire O, Wiehe T, Tautz D, Genetics 180, 1537-1545 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(163);">abstract <img id="abstractI163" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18791245&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP163" 
  class="publication01Abstract" style="display:none;">Genome scans of polymorphisms promise to provide insights into the patterns and frequencies of positive selection under natural conditions. The use of microsatellites as markers has the potential to focus on very recent events, since in contrast to SNPs, their high mutation rates should remove signatures of older events. We assess this concept here in a large-scale study. We have analyzed two population pairs of the house mouse, one pair of the subspecies Mus musculus domesticus and the other of M. m. musculus. A total of 915 microsatellite loci chosen to cover the whole genome were assessed in a prescreening procedure, followed by individual typing of candidate loci. Schlötterer&apos;s ratio statistics (lnRH) were applied to detect loci with significant deviations from patterns of neutral expectation. For eight loci from each population pair we have determined the size of the potential sweep window and applied a second statistical procedure (linked locus statistics). For the two population pairs, we find five and four significant sweep loci, respectively, with an average estimated window size of 120 kb. On the basis of the analysis of individual allele frequencies, it is possible to identify the most recent sweep, for which we estimate an onset of 400-600 years ago. Given the known population history for the French-German population pair, we infer that the average frequency of selective sweeps in these populations is higher than 1 in 100 generations across the whole genome. We discuss the implications for adaptation processes in natural populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(163);">hide</a></div></div></div><div id="publication164" class="publication01"><div 
  class="publication01Title">Regulatory interactions between IRG resistance GTPases in the cellular response to Toxoplasma gondii.</div><div 
  class="publication01Info">Hunn JP, Koenen-Waisman S, Papic N, Schroeder N, Pawlowski N, Lange R, Kaiser F, Zerrahn J, Martens S, Howard JC, The EMBO Journal 27, 2495-2509 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(164);">abstract <img id="abstractI164" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18772884&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP164" 
  class="publication01Abstract" style="display:none;">Members of the immunity-related GTPase (IRG) family are interferon-inducible resistance factors against a broad spectrum of intracellular pathogens including Toxoplasma gondii. The molecular mechanisms governing the function and regulation of the IRG resistance system are largely unknown. We find that IRG proteins function in a system of direct, nucleotide-dependent regulatory interactions between family members. After interferon induction but before infection, the three members of the GMS subfamily of IRG proteins, Irgm1, Irgm2 and Irgm3, which possess an atypical nucleotide-binding site, regulate the intracellular positioning of the conventional GKS subfamily members, Irga6 and Irgb6. Following infection, the normal accumulation of Irga6 protein at the parasitophorous vacuole membrane (PVM) is nucleotide dependent and also depends on the presence of all three GMS proteins. We present evidence that an essential role of the GMS proteins in this response is control of the nucleotide-bound state of the GKS proteins, preventing their GTP-dependent activation before infection. Accumulation of IRG proteins at the PVM has previously been shown to be associated with a block in pathogen replication: our results relate for the first time the enzymatic properties of IRG proteins to their role in pathogen resistance.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(164);">hide</a></div></div></div><div id="publication165" class="publication01"><div 
  class="publication01Title">Second-order moments of segregating sites under variable population size.</div><div 
  class="publication01Info">Zivković D, Wiehe T, Genetics 180, 341-357 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(165);">abstract <img id="abstractI165" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18716326&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP165" 
  class="publication01Abstract" style="display:none;">The identification of genomic regions that have been exposed to positive selection is a major challenge in population genetics. Since selective sweeps are expected to occur during environmental changes or when populations are colonizing a new habitat, statistical tests constructed on the assumption of constant population size are biased by the co-occurrence of population size changes and selection. To delimit this problem and gain better insights into demographic factors, theoretical results regarding the second-order moments of segregating sites, such as the variance of segregating sites, have been derived. Driven by emerging genomewide surveys, which allow the estimation of demographic parameters, a generalized version of Tajima&apos;s D has been derived that takes into account a previously estimated demographic scenario to test single loci for traces of selection against the null hypothesis of neutral evolution under variable population size.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(165);">hide</a></div></div></div><div id="publication166" class="publication01"><div 
  class="publication01Title">The role of the segmentation gene hairy in Tribolium.</div><div 
  class="publication01Info">Aranda M, Marques-Souza H, Bayer T, Tautz D, Development Genes and Evolution 218, 465-477 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(166);">abstract <img id="abstractI166" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18679713&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP166" 
  class="publication01Abstract" style="display:none;">Hairy stripes in Tribolium are generated during blastoderm and germ band extension, but a direct role for Tc-h in trunk segmentation was not found. We have studied here several aspects of hairy function and expression in Tribolium, to further elucidate its role. First, we show that there is no functional redundancy with other hairy paralogues in Tribolium. Second, we cloned the hairy orthologue from Tribolium confusum and show that its expression mimics that of Tribolium castaneum, implying that stripe expression should be functional in some way. Third, we show that the dynamics of stripe formation in the growth zone is not compatible with an oscillatory mechanism comparable to the one driving the expression of hairy homologues in vertebrates. Fourth, we use parental RNAi experiments to study Tc-h function and we find that mandible and labium are particularly sensitive to loss of Tc-h, reminiscent of a pair-rule function in the head region. In addition, lack of Tc-h leads to cell death in the gnathal region at later embryonic stages, resulting in a detachment of the head. Cell death patterns are also altered in the midline. Finally, we have analysed the effect of Tc-h knockdown on two of the target genes of hairy in Drosophila, namely fushi tarazu and paired. We find that the trunk expression of Tc-h is required to regulate Tc-ftz, although Tc-ftz is itself also not required for trunk segmentation in Tribolium. Our results imply that there is considerable divergence in hairy function between Tribolium and Drosophila.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(166);">hide</a></div></div></div><div id="publication167" class="publication01"><div 
  class="publication01Title">Energy-dependent fitness: a quantitative model for the evolution of yeast transcription factor binding sites.</div><div 
  class="publication01Info">Mustonen V, Kinney J, Callan CG, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 105, 12376-12381 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(167);">abstract <img id="abstractI167" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18723669&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP167" 
  class="publication01Abstract" style="display:none;">We present a genomewide cross-species analysis of regulation for broad-acting transcription factors in yeast. Our model for binding site evolution is founded on biophysics: the binding energy between transcription factor and site is a quantitative phenotype of regulatory function, and selection is given by a fitness landscape that depends on this phenotype. The model quantifies conservation, as well as loss and gain, of functional binding sites in a coherent way. Its predictions are supported by direct cross-species comparison between four yeast species. We find ubiquitous compensatory mutations within functional sites, such that the energy phenotype and the function of a site evolve in a significantly more constrained way than does its sequence. We also find evidence for substantial evolution of regulatory function involving point mutations as well as sequence insertions and deletions within binding sites. Genes lose their regulatory link to a given transcription factor at a rate similar to the neutral point mutation rate, from which we infer a moderate average fitness advantage of functional over nonfunctional sites. In a wider context, this study provides an example of inference of selection acting on a quantitative molecular trait.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(167);">hide</a></div></div></div><div id="publication168" class="publication01"><div 
  class="publication01Title">Structurally different alleles of the ath-MIR824 microRNA precursor are maintained at high frequency in Arabidopsis thaliana.</div><div 
  class="publication01Info">de Meaux J, Hu J, Tartler U, Goebel U, Proceedings Of The National Academy Of Sciences Of The United States Of America 105, 8994-8999 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(168);">abstract <img id="abstractI168" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18579782&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP168" 
  class="publication01Abstract" style="display:none;">In plants and animals, gene expression can be down-regulated at the posttranscriptional level by microRNAs (miRNAs), a class of small endogenous RNA. Comparative analysis of miRNA content across species indicates continuous birth and death of these loci in the course of evolution. However, little is known about the microevolutionary dynamics of these genetic elements, especially in plants. In this article we examine polymorphism at two miRNA-encoding loci in Arabidopsis thaliana, miR856 and miR824, which are not found in rice or poplar. We compare their diversity to other miRNA-encoding loci conserved across distant taxa. We find that levels of variation vary significantly across loci and that the two recently derived loci harbor patterns of diversity deviating from neutrality. miRNA miR856 shows a weak signature of a selective sweep whereas miR824 displays signs of balancing selection. A detailed examination of structural variation among alleles found at the miR824-encoding locus suggests nonrandom evolution of a thermoresistant substructure in the precursor. Expression analysis of pre-miR824 and its target, AGL16, indicates that these structural differences likely impact the processing of mature miR824. Our work highlights the relevance of RNA structure in precursor sequence evolution, suggesting that the evolutionary dynamics of miRNA-encoding loci is more complex than suggested by the constraints exerted on the interaction between mature miRNA fragments and their target exon.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(168);">hide</a></div></div></div><div id="publication169" class="publication01"><div 
  class="publication01Title">Chlamydia muridarum evades growth restriction by the IFN-gamma-inducible host resistance factor Irgb10.</div><div 
  class="publication01Info">Coers J, Bernstein-Hanley I, Grotsky D, Parvanova I, Howard JC, Taylor GA, Dietrich WF, Starnbach MN, Journal Of Immunology (Baltimore, Md. : 1950) 180, 6237-6245 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(169);">abstract <img id="abstractI169" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18424746&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP169" 
  class="publication01Abstract" style="display:none;">Chlamydiae are obligate intracellular bacterial pathogens that exhibit a broad range of host tropism. Differences in host tropism between Chlamydia species have been linked to host variations in IFN-gamma-mediated immune responses. In mouse cells, IFN-gamma can effectively restrict growth of the human pathogen Chlamydia trachomatis but fails to control growth of the closely related mouse pathogen Chlamydia muridarum. The ability of mouse cells to resist C. trachomatis replication is largely dependent on the induction of a family of IFN-gamma-inducible GTPases called immunity-related GTPases or IRGs. In this study we demonstrate that C. muridarum can specifically evade IRG-mediated host resistance. It has previously been suggested that C. muridarum inactivates the IRG protein Irga6 (Iigp1) to dampen the murine immune response. However, we show that Irga6 is dispensable for the control of C. trachomatis replication. Instead, an effective IFN-gamma response to C. trachomatis requires the IRG proteins Irgm1 (Lrg47), Irgm3 (Igtp), and Irgb10. Ectopic expression of Irgb10 in the absence of IFN-gamma is sufficient to reduce intracellular growth of C. trachomatis but fails to restrict growth of C. muridarum, indicating that C. muridarum can specifically evade Irgb10-driven host responses. Importantly, we find that Irgb10 protein intimately associates with inclusions harboring C. trachomatis but is absent from inclusions formed by C. muridarum. These data suggest that C. muridarum has evolved a mechanism to escape the murine IFN-gamma response by restricting access of Irgb10 and possibly other IRG proteins to the inclusion.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(169);">hide</a></div></div></div><div id="publication170" class="publication01"><div 
  class="publication01Title">The genome of the model beetle and pest Tribolium castaneum.</div><div 
  class="publication01Info">Tribolium Genome Sequencing Consortium, Richards S, Gibbs RA, Weinstock GM, Brown SJ, Denell R, Beeman RW, Gibbs R, Beeman RW, Brown SJ, Bucher G et al., Nature 452, 949-955 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(170);">abstract <img id="abstractI170" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18362917&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP170" 
  class="publication01Abstract" style="display:none;">Tribolium castaneum is a member of the most species-rich eukaryotic order, a powerful model organism for the study of generalized insect development, and an important pest of stored agricultural products. We describe its genome sequence here. This omnivorous beetle has evolved the ability to interact with a diverse chemical environment, as shown by large expansions in odorant and gustatory receptors, as well as P450 and other detoxification enzymes. Development in Tribolium is more representative of other insects than is Drosophila, a fact reflected in gene content and function. For example, Tribolium has retained more ancestral genes involved in cell-cell communication than Drosophila, some being expressed in the growth zone crucial for axial elongation in short-germ development. Systemic RNA interference in T. castaneum functions differently from that in Caenorhabditis elegans, but nevertheless offers similar power for the elucidation of gene function and identification of targets for selective insect control.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(170);">hide</a></div></div></div><div id="publication171" class="publication01"><div 
  class="publication01Title">Self-regulatory circuits in dorsoventral axis formation of the short-germ beetle Tribolium castaneum.</div><div 
  class="publication01Info">Nunes da Fonseca R, von Levetzow C, Kalscheuer P, Basal A, van der Zee M, Roth S, Developmental cell 14, 605-615 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(171);">abstract <img id="abstractI171" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18410735&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP171" 
  class="publication01Abstract" style="display:none;">The rel/NF-kappaB transcription factor Dorsal controls dorsoventral (DV) axis formation in Drosophila. A stable nuclear gradient of Dorsal directly regulates approximately 50 target genes. In Tribolium castaneum (Tc), a beetle with an ancestral type of embryogenesis, the Dorsal nuclear gradient is not stable, but rapidly shrinks and disappears. We find that negative feedback accounts for this dynamic behavior: Tc-Dorsal and one of its target genes activate transcription of the IkB homolog Tc-cactus, terminating Dorsal function. Despite its transient role, Tc-Dorsal is strictly required to initiate DV polarity, as in Drosophila. However, unlike in Drosophila, embryos lacking Tc-Dorsal display a periodic pattern of DV cell fates along the AP axis, indicating that a self-organizing ectodermal patterning system operates independently of mesoderm or maternal DV polarity cues. Our results also elucidate how extraembryonic tissues are organized in short-germ embryos, and how patterning information is transmitted from the early embryo to the growth zone.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(171);">hide</a></div></div></div><div id="publication172" class="publication01"><div 
  class="publication01Title">TGFbeta signaling in Tribolium: vertebrate-like components in a beetle.</div><div 
  class="publication01Info">van der Zee M, da Fonseca RN, Roth S, Development Genes and Evolution 218, 203-213 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(172);">abstract <img id="abstractI172" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18392881&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP172" 
  class="publication01Abstract" style="display:none;">The cytokines of the TGFbeta superfamily are highly conserved in evolution and elicit a diverse range of cellular responses in all metazoa. In Drosophila, the signaling pathways of the two TGFbeta subfamilies, Activins and Bone Morphogenetic Proteins (BMPs), have been well studied. To address the question of whether the findings from Drosophila are representative of insects in general, we analyzed the components of TGFbeta-signaling present in the genome of the beetle Tribolium castaneum. We were able to identify orthologs of the BMPs Decapentaplegic and Glass bottom boat, of the Activins Activinbeta and Dawdle, as well as orthologs of the less well-known ligands Myoglianin and Maverick, together with orthologs of all TGFbeta receptors and cytoplasmic signal transducers present in Drosophila. This indicates that the diversity of TGFbeta signaling components is generally well conserved between Drosophila and Tribolium. However, the genome of the beetle-and of the bee Apis mellifera-lacks an ortholog of the Drosophila BMP Screw but does contain a vertebrate-like BMP10 homolog which is not found in Drosophila. Concerning BMP inhibitors, Tribolium displays an even more vertebrate-like ensemble of components. We found two orthologs of the vertebrate DAN family, Dan and Gremlin, and show embryonic expression of a vertebrate-like BAMBI ortholog, all of which are absent in Drosophila. This suggests that Tribolium might have retained a more ancestral composition of TGFbeta signaling components and that TGFbeta signaling underwent considerable change in the Drosophila lineage. Tribolium is an excellent model to study the function of these ancestral signaling components in insects.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(172);">hide</a></div></div></div><div id="publication173" class="publication01"><div 
  class="publication01Title">Molecular evolution under fitness fluctuations.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Physical Review Letters 100, 108101 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(173);">abstract <img id="abstractI173" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18352233&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP173" 
  class="publication01Abstract" style="display:none;">Molecular evolution is a stochastic process governed by fitness, mutations, and reproductive fluctuations in a population. Here, we study evolution where fitness itself is stochastic, with random switches in the direction of selection at individual genomic loci. As the correlation time of these fluctuations becomes larger than the diffusion time of mutations within the population, fitness changes from an annealed to a quenched random variable. We show that the rate of evolution has its maximum in the crossover regime, where both time scales are comparable. Adaptive evolution emerges in the quenched fitness regime (evidence for such fitness fluctuations has recently been found in genomic data). The joint statistical theory of reproductive and fitness fluctuations establishes a conceptual connection between evolutionary genetics and statistical physics of disordered systems.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(173);">hide</a></div></div></div><div id="publication174" class="publication01"><div 
  class="publication01Title">Delimiting the conserved features of hunchback function for the trunk organization of insects.</div><div 
  class="publication01Info">Marques-Souza H, Aranda M, Tautz D, Development (Cambridge, England) 135, 881-888 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(174);">abstract <img id="abstractI174" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18216167&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP174" 
  class="publication01Abstract" style="display:none;">The gap gene hunchback in Drosophila acts during syncytial blastoderm stage via a short-range gradient and concentration-dependent activation or repression of target genes. Orthologues of hunchback can be easily found in other insects, but it has been unclear how well its functions are conserved. The segmentation process in most insect embryos occurs under cellular conditions, which should not allow the formation of diffusion-controlled transcription factor gradients. We have studied here in detail the function of hunchback in the short germ embryo of Tribolium using parental RNAi and interaction with possible target genes. We find that hunchback is a major regulator of the trunk gap genes and Hox genes in Tribolium, but may only indirectly be required to regulate other segmentation genes. The core function of hunchback appears to be the setting of the Ultrabithorax expression border via a repression effect, and the activation of the Krüppel expression domain. These regulatory effects are likely to be direct and are conserved between Drosophila and Tribolium. We find no evidence for a classical gap phenotype in the form of loss of segments in the region of expression of hunchback. However, the phenotypic effects in Tribolium are highly comparable with those found for other short germ embryos, i.e. the core functions of hunchback in Tribolium appear to be the same in these other insects, although they are evolutionarily more distant to Tribolium, than Tribolium is to Drosophila. These results allow the disentanglement of the conserved role of hunchback in insects from the derived features that have been acquired in the lineage towards Drosophila. Given that the gap phenotype appears to occur only in long germ embryos and that the main role of hunchback appears to be the regionalization of the embryo, it may be appropriate to revive an alternative name for the class of gap genes, namely &apos;cardinal genes&apos;.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(174);">hide</a></div></div></div><div id="publication175" class="publication01"><div 
  class="publication01Title">Integration of horizontally transferred genes into regulatory interaction networks takes many million years.</div><div 
  class="publication01Info">Lercher MJ, Pal C, Molecular Biology And Evolution 25, 559-567 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(175);">abstract <img id="abstractI175" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 18158322&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP175" 
  class="publication01Abstract" style="display:none;">Adaptation of bacteria to new or changing environments is often associated with the uptake of foreign genes through horizontal gene transfer. However, it has remained unclear how (and how fast) new genes are integrated into their host&apos;s cellular networks. Combining the regulatory and protein interaction networks of Escherichia coli with comparative genomics tools, we provide the first systematic analysis of this issue. Genes transferred recently have fewer interaction partners compared to nontransferred genes in both regulatory and protein interaction networks. Thus, horizontally transferred genes involved in complex regulatory and protein-protein interactions are rarely favored by selection. Only few protein-protein interactions are gained after the initial integration of genes following the transfer event. In contrast, transferred genes are gradually integrated into the regulatory network of their host over evolutionary time. During adaptation to the host cellular environment, horizontally transferred genes recruit existing transcription factors of the host, reflected in the fast evolutionary rates of the cis-regulatory regions of transferred genes. Further, genes resulting from increasingly ancient transfer events show increasing numbers of transcriptional regulators as well as improved coregulation with interacting proteins. Fine-tuned integration of horizontally transferred genes into the regulatory network spans more than 8-22 million years and encompasses accelerated evolution of regulatory regions, stabilization of protein-protein interactions, and changes in codon usage.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(175);">hide</a></div></div></div><div id="publication176" class="publication01"><div 
  class="publication01Title">Plant development revolves around axes.</div><div 
  class="publication01Info">Chandler J, Nardmann J, Werr W, Trends In Plant Science 13, 78-84 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(176);">abstract <img id="abstractI176" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18262821&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP176" 
  class="publication01Abstract" style="display:none;">Arabidopsis thaliana has become a paradigm for dicot embryo development, despite its embryology being non-representative of dicots in general. The recent cloning of heterologous genes involved in embryonic development from maize and construction of robust phylogenies has shed light on the conservation of transcription factor function and now facilitates a comparison of maize and Arabidopsis embryogenesis orthology. In this review, we focus on a comparison of expression domains of WUSCHEL HOMEOBOX LIKE (WOX), SHOOTMERISTEMLESS (STM), DORNROESCHEN (DRN) and CUP-SHAPED COTYLEDON (CUC) genes and their role in axialization in both species, showing that despite significantly divergent modes of embryogenesis, most notably in terms of axes and planes of symmetry, there is considerable conservation of function as well as notable differences between maize and Arabidopsis.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(176);">hide</a></div></div></div><div id="publication177" class="publication01"><div 
  class="publication01Title">Genome-wide analysis indicates more Asian than Melanesian ancestry of Polynesians.</div><div 
  class="publication01Info">Kayser M, Lao O, Saar K, Brauer S, Wang X, Nürnberg P, Trent RJ, Stoneking M, American Journal Of Human Genetics 82, 194-198 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(177);">abstract <img id="abstractI177" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18179899&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP177" 
  class="publication01Abstract" style="display:none;">Analyses of mitochondrial DNA (mtDNA) and nonrecombining Y chromosome (NRY) variation in the same populations are sometimes concordant but sometimes discordant. Perhaps the most dramatic example known of the latter concerns Polynesians, in which about 94% of Polynesian mtDNAs are of East Asian origin, while about 66% of Polynesian Y chromosomes are of Melanesian origin. Here we analyze on a genome-wide scale, to our knowledge for the first time, the origins of the autosomal gene pool of Polynesians by screening 377 autosomal short tandem repeat (STR) loci in 47 Pacific Islanders and compare the results with those obtained from 44 Chinese and 24 individuals from Papua New Guinea. Our data indicate that on average about 79% of the Polynesian autosomal gene pool is of East Asian origin and 21% is of Melanesian origin. The genetic data thus suggest a dual origin of Polynesians with a high East Asian but also considerable Melanesian component, reflecting sex-biased admixture in Polynesian history in agreement with the Slow Boat model. More generally, these results also demonstrate that conclusions based solely on uniparental markers, which are frequently used in population history studies, may not accurately reflect the history of the autosomal gene pool of a population.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(177);">hide</a></div></div></div><div id="publication178" class="publication01"><div 
  class="publication01Title">Evolution in random fitness landscapes: the infinite sites model</div><div 
  class="publication01Info">Park S, Krug J, J. Stat. Mech. P04014 doi:10.1088/1742-5468/2008/04/P04014 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(178);">abstract <img id="abstractI178" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1742-5468/2008/04/P04014">paper [pdf]</a> &nbsp; </div><div id="abstractP178" 
  class="publication01Abstract" style="display:none;">We consider the evolution of an asexually reproducing population in an uncorrelated random fitness landscape in the limit of infinite genome size, which implies that each mutation generates a new fitness value drawn from a probability distribution g(w). This is the finite population version of Kingman's house of cards model (Kingman 1978 J. Appl. Probab. 15 1). In contrast to Kingman's work, the focus here is on unbounded distributions g(w) which lead to an indefinite growth of the population fitness. The model is solved analytically in the limit of infinite population size  and simulated numerically for finite N. When the genome-wide mutation probability U is small, the long-time behavior of the model reduces to a point process of fixation events, which is referred to as a diluted record process (DRP). The DRP is similar to the standard record process except that a new record candidate (a number that exceeds all previous entries in the sequence) is accepted only with a certain probability that depends on the values of the current record and the candidate. We develop a systematic analytic approximation scheme for the DRP. At finite U the fitness frequency distribution of the population decomposes into a stationary part due to mutations and a traveling wave component due to selection, which is shown to imply a reduction of the mean fitness by a factor of 1−U compared to the  limit.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(178);">hide</a></div></div></div><div id="publication179" class="publication01"><div 
  class="publication01Title">From protein interactions to functional annotation: graph alignment in Herpes.</div><div 
  class="publication01Info">Kolár M, Lässig M, Berg J, BMC Systems Biology 2, 90 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(179);">abstract <img id="abstractI179" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18957106&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP179" 
  class="publication01Abstract" style="display:none;">BACKGROUND:Sequence alignment is a prolific basis of functional annotation, but remains a challenging problem in the &apos;twilight zone&apos; of high sequence divergence or short gene length. Here we demonstrate how information on gene interactions can help to resolve ambiguous sequence alignments. We compare two distant Herpes viruses by constructing a graph alignment, which is based jointly on the similarity of their protein interaction networks and on sequence similarity. This hybrid method provides functional associations between proteins of the two organisms that cannot be obtained from sequence or interaction data alone.RESULTS:We find proteins where interaction similarity and sequence similarity are individually weak, but together provide significant evidence of orthology. There are also proteins with high interaction similarity but without any detectable sequence similarity, providing evidence of functional association beyond sequence homology. The functional predictions derived from our alignment are consistent with genomic position and gene expression data.CONCLUSION:Our approach shows that evolutionary conservation is a powerful filter to make protein interaction data informative about functional similarities between the interacting proteins, and it establishes graph alignment as a powerful tool for the comparative analysis of data from highly diverged species.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(179);">hide</a></div></div></div><div id="publication180" class="publication01"><div 
  class="publication01Title">Many roads lead to Rome: different ways to construct a nematode</div><div 
  class="publication01Info">Schierenberg E, Schulze J, Evolving Pathways: Key Themes In Evolutionary Developmental Biology. Cambridge University Press, Cambridge, 261-280 (2008)</div><div 
  class="publication01Links"><a href="http://www.sfb680.uni-koeln.de/docs/A6Schierenberg08SbgSchulzeCAB.pdf">paper [pdf]</a> &nbsp; </div></div><div id="publication181" class="publication01"><div 
  class="publication01Title">The IRG proteins: a function in search of a mechanism.</div><div 
  class="publication01Info">Howard J, Immunobiology 213, 367-375 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(181);">abstract <img id="abstractI181" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18406381&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP181" 
  class="publication01Abstract" style="display:none;">The IRG proteins (p47 GTPases) constitute one of the strongest resistance systems known to be active against intracellular pathogens in mice. The proteins are induced by interferons and assemble on phagosomes and parasitophorous vacuoles of a number of different micro-organisms in all cell types assayed. There are presently three experimentally based views as to how they exert their cell-autonomous activity against intracellular pathogens: blocking of interferon-mediated acceleration of phagosome maturation, induction of autophagic membranes, and direct destruction of the parasitophorous vacuole membrane. Failure of hemopoietic stem cells during infection is associated with targeted deletion of one IRG protein, Irgm1. The significance of this non-cell-autonomous phenotype is discussed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(181);">hide</a></div></div></div>
          </div>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2007</div>
          <div id="publication182" class="publication01"><div 
  class="publication01Title">The evolution of plant regulatory networks: what Arabidopsis cannot say for itself.</div><div 
  class="publication01Info">Nardmann J, Werr W, Current Opinion In Plant Biology 10, 653-659 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(182);">abstract <img id="abstractI182" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17720614&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP182" 
  class="publication01Abstract" style="display:none;">Genetic and molecular analyses in the dicot model plant Arabidopsis thaliana have begun to shed some light on regulatory networks in plants. However, comparisons with other species are necessary to validate networks identified in model species on the evolutionary scale. Many key regulatory proteins are encoded by members of transcription factor gene families. Orthologous genes can be identified by phylogenetic reconstructions based on conserved protein domains and functionally substantiated by gene expression patterns and mutant analyses. Recent comparative analyses of different pathways involved in shoot meristem development reveal not only conservation from basal land plants to angiosperms but also evolutionary freedom for significant adaptations in the course of plant speciation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(182);">hide</a></div></div></div><div id="publication183" class="publication01"><div 
  class="publication01Title">Clonal interference in large populations.</div><div 
  class="publication01Info">Park S, Krug J, Proceedings Of The National Academy Of Sciences Of The United States Of America 104, 18135-18140 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(183);">abstract <img id="abstractI183" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17984061&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP183" 
  class="publication01Abstract" style="display:none;">Clonal interference, the competition between lineages arising from different beneficial mutations in an asexually reproducing population, is an important factor determining the tempo and mode of microbial adaptation. The standard theory of this phenomenon neglects the occurrence of multiple mutations as well as the correlation between loss by genetic drift and clonal competition, which is questionable in large populations. Working within the Wright-Fisher model with multiplicative fitness (no epistasis), we determine the rate of adaptation asymptotically for very large population sizes and show that the standard theory fails in this regime. Our study also explains the success of the standard theory in predicting the rate of adaptation for moderately large populations. Furthermore, we show that the nature of the substitution process changes qualitatively when multiple mutations are allowed for, because several mutations can be fixed in a single fixation event. As a consequence, the index of dispersion for counts of the fixation process displays a minimum as a function of population size, whereas the origination process of fixed mutations becomes completely regular for very large populations. We find that the number of mutations fixed in a single event is geometrically distributed as in the neutral case. These conclusions are based on extensive simulations combined with analytic results for the limit of infinite population size.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(183);">hide</a></div></div></div><div id="publication184" class="publication01"><div 
  class="publication01Title">Cell-autonomous immunity to Toxoplasma gondii in mouse and man.</div><div 
  class="publication01Info">Könen-Waisman S, Howard JC, Microbes And Infection 9, 1652-1661 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(184);">abstract <img id="abstractI184" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18024121&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP184" 
  class="publication01Abstract" style="display:none;">The protozoan, Toxoplasma gondii, is a natural pathogen of mouse and a zoonosis of man. Immunity against the pathogen is largely mediated by interferon-stimulated cell-autonomous mechanisms that are strikingly different between man and mouse. There are many poorly understood host and pathogen variables that affect the outcome of infection.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(184);">hide</a></div></div></div><div id="publication185" class="publication01"><div 
  class="publication01Title">Extensive additivity of gene expression differentiates subspecies of the house mouse.</div><div 
  class="publication01Info">Rottscheidt R, Harr B, Genetics 177, 1553-1567 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(185);">abstract <img id="abstractI185" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17947435&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP185" 
  class="publication01Abstract" style="display:none;">We have studied different subspecies of the house mouse and their reciprocal F(1) hybrids to estimate the within-locus mode of inheritance for subspecies differences in gene expression in three tissues (brain, liver, and testis) of male mice. This study investigates the mode of inheritance in crosses at a larger taxonomic distance than has been previously systematically investigated. We found the vast majority of transcripts to be additively expressed with only a few transcripts showing dominance or overdominance in expression, except for one direction of one cross, which showed large mis-expression in the testis. We suggest that, as time passes, more genes come to influence expression, and if there is no directional dominance, additivity becomes increasingly more likely, up to a threshold beyond which there is F(1) hybrid breakdown. Some previous studies on different organisms have found a large degree of dominance, commonly at shorter taxonomic differences. We surveyed these findings and show that the most consistent association exists between the amount of additivity detected in a study and the expression analysis method (in particular microarray platform), suggesting that at least some of the differences among studies might be methodological. Most studies agree with ours in that within-locus additivity seems to be general mode of inheritance for transcript expression. Differentially expressed transcripts identified in our screen among subspecies of house mice are candidate genes that could be involved in reproductive isolation between these subspecies.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(185);">hide</a></div></div></div><div id="publication186" class="publication01"><div 
  class="publication01Title">WOX gene phylogeny in Poaceae: a comparative approach addressing leaf and embryo development.</div><div 
  class="publication01Info">Nardmann J, Zimmermann R, Durantini D, Kranz E, Werr W, Molecular Biology And Evolution 24, 2474-2484 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(186);">abstract <img id="abstractI186" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17768306&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP186" 
  class="publication01Abstract" style="display:none;">The phylogeny based on the homeodomain (HD) amino acid sequence of the WOX (WUSCHEL-related homeobox gene family) was established in the 3 major radiations of the Poaceae family: Pooideae (Brachypodium distachyon), Bambusoideae (Oryza sativa), and Panicoideae (Zea mays). The genomes of all 3 grasses contain an ancient duplication in the WOX3 branch, and the cellular expression patterns in maize and rice indicate subfunctionalization of paralogues during leaf development, which may relate to the architecture of the grass leaf and the encircling of the stem. The use of maize WOX gene family members as molecular markers in maize embryo development for the first time allowed us to visualize cellular decisions in the maize proembryo, including specification of the shoot/root axis at an oblique angle to the apical-basal polarity of the zygote. All molecular marker data are compatible with the conclusion that the embryonic shoot/root axis comprises a discrete domain from early proembryo stages onward. Novel cell fates of the shoot and the root are acquired within this distinct morphogenic axis domain, which elongates and thus separates the shoot apical meristem and root apical meristem (RAM) anlagen in the maize embryo.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(186);">hide</a></div></div></div><div id="publication187" class="publication01"><div 
  class="publication01Title">The MADS-domain protein PPM2 preferentially occurs in gametangia and sporophytes of the moss Physcomitrella patens.</div><div 
  class="publication01Info">Quodt V, Faigl W, Saedler H, Münster T, Gene 400, 25-34 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(187);">abstract <img id="abstractI187" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17614216&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP187" 
  class="publication01Abstract" style="display:none;">To date, the function of MADS-domain transcription factors in non-seed plants remains largely elusive, although a number of genes have been isolated and characterized from a variety of species. In our study we analyzed PPM2, a classical MIKC-type MADS-box gene from the moss Physcomitrella patens, taking advantage of the unique technical properties Physcomitrella offers in terms of efficient homologous recombination. We determined mRNA and protein distribution and performed targeted disruption of the genomic locus for functional analysis of PPM2. Despite weak ubiquitous expression, PPM2 protein is mostly found in male and female gametangia and basal parts of developing sporophytes. Therefore, PPM2 seems to function in both the haploid and the diploid phase of the moss life cycle. This situation reflects an evolutionary transition state of gene recruitment from an ancestral gametophytic generation into a derived sporophytic generation which became dominating in tracheophytes. However, a knock-out of the PPM2 gene did not cause visible phenotypical changes in the respective structures. The implications of our findings for the understanding of the evolutionary history of MADS-box transcription factors in plants are discussed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(187);">hide</a></div></div></div><div id="publication188" class="publication01"><div 
  class="publication01Title">Evolution of immunity and pathogens.</div><div 
  class="publication01Info">Howard JC, Jack RS, European Journal Of Immunology 37, 1721-1723 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(188);">abstract <img id="abstractI188" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17587195&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP188" 
  class="publication01Abstract" style="display:none;">The Cologne Spring Meeting of the Institute of Genetics focuses on a different topic each year. In Spring 2007, the meeting focused on the complex evolutionary relationships between hosts and pathogens.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(188);">hide</a></div></div></div><div id="publication189" class="publication01"><div 
  class="publication01Title">Identification of a candidate genetic variant for the high prevalence of type II diabetes in Polynesians.</div><div 
  class="publication01Info">Myles S, Hradetzky E, Engelken J, Lao O, Nürnberg P, Trent RJ, Wang X, Kayser M, Stoneking M, European Journal Of Human Genetics 15, 584-589 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(189);">abstract <img id="abstractI189" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17327880&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP189" 
  class="publication01Abstract" style="display:none;">The prevalence of non-insulin-dependent diabetes mellitus (type II diabetes) in Polynesia is among the highest recorded worldwide and is substantially higher than in neighboring human populations. Such large differences in the frequency of a phenotype between populations may be explained by large allele frequency differences between populations in genes associated with the phenotype. To identify genes that may explain the high between-population variation in type II diabetes prevalence in the Pacific, we determined the frequency of 10 type II diabetes-associated alleles in 23 Polynesians, 23 highland New Guineans and 19 Han Chinese, calculated population-pairwise Fst values for each allele and compared these values to the distribution of Fst values from approximately 100,000 SNPs from the same populations. The susceptibility allele in the PPARGC1A gene is at a frequency of 0.717 in Polynesians, 0.368 in Chinese but is absent in the New Guineans. The striking frequency difference between Polynesians and New Guineans is highly unusual (Fst=0.703, P=0.007) and we therefore suggest that this allele may play a role in the large difference in type II diabetes prevalence between Polynesians and neighboring populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(189);">hide</a></div></div></div><div id="publication190" class="publication01"><div 
  class="publication01Title">A pooling approach to detect signatures of selective sweeps in genome scans using microsatellites</div><div 
  class="publication01Info">Thomas M, Möller F, Wiehe T, Tautz D, Molecular Ecology Notes 7, 400-403 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(190);">abstract <img id="abstractI190" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2007.01697.x/abstract">paper [pdf]</a> &nbsp; </div><div id="abstractP190" 
  class="publication01Abstract" style="display:none;">We have evaluated a pooling approach that can reduce the number of polymerase chain reactions in a screen for selective sweeps by more than an order of magnitude. We show that the complex peak pattern that results from pooling of all samples from a given population is a faithful reflection of the composite pattern of the individual alleles, although with an under-representation of the larger alleles. Candidate loci for selective sweeps can be identified by visual inspection of the pool patterns. We have also implemented a software tool, which can find suitable microsatellite loci in the vicinity of annotated genes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(190);">hide</a></div></div></div><div id="publication191" class="publication01"><div 
  class="publication01Title">Reduced X-linked diversity in derived populations of house mice.</div><div 
  class="publication01Info">Baines JF, Harr B, Genetics 175, 1911-1921 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(191);">abstract <img id="abstractI191" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17287527&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP191" 
  class="publication01Abstract" style="display:none;">Contrasting patterns of X-linked vs. autosomal diversity may be indicative of the mode of selection operating in natural populations. A number of observations have shown reduced X-linked (or Z-linked) diversity relative to autosomal diversity in various organisms, suggesting a large impact of genetic hitchhiking. However, the relative contribution of other forces such as population bottlenecks, variation in reproductive success of the two sexes, and differential introgression remains unclear. Here, we survey 13 loci, 6 X-linked and 7 autosomal, in natural populations of the house mouse (Mus musculus) subspecies complex. We studied seven populations of three different subspecies, the eastern house mouse M. musculus castaneus, the central house mouse M. m. musculus, and the western house mouse M. m. domesticus, including putatively ancestral and derived populations for each. All populations display lower diversity on the X chromosomes relative to autosomes, and this effect is most pronounced in derived populations. To assess the role of demography, we fit the demographic parameters that gave the highest likelihood of the data using coalescent simulations. We find that the reduction in X-linked diversity is too large to be explained by a simple demographic model in at least two of four derived populations. These observations are also not likely to be explained by differences in reproductive success between males and females. They are consistent with a greater impact of positive selection on the X chromosome, and this is supported by the observation of an elevated K(A) and elevated K(A)/K(S) ratios on the rodent X chromosome. A second contribution may be that the X chromosome less readily introgresses across subspecies boundaries.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(191);">hide</a></div></div></div><div id="publication192" class="publication01"><div 
  class="publication01Title">Deterministic and stochastic regimes of asexual evolution on rugged fitness landscapes.</div><div 
  class="publication01Info">Jain K, Krug J, Genetics 175, 1275-1288 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(192);">abstract <img id="abstractI192" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17179085&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP192" 
  class="publication01Abstract" style="display:none;">We study the adaptation dynamics of an initially maladapted asexual population with genotypes represented by binary sequences of length L. The population evolves in a maximally rugged fitness landscape with a large number of local optima. We find that whether the evolutionary trajectory is deterministic or stochastic depends on the effective mutational distance d(eff) up to which the population can spread in genotype space. For d(eff) = L, the deterministic quasi-species theory operates while for d(eff) &lt; 1, the evolution is completely stochastic. Between these two limiting cases, the dynamics are described by a local quasi-species theory below a crossover time T(x) while above T(x) the population gets trapped at a local fitness peak and manages to find a better peak via either stochastic tunneling or double mutations. In the stochastic regime d(eff) &lt; 1, we identify two subregimes associated with clonal interference and uphill adaptive walks, respectively. We argue that our findings are relevant to the interpretation of evolution experiments with microbial populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(192);">hide</a></div></div></div><div id="publication193" class="publication01"><div 
  class="publication01Title">Adaptations to fluctuating selection in Drosophila.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 104, 2277-2282 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(193);">abstract <img id="abstractI193" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17287357&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP193" 
  class="publication01Abstract" style="display:none;">Time-dependent selection causes the adaptive evolution of new phenotypes, and this dynamics can be traced in genomic data. We have analyzed polymorphisms and substitutions in Drosophila, using a more sensitive inference method for adaptations than the standard population-genetic tests. We find evidence that selection itself is strongly time-dependent, with changes occurring at nearly the rate of neutral evolution. At the same time, higher than previously estimated levels of selection make adaptive responses by a factor 10-100 faster than the pace of selection changes, ensuring that adaptations are an efficient mode of evolution under time-dependent selection. The rate of selection changes is faster in noncoding DNA, i.e., the inference of functional elements can less be based on sequence conservation than for proteins. Our results suggest that selection acts not only as a constraint but as a major driving force of genomic change.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(193);">hide</a></div></div></div><div id="publication194" class="publication01"><div 
  class="publication01Title">Contrasting evolution of expression differences in the testis between species and subspecies of the house mouse.</div><div 
  class="publication01Info">Voolstra C, Tautz D, Farbrother P, Eichinger L, Harr B, Genome Research 17, 42-49 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(194);">abstract <img id="abstractI194" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17038563&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP194" 
  class="publication01Abstract" style="display:none;">Regulatory changes in genes involved in reproduction are thought to be prime targets for divergence during speciation, since they are expected to play an important role in sexual selection and sexual conflict. We used microarray analysis of RNA from different wild populations of house mouse subspecies (including Mus m. musculus, Mus m. domesticus, and Mus m. castaneus) and from the sister species Mus spretus to test this assumption. A comparison of expression divergence in brain, liver/kidney, and testis shows a major difference in the evolutionary dynamics of testis-related genes. While the comparison between species confirms an excess in divergence in testis genes, we find that all comparisons between subspecies yield only a very small number of genes with significantly different expression levels in the testis. These results suggest that the early phase of the speciation process may not be driven by regulatory changes in genes that are potential targets of sexual selection, and that the divergence in these genes is only established during a later phase of the speciation process.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(194);">hide</a></div></div></div><div id="publication195" class="publication01"><div 
  class="publication01Title">Identification of selective sweeps using a dynamically adjusted number of linked microsatellites.</div><div 
  class="publication01Info">Wiehe T, Nolte V, Zivković D, Schlötterer C, Genetics 175, 207-218 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(195);">abstract <img id="abstractI195" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17057237&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP195" 
  class="publication01Abstract" style="display:none;">There is currently large interest in distinguishing the signatures of genetic variation produced by demographic events from those produced by natural selection. We propose a simple multilocus statistical test to identify candidate sites of selective sweeps with high power. The test is based on the variability profile measured in an array of linked microsatellites. We also show that the analysis of flanking markers drastically reduces the number of false positives among the candidates that are identified in a genomewide survey of unlinked loci and find that this property is maintained in many population-bottleneck scenarios. However, for a certain range of intermediately severe population bottlenecks we find genomic signatures that are very similar to those produced by a selective sweep. While in these worst-case scenarios the power of the proposed test remains high, the false-positive rate reaches values close to 50%. Hence, selective sweeps may be hard to identify even if multiple linked loci are analyzed. Nevertheless, the integration of information from multiple linked loci always leads to a considerable reduction of the false-positive rate compared to a genome scan of unlinked loci. We discuss the application of this test to experimental data from Drosophila melanogaster.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(195);">hide</a></div></div></div><div id="publication196" class="publication01"><div 
  class="publication01Title">MIKC* MADS-protein complexes bind motifs enriched in the proximal region of late pollen-specific Arabidopsis promoters.</div><div 
  class="publication01Info">Verelst W, Saedler H, Münster T, Plant Physiology 143, 447-460 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(196);">abstract <img id="abstractI196" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17071640&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP196" 
  class="publication01Abstract" style="display:none;">The genome of Arabidopsis (Arabidopsis thaliana) encodes over 100 MADS-domain transcription factors, categorized into five phylogenetic subgroups. Most research efforts have focused on just one of these subgroups (MIKC(c)), whereas the other four remain largely unexplored. Here, we report on five members of the so-called Mdelta or Arabidopsis MIKC* (AtMIKC*) subgroup, which are predominantly expressed during the late stages of pollen development. Very few MADS-box genes function in mature pollen, and from this perspective, the AtMIKC* genes are therefore highly exceptional. We found that the AtMIKC* proteins are able to form multiple heterodimeric complexes in planta, and that these protein complexes exhibit a for the MADS-family unusual and high DNA binding specificity in vitro. Compared to their occurrence in promoters genome wide, AtMIKC* binding sites are strongly overrepresented in the proximal region of late pollen-specific promoters. By combining our experimental data with in silico genomics and pollen transcriptomics approaches, we identified a considerable number of putative direct target genes of the AtMIKC* transcription factor complexes in pollen, many of which have known or proposed functions in pollen tube growth. The expression of several of these predicted targets is altered in mutant pollen in which all AtMIKC* complexes are affected, and in vitro germination of this mutant pollen is severely impaired. Our data therefore suggest that the AtMIKC* protein complexes play an essential role in transcriptional regulation during late pollen development.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(196);">hide</a></div></div></div><div id="publication197" class="publication01"><div 
  class="publication01Title">Transcription of the putative maize orthologue of the Arabidopsis DORNROSCHEN gene marks early asymmetry in the proembryo and during leaf initiation in the shoot apical meristem.</div><div 
  class="publication01Info">Zimmermann R, Werr W, Gene Expression Patterns 7, 158-164 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(197);">abstract <img id="abstractI197" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16880009&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP197" 
  class="publication01Abstract" style="display:none;">A potential orthologue of the Arabidopsis DORNROSCHEN (DRN) gene was isolated from maize based on phylogeny and expression patterns. ZmDRN transcription provides a new marker for embryonic patterning and cellular differentiation in the shoot apical meristem. In contrast to DRN expression in the 2-4-cell Arabidopsis embryo, transcription of the maize orthologue is activated only in the late proembryo stage where expression, however, marks the prospective scutellum domain such as DRN transcription prepatterns cotyledon development in the Arabidopsis globular embryo. The scutellum is commonly considered to be the grass specific organ, which is homologous to the pair of cotyledons in dicots. Such as in Arabidopsis, ZmDRN transcriptional activity is linked to the anlagen of new lateral organs in the maize apex. Striking with respect to the timing of cellular decisions during leaf initiation is asymmetry established between adjacent cells at the very tip of the shoot apical meristem.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(197);">hide</a></div></div></div><div id="publication198" class="publication01"><div 
  class="publication01Title">From biophysics to evolutionary genetics: statistical aspects of gene regulation.</div><div 
  class="publication01Info">Lässig M, BMC Bioinformatics 8 Suppl 6, S7 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(198);">abstract <img id="abstractI198" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17903288&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP198" 
  class="publication01Abstract" style="display:none;">This is an introductory review on how genes interact to produce biological functions. Transcriptional interactions involve the binding of proteins to regulatory DNA. Specific binding sites can be identified by genomic analysis, and these undergo a stochastic evolution process governed by selection, mutations, and genetic drift. We focus on the links between the biophysical function and the evolution of regulatory elements. In particular, we infer fitness landscapes of binding sites from genomic data, leading to a quantitative evolutionary picture of regulation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(198);">hide</a></div></div></div><div id="publication199" class="publication01"><div 
  class="publication01Title">MADS-complexes regulate transcriptome dynamics during pollen maturation.</div><div 
  class="publication01Info">Verelst W, Twell D, de Folter S, Immink R, Saedler H, Münster T, Genome Biology 8, R249 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(199);">abstract <img id="abstractI199" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18034896&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP199" 
  class="publication01Abstract" style="display:none;">BACKGROUND:Differentiation processes are responsible for the diversity and functional specialization of the cell types that compose an organism. The outcome of these processes can be studied at molecular, physiologic, and biochemical levels by comparing different cell types, but the complexity and dynamics of the regulatory processes that specify the differentiation are largely unexplored.RESULTS:Here we identified the pollen-specific MIKC* class of MADS-domain transcription factors as major regulators of transcriptome dynamics during male reproductive cell development in Arabidopsis thaliana. Pollen transcript profiling of mutants deficient in different MIKC* protein complexes revealed that they control a transcriptional switch that directs pollen maturation and that is essential for pollen competitive ability. We resolved the functional redundancy among the MIKC* proteins and uncovered part of the underlying network by identifying the non-MIKC* MADS-box genes AGL18 and AGL29 as downstream regulators of a subset of the MIKC* MADS-controlled genes.CONCLUSION:Our results provide a first, unique, and compelling insight into the complexity of a transcription factor network that directs cellular differentiation during pollen maturation, a process that is essential for male reproductive fitness in flowering plants.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(199);">hide</a></div></div></div>
          </div>
          <div 
            class="eventGBox01">&nbsp;</div><div class="eventBox02"><div class="eventTitle01">2006</div>
          <div id="publication200" class="publication01"><div 
  class="publication01Title">The shoot stem cell niche in angiosperms: expression patterns of WUS orthologues in rice and maize imply major modifications in the course of mono- and dicot evolution.</div><div 
  class="publication01Info">Nardmann J, Werr W, Molecular Biology And Evolution 23, 2492-2504 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(200);">abstract <img id="abstractI200" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16987950&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP200" 
  class="publication01Abstract" style="display:none;">In Arabidopsis, stem cell homeostasis in the shoot apical meristem (SAM) is controlled by a feedback loop between WUS and CLV functions. We have identified WUS orthologues in maize and rice by a detailed phylogenetic analysis of the WOX gene family and subsequent cloning. A single WUS orthologue is present in the rice genome (OsWUS), whereas the allotetraploid maize genome contains 2 WUS paralogues (ZmWUS1 and ZmWUS2). None of the isolated grass WUS orthologues displays an organizing center-type expression pattern in the vegetative SAM as in Arabidopsis. In contrast, the grass-specific expression patterns relate to the specification of new phytomers consistent with the transcriptional expression patterns of TD1 and FON1 (CLV1 orthologues of maize and rice, respectively). Moreover, the grass WUS and CLV1 orthologues are coexpressed in all reproductive meristems, where fasciation and supernumerary floral organs occur in td1 or fon1 loss-of-function mutants. The expression patterns of WUS orthologues in both grass species compared with those of dicots imply that major changes in WUS function, which are correlated with changes in CLV1 signaling, have occurred during angiosperm evolution and raise doubts about the uniqueness of the WUS/CLV antagonism in the maintenance of the shoot stem cell niche in grasses.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(200);">hide</a></div></div></div><div id="publication201" class="publication01"><div 
  class="publication01Title">Cloning of DOG1, a quantitative trait locus controlling seed dormancy in Arabidopsis.</div><div 
  class="publication01Info">Bentsink L, Jowett J, Hanhart CJ, Koornneef M, Proceedings Of The National Academy Of Sciences Of The United States Of America 103, 17042-17047 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(201);">abstract <img id="abstractI201" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17065317&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP201" 
  class="publication01Abstract" style="display:none;">Genetic variation for seed dormancy in nature is a typical quantitative trait controlled by multiple loci on which environmental factors have a strong effect. Finding the genes underlying dormancy quantitative trait loci is a major scientific challenge, which also has relevance for agriculture and ecology. In this study we describe the identification of the DELAY OF GERMINATION 1 (DOG1) gene previously identified as a quantitative trait locus involved in the control of seed dormancy. This gene was isolated by a combination of positional cloning and mutant analysis and is absolutely required for the induction of seed dormancy. DOG1 is a member of a small gene family of unknown molecular function, with five members in Arabidopsis. The functional natural allelic variation present in Arabidopsis is caused by polymorphisms in the cis-regulatory region of the DOG1 gene and results in considerable expression differences between the DOG1 alleles of the accessions analyzed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(201);">hide</a></div></div></div><div id="publication202" class="publication01"><div 
  class="publication01Title">Melanesian and Asian origins of Polynesians: mtDNA and Y chromosome gradients across the Pacific.</div><div 
  class="publication01Info">Kayser M, Brauer S, Cordaux R, Casto A, Lao O, Zhivotovsky LA, Moyse-Faurie C, Rutledge RB, Schiefenhoevel W, Gil D, Lin AA et al., Molecular Biology And Evolution 23, 2234-2244 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(202);">abstract <img id="abstractI202" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16923821&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP202" 
  class="publication01Abstract" style="display:none;">The human settlement of the Pacific Islands represents one of the most recent major migration events of mankind. Polynesians originated in Asia according to linguistic evidence or in Melanesia according to archaeological evidence. To shed light on the genetic origins of Polynesians, we investigated over 400 Polynesians from 8 island groups, in comparison with over 900 individuals from potential parental populations of Melanesia, Southeast and East Asia, and Australia, by means of Y chromosome (NRY) and mitochondrial DNA (mtDNA) markers. Overall, we classified 94.1% of Polynesian Y chromosomes and 99.8% of Polynesian mtDNAs as of either Melanesian (NRY-DNA: 65.8%, mtDNA: 6%) or Asian (NRY-DNA: 28.3%, mtDNA: 93.8%) origin, suggesting a dual genetic origin of Polynesians in agreement with the &quot;Slow Boat&quot; hypothesis. Our data suggest a pronounced admixture bias in Polynesians toward more Melanesian men than women, perhaps as a result of matrilocal residence in the ancestral Polynesian society. Although dating methods are consistent with somewhat similar entries of NRY/mtDNA haplogroups into Polynesia, haplotype sharing suggests an earlier appearance of Melanesian haplogroups than those from Asia. Surprisingly, we identified gradients in the frequency distribution of some NRY/mtDNA haplogroups across Polynesia and a gradual west-to-east decrease of overall NRY/mtDNA diversity, not only providing evidence for a west-to-east direction of Polynesian settlements but also suggesting that Pacific voyaging was regular rather than haphazard. We also demonstrate that Fiji played a pivotal role in the history of Polynesia: humans probably first migrated to Fiji, and subsequent settlement of Polynesia probably came from Fiji.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(202);">hide</a></div></div></div><div id="publication203" class="publication01"><div 
  class="publication01Title">Sog/Chordin is required for ventral-to-dorsal Dpp/BMP transport and head formation in a short germ insect.</div><div 
  class="publication01Info">van der Zee M, Stockhammer O, von Levetzow C, Nunes da Fonseca R, Roth S, Proceedings Of The National Academy Of Sciences Of The United States Of America 103, 16307-16312 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(203);">abstract <img id="abstractI203" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17050690&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP203" 
  class="publication01Abstract" style="display:none;">Bone morphogenetic protein (BMP) signaling plays a major role in dorsoventral patterning in vertebrates and in Drosophila. Remarkably, in Tribolium, a beetle with an ancestral type of insect development, early BMP/dpp exhibits differential expression along the anteroposterior axis. However, the BMP/Dpp inhibitor Sog/chordin is expressed ventrally and establishes a dorsal domain of BMP/Dpp activity by transporting BMPs toward the dorsal side, like in Drosophila. Loss of Tribolium Sog not only abolishes dorsoventral polarity in the ectoderm, but also leads to the complete absence of the CNS. This phenotype suggests that sog is the main BMP antagonist in Tribolium, in contrast to vertebrates and Drosophila, which possess redundant antagonists. Surprisingly, Sog also is required for head formation in Tribolium, as are the BMP antagonists in vertebrates. Thus, in Tribolium, the system of BMP and its antagonists is less complex than in Drosophila or vertebrates and combines features from both, suggesting that it might represent an ancestral state.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(203);">hide</a></div></div></div><div id="publication204" class="publication01"><div 
  class="publication01Title">A change of expression in the conserved signaling gene MKK7 is associated with a selective sweep in the western house mouse Mus musculus domesticus.</div><div 
  class="publication01Info">Harr  B, Voolstra  C, Heinen  T J A J, Baines  J F, Rottscheidt  R, Ihle  S, Müller  W, Bonhomme  F, Tautz  D, Journal Of Evolutionary Biology 19, 1486-1496 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(204);">abstract <img id="abstractI204" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16910979&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP204" 
  class="publication01Abstract" style="display:none;">Changes in gene expression are known to occur between closely related species, but it is not yet clear how many of these are due to random fixation of allelic variants or due to adaptive events. In a microarray survey between subspecies of the Mus musculus complex, we identified the mitogen-activated protein-kinase-kinase MKK7 as a candidate for change in gene expression. Quantitative PCR experiments with multiple individuals from each subspecies confirmed a specific and significant up-regulation in the testis of M. m. domesticus. Northern blot analysis shows that this is due to a new transcript that is not found in other tissues, nor in M. m. musculus. A cis-trans test via allele specific expression analysis of the MKK7 gene in F1 hybrids between domesticus and musculus shows that the expression change is mainly caused by a mutation located in cis. Nucleotide diversity was found to be significantly reduced in a window of at least 20 kb around the MKK7 locus in domesticus, indicative of a selective sweep. Because the MKK7 gene is involved in modulating a kinase signalling cascade in a stress response pathway, it seems a plausible target for adaptive differences between subspecies, although the functional role of the new testis-specific transcripts will need to be further studied.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(204);">hide</a></div></div></div><div id="publication205" class="publication01"><div 
  class="publication01Title">A segmentation gene in tribolium produces a polycistronic mRNA that codes for multiple conserved peptides.</div><div 
  class="publication01Info">Savard J, Marques-Souza H, Aranda M, Tautz D, Cell 126, 559-569 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(205);">abstract <img id="abstractI205" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16901788&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP205" 
  class="publication01Abstract" style="display:none;">Segmentation genes in insects are required for generating the subdivisions of the early embryo. We describe here a new member of the gap family of segmentation genes in the flour beetle Tribolium, mille-pattes (mlpt). mlpt knockdown leads to transformation of the abdominal segments into thoracic segments, providing embryos with up to ten pairs of legs. We show that there are crossregulatory interactions between mlpt and the known gap genes in Tribolium, suggesting that mlpt is itself a gap gene. The mlpt gene reveals an unusual structure, as it encodes a polycistronic mRNA that codes for four peptides. mlpt appears to be the prototype of this previously unknown gene structure in eukaryotes, as we find homologous genes with the same polycistronic arrangement in other insect genomes as well.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(205);">hide</a></div></div></div><div id="publication206" class="publication01"><div 
  class="publication01Title">Cross-species analysis of biological networks by Bayesian alignment.</div><div 
  class="publication01Info">Berg J, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 103, 10967-10972 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(206);">abstract <img id="abstractI206" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16835301&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP206" 
  class="publication01Abstract" style="display:none;">Complex interactions between genes or proteins contribute a substantial part to phenotypic evolution. Here we develop an evolutionarily grounded method for the cross-species analysis of interaction networks by alignment, which maps bona fide functional relationships between genes in different organisms. Network alignment is based on a scoring function measuring mutual similarities between networks, taking into account their interaction patterns as well as sequence similarities between their nodes. High-scoring alignments and optimal alignment parameters are inferred by a systematic Bayesian analysis. We apply this method to analyze the evolution of coexpression networks between humans and mice. We find evidence for significant conservation of gene expression clusters and give network-based predictions of gene function. We discuss examples where cross-species functional relationships between genes do not concur with sequence similarity.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(206);">hide</a></div></div></div><div id="publication207" class="publication01"><div 
  class="publication01Title">Genomic islands of differentiation between house mouse subspecies.</div><div 
  class="publication01Info">Harr B, Genome Research 16, 730-737 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(207);">abstract <img id="abstractI207" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16687734&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP207" 
  class="publication01Abstract" style="display:none;">Understanding the genes that contribute to reproductive isolation is essential to understanding speciation, but isolating such genes has proven very difficult. In this study I apply a multilocus test statistic to &gt;10,000 SNP markers assayed in wild-derived inbred strains of house mice to identify genomic regions of elevated differentiation between two subspecies of house mice, Mus musculus musculus and M. m. domesticus. Differentiation was high through approximately 90% of the X chromosome. In addition, eight regions of high differentiation were identified on the autosomes, totaling 7.5% of the autosomal genome. Regions of high differentiation were confirmed by direct sequencing of samples collected from the wild. Some regions of elevated differentiation have an overrepresentation of genes with host-pathogen interactions and olfaction. The most strongly differentiated region on the X has previously been shown to fail to introgress across a hybrid zone between the two subspecies. This survey indicates autosomal regions that should also be examined for differential introgression across the hybrid zone, as containing potential genes causing hybrid unfitness.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(207);">hide</a></div></div></div><div id="publication208" class="publication01"><div 
  class="publication01Title">An integrated view of protein evolution.</div><div 
  class="publication01Info">Pal C, Papp B, Lercher MJ, Nature Reviews. Genetics 7, 337-348 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(208);">abstract <img id="abstractI208" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 16619049&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP208" 
  class="publication01Abstract" style="display:none;">Why do proteins evolve at different rates? Advances in systems biology and genomics have facilitated a move from studying individual proteins to characterizing global cellular factors. Systematic surveys indicate that protein evolution is not determined exclusively by selection on protein structure and function, but is also affected by the genomic position of the encoding genes, their expression patterns, their position in biological networks and possibly their robustness to mistranslation. Recent work has allowed insights into the relative importance of these factors. We discuss the status of a much-needed coherent view that integrates studies on protein evolution with biochemistry and functional and structural genomics.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(208);">hide</a></div></div></div><div id="publication209" class="publication01"><div 
  class="publication01Title">An analysis of signatures of selective sweeps in natural populations of the house mouse.</div><div 
  class="publication01Info">Ihle S, Ravaoarimanana I, Thomas M, Tautz D, Molecular Biology And Evolution 23, 790-797 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(209);">abstract <img id="abstractI209" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16421176&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP209" 
  class="publication01Abstract" style="display:none;">Population and locus-specific reduction of variability of polymorphic loci could be an indication of positive selection at a linked site (selective sweep) and therefore point toward genes that have been involved in recent adaptations. Analysis of microsatellite variability offers a way to identify such regions and to ask whether they occur more often than expected by chance. We studied four populations of the house mouse (Mus musculus) to assess the frequency of such signatures of selective sweeps under natural conditions. Three samples represent the subspecies Mus m. dometicus [corrected] and came from Germany, France, and Cameroon. One sample came from Kazakhstan and constitutes a population of the subspecies Mus m. [corrected] musculus. Mitochondrial D-loop sequences from all animals confirm their respective assignments. Approximately 200 microsatellite loci were typed for up to 60 unrelated individuals from each population and evaluated for signs of selective sweeps on the basis of Schlötterer&apos;s ln RV and ln RH statistics. Our data suggest that there are slightly more signs of selective sweeps than would have been expected by chance alone in each of the populations and also highlights some of the statistical challenges faced in genome scans for detecting selection. Single-nucleotide polymorphism typing of one sweep signature in the M. m. domesticus populations around the beta-defensin 6 locus confirms a lowered nucleotide diversity in this region and limits the potential sweep region to about 20 kb. However, no amino acid exchange has occurred in the coding region when compared to M. m. musculus. If this sweep signature is due to a recent adaptation, it is expected that a regulatory change would have caused it. Our data provide a framework for conducting a systematic whole genome scan for signatures of selective sweeps in the mouse genome.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(209);">hide</a></div></div></div><div id="publication210" class="publication01"><div 
  class="publication01Title">Chance and necessity in the evolution of minimal metabolic networks.</div><div 
  class="publication01Info">Pal C, Papp B, Lercher MJ, Csermely P, Oliver SG, Hurst LD, Nature 440, 667-670 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(210);">abstract <img id="abstractI210" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id= 16572170&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP210" 
  class="publication01Abstract" style="display:none;">It is possible to infer aspects of an organism&apos;s lifestyle from its gene content. Can the reverse also be done? Here we consider this issue by modelling evolution of the reduced genomes of endosymbiotic bacteria. The diversity of gene content in these bacteria may reflect both variation in selective forces and contingency-dependent loss of alternative pathways. Using an in silico representation of the metabolic network of Escherichia coli, we examine the role of contingency by repeatedly simulating the successive loss of genes while controlling for the environment. The minimal networks that result are variable in both gene content and number. Partially different metabolisms can thus evolve owing to contingency alone. The simulation outcomes do preserve a core metabolism, however, which is over-represented in strict intracellular bacteria. Moreover, differences between minimal networks based on lifestyle are predictable: by simulating their respective environmental conditions, we can model evolution of the gene content in Buchnera aphidicola and Wigglesworthia glossinidia with over 80% accuracy. We conclude that, at least for the particular cases considered here, gene content of an organism can be predicted with knowledge of its distant ancestors and its current lifestyle.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(210);">hide</a></div></div></div><div id="publication211" class="publication01"><div 
  class="publication01Title">Egg development in parthenogenetic nematodes: variations in meiosis and axis formation.</div><div 
  class="publication01Info">Lahl V, Sadler B, Schierenberg E, The International Journal Of Developmental Biology 50, 393-398 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(211);">abstract <img id="abstractI211" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16525934&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP211" 
  class="publication01Abstract" style="display:none;">In the well studied model nematode Caenorhabditis elegans entrance of the sperm induces an anterior-posterior polarity in the egg and determines the orientation of the primary embryonic axis. Subsequently, fusion of two haploid gamete nuclei results in a diploid zygote as a prerequisite for normal embryogenesis. Here we analyze the establishment of embryonic polarity and diploidy in the absence of sperm in three parthenogenetic nematode species from three different families, Diploscapter coronatus (Diploscapteridae), Acrobeloides nanus (Cephalobidae) and Plectus sp. (Plectidae). We find that they not only differ from C. elegans in these two aspects but also from each other, indicating variant solutions for the same developmental challenges and supporting the view that the parthenogenetic mode of reproduction has been acquired multiple times independently.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(211);">hide</a></div></div></div><div id="publication212" class="publication01"><div 
  class="publication01Title">Embryological variation during nematode development.</div><div 
  class="publication01Info">Schierenberg E, WormBook : The Online Review Of C. Elegans Biology, 1-13 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(212);">abstract <img id="abstractI212" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18050475&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP212" 
  class="publication01Abstract" style="display:none;">Early cell lineages and arrangement of blastomeres in C. elegans are similar to the pattern found in Ascaris and other studied nematodes leading to the assumption that embryonic development shows little variation within the phylum Nematoda. However, analysis of a larger variety of species from various branches of the phylogenetic tree demonstrate that prominent variations in crucial steps of early embryogenesis exist among representatives of this taxon. So far, most of these variations have only been studied on a descriptive level and thus essentially nothing is known about their molecular or genetic basis. Nevertheless, it is obvious that the limited morphological diversity of the freshly hatched juvenile and the uniformity of the basic body plan contrast with the many modifications in the way a worm is generated from the egg cell. This chapter focuses on the initial phase between egg activation and gastrulation and deals with the following aspects: reproduction and diploidy, polarity, cleavage and germ line, cell lineages; cell cycles and maternal contribution, cell-cell communication and cell specification, gastrulation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(212);">hide</a></div></div></div><div id="publication213" class="publication01"><div 
  class="publication01Title">The interferon-inducible GTPases.</div><div 
  class="publication01Info">Martens S, Howard J, Annual Review Of Cell And Developmental Biology 22, 559-589 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(213);">abstract <img id="abstractI213" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16824009&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP213" 
  class="publication01Abstract" style="display:none;">Mammalian cells respond to interferons (IFNs) secreted during infection by the transcriptional upregulation of as many as a thousand genes. This remarkable transition prepares cells and organisms for resistance to infection, and many IFN-regulated gene products are players in well-understood resistance programs. Oddly, however, many of the most abundantly induced proteins are GTPases whose functions are not well understood. Here we review the progress that has been made toward understanding the roles of individual GTPase families in disease resistance and the hints of common mechanisms that are now available.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(213);">hide</a></div></div></div>
          </div>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
