<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
								 id="contentPane"><h2>C9: Publications</h2>

	
<div id="publication220" class="publication01">
  <div class="publication01Title">
    Annotation of genomics data using bidirectional hidden Markov models unveils variations in Pol II transcription cycle.
  </div>
  <div class="publication01Info">
    Zacher B, Lidschreiber M, Cramer P, Gagneur J, Tresch A.
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(220);">abstract <img id="abstractI220" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.ncbi.nlm.nih.gov/pubmed/25527639">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP220" class="publication01Abstract" style="display:none;">
    <p>
      DNA replication, transcription and repair involve the recruitment of protein complexes that change their composition as they progress along the genome in a directed or strand-specific manner. Chromatin immunoprecipitation in conjunction with hidden Markov models (HMMs) has been instrumental in understanding these processes, as they segment the genome into discrete states that can be related to DNA-associated protein complexes. However, current HMM-based approaches are not able to assign forward or reverse direction to states or properly integrate strand-specific (e.g., RNA expression) with non-strand-specific (e.g., ChIP) data, which is indispensable to accurately characterize directed processes. To overcome these limitations, we introduce bidirectional HMMs which infer directed genomic states from occupancy profiles de novo. Application to RNA polymerase II-associated factors in yeast and chromatin modifications in human T cells recovers the majority of transcribed loci, reveals gene-specific variations in the yeast transcription cycle and indicates the existence of directed chromatin state patterns at transcribed, but not at repressed, regions in the human genome. In yeast, we identify 32 new transcribed loci, a regulated initiation-elongation transition, the absence of elongation factors Ctk1 and Paf1 from a class of genes, a distinct transcription mechanism for highly expressed genes and novel DNA sequence motifs associated with transcription termination. We anticipate bidirectional HMMs to significantly improve the analyses of genome-associated directed processes.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(220);">hide</a>
    </div>
  </div>
</div>

	
	
<div id="publication219" class="publication01">
  <div class="publication01Title">
    Annotation of genomics data using bidirectional hidden Markov models unveils variations in Pol II transcription cycle
  </div>
  <div class="publication01Info">
Zacher B, Lidschreiber M, Cramer P, Gagneur J, Tresch A, DOI 10.15252/msb.20145654 | Published online 19.12.2014 Molecular Systems Biology(2014)10:768
  </div>
  <div class="publication01Links"><a href="javascript:Publications.toggleAbstract(219);">abstract <img id="abstractI219" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://msb.embopress.org/content/10/12/768">paper [pdf]</a> &nbsp; 
  </div>
  <div id="abstractP219" class="publication01Abstract" style="display:none;">
    <p>DNA replication, transcription and repair involve the recruitment of protein complexes that change their composition as they progress along the genome in a directed or strand‐specific manner. Chromatin immunoprecipitation in conjunction with hidden Markov models (HMMs) has been instrumental in understanding these processes, as they segment the genome into discrete states that can be related to DNA‐associated protein complexes. However, current HMM‐based approaches are not able to assign forward or reverse direction to states or properly integrate strand‐specific (e.g., RNA expression) with non‐strand‐specific (e.g., ChIP) data, which is indispensable to accurately characterize directed processes. To overcome these limitations, we introduce bidirectional HMMs which infer directed genomic states from occupancy profiles de novo. Application to RNA polymerase II‐associated factors in yeast and chromatin modifications in human T cells recovers the majority of transcribed loci, reveals gene‐specific variations in the yeast transcription cycle and indicates the existence of directed chromatin state patterns at transcribed, but not at repressed, regions in the human genome. In yeast, we identify 32 new transcribed loci, a regulated initiation–elongation transition, the absence of elongation factors Ctk1 and Paf1 from a class of genes, a distinct transcription mechanism for highly expressed genes and novel DNA sequence motifs associated with transcription termination. We anticipate bidirectional HMMs to significantly improve the analyses of genome‐associated directed processes.
    </p>
    <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(219);">hide</a>
    </div>
  </div>
</div>

	
	    <div id="publication7" class="publication01"><div 
  class="publication01Title">Genome-wide quantitative analysis of DNA methylation from bisulfite sequencing data</div><div 
  class="publication01Info">Akman K, Haaf T, Gravina S, Vijg J, Tresch A, Bioinformatics, 30(13) (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(7);">abstract <img id="abstractI7" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://bioinformatics.oxfordjournals.org/content/30/13/1933.long">paper [pdf]</a> &nbsp; </div><div id="abstractP7" 
  class="publication01Abstract" style="display:none;">Although many computer programs can perform population genetics calculations, they are typically limited in the analyses and data input formats they offer; few applications can process the large data sets produced by whole-genome resequencing projects. Furthermore, there is no coherent framework for the easy integration of new statistics into existing pipelines, hindering the development and application of new population genetics and genomics approaches. Here, we present PopGenome, a population genomics package for the R software environment (a de facto standard for statistical analyses). PopGenome can efficiently process genome-scale data as well as large sets of individual loci. It reads DNA alignments and single-nucleotide polymorphism (SNP) data sets in most common formats, including those used by the HapMap, 1000 human genomes, and 1001 Arabidopsis genomes projects. PopGenome also reads associated annotation files in GFF format, enabling users to easily define regions or classify SNPs based on their annotation; all analyses can also be applied to sliding windows. PopGenome offers a wide range of diverse population genetics analyses, including neutrality tests as well as statistics for population differentiation, linkage disequilibrium, and recombination. PopGenome is linked to Hudson's MS and Ewing's MSMS programs to assess statistical significance based on coalescent simulations. PopGenome's integration in R facilitates effortless and reproducible downstream analyses as well as the production of publication-quality graphics. Developers can easily incorporate new analyses methods into the PopGenome framework. PopGenome and R are freely available from CRAN (http://cran.r-project.org/) for all major operating systems under the GNU General Public License. <div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(7);">hide</a></div></div></div><div id="publication17" class="publication01"><div 
  class="publication01Title">Periodic mRNA synthesis and degradation co‐operate during cell cycle gene expression</div><div 
  class="publication01Info">Eser P, Demel C, Maier KC, Schwalb B, Pirkl N, Martin DE, Cramer P, Tresch A, Mol Syst Biol. 10:717 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(17);">abstract <img id="abstractI17" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://msb.embopress.org/content/10/1/717">paper [pdf]</a> &nbsp; </div><div id="abstractP17" 
  class="publication01Abstract" style="display:none;">During the cell cycle, the levels of hundreds of mRNAs change in a periodic manner, but how this is achieved by alterations in the rates of mRNA synthesis and degradation has not been studied systematically. Here, we used metabolic RNA labeling and comparative dynamic transcriptome analysis (cDTA) to derive mRNA synthesis and degradation rates every 5 min during three cell cycle periods of the yeast <em>Saccharomyces cerevisiae</em>. A novel statistical model identified 479 genes that show periodic changes in mRNA synthesis and generally also periodic changes in their mRNA degradation rates. Peaks of mRNA degradation generally follow peaks of mRNA synthesis, resulting in sharp and high peaks of mRNA levels at defined times during the cell cycle. Whereas the timing of mRNA synthesis is set by upstream DNA motifs and their associated transcription factors (TFs), the synthesis rate of a periodically expressed gene is apparently set by its core promoter.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(17);">hide</a></div></div></div>
	    <p><a href="project_area_c.html">back to project area C</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
