<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>C2: Publications</h2>
	    <div class="publication01"><div 
  class="publication01Title">A predictive fitness model for influenza</div><div 
  class="publication01Info">Łuksza M, Lässig M, Nature, 507, 57-61 (2014)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(2);">abstract <img id="abstractI2" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.nature.com/nature/journal/vaop/ncurrent/abs/nature13087.html">paper [pdf]</a> &nbsp; </div><div id="abstractP2" 
  class="publication01Abstract" style="display:none;">The seasonal human influenza A (H3N2) virus undergoes rapid evolution, which produces significant year-to-year sequence turnover in the population of circulating strains. Adaptive mutations respond to human immune challenge and occur primarily in antigenic epitopes, the antibody-binding domains of the viral surface protein haemagglutinin. Here we develop a fitness model for haemagglutinin that predicts the evolution of the viral population from one year to the next. Two factors are shown to determine the fitness of a strain: adaptive epitope changes and deleterious mutations outside the epitopes. We infer both fitness components for the strains circulating in a given year, using population-genetic data of all previous strains. From fitness and frequency of each strain, we predict the frequency of its descendent strains in the following year. This fitness model maps the adaptive history of influenza A and suggests a principled method for vaccine selection. Our results call for a more comprehensive  epidemiology of influenza and other fast-evolving pathogens that integrates antigenic phenotypes with other viral functions coupled by genetic linkage.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(2);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Universality and predictability in molecular quantitative genetics</div><div 
  class="publication01Info">Nourmohammad A, Held T, Lässig M, Current Opinion in Genetics and Development, Volume 23, Issue 6, Pages 684–693 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(6);">abstract <img id="abstractI6" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.thp.uni-koeln.de/~lassig/docs/2013d.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP6" 
  class="publication01Abstract" style="display:none;">Molecular traits, such as gene expression levels or protein binding affinities, are increasingly accessible to quantitative measurement by modern high-throughput techniques. Such traits measure molecular functions and, from an evolutionary point of view, are important as targets of natural selection. We review recent developments in evolutionary theory and experiments that are expected to become building blocks of a quantitative genetics of molecular traits. We focus on universal evolutionary characteristics: these are largely independent of a trait's genetic basis, which is often at least partially unknown. We show that universal measurements can be used to infer selection on a quantitative trait, which determines its evolutionary mode of conservation or adaptation. Furthermore, universality is closely linked to predictability of trait evolution across lineages. We argue that universal trait statistics extends over a range of cellular scales and opens new avenues of quantitative evolutionary systems biology.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(6);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Fitness landscape of nucleosome positioning</div><div 
  class="publication01Info">Weghorn D, Lässig M, Proc. Natl. Acad. Sci., vol. 110 no. 27 10988-10993 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(14);">abstract <img id="abstractI14" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.pnas.org/content/110/27/10988.full.pdf+html">paper [pdf]</a> &nbsp; </div><div id="abstractP14" 
  class="publication01Abstract" style="display:none;">Histone–DNA complexes, so-called nucleosomes, are the building blocks of DNA packaging in eukaryotic cells. The histone-binding affinity of a local DNA segment depends on its elastic properties and determines its accessibility within the nucleus, which plays an important role in the regulation of gene expression. Here, we derive a fitness landscape for intergenic DNA segments in yeast as a function of two molecular phenotypes: their elasticity-dependent histone affinity and their coverage with transcription factor binding sites. This landscape reveals substantial selection against nucleosome formation over a wide range of both phenotypes. We use it as the core component of a quantitative evolutionary model for intergenic DNA segments. This model consistently predicts the observed diversity of histone affinities within wild Saccharomyces paradoxus populations, as well as the affinity divergence between neighboring Saccharomyces species. Our analysis establishes histone binding and transcription factor binding as two separable modes of sequence evolution, each of which is a direct target of natural selection.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(14);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Evolutionary dynamics and statistical physics</div><div 
  class="publication01Info">Fisher DS, Lässig M, Shraiman B, J. Stat. Mech. N01001 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(33);">abstract <img id="abstractI33" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1742-5468/2013/01/N01001">paper [pdf]</a> &nbsp; </div><div id="abstractP33" 
  class="publication01Abstract" style="display:none;">This introductory article provides the background to and motivation for this special issue and the relationship between evolutionary dynamics and statistical physics.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(33);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Clonal interference in the evolution of influenza.</div><div 
  class="publication01Info">Strelkowa N, Lässig M, Genetics 192, 671-682 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(43);">abstract <img id="abstractI43" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22851649&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP43" 
  class="publication01Abstract" style="display:none;">The seasonal influenza A virus undergoes rapid evolution to escape human immune response. Adaptive changes occur primarily in antigenic epitopes, the antibody-binding domains of the viral hemagglutinin. This process involves recurrent selective sweeps, in which clusters of simultaneous nucleotide fixations in the hemagglutinin coding sequence are observed about every 4 years. Here, we show that influenza A (H3N2) evolves by strong clonal interference. This mode of evolution is a red queen race between viral strains with different beneficial mutations. Clonal interference explains and quantifies the observed sweep pattern: we find an average of at least one strongly beneficial amino acid substitution per year, and a given selective sweep has three to four driving mutations on average. The inference of selection and clonal interference is based on frequency time series of single-nucleotide polymorphisms, which are obtained from a sample of influenza genome sequences over 39 years. Our results imply that mode and speed of influenza evolution are governed not only by positive selection within, but also by background selection outside antigenic epitopes: immune adaptation and conservation of other viral functions interfere with each other. Hence, adapting viral proteins are predicted to be particularly brittle. We conclude that a quantitative understanding of influenza&apos;s evolutionary and epidemiological dynamics must be based on all genomic domains and functions coupled by clonal interference.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(43);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Chance and risk in adaptive evolution.</div><div 
  class="publication01Info">Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 109, 4719-4720 (2012)</div><div 
  class="publication01Links"><a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=22411817&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div></div><div class="publication01"><div 
  class="publication01Title">Emergent neutrality in adaptive asexual evolution.</div><div 
  class="publication01Info">Schiffels S, Szöllosi GJ, Mustonen V, Lässig M, Genetics 189, 1361-1375 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(68);">abstract <img id="abstractI68" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21926305&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP68" 
  class="publication01Abstract" style="display:none;">In nonrecombining genomes, genetic linkage can be an important evolutionary force. Linkage generates interference interactions, by which simultaneously occurring mutations affect each other&apos;s chance of fixation. Here, we develop a comprehensive model of adaptive evolution in linked genomes, which integrates interference interactions between multiple beneficial and deleterious mutations into a unified framework. By an approximate analytical solution, we predict the fixation rates of these mutations, as well as the probabilities of beneficial and deleterious alleles at fixed genomic sites. We find that interference interactions generate a regime of emergent neutrality: all genomic sites with selection coefficients smaller in magnitude than a characteristic threshold have nearly random fixed alleles, and both beneficial and deleterious mutations at these sites have nearly neutral fixation rates. We show that this dynamic limits not only the speed of adaptation, but also a population&apos;s degree of adaptation in its current environment. We apply the model to different scenarios: stationary adaptation in a time-dependent environment and approach to equilibrium in a fixed environment. In both cases, the analytical predictions are in good agreement with numerical simulations. Our results suggest that interference can severely compromise biological functions in an adapting population, which sets viability limits on adaptive evolution under linkage.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(68);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Formation of regulatory modules by local sequence duplication.</div><div 
  class="publication01Info">Nourmohammad A, Lässig M, PLoS Computational Biology 7, e1002167 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(73);">abstract <img id="abstractI73" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=21998564&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP73" 
  class="publication01Abstract" style="display:none;">Turnover of regulatory sequence and function is an important part of molecular evolution. But what are the modes of sequence evolution leading to rapid formation and loss of regulatory sites? Here we show that a large fraction of neighboring transcription factor binding sites in the fly genome have formed from a common sequence origin by local duplications. This mode of evolution is found to produce regulatory information: duplications can seed new sites in the neighborhood of existing sites. Duplicate seeds evolve subsequently by point mutations, often towards binding a different factor than their ancestral neighbor sites. These results are based on a statistical analysis of 346 cis-regulatory modules in the Drosophila melanogaster genome, and a comparison set of intergenic regulatory sequence in Saccharomyces cerevisiae. In fly regulatory modules, pairs of binding sites show significantly enhanced sequence similarity up to distances of about 50 bp. We analyze these data in terms of an evolutionary model with two distinct modes of site formation: (i) evolution from independent sequence origin and (ii) divergent evolution following duplication of a common ancestor sequence. Our results suggest that pervasive formation of binding sites by local sequence duplications distinguishes the complex regulatory architecture of higher eukaryotes from the simpler architecture of unicellular organisms.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(73);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Fitness flux and ubiquity of adaptive evolution.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 107, 4248-4253 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(109);">abstract <img id="abstractI109" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=20145113&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP109" 
  class="publication01Abstract" style="display:none;">Natural selection favors fitter variants in a population, but actual evolutionary processes may decrease fitness by mutations and genetic drift. How is the stochastic evolution of molecular biological systems shaped by natural selection? Here, we derive a theorem on the fitness flux in a population, defined as the selective effect of its genotype frequency changes. The fitness-flux theorem generalizes Fisher&apos;s fundamental theorem of natural selection to evolutionary processes including mutations, genetic drift, and time-dependent selection. It shows that a generic state of populations is adaptive evolution: there is a positive fitness flux resulting from a surplus of beneficial over deleterious changes. In particular, stationary nonequilibrium evolution processes are predicted to be adaptive. Under specific nonstationary conditions, notably during a decrease in population size, the average fitness flux can become negative. We show that these predictions are in accordance with experiments in bacteria and bacteriophages and with genomic data in Drosophila. Our analysis establishes fitness flux as a universal measure of adaptation in molecular evolution.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(109);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">From fitness landscapes to seascapes: non-equilibrium dynamics of selection and adaptation.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Trends In Genetics 25, 111-119 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(134);">abstract <img id="abstractI134" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19232770&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP134" 
  class="publication01Abstract" style="display:none;">Evolution is a quest for innovation. Organisms adapt to changing natural selection by evolving new phenotypes. Can we read this dynamics in their genomes? Not every mutation under positive selection responds to a change in selection: beneficial changes also occur at evolutionary equilibrium, repairing previous deleterious changes and restoring existing functions. Adaptation, by contrast, is viewed here as a non-equilibrium phenomenon: the genomic response to time-dependent selection. Our approach extends the static concept of fitness landscapes to dynamic fitness seascapes. It shows that adaptation requires a surplus of beneficial substitutions over deleterious ones. Here, we focus on the evolution of yeast and Drosophila genomes, providing examples where adaptive evolution can and cannot be inferred, despite the presence of positive selection.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(134);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Energy-dependent fitness: a quantitative model for the evolution of yeast transcription factor binding sites.</div><div 
  class="publication01Info">Mustonen V, Kinney J, Callan CG, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 105, 12376-12381 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(152);">abstract <img id="abstractI152" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18723669&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP152" 
  class="publication01Abstract" style="display:none;">We present a genomewide cross-species analysis of regulation for broad-acting transcription factors in yeast. Our model for binding site evolution is founded on biophysics: the binding energy between transcription factor and site is a quantitative phenotype of regulatory function, and selection is given by a fitness landscape that depends on this phenotype. The model quantifies conservation, as well as loss and gain, of functional binding sites in a coherent way. Its predictions are supported by direct cross-species comparison between four yeast species. We find ubiquitous compensatory mutations within functional sites, such that the energy phenotype and the function of a site evolve in a significantly more constrained way than does its sequence. We also find evidence for substantial evolution of regulatory function involving point mutations as well as sequence insertions and deletions within binding sites. Genes lose their regulatory link to a given transcription factor at a rate similar to the neutral point mutation rate, from which we infer a moderate average fitness advantage of functional over nonfunctional sites. In a wider context, this study provides an example of inference of selection acting on a quantitative molecular trait.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(152);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Molecular evolution under fitness fluctuations.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Physical Review Letters 100, 108101 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(158);">abstract <img id="abstractI158" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18352233&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP158" 
  class="publication01Abstract" style="display:none;">Molecular evolution is a stochastic process governed by fitness, mutations, and reproductive fluctuations in a population. Here, we study evolution where fitness itself is stochastic, with random switches in the direction of selection at individual genomic loci. As the correlation time of these fluctuations becomes larger than the diffusion time of mutations within the population, fitness changes from an annealed to a quenched random variable. We show that the rate of evolution has its maximum in the crossover regime, where both time scales are comparable. Adaptive evolution emerges in the quenched fitness regime (evidence for such fitness fluctuations has recently been found in genomic data). The joint statistical theory of reproductive and fitness fluctuations establishes a conceptual connection between evolutionary genetics and statistical physics of disordered systems.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(158);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Adaptations to fluctuating selection in Drosophila.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 104, 2277-2282 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(178);">abstract <img id="abstractI178" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17287357&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP178" 
  class="publication01Abstract" style="display:none;">Time-dependent selection causes the adaptive evolution of new phenotypes, and this dynamics can be traced in genomic data. We have analyzed polymorphisms and substitutions in Drosophila, using a more sensitive inference method for adaptations than the standard population-genetic tests. We find evidence that selection itself is strongly time-dependent, with changes occurring at nearly the rate of neutral evolution. At the same time, higher than previously estimated levels of selection make adaptive responses by a factor 10-100 faster than the pace of selection changes, ensuring that adaptations are an efficient mode of evolution under time-dependent selection. The rate of selection changes is faster in noncoding DNA, i.e., the inference of functional elements can less be based on sequence conservation than for proteins. Our results suggest that selection acts not only as a constraint but as a major driving force of genomic change.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(178);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">From biophysics to evolutionary genetics: statistical aspects of gene regulation.</div><div 
  class="publication01Info">Lässig M, BMC Bioinformatics 8 Suppl 6, S7 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(183);">abstract <img id="abstractI183" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17903288&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP183" 
  class="publication01Abstract" style="display:none;">This is an introductory review on how genes interact to produce biological functions. Transcriptional interactions involve the binding of proteins to regulatory DNA. Specific binding sites can be identified by genomic analysis, and these undergo a stochastic evolution process governed by selection, mutations, and genetic drift. We focus on the links between the biophysical function and the evolution of regulatory elements. In particular, we infer fitness landscapes of binding sites from genomic data, leading to a quantitative evolutionary picture of regulation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(183);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Evolutionary population genetics of promoters: predicting binding sites and functional phylogenies.</div><div 
  class="publication01Info">Mustonen V, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 102, 15936-15941 (2005)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(201);">abstract <img id="abstractI201" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16236723&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP201" 
  class="publication01Abstract" style="display:none;">We study the evolution of transcription factor-binding sites in prokaryotes, using an empirically grounded model with point mutations and genetic drift. Selection acts on the site sequence via its binding affinity to the corresponding transcription factor. Calibrating the model with populations of functional binding sites, we verify this form of selection and show that typical sites are under substantial selection pressure for functionality: for cAMP response protein sites in Escherichia coli, the product of fitness difference and effective population size takes values 2NDeltaF of order 10. We apply this model to cross-species comparisons of binding sites in bacteria and obtain a prediction method for binding sites that uses evolutionary information in a quantitative way. At the same time, this method predicts the functional histories of orthologous sites in a phylogeny, evaluating the likelihood for conservation or loss or gain of function during evolution. We have performed, as an example, a cross-species analysis of E. coli, Salmonella typhimurium, and Yersinia pseudotuberculosis. Detailed lists of predicted sites and their functional phylogenies are available.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(201);">hide</a></div></div></div>
	    <p><a href="project_area_c.html">back to project area C</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
