<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>B2: Publications</h2>
	    <div class="publication01"><div 
  class="publication01Title">A common and unstable copy number variant is associated with differences in Glo1 expression and anxiety-like behavior.</div><div 
  class="publication01Info">Williams R, Lim JE, Harr B, Wing C, Walters R, Distler MG, Teschke M, Wu C, Wiltshire T, Su AI, Sokoloff GEA, PloS One 4, e4649 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(124);">abstract <img id="abstractI124" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19266052&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP124" 
  class="publication01Abstract" style="display:none;">Glyoxalase 1 (Glo1) has been implicated in anxiety-like behavior in mice and in multiple psychiatric diseases in humans. We used mouse Affymetrix exon arrays to detect copy number variants (CNV) among inbred mouse strains and thereby identified a approximately 475 kb tandem duplication on chromosome 17 that includes Glo1 (30,174,390-30,651,226 Mb; mouse genome build 36). We developed a PCR-based strategy and used it to detect this duplication in 23 of 71 inbred strains tested, and in various outbred and wild-caught mice. Presence of the duplication is associated with a cis-acting expression QTL for Glo1 (LOD&gt;30) in BXD recombinant inbred strains. However, evidence for an eQTL for Glo1 was not obtained when we analyzed single SNPs or 3-SNP haplotypes in a panel of 27 inbred strains. We conclude that association analysis in the inbred strain panel failed to detect an eQTL because the duplication was present on multiple highly divergent haplotypes. Furthermore, we suggest that non-allelic homologous recombination has led to multiple reversions to the non-duplicated state among inbred strains. We show associations between multiple duplication-containing haplotypes, Glo1 expression and anxiety-like behavior in both inbred strain panels and outbred CD-1 mice. Our findings provide a molecular basis for differential expression of Glo1 and further implicate Glo1 in anxiety-like behavior. More broadly, these results identify problems with commonly employed tests for association in inbred strains when CNVs are present. Finally, these data provide an example of biologically significant phenotypic variability in model organisms that can be attributed to CNVs.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(124);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Inferring the history of speciation in house mice from autosomal, X-linked, Y-linked and mitochondrial genes.</div><div 
  class="publication01Info">Geraldes A, Basset P, Gibson B, Smith KL, Harr B, Yu H, Bulatova N, Ziv Y, Nachman MW, Molecular Ecology 17, 5349-5363 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(132);">abstract <img id="abstractI132" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19121002&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP132" 
  class="publication01Abstract" style="display:none;">Patterns of genetic differentiation among taxa at early stages of divergence provide an opportunity to make inferences about the history of speciation. Here, we conduct a survey of DNA-sequence polymorphism and divergence at loci on the autosomes, X chromosome, Y chromosome and mitochondrial DNA in samples of Mus domesticus, M. musculus and M. castaneus. We analyzed our data under a divergence with gene flow model and estimate that the effective population size of M. castaneus is 200,000-400,000, of M. domesticus is 100,000-200,000 and of M. musculus is 60,000-120,000. These data also suggest that these species started to diverge approximately 500,000 years ago. Consistent with this recent divergence, we observed considerable variation in the genealogical patterns among loci. For some loci, all alleles within each species formed a monophyletic group, while at other loci, species were intermingled on the phylogeny of alleles. This intermingling probably reflects both incomplete lineage sorting and gene flow after divergence. Likelihood ratio tests rejected a strict allopatric model with no gene flow in comparisons between each pair of species. Gene flow was asymmetric: no gene flow was detected into M. domesticus, while significant gene flow was detected into both M. castaneus and M. musculus. Finally, most of the gene flow occurred at autosomal loci, resulting in a significantly higher ratio of fixed differences to polymorphisms at the X and Y chromosomes relative to autosomes in some comparisons, or just the X chromosome in others, emphasizing the important role of the sex chromosomes in general and the X chromosome in particular in speciation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(132);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Extensive additivity of gene expression differentiates subspecies of the house mouse.</div><div 
  class="publication01Info">Rottscheidt R, Harr B, Genetics 177, 1553-1567 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(155);">abstract <img id="abstractI155" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17947435&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP155" 
  class="publication01Abstract" style="display:none;">We have studied different subspecies of the house mouse and their reciprocal F(1) hybrids to estimate the within-locus mode of inheritance for subspecies differences in gene expression in three tissues (brain, liver, and testis) of male mice. This study investigates the mode of inheritance in crosses at a larger taxonomic distance than has been previously systematically investigated. We found the vast majority of transcripts to be additively expressed with only a few transcripts showing dominance or overdominance in expression, except for one direction of one cross, which showed large mis-expression in the testis. We suggest that, as time passes, more genes come to influence expression, and if there is no directional dominance, additivity becomes increasingly more likely, up to a threshold beyond which there is F(1) hybrid breakdown. Some previous studies on different organisms have found a large degree of dominance, commonly at shorter taxonomic differences. We surveyed these findings and show that the most consistent association exists between the amount of additivity detected in a study and the expression analysis method (in particular microarray platform), suggesting that at least some of the differences among studies might be methodological. Most studies agree with ours in that within-locus additivity seems to be general mode of inheritance for transcript expression. Differentially expressed transcripts identified in our screen among subspecies of house mice are candidate genes that could be involved in reproductive isolation between these subspecies.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(155);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Reduced X-linked diversity in derived populations of house mice.</div><div 
  class="publication01Info">Baines JF, Harr B, Genetics 175, 1911-1921 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(161);">abstract <img id="abstractI161" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17287527&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP161" 
  class="publication01Abstract" style="display:none;">Contrasting patterns of X-linked vs. autosomal diversity may be indicative of the mode of selection operating in natural populations. A number of observations have shown reduced X-linked (or Z-linked) diversity relative to autosomal diversity in various organisms, suggesting a large impact of genetic hitchhiking. However, the relative contribution of other forces such as population bottlenecks, variation in reproductive success of the two sexes, and differential introgression remains unclear. Here, we survey 13 loci, 6 X-linked and 7 autosomal, in natural populations of the house mouse (Mus musculus) subspecies complex. We studied seven populations of three different subspecies, the eastern house mouse M. musculus castaneus, the central house mouse M. m. musculus, and the western house mouse M. m. domesticus, including putatively ancestral and derived populations for each. All populations display lower diversity on the X chromosomes relative to autosomes, and this effect is most pronounced in derived populations. To assess the role of demography, we fit the demographic parameters that gave the highest likelihood of the data using coalescent simulations. We find that the reduction in X-linked diversity is too large to be explained by a simple demographic model in at least two of four derived populations. These observations are also not likely to be explained by differences in reproductive success between males and females. They are consistent with a greater impact of positive selection on the X chromosome, and this is supported by the observation of an elevated K(A) and elevated K(A)/K(S) ratios on the rodent X chromosome. A second contribution may be that the X chromosome less readily introgresses across subspecies boundaries.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(161);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Contrasting evolution of expression differences in the testis between species and subspecies of the house mouse.</div><div 
  class="publication01Info">Voolstra C, Tautz D, Farbrother P, Eichinger L, Harr B, Genome Research 17, 42-49 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(164);">abstract <img id="abstractI164" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17038563&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP164" 
  class="publication01Abstract" style="display:none;">Regulatory changes in genes involved in reproduction are thought to be prime targets for divergence during speciation, since they are expected to play an important role in sexual selection and sexual conflict. We used microarray analysis of RNA from different wild populations of house mouse subspecies (including Mus m. musculus, Mus m. domesticus, and Mus m. castaneus) and from the sister species Mus spretus to test this assumption. A comparison of expression divergence in brain, liver/kidney, and testis shows a major difference in the evolutionary dynamics of testis-related genes. While the comparison between species confirms an excess in divergence in testis genes, we find that all comparisons between subspecies yield only a very small number of genes with significantly different expression levels in the testis. These results suggest that the early phase of the speciation process may not be driven by regulatory changes in genes that are potential targets of sexual selection, and that the divergence in these genes is only established during a later phase of the speciation process.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(164);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">A change of expression in the conserved signaling gene MKK7 is associated with a selective sweep in the western house mouse Mus musculus domesticus.</div><div 
  class="publication01Info">Harr  B, Voolstra  C, Heinen  T J A J, Baines  J F, Rottscheidt  R, Ihle  S, Müller  W, Bonhomme  F, Tautz  D, Journal Of Evolutionary Biology 19, 1486-1496 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(174);">abstract <img id="abstractI174" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16910979&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP174" 
  class="publication01Abstract" style="display:none;">Changes in gene expression are known to occur between closely related species, but it is not yet clear how many of these are due to random fixation of allelic variants or due to adaptive events. In a microarray survey between subspecies of the Mus musculus complex, we identified the mitogen-activated protein-kinase-kinase MKK7 as a candidate for change in gene expression. Quantitative PCR experiments with multiple individuals from each subspecies confirmed a specific and significant up-regulation in the testis of M. m. domesticus. Northern blot analysis shows that this is due to a new transcript that is not found in other tissues, nor in M. m. musculus. A cis-trans test via allele specific expression analysis of the MKK7 gene in F1 hybrids between domesticus and musculus shows that the expression change is mainly caused by a mutation located in cis. Nucleotide diversity was found to be significantly reduced in a window of at least 20 kb around the MKK7 locus in domesticus, indicative of a selective sweep. Because the MKK7 gene is involved in modulating a kinase signalling cascade in a stress response pathway, it seems a plausible target for adaptive differences between subspecies, although the functional role of the new testis-specific transcripts will need to be further studied.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(174);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Genomic islands of differentiation between house mouse subspecies.</div><div 
  class="publication01Info">Harr B, Genome Research 16, 730-737 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(177);">abstract <img id="abstractI177" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16687734&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP177" 
  class="publication01Abstract" style="display:none;">Understanding the genes that contribute to reproductive isolation is essential to understanding speciation, but isolating such genes has proven very difficult. In this study I apply a multilocus test statistic to &gt;10,000 SNP markers assayed in wild-derived inbred strains of house mice to identify genomic regions of elevated differentiation between two subspecies of house mice, Mus musculus musculus and M. m. domesticus. Differentiation was high through approximately 90% of the X chromosome. In addition, eight regions of high differentiation were identified on the autosomes, totaling 7.5% of the autosomal genome. Regions of high differentiation were confirmed by direct sequencing of samples collected from the wild. Some regions of elevated differentiation have an overrepresentation of genes with host-pathogen interactions and olfaction. The most strongly differentiated region on the X has previously been shown to fail to introgress across a hybrid zone between the two subspecies. This survey indicates autosomal regions that should also be examined for differential introgression across the hybrid zone, as containing potential genes causing hybrid unfitness.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(177);">hide</a></div></div></div>
	    <p><a href="project_area_b.html">back to project area B</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
