<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
								 id="contentPane"><h2>C1: Publications</h2>


		    <div class="publication01"><div 
  class="publication01Title">Multiple-Line Inference of Selection on Quantitative Traits</div><div 
  class="publication01Info"> N. Riedel, B. S. Khatri, M. Lässig, and J. Berg; Genetics September 2015 201:305-322; Early online July 2, 2015</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(4);">abstract <img id="abstractI4" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.genetics.org/content/201/1/305.full">paper [pdf]</a> &nbsp; </div><div id="abstractP4" 
  class="publication01Abstract" style="display:none;">To Be Added<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(4);">hide</a></div></div></div>



		    <div class="publication01"><div 
  class="publication01Title">Dynamic BMP signaling polarized by Toll patterns the dorsoventral axis in a hemimetabolous insect</div><div 
  class="publication01Info"> L. Sachs, Y.-T. Chen, A. Drechsler, J. A. Lynch, K. A. Panfilio, M. Lässig, J. Berg, and S. Roth, eLife 4:e05502 (2015)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(3);">abstract <img id="abstractI3" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://elifesciences.org/content/4/e05502">paper [pdf]</a> &nbsp; </div><div id="abstractP3" 
  class="publication01Abstract" style="display:none;">Toll-dependent patterning of the dorsoventral axis in Drosophila represents one of the best understood gene regulatory networks. However, its evolutionary origin has remained elusive. Outside the insects Toll is not known for a patterning function, but rather for a role in pathogen defense. Here, we show that in the milkweed bug Oncopeltus fasciatus, whose lineage split from Drosophila's more than 350 million years ago, Toll is only required to polarize a dynamic BMP signaling network. A theoretical model reveals that this network has self-regulatory properties and that shallow Toll signaling gradients are sufficient to initiate axis formation. Such gradients can account for the experimentally observed twinning of insect embryos upon egg fragmentation and might have evolved from a state of uniform Toll activity associated with protecting insect eggs against pathogens.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(3);">hide</a></div></div></div>
	
	    <div class="publication01"><div 
  class="publication01Title">A Genomics-Based Classification of Human Lung Tumors</div><div 
  class="publication01Info">Seidel D et al., Science Translational Medicine, 5 (209) p. 209ra153 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(2);">abstract <img id="abstractI2" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://stm.sciencemag.org/content/5/209/209ra153.full.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP2" 
  class="publication01Abstract" style="display:none;">We characterized genome alterations in 1255 clinically annotated lung tumors of all histological subgroups to identify genetically defined and clinically relevant subtypes. More than 55% of all cases had at least one oncogenic genome alteration potentially amenable to specific therapeutic intervention, including several personalized treatment approaches that are already in clinical evaluation. Marked differences in the pattern of genomic alterations existed between and within histological subtypes, thus challenging the original histomorphological diagnosis. Immunohistochemical studies confirmed many of these reassigned subtypes. The reassignment eliminated almost all cases of large cell carcinomas, some of which had therapeutically relevant alterations. Prospective testing of our genomics-based diagnostic algorithm in 5145 lung cancer patients enabled a genome-based diagnosis in 3863 (75%) patients, confirmed the feasibility of rational reassignments of large cell lung cancer, and led to improvement in overall survival in patients with EGFR-mutant or ALK-rearranged cancers. Thus, our findings provide support for broad implementation of genome-based diagnosis of lung cancer.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(2);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">How epigenetic mutations can affect genetic evolution: Model and mechanism</div><div 
  class="publication01Info">Klironomos F, Berg J, Collins S, Bioessays, 35 (6), 571-578 (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(13);">abstract <img id="abstractI13" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1002/bies.201200169/pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP13" 
  class="publication01Abstract" style="display:none;">We hypothesize that heritable epigenetic changes can affect rates of fitness increase as well as patterns of genotypic and phenotypic change during adaptation. In particular, we suggest that when natural selection acts on pure epigenetic variation in addition to genetic variation, populations adapt faster, and adaptive phenotypes can arise before any genetic changes. This may make it difficult to reconcile the timing of adaptive events detected using conventional population genetics tools based on DNA sequence data with environmental drivers of adaptation, such as changes in climate. Epigenetic modifications are frequently associated with somatic cell differentiation, but recently epigenetic changes have been found that can be transmitted over many generations. Here, we show how the interplay of these heritable epigenetic changes with genetic changes can affect adaptive evolution, and how epigenetic changes affect the signature of selection in the genetic record.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(13);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Evolution of molecular phenotypes under stabilizing selection</div><div 
  class="publication01Info">Nourmohammad A, Schiffels S, Lässig M, Journal Of Statistical Mechanics, P01012 (34 pages) (2013)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(30);">abstract <img id="abstractI30" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/1742-5468/2013/01/P01012/pdf/1742-5468_2013_01_P01012.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP30" 
  class="publication01Abstract" style="display:none;">Molecular phenotypes are important links between genomic information and organismic functions, fitness, and evolution. Complex phenotypes, which are also called quantitative traits, often depend on multiple genomic loci. Their evolution builds on genome evolution in a complicated way, which involves selection, genetic drift, mutations and recombination. Here we develop a coarse-grained evolutionary statistics for phenotypes, which decouples from details of the underlying genotypes. We derive approximate evolution equations for the distribution of phenotype values within and across populations. This dynamics covers evolutionary processes at high and low recombination rates, that is, it applies to sexual and asexual populations. In a fitness landscape with a single optimal phenotype value, the phenotypic diversity within populations and the divergence between populations reach evolutionary equilibria, which describe stabilizing selection. We compute the equilibrium distributions of both quantities analytically and we show that the ratio of mean divergence and diversity depends on the strength of selection in a universal way: it is largely independent of the phenotype's genomic encoding and of the recombination rate. This establishes a new method for the inference of selection on molecular phenotypes beyond the genome level. We discuss the implications of our findings for the predictability of evolutionary processes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(30);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">GraphAlignment: Bayesian pairwise alignment of biological networks</div><div 
  class="publication01Info">Kolář M, Meier J, Mustonen V, Lässig M, Berg J, BMC Systems Biology, 6:144 doi:10.1186/1752-0509-6-144 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(36);">abstract <img id="abstractI36" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.biomedcentral.com/content/pdf/1752-0509-6-144.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP36" 
  class="publication01Abstract" style="display:none;"><p><strong>Background</strong></p>
<p>With increased experimental availability and accuracy of bio-molecular networks, tools for their comparative and evolutionary analysis are needed. A key component for such studies is the alignment of networks.<p>
<p><strong>Results</strong></p>
<p>We introduce the Bioconductor package GraphAlignment for pairwise alignment of bio-molecular networks. The alignment incorporates information both from network vertices and network edges and is based on an explicit evolutionary model, allowing inference of all scoring parameters directly from empirical data. We compare the performance of our algorithm to an alternative algorithm, Græmlin 2.0.</p>
<p>On simulated data, GraphAlignment outperforms Græmlin 2.0 in several benchmarks except for computational complexity. When there is little or no noise in the data, GraphAlignment is slower than Græmlin 2.0. It is faster than Græmlin 2.0 when processing noisy data containing spurious vertex associations. Its typical case complexity grows approximately as 𝒪(N2.6).</p>
<p>On empirical bacterial protein-protein interaction networks (PIN) and gene co-expression networks, GraphAlignment outperforms Græmlin 2.0 with respect to coverage and specificity, albeit by a small margin. On large eukaryotic PIN, Græmlin 2.0 outperforms GraphAlignment.</p>
<p><strong>Conclusions</strong></p>
<p>The GraphAlignment algorithm is robust to spurious vertex associations, correctly resolves paralogs, and shows very good performance in identification of homologous vertices defined by high vertex and/or interaction similarity. The simplicity and generality of GraphAlignment edge scoring makes the algorithm an appropriate choice for global alignment of networks.</p><div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(36);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Mean-field theory for the inverse Ising problem at low temperatures</div><div 
  class="publication01Info">Nguyen HC, Berg J, Phys. Rev. Lett. 109, 050602 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(43);">abstract <img id="abstractI43" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://prl.aps.org/abstract/PRL/v109/i5/e050602">paper [pdf]</a> &nbsp; </div><div id="abstractP43" 
  class="publication01Abstract" style="display:none;">The large amounts of data from molecular biology and neuroscience have lead to a renewed interest in the inverse Ising problem: how to reconstruct parameters of the Ising model (couplings between spins and external fields) from a number of spin configurations sampled from the Boltzmann measure. To invert the relationship between model parameters and observables (magnetizations and correlations), mean-field approximations are often used, allowing the determination of model parameters from data. However, all known mean-field methods fail at low temperatures with the emergence of multiple thermodynamic states. Here, we show how clustering spin configurations can approximate these thermodynamic states and how mean-field methods applied to thermodynamic states allow an efficient reconstruction of Ising models also at low temperatures.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(43);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Bethe-Peierls approximation and the inverse Ising problem</div><div 
  class="publication01Info">Chau Nguyen H, Berg J, Journal Of Statistical Mechanics 03, 004 (2012)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(54);">abstract <img id="abstractI54" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2012JSMTE..03..004C&amp;link_type=ABSTRACT">paper [pdf]</a> &nbsp; </div><div id="abstractP54" 
  class="publication01Abstract" style="display:none;">We apply the Bethe-Peierls approximation to the inverse Ising problem and show how the linear response relation leads to a simple method for reconstructing couplings and fields of the Ising model. This reconstruction is exact on tree graphs, yet its computational expense is comparable to those of other mean-field methods. We compare the performance of this method to the independent-pair, naive mean-field, and Thouless-Anderson-Palmer approximations, the Sessak-Monasson expansion, and susceptibility propagation on the Cayley tree, SK model and random graph with fixed connectivity. At low temperatures, Bethe reconstruction outperforms all of these methods, while at high temperatures it is comparable to the best method available so far (the Sessak-Monasson method). The relationship between Bethe reconstruction and other mean-field methods is discussed.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(54);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Nonlinear fitness landscape of a molecular pathway.</div><div 
  class="publication01Info">Perfeito L, Ghozzi S, Berg J, Schnetz K, Lässig M, PLoS Genetics 7(7): e1002160 (2011)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(74);">abstract <img id="abstractI74" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://link.springer.com/article/10.1007%2Fs00442-012-2479-5">paper [pdf]</a> &nbsp; </div><div id="abstractP74" 
  class="publication01Abstract" style="display:none;">Genes are regulated because their expression involves a fitness cost to the organism. The production of proteins by transcription and translation is a well-known cost factor, but the enzymatic activity of the proteins produced can also reduce fitness, depending on the internal state and the environment of the cell. Here, we map the fitness costs of a key metabolic network, the lactose utilization pathway in <em>Escherichia coli.</em> We measure the growth of several regulatory lac operon mutants in different environments inducing expression of the lac genes. We find a strikingly nonlinear fitness landscape, which depends on the production rate and on the activity rate of the lac proteins. A simple fitness model of the lac pathway, based on elementary biophysical processes, predicts the growth rate of all observed strains. The nonlinearity of fitness is explained by a feedback loop: production and activity of the lac proteins reduce growth, but growth also affects the density of these molecules. This nonlinearity has important consequences for molecular function and evolution. It generates a cliff in the fitness landscape, beyond which populations cannot maintain growth. In viable populations, there is an expression barrier of the lac genes, which cannot be exceeded in any stationary growth process. Furthermore, the nonlinearity determines how the fitness of operon mutants depends on the inducer environment. We argue that fitness nonlinearities, expression barriers, and gene–environment interactions are generic features of fitness landscapes for metabolic pathways, and we discuss their implications for the evolution of regulation.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(74);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Significance analysis and statistical mechanics: an application to clustering.</div><div 
  class="publication01Info">Łuksza M, Lässig M, Berg J, Physical Review Letters 105, 220601 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(91);">abstract <img id="abstractI91" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2010PhRvL.105v0601L&amp;link_type=ABSTRACT">paper [pdf]</a> &nbsp; </div><div id="abstractP91" 
  class="publication01Abstract" style="display:none;">This Letter addresses the statistical significance of structures in random data: given a set of vectors and a measure of mutual similarity, how likely is it that a subset of these vectors forms a cluster with enhanced similarity among its elements? The computation of this cluster p value for randomly distributed vectors is mapped onto a well-defined problem of statistical mechanics. We solve this problem analytically, establishing a connection between the physics of quenched disorder and multiple-testing statistics in clustering and related problems. In an application to gene expression data, we find a remarkable link between the statistical significance of a cluster and the functional relationships between its genes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(91);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Bayesian analysis of biological networks: clusters, motifs, cross-species correlations</div><div 
  class="publication01Info">Berg J, Lässig M, in Statistical And Evolutionary Analysis Of Biological Network Data, M. Stumpf and C. Wiuf (Eds.), Imperial College Press (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(107);">abstract <img id="abstractI107" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://www.thp.uni-koeln.de/~lassig/docs/2009a.pdf">paper [pdf]</a> &nbsp; </div><div id="abstractP107" 
  class="publication01Abstract" style="display:none;">An important part of the analysis of bio-molecular networks is to detect different functional units. Different functions are reflected in a different evolutionary dynamics, and hence in different statistical characteristics of network parts. In this sense, the {\em global statistics} of a biological network, e.g., its connectivity distribution, provides a background, and {\em local deviations} from this background signal functional units. In the computational analysis of biological networks, we thus typically have to discriminate between different statistical models governing different parts of the dataset. The nature of these models depends on the biological question asked. We illustrate this rationale here with three examples: identification of functional parts as highly connected 	extit{network clusters}, finding 	extit{network motifs}, which occur in a similar form at different places in the network, and the analysis of 	extit{cross-species network correlations}, which reflect evolutionary dynamics between species.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(107);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Adaptive gene regulatory networks</div><div 
  class="publication01Info">Stauffer F, Berg J, EPL 88, 48004 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(117);">abstract <img id="abstractI117" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://iopscience.iop.org/0295-5075/88/4/48004">paper [pdf]</a> &nbsp; </div><div id="abstractP117" 
  class="publication01Abstract" style="display:none;">Regulatory interactions between genes show a large amount of cross-species variability, even when the underlying functions are conserved: there are many ways to achieve the same function. Here we investigate the ability of regulatory networks to reproduce given expression levels within a simple model of gene regulation. We find an exponentially large space of regulatory networks compatible with a given set of expression levels, giving rise to an extensive entropy of networks. Typical realisations of regulatory networks are found to share a bias towards symmetric interactions, in line with empirical evidence.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(117);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">From protein interactions to functional annotation: graph alignment in Herpes.</div><div 
  class="publication01Info">Kolár M, Lässig M, Berg J, BMC Systems Biology 2, 90 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(159);">abstract <img id="abstractI159" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18957106&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP159" 
  class="publication01Abstract" style="display:none;">BACKGROUND:Sequence alignment is a prolific basis of functional annotation, but remains a challenging problem in the &apos;twilight zone&apos; of high sequence divergence or short gene length. Here we demonstrate how information on gene interactions can help to resolve ambiguous sequence alignments. We compare two distant Herpes viruses by constructing a graph alignment, which is based jointly on the similarity of their protein interaction networks and on sequence similarity. This hybrid method provides functional associations between proteins of the two organisms that cannot be obtained from sequence or interaction data alone.RESULTS:We find proteins where interaction similarity and sequence similarity are individually weak, but together provide significant evidence of orthology. There are also proteins with high interaction similarity but without any detectable sequence similarity, providing evidence of functional association beyond sequence homology. The functional predictions derived from our alignment are consistent with genomic position and gene expression data.CONCLUSION:Our approach shows that evolutionary conservation is a powerful filter to make protein interaction data informative about functional similarities between the interacting proteins, and it establishes graph alignment as a powerful tool for the comparative analysis of data from highly diverged species.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(159);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Cross-species analysis of biological networks by Bayesian alignment.</div><div 
  class="publication01Info">Berg J, Lässig M, Proceedings Of The National Academy Of Sciences Of The United States Of America 103, 10967-10972 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(186);">abstract <img id="abstractI186" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16835301&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP186" 
  class="publication01Abstract" style="display:none;">Complex interactions between genes or proteins contribute a substantial part to phenotypic evolution. Here we develop an evolutionarily grounded method for the cross-species analysis of interaction networks by alignment, which maps bona fide functional relationships between genes in different organisms. Network alignment is based on a scoring function measuring mutual similarities between networks, taking into account their interaction patterns as well as sequence similarities between their nodes. High-scoring alignments and optimal alignment parameters are inferred by a systematic Bayesian analysis. We apply this method to analyze the evolution of coexpression networks between humans and mice. We find evidence for significant conservation of gene expression clusters and give network-based predictions of gene function. We discuss examples where cross-species functional relationships between genes do not concur with sequence similarity.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(186);">hide</a></div></div></div>
	    <p><a href="project_area_c.html">back to project area C</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
