<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>SFB 680 | Molecular Basis of Evolutionary Innovations | publications</title>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
  <link rel="stylesheet" href="css/publications.css" type="text/css" />
  <meta name="description" content="SFB 680, Molecular Basis of Evolutionary Innovations"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <script id="prototype" type="text/javascript" charset="utf-8" 
    src="scripts/prototype.js"></script>
  <script id="publications" type="text/javascript" charset="utf-8" 
    src="scripts/publications.js"></script>
</head>
<body>
  <div id="main">
    <div id="content">
      <div id="pageHeader"><img id="pageHeaderText"
        src="images/sfb680_header_text_2013-04-3.png"></img><img id="mainHeaderImageR"
        src="images/header-01.jpg"></img></div><div 
        id="headerLine"></div><div id="pageMenuPane"><ul 
        id="mainMenu"><li class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="index.html">home</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="people.html">people and institutions</a></li><li 
        class="mainMenuItem01Active"><img class="menuItemPrefix" 
        src="images/arrow_d_01.png" /><a class="mainMenuLink01" href="projects_and_publications.html">projects and publications</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_a.html">project area A</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_b.html">project area B</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="project_area_c.html">project area C</a></li><li 
        class="mainMenuItem02"><a class="mainMenuLink02" 
        href="publication_list.html">publication list</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="events.html">events</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/arrow_r_01.png" /><a class="mainMenuLink01" 
        href="working_with_us.html">working with us</a></li><li 
        class="mainMenuItem01"><img class="menuItemPrefix" 
        src="images/blank_01.png" /><a class="mainMenuLink01" 
        href="gender.html">gender equality</a></li></ul></div><div 
	id="contentPane"><h2>B1: Publications</h2>
	    <div class="publication01"><div 
  class="publication01Title">A test of the neutral model of expression change in natural populations of house mouse subspecies.</div><div 
  class="publication01Info">Staubach F, Teschke M, Voolstra CR, Wolf JBW, Tautz D, Evolution 64, 549-560 (2010)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(95);">abstract <img id="abstractI95" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19674092&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP95" 
  class="publication01Abstract" style="display:none;">Changes in expression of genes are thought to contribute significantly to evolutionary divergence. To study the relative role of selection and neutrality in shaping expression changes, we analyzed 24 genes in three different tissues of the house mouse (Mus musculus). Samples from two natural populations of the subspecies M. m. domesticus and M. m. musculus were investigated using quantitative PCR assays and sequencing of the upstream region. We have developed an approach to quantify expression polymorphism within such populations and to disentangle technical from biological variation in the data. We found a correlation between expression polymorphism within populations and divergence between populations. Furthermore, we found a correlation between expression polymorphism and sequence polymorphism of the respective genes. These data are most easily interpreted within a framework of a predominantly neutral model of gene expression change, where only a fraction of the changes may have been driven by positive selection. Although most genes investigated were expressed in all three tissues analyzed, significant changes of expression levels occurred predominantly in a single tissue only. This adds to the notion that enhancer-specific effects or transregulatory effects can modulate the evolution of gene expression in a tissue-specific way.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(95);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Emergence of a new gene from an intergenic region.</div><div 
  class="publication01Info">Heinen TJAJ, Staubach F, Häming D, Tautz D, Current Biology 19, 1527-1531 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(110);">abstract <img id="abstractI110" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19733073&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP110" 
  class="publication01Abstract" style="display:none;">It is generally assumed that new genes would arise by gene duplication mechanisms, because the signals for regulation and transcript processing would be unlikely to evolve in parallel with a new gene function. We have identified here a transcript in the house mouse (Mus musculus) that has arisen within the past 2.5-3.5 million years in a large intergenic region. The region is present in many mammals, including humans, allowing us to exclude the involvement of gene duplication, transposable elements, or other genome rearrangements, which are typically found for other cases of newly evolved genes. The gene has three exons, shows alternative splicing, and is specifically expressed in postmeiotic cells of the testis. The transcript is restricted to species within the genus Mus and its emergence correlates with indel mutations in the 5&apos; regulatory region of the transcript. A recent selective sweep is associated with the transcript region in M. m. musculus populations. A knockout in the laboratory strain BL6 results in reduced sperm motility and reduced testis weight. Our results show that cryptic signals for transcript regulation and processing exist in intergenic regions and can become the basis for the evolution of a new functional gene.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(110);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Selection on cis-regulatory variation at B4galnt2 and its influence on von Willebrand factor in house mice.</div><div 
  class="publication01Info">Johnsen JM, Teschke M, Pavlidis P, McGee BM, Tautz D, Ginsburg D, Baines JF, Molecular Biology And Evolution 26, 567-578 (2009)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(120);">abstract <img id="abstractI120" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=19088380&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP120" 
  class="publication01Abstract" style="display:none;">The RIIIS/J inbred mouse strain is a model for type 1 von Willebrand disease (VWD), a common human bleeding disorder. Low von Willebrand factor (VWF) levels in RIIIS/J are due to a regulatory mutation, Mvwf1, which directs a tissue-specific switch in expression of a glycosyltransferase, B4GALNT2, from intestine to blood vessel. We recently found that Mvwf1 lies on a founder allele common among laboratory mouse strains. To investigate the evolutionary forces operating at B4galnt2, we conducted a survey of DNA sequence polymorphism and microsatellite variation spanning the B4galnt2 gene region in natural Mus musculus domesticus populations. Two divergent haplotypes segregate in these natural populations, one of which corresponds to the RIIIS/J sequence. Different local populations display dramatic differences in the frequency of these haplotypes, and reduced microsatellite variability near B4galnt2 within the RIIIS/J haplotype is consistent with the recent action of natural selection. The level and pattern of DNA sequence polymorphism in the 5&apos; flanking region of the gene significantly deviates from the neutral expectation and suggests that variation in B4galnt2 expression may be under balancing selection and/or arose from a recently introgressed allele that subsequently increased in frequency due to natural selection. However, coalescent simulations indicate that the heterogeneity in divergence between haplotypes is greater than expected under an introgression model. Analysis of a population where the RIIIS/J haplotype is in high frequency reveals an association between this haplotype, the B4galnt2 tissue-specific switch, and a significant decrease in plasma VWF levels. Given these observations, we propose that low VWF levels may represent a fitness cost that is offset by a yet unknown benefit of the B4galnt2 tissue-specific switch. Similar mechanisms may account for the variability in VWF levels and high prevalence of VWD in other mammals, including humans.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(120);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Identification of selective sweeps in closely related populations of the house mouse based on microsatellite scans.</div><div 
  class="publication01Info">Teschke M, Mukabayire O, Wiehe T, Tautz D, Genetics 180, 1537-1545 (2008)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(133);">abstract <img id="abstractI133" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=18791245&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP133" 
  class="publication01Abstract" style="display:none;">Genome scans of polymorphisms promise to provide insights into the patterns and frequencies of positive selection under natural conditions. The use of microsatellites as markers has the potential to focus on very recent events, since in contrast to SNPs, their high mutation rates should remove signatures of older events. We assess this concept here in a large-scale study. We have analyzed two population pairs of the house mouse, one pair of the subspecies Mus musculus domesticus and the other of M. m. musculus. A total of 915 microsatellite loci chosen to cover the whole genome were assessed in a prescreening procedure, followed by individual typing of candidate loci. Schlötterer&apos;s ratio statistics (lnRH) were applied to detect loci with significant deviations from patterns of neutral expectation. For eight loci from each population pair we have determined the size of the potential sweep window and applied a second statistical procedure (linked locus statistics). For the two population pairs, we find five and four significant sweep loci, respectively, with an average estimated window size of 120 kb. On the basis of the analysis of individual allele frequencies, it is possible to identify the most recent sweep, for which we estimate an onset of 400-600 years ago. Given the known population history for the French-German population pair, we infer that the average frequency of selective sweeps in these populations is higher than 1 in 100 generations across the whole genome. We discuss the implications for adaptation processes in natural populations.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(133);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">A pooling approach to detect signatures of selective sweeps in genome scans using microsatellites</div><div 
  class="publication01Info">Thomas M, Möller F, Wiehe T, Tautz D, Molecular Ecology Notes 7, 400-403 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(160);">abstract <img id="abstractI160" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2007.01697.x/abstract">paper [pdf]</a> &nbsp; </div><div id="abstractP160" 
  class="publication01Abstract" style="display:none;">We have evaluated a pooling approach that can reduce the number of polymerase chain reactions in a screen for selective sweeps by more than an order of magnitude. We show that the complex peak pattern that results from pooling of all samples from a given population is a faithful reflection of the composite pattern of the individual alleles, although with an under-representation of the larger alleles. Candidate loci for selective sweeps can be identified by visual inspection of the pool patterns. We have also implemented a software tool, which can find suitable microsatellite loci in the vicinity of annotated genes.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(160);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">Contrasting evolution of expression differences in the testis between species and subspecies of the house mouse.</div><div 
  class="publication01Info">Voolstra C, Tautz D, Farbrother P, Eichinger L, Harr B, Genome Research 17, 42-49 (2007)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(164);">abstract <img id="abstractI164" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=17038563&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP164" 
  class="publication01Abstract" style="display:none;">Regulatory changes in genes involved in reproduction are thought to be prime targets for divergence during speciation, since they are expected to play an important role in sexual selection and sexual conflict. We used microarray analysis of RNA from different wild populations of house mouse subspecies (including Mus m. musculus, Mus m. domesticus, and Mus m. castaneus) and from the sister species Mus spretus to test this assumption. A comparison of expression divergence in brain, liver/kidney, and testis shows a major difference in the evolutionary dynamics of testis-related genes. While the comparison between species confirms an excess in divergence in testis genes, we find that all comparisons between subspecies yield only a very small number of genes with significantly different expression levels in the testis. These results suggest that the early phase of the speciation process may not be driven by regulatory changes in genes that are potential targets of sexual selection, and that the divergence in these genes is only established during a later phase of the speciation process.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(164);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">A change of expression in the conserved signaling gene MKK7 is associated with a selective sweep in the western house mouse Mus musculus domesticus.</div><div 
  class="publication01Info">Harr  B, Voolstra  C, Heinen  T J A J, Baines  J F, Rottscheidt  R, Ihle  S, Müller  W, Bonhomme  F, Tautz  D, Journal Of Evolutionary Biology 19, 1486-1496 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(174);">abstract <img id="abstractI174" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16910979&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP174" 
  class="publication01Abstract" style="display:none;">Changes in gene expression are known to occur between closely related species, but it is not yet clear how many of these are due to random fixation of allelic variants or due to adaptive events. In a microarray survey between subspecies of the Mus musculus complex, we identified the mitogen-activated protein-kinase-kinase MKK7 as a candidate for change in gene expression. Quantitative PCR experiments with multiple individuals from each subspecies confirmed a specific and significant up-regulation in the testis of M. m. domesticus. Northern blot analysis shows that this is due to a new transcript that is not found in other tissues, nor in M. m. musculus. A cis-trans test via allele specific expression analysis of the MKK7 gene in F1 hybrids between domesticus and musculus shows that the expression change is mainly caused by a mutation located in cis. Nucleotide diversity was found to be significantly reduced in a window of at least 20 kb around the MKK7 locus in domesticus, indicative of a selective sweep. Because the MKK7 gene is involved in modulating a kinase signalling cascade in a stress response pathway, it seems a plausible target for adaptive differences between subspecies, although the functional role of the new testis-specific transcripts will need to be further studied.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(174);">hide</a></div></div></div><div class="publication01"><div 
  class="publication01Title">An analysis of signatures of selective sweeps in natural populations of the house mouse.</div><div 
  class="publication01Info">Ihle S, Ravaoarimanana I, Thomas M, Tautz D, Molecular Biology And Evolution 23, 790-797 (2006)</div><div 
  class="publication01Links"><a href="javascript:Publications.toggleAbstract(179);">abstract <img id="abstractI179" class="publication01Arrow" src="images/arrow_r_02.png"></img></a> &nbsp; <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;id=16421176&amp;retmode=ref&amp;cmd=prlinks">paper [pdf]</a> &nbsp; </div><div id="abstractP179" 
  class="publication01Abstract" style="display:none;">Population and locus-specific reduction of variability of polymorphic loci could be an indication of positive selection at a linked site (selective sweep) and therefore point toward genes that have been involved in recent adaptations. Analysis of microsatellite variability offers a way to identify such regions and to ask whether they occur more often than expected by chance. We studied four populations of the house mouse (Mus musculus) to assess the frequency of such signatures of selective sweeps under natural conditions. Three samples represent the subspecies Mus m. dometicus [corrected] and came from Germany, France, and Cameroon. One sample came from Kazakhstan and constitutes a population of the subspecies Mus m. [corrected] musculus. Mitochondrial D-loop sequences from all animals confirm their respective assignments. Approximately 200 microsatellite loci were typed for up to 60 unrelated individuals from each population and evaluated for signs of selective sweeps on the basis of Schlötterer&apos;s ln RV and ln RH statistics. Our data suggest that there are slightly more signs of selective sweeps than would have been expected by chance alone in each of the populations and also highlights some of the statistical challenges faced in genome scans for detecting selection. Single-nucleotide polymorphism typing of one sweep signature in the M. m. domesticus populations around the beta-defensin 6 locus confirms a lowered nucleotide diversity in this region and limits the potential sweep region to about 20 kb. However, no amino acid exchange has occurred in the coding region when compared to M. m. musculus. If this sweep signature is due to a recent adaptation, it is expected that a regulatory change would have caused it. Our data provide a framework for conducting a systematic whole genome scan for signatures of selective sweeps in the mouse genome.<div class="publication01HideAbstract"><a href="javascript:Publications.hideAbstract(179);">hide</a></div></div></div>
	    <p><a href="project_area_b.html">back to project area B</a> 
    &nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;<br /> 
	&nbsp;</p>
	    </div>
      </div>
    </div>
  </div>
</body>
</html>
